{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Diatom Analysis Pipeline - OTU\n",
    "\n",
    "#### A notebook  for the diatom analysis pipeline using OTU classification."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1. Create BLAST reference database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 06/03/2019 09:21:25\n",
      "New DB name:   /code/diatoms\n",
      "New DB title:  diatoms.sequences.FINAL2017.fasta\n",
      "Sequence type: Nucleotide\n",
      "Keep Linkouts: T\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2701 sequences in 0.207415 seconds.\n",
      "CPU times: user 0 ns, sys: 20 ms, total: 20 ms\n",
      "Wall time: 1.1 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!makeblastdb -in diatoms.sequences.FINAL2017.fasta -out diatoms -dbtype nucl"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Quality Control\n",
    "* *Cutadapt:* Trim primers from sequences\n",
    "* *SicklePE:* Trim off bad quality 3' bases\n",
    "* *Pear:* Merge R1 and R2 reads\n",
    "* *SickleSE:* Remove post-merging bad quality sequences\n",
    "* *Histogram Generation:* Plots sequence length against number of sequences\n",
    "* *QIIME Prep:* Pepare passed QC files for QIIME processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1040.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1040.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.64 s (38 us/read; 1.59 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  96,518\r\n",
      "Reads with adapters:                    90,759 (94.0%)\r\n",
      "Reads written (passing filters):        96,518 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,184,439 bp\r\n",
      "Total written (filtered):     21,576,862 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 73209 times.\r\n",
      "73157 times, it overlapped the 5' end of a read\r\n",
      "52 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1508.1\t0\t1\r\n",
      "4\t1\t377.0\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t99\t0.0\t0\t99\r\n",
      "21\t73029\t0.0\t0\t73029\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t1508.1\t0\t46\r\n",
      "4\t1\t377.0\t0\t1\r\n",
      "51\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17550 times.\r\n",
      "59 times, it overlapped the 5' end of a read\r\n",
      "17491 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1508.1\t0\t59\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1508.1\t0\t48\r\n",
      "4\t17442\t377.0\t0\t17442\r\n",
      "5\t1\t94.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1060.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1060.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.90 s (38 us/read; 1.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,555\r\n",
      "Reads with adapters:                     3,110 (3.1%)\r\n",
      "Reads written (passing filters):       101,555 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,265,499 bp\r\n",
      "Total written (filtered):     17,215,433 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1242 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1234 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1586.8\t0\t4\r\n",
      "4\t4\t396.7\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1224\t1586.8\t0\t1224\r\n",
      "4\t4\t396.7\t0\t4\r\n",
      "5\t4\t99.2\t0\t4\r\n",
      "6\t2\t24.8\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1868 times.\r\n",
      "1698 times, it overlapped the 5' end of a read\r\n",
      "170 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1586.8\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1692\t0.0\t1\t0 1692\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t158\t1586.8\t0\t158\r\n",
      "4\t8\t396.7\t0\t8\r\n",
      "5\t4\t99.2\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1006.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1006.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.15 s (36 us/read; 1.68 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 116,297\r\n",
      "Reads with adapters:                     4,690 (4.0%)\r\n",
      "Reads written (passing filters):       116,297 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,552,210 bp\r\n",
      "Total written (filtered):     27,484,054 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2327 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2321 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1817.1\t0\t3\r\n",
      "4\t3\t454.3\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2312\t1817.1\t0\t2312\r\n",
      "4\t6\t454.3\t0\t6\r\n",
      "5\t2\t113.6\t0\t2\r\n",
      "6\t1\t28.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2363 times.\r\n",
      "2230 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1817.1\t0\t3\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2224\t0.0\t1\t0 2224\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t116\t1817.1\t0\t116\r\n",
      "4\t4\t454.3\t0\t4\r\n",
      "5\t6\t113.6\t0\t6\r\n",
      "6\t5\t28.4\t0\t5\r\n",
      "300\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1060.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1060.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.86 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,555\r\n",
      "Reads with adapters:                    90,191 (88.8%)\r\n",
      "Reads written (passing filters):       101,555 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,603,710 bp\r\n",
      "Total written (filtered):     16,270,489 bp (92.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 57212 times.\r\n",
      "57132 times, it overlapped the 5' end of a read\r\n",
      "80 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1586.8\t0\t1\r\n",
      "6\t1\t24.8\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t92\t0.0\t0\t92\r\n",
      "21\t57011\t0.0\t0\t57011\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1586.8\t0\t71\r\n",
      "4\t3\t396.7\t0\t3\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 32979 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "32953 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1586.8\t0\t24\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1586.8\t0\t80\r\n",
      "4\t32870\t396.7\t0\t32870\r\n",
      "5\t3\t99.2\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1040.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1040.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.69 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  96,518\r\n",
      "Reads with adapters:                     4,157 (4.3%)\r\n",
      "Reads written (passing filters):        96,518 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,014,691 bp\r\n",
      "Total written (filtered):     22,957,397 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2194 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2188 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1508.1\t0\t5\r\n",
      "4\t1\t377.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2181\t1508.1\t0\t2181\r\n",
      "4\t4\t377.0\t0\t4\r\n",
      "5\t3\t94.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1963 times.\r\n",
      "1857 times, it overlapped the 5' end of a read\r\n",
      "106 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1508.1\t0\t1\r\n",
      "4\t2\t377.0\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1852\t0.0\t1\t0 1852\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t92\t1508.1\t0\t92\r\n",
      "4\t8\t377.0\t0\t8\r\n",
      "5\t4\t94.3\t0\t4\r\n",
      "6\t1\t23.6\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1006.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1006.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.37 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 116,297\r\n",
      "Reads with adapters:                    99,126 (85.2%)\r\n",
      "Reads written (passing filters):       116,297 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,795,796 bp\r\n",
      "Total written (filtered):     26,035,384 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80219 times.\r\n",
      "80145 times, it overlapped the 5' end of a read\r\n",
      "74 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t105\t0.0\t0\t105\r\n",
      "21\t80019\t0.0\t0\t80019\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t1817.1\t0\t66\r\n",
      "5\t1\t113.6\t0\t1\r\n",
      "6\t1\t28.4\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 18907 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "18799 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t106\t1817.1\t0\t106\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1817.1\t0\t52\r\n",
      "4\t18747\t454.3\t0\t18747\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1114.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1114.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.49 s (48 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 115,076\r\n",
      "Reads with adapters:                     4,067 (3.5%)\r\n",
      "Reads written (passing filters):       115,076 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,011,594 bp\r\n",
      "Total written (filtered):     24,948,693 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1782 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "1768 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1798.1\t0\t12\r\n",
      "4\t2\t449.5\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1753\t1798.1\t0\t1753\r\n",
      "4\t8\t449.5\t0\t8\r\n",
      "5\t6\t112.4\t0\t6\r\n",
      "7\t1\t7.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2285 times.\r\n",
      "2114 times, it overlapped the 5' end of a read\r\n",
      "171 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1798.1\t0\t2\r\n",
      "4\t1\t449.5\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2108\t0.0\t1\t0 2108\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t162\t1798.1\t0\t162\r\n",
      "4\t6\t449.5\t0\t6\r\n",
      "5\t3\t112.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1114.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1114.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.48 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 115,076\r\n",
      "Reads with adapters:                   108,831 (94.6%)\r\n",
      "Reads written (passing filters):       115,076 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,334,314 bp\r\n",
      "Total written (filtered):     23,472,442 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 83939 times.\r\n",
      "83855 times, it overlapped the 5' end of a read\r\n",
      "84 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1798.1\t0\t1\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t138\t0.0\t0\t138\r\n",
      "21\t83696\t0.0\t0\t83696\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1798.1\t0\t74\r\n",
      "4\t1\t449.5\t0\t1\r\n",
      "6\t1\t28.1\t0\t1\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 24892 times.\r\n",
      "76 times, it overlapped the 5' end of a read\r\n",
      "24816 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1798.1\t0\t71\r\n",
      "4\t2\t449.5\t0\t2\r\n",
      "5\t1\t112.4\t0\t1\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t51\t1798.1\t0\t51\r\n",
      "4\t24765\t449.5\t0\t24765\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1116.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1116.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.73 s (45 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,309\r\n",
      "Reads with adapters:                     3,027 (3.7%)\r\n",
      "Reads written (passing filters):        82,309 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,725,906 bp\r\n",
      "Total written (filtered):     19,683,705 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1531 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "1514 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1286.1\t0\t16\r\n",
      "4\t1\t321.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1505\t1286.1\t0\t1505\r\n",
      "4\t5\t321.5\t0\t5\r\n",
      "5\t1\t80.4\t0\t1\r\n",
      "6\t3\t20.1\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1496 times.\r\n",
      "1379 times, it overlapped the 5' end of a read\r\n",
      "117 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t1379\t0.0\t1\t0 1379\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t113\t1286.1\t0\t113\r\n",
      "4\t1\t321.5\t0\t1\r\n",
      "5\t3\t80.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1243.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1243.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.46 s (37 us/read; 1.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,917\r\n",
      "Reads with adapters:                     3,329 (3.5%)\r\n",
      "Reads written (passing filters):        93,917 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,638,297 bp\r\n",
      "Total written (filtered):     20,591,943 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1664 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "1644 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1467.5\t0\t16\r\n",
      "4\t4\t366.9\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1633\t1467.5\t0\t1633\r\n",
      "4\t4\t366.9\t0\t4\r\n",
      "5\t7\t91.7\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1665 times.\r\n",
      "1515 times, it overlapped the 5' end of a read\r\n",
      "150 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1467.5\t0\t2\r\n",
      "4\t1\t366.9\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t1506\t0.0\t1\t0 1506\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t137\t1467.5\t0\t137\r\n",
      "4\t4\t366.9\t0\t4\r\n",
      "5\t6\t91.7\t0\t6\r\n",
      "6\t2\t22.9\t0\t2\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1243.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1243.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.26 s (45 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,917\r\n",
      "Reads with adapters:                    88,775 (94.5%)\r\n",
      "Reads written (passing filters):        93,917 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,805,584 bp\r\n",
      "Total written (filtered):     19,311,986 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 66868 times.\r\n",
      "66790 times, it overlapped the 5' end of a read\r\n",
      "78 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t94\t0.0\t0\t94\r\n",
      "21\t66677\t0.0\t0\t66677\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t67\t1467.5\t0\t67\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 21907 times.\r\n",
      "46 times, it overlapped the 5' end of a read\r\n",
      "21861 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1467.5\t0\t41\r\n",
      "27\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t53\t1467.5\t0\t53\r\n",
      "4\t21808\t366.9\t0\t21808\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1116.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1116.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.84 s (59 us/read; 1.02 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,309\r\n",
      "Reads with adapters:                    76,552 (93.0%)\r\n",
      "Reads written (passing filters):        82,309 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,856,185 bp\r\n",
      "Total written (filtered):     18,480,172 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62861 times.\r\n",
      "62821 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1286.1\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t82\t0.0\t0\t82\r\n",
      "21\t62722\t0.0\t0\t62722\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1286.1\t0\t33\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 13691 times.\r\n",
      "50 times, it overlapped the 5' end of a read\r\n",
      "13641 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1286.1\t0\t48\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1286.1\t0\t33\r\n",
      "4\t13605\t321.5\t0\t13605\r\n",
      "5\t3\t80.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1248.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1248.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.05 s (25 us/read; 2.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,659\r\n",
      "Reads with adapters:                     5,821 (4.7%)\r\n",
      "Reads written (passing filters):       122,659 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,342,550 bp\r\n",
      "Total written (filtered):     32,264,439 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3156 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "3153 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1916.5\t0\t1\r\n",
      "4\t2\t479.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3138\t1916.5\t0\t3138\r\n",
      "4\t8\t479.1\t0\t8\r\n",
      "5\t7\t119.8\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2665 times.\r\n",
      "2526 times, it overlapped the 5' end of a read\r\n",
      "139 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t479.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2523\t0.0\t1\t0 2523\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t125\t1916.5\t0\t125\r\n",
      "4\t5\t479.1\t0\t5\r\n",
      "5\t8\t119.8\t0\t8\r\n",
      "7\t1\t7.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1257.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1257.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.11 s (41 us/read; 1.45 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  75,074\r\n",
      "Reads with adapters:                     2,598 (3.5%)\r\n",
      "Reads written (passing filters):        75,074 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,855,586 bp\r\n",
      "Total written (filtered):     15,818,448 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1233 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1232 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1173.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1225\t1173.0\t0\t1225\r\n",
      "4\t4\t293.3\t0\t4\r\n",
      "5\t3\t73.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1365 times.\r\n",
      "1224 times, it overlapped the 5' end of a read\r\n",
      "141 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1173.0\t0\t2\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "27\t1220\t0.0\t1\t0 1220\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t134\t1173.0\t0\t134\r\n",
      "4\t4\t293.3\t0\t4\r\n",
      "5\t3\t73.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1248.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1248.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.53 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,659\r\n",
      "Reads with adapters:                   113,876 (92.8%)\r\n",
      "Reads written (passing filters):       122,659 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,367,837 bp\r\n",
      "Total written (filtered):     30,167,394 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102636 times.\r\n",
      "102602 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1916.5\t0\t1\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t135\t0.0\t0\t135\r\n",
      "21\t102443\t0.0\t0\t102443\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1916.5\t0\t29\r\n",
      "4\t1\t479.1\t0\t1\r\n",
      "55\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11240 times.\r\n",
      "58 times, it overlapped the 5' end of a read\r\n",
      "11182 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1916.5\t0\t54\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t1 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1916.5\t0\t30\r\n",
      "4\t11152\t479.1\t0\t11152\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1257.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1257.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.05 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  75,074\r\n",
      "Reads with adapters:                    67,747 (90.2%)\r\n",
      "Reads written (passing filters):        75,074 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,134,958 bp\r\n",
      "Total written (filtered):     14,992,695 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 51243 times.\r\n",
      "51192 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t73\t0.0\t0\t73\r\n",
      "21\t51106\t0.0\t0\t51106\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t1173.0\t0\t46\r\n",
      "57\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16504 times.\r\n",
      "79 times, it overlapped the 5' end of a read\r\n",
      "16425 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t77\t1173.0\t0\t77\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1173.0\t0\t45\r\n",
      "4\t16378\t293.3\t0\t16378\r\n",
      "5\t2\t73.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1302.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1302.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.23 s (40 us/read; 1.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,946\r\n",
      "Reads with adapters:                     3,844 (4.7%)\r\n",
      "Reads written (passing filters):        80,946 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,199,585 bp\r\n",
      "Total written (filtered):     20,147,717 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2064 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "2064 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2059\t1264.8\t0\t2059\r\n",
      "4\t2\t316.2\t0\t2\r\n",
      "5\t3\t79.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1780 times.\r\n",
      "1682 times, it overlapped the 5' end of a read\r\n",
      "98 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1264.8\t0\t1\r\n",
      "4\t1\t316.2\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1676\t0.0\t1\t0 1676\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t90\t1264.8\t0\t90\r\n",
      "4\t4\t316.2\t0\t4\r\n",
      "5\t4\t79.0\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1255.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1255.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.19 s (42 us/read; 1.43 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 123,677\r\n",
      "Reads with adapters:                     4,757 (3.8%)\r\n",
      "Reads written (passing filters):       123,677 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,529,119 bp\r\n",
      "Total written (filtered):     28,458,852 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2237 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2236 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t483.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2220\t1932.5\t0\t2220\r\n",
      "4\t10\t483.1\t0\t10\r\n",
      "5\t5\t120.8\t0\t5\r\n",
      "6\t1\t30.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2520 times.\r\n",
      "2335 times, it overlapped the 5' end of a read\r\n",
      "185 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1932.5\t0\t3\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2328\t0.0\t1\t0 2328\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t1932.5\t0\t180\r\n",
      "4\t2\t483.1\t0\t2\r\n",
      "5\t3\t120.8\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1255.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1255.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.52 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 123,677\r\n",
      "Reads with adapters:                   110,983 (89.7%)\r\n",
      "Reads written (passing filters):       123,677 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,889,708 bp\r\n",
      "Total written (filtered):     26,907,672 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90506 times.\r\n",
      "90446 times, it overlapped the 5' end of a read\r\n",
      "60 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t154\t0.0\t0\t154\r\n",
      "21\t90256\t0.0\t0\t90256\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1932.5\t0\t54\r\n",
      "7\t1\t7.5\t0\t1\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 20477 times.\r\n",
      "111 times, it overlapped the 5' end of a read\r\n",
      "20366 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t111\t1932.5\t0\t111\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1932.5\t0\t54\r\n",
      "4\t20308\t483.1\t0\t20308\r\n",
      "5\t4\t120.8\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1302.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1302.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.61 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,946\r\n",
      "Reads with adapters:                    69,180 (85.5%)\r\n",
      "Reads written (passing filters):        80,946 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,355,220 bp\r\n",
      "Total written (filtered):     19,075,590 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58934 times.\r\n",
      "58892 times, it overlapped the 5' end of a read\r\n",
      "42 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1264.8\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t86\t0.0\t0\t86\r\n",
      "21\t58788\t0.0\t0\t58788\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1264.8\t0\t33\r\n",
      "4\t1\t316.2\t0\t1\r\n",
      "7\t1\t4.9\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 10246 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "10199 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1264.8\t0\t40\r\n",
      "27\t7\t0.0\t1\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1264.8\t0\t31\r\n",
      "4\t10166\t316.2\t0\t10166\r\n",
      "5\t2\t79.0\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1337.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1337.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.49 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,772\r\n",
      "Reads with adapters:                    76,686 (94.9%)\r\n",
      "Reads written (passing filters):        80,772 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,219,032 bp\r\n",
      "Total written (filtered):     19,727,345 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 69635 times.\r\n",
      "69612 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t95\t0.0\t0\t95\r\n",
      "21\t69495\t0.0\t0\t69495\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1262.1\t0\t16\r\n",
      "4\t1\t315.5\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7051 times.\r\n",
      "58 times, it overlapped the 5' end of a read\r\n",
      "6993 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1262.1\t0\t58\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1262.1\t0\t15\r\n",
      "4\t6978\t315.5\t0\t6978\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1337.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1337.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.72 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,772\r\n",
      "Reads with adapters:                     3,948 (4.9%)\r\n",
      "Reads written (passing filters):        80,772 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,086,822 bp\r\n",
      "Total written (filtered):     21,033,614 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2159 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2157 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1262.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2149\t1262.1\t0\t2149\r\n",
      "4\t5\t315.5\t0\t5\r\n",
      "5\t2\t78.9\t0\t2\r\n",
      "6\t1\t19.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1789 times.\r\n",
      "1726 times, it overlapped the 5' end of a read\r\n",
      "63 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1262.1\t0\t2\r\n",
      "4\t1\t315.5\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1722\t0.0\t1\t0 1722\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1262.1\t0\t63\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1342.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1342.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.48 s (44 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  78,530\r\n",
      "Reads with adapters:                     3,532 (4.5%)\r\n",
      "Reads written (passing filters):        78,530 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,957,601 bp\r\n",
      "Total written (filtered):     19,907,827 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1817 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "1817 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1813\t1227.0\t0\t1813\r\n",
      "4\t2\t306.8\t0\t2\r\n",
      "5\t2\t76.7\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1715 times.\r\n",
      "1633 times, it overlapped the 5' end of a read\r\n",
      "82 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t306.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1631\t0.0\t1\t0 1631\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1227.0\t0\t80\r\n",
      "5\t2\t76.7\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1342.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1342.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.31 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  78,530\r\n",
      "Reads with adapters:                    74,782 (95.2%)\r\n",
      "Reads written (passing filters):        78,530 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,085,000 bp\r\n",
      "Total written (filtered):     18,683,399 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64833 times.\r\n",
      "64797 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1227.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t99\t0.0\t0\t99\r\n",
      "21\t64674\t0.0\t0\t64674\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1227.0\t0\t32\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9949 times.\r\n",
      "55 times, it overlapped the 5' end of a read\r\n",
      "9894 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1227.0\t0\t54\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1227.0\t0\t25\r\n",
      "4\t9868\t306.8\t0\t9868\r\n",
      "5\t1\t76.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1415.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1415.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.03 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,335\r\n",
      "Reads with adapters:                     3,372 (3.2%)\r\n",
      "Reads written (passing filters):       104,335 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,560,520 bp\r\n",
      "Total written (filtered):     19,513,293 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1681 times.\r\n",
      "27 times, it overlapped the 5' end of a read\r\n",
      "1654 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1630.2\t0\t21\r\n",
      "4\t6\t407.6\t0\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1642\t1630.2\t0\t1642\r\n",
      "4\t7\t407.6\t0\t7\r\n",
      "5\t5\t101.9\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1691 times.\r\n",
      "1546 times, it overlapped the 5' end of a read\r\n",
      "145 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t407.6\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "27\t1543\t0.0\t1\t0 1543\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t127\t1630.2\t0\t127\r\n",
      "4\t3\t407.6\t0\t3\r\n",
      "5\t15\t101.9\t0\t15\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1415.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1415.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.43 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,335\r\n",
      "Reads with adapters:                    97,954 (93.9%)\r\n",
      "Reads written (passing filters):       104,335 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,719,415 bp\r\n",
      "Total written (filtered):     18,254,359 bp (92.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 63219 times.\r\n",
      "63113 times, it overlapped the 5' end of a read\r\n",
      "106 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t78\t0.0\t0\t78\r\n",
      "21\t63013\t0.0\t0\t63013\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t1630.2\t0\t102\r\n",
      "5\t1\t101.9\t0\t1\r\n",
      "60\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 34735 times.\r\n",
      "16 times, it overlapped the 5' end of a read\r\n",
      "34719 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1630.2\t0\t16\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t81\t1630.2\t0\t81\r\n",
      "4\t34637\t407.6\t0\t34637\r\n",
      "5\t1\t101.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1567.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1567.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.40 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,223\r\n",
      "Reads with adapters:                    75,960 (95.9%)\r\n",
      "Reads written (passing filters):        79,223 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,974,321 bp\r\n",
      "Total written (filtered):     18,519,119 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67636 times.\r\n",
      "67590 times, it overlapped the 5' end of a read\r\n",
      "46 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t90\t0.0\t0\t90\r\n",
      "21\t67478\t0.0\t0\t67478\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1237.9\t0\t35\r\n",
      "57\t1\t0.0\t0\t1\r\n",
      "58\t2\t0.0\t0\t2\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8324 times.\r\n",
      "70 times, it overlapped the 5' end of a read\r\n",
      "8254 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t67\t1237.9\t0\t67\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1237.9\t0\t19\r\n",
      "4\t8235\t309.5\t0\t8235\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1374.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1374.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.72 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 144,106\r\n",
      "Reads with adapters:                     6,187 (4.3%)\r\n",
      "Reads written (passing filters):       144,106 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,030,158 bp\r\n",
      "Total written (filtered):     32,953,274 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3568 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "3542 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t2251.7\t0\t21\r\n",
      "4\t3\t562.9\t0\t3\r\n",
      "5\t2\t140.7\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3528\t2251.7\t0\t3528\r\n",
      "4\t7\t562.9\t0\t7\r\n",
      "5\t6\t140.7\t0\t6\r\n",
      "6\t1\t35.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2619 times.\r\n",
      "2431 times, it overlapped the 5' end of a read\r\n",
      "188 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2251.7\t0\t2\r\n",
      "4\t1\t562.9\t0\t1\r\n",
      "27\t2428\t0.0\t1\t0 2428\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t176\t2251.7\t0\t176\r\n",
      "4\t4\t562.9\t0\t4\r\n",
      "5\t7\t140.7\t0\t7\r\n",
      "6\t1\t35.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1567.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1567.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.65 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,223\r\n",
      "Reads with adapters:                     3,832 (4.8%)\r\n",
      "Reads written (passing filters):        79,223 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,847,823 bp\r\n",
      "Total written (filtered):     19,795,628 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2077 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "2064 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1237.9\t0\t9\r\n",
      "4\t4\t309.5\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2059\t1237.9\t0\t2059\r\n",
      "4\t1\t309.5\t0\t1\r\n",
      "5\t4\t77.4\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1755 times.\r\n",
      "1695 times, it overlapped the 5' end of a read\r\n",
      "60 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1694\t0.0\t1\t0 1694\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t1237.9\t0\t55\r\n",
      "4\t3\t309.5\t0\t3\r\n",
      "5\t2\t77.4\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1374.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1374.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.73 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 144,106\r\n",
      "Reads with adapters:                   135,912 (94.3%)\r\n",
      "Reads written (passing filters):       144,106 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,136,248 bp\r\n",
      "Total written (filtered):     30,740,868 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 108730 times.\r\n",
      "108639 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2251.7\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t151\t0.0\t0\t151\r\n",
      "21\t108462\t0.0\t0\t108462\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t69\t2251.7\t0\t69\r\n",
      "4\t1\t562.9\t0\t1\r\n",
      "7\t1\t8.8\t0\t1\r\n",
      "45\t1\t0.0\t0\t1\r\n",
      "56\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 27182 times.\r\n",
      "36 times, it overlapped the 5' end of a read\r\n",
      "27146 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t2251.7\t0\t32\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t69\t2251.7\t0\t69\r\n",
      "4\t27076\t562.9\t0\t27076\r\n",
      "5\t1\t140.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1735.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1735.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.83 s (23 us/read; 2.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,303\r\n",
      "Reads with adapters:                     3,349 (4.2%)\r\n",
      "Reads written (passing filters):        80,303 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,305,439 bp\r\n",
      "Total written (filtered):     19,260,492 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1792 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "1774 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1254.7\t0\t13\r\n",
      "4\t5\t313.7\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1763\t1254.7\t0\t1763\r\n",
      "4\t5\t313.7\t0\t5\r\n",
      "5\t6\t78.4\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1557 times.\r\n",
      "1443 times, it overlapped the 5' end of a read\r\n",
      "114 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1254.7\t0\t2\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1439\t0.0\t1\t0 1439\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t1254.7\t0\t112\r\n",
      "4\t1\t313.7\t0\t1\r\n",
      "299\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1670.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1670.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.98 s (32 us/read; 1.89 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,690\r\n",
      "Reads with adapters:                     4,096 (4.4%)\r\n",
      "Reads written (passing filters):        93,690 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,645,422 bp\r\n",
      "Total written (filtered):     23,587,886 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2110 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "2093 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1463.9\t0\t16\r\n",
      "4\t1\t366.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2079\t1463.9\t0\t2079\r\n",
      "4\t8\t366.0\t0\t8\r\n",
      "5\t6\t91.5\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1986 times.\r\n",
      "1874 times, it overlapped the 5' end of a read\r\n",
      "112 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1463.9\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1870\t0.0\t1\t0 1870\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t109\t1463.9\t0\t109\r\n",
      "5\t2\t91.5\t0\t2\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1576.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1576.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.29 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 120,112\r\n",
      "Reads with adapters:                     4,716 (3.9%)\r\n",
      "Reads written (passing filters):       120,112 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,611,880 bp\r\n",
      "Total written (filtered):     26,542,470 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2273 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "2251 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1876.8\t0\t16\r\n",
      "4\t5\t469.2\t0\t5\r\n",
      "5\t1\t117.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2239\t1876.8\t0\t2239\r\n",
      "4\t5\t469.2\t0\t5\r\n",
      "5\t6\t117.3\t0\t6\r\n",
      "6\t1\t29.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2443 times.\r\n",
      "2302 times, it overlapped the 5' end of a read\r\n",
      "141 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t469.2\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2299\t0.0\t1\t0 2299\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t132\t1876.8\t0\t132\r\n",
      "4\t5\t469.2\t0\t5\r\n",
      "5\t4\t117.3\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1576.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1576.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.53 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 120,112\r\n",
      "Reads with adapters:                   114,115 (95.0%)\r\n",
      "Reads written (passing filters):       120,112 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,687,123 bp\r\n",
      "Total written (filtered):     24,694,944 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90374 times.\r\n",
      "90313 times, it overlapped the 5' end of a read\r\n",
      "61 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1876.8\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t118\t0.0\t0\t118\r\n",
      "21\t90158\t0.0\t0\t90158\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1876.8\t0\t58\r\n",
      "8\t1\t1.8\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 23741 times.\r\n",
      "39 times, it overlapped the 5' end of a read\r\n",
      "23702 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1876.8\t0\t32\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1876.8\t0\t43\r\n",
      "4\t23658\t469.2\t0\t23658\r\n",
      "5\t1\t117.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1670.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1670.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.23 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,690\r\n",
      "Reads with adapters:                    89,112 (95.1%)\r\n",
      "Reads written (passing filters):        93,690 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,740,674 bp\r\n",
      "Total written (filtered):     22,087,073 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 76238 times.\r\n",
      "76189 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t100\t0.0\t0\t100\r\n",
      "21\t76068\t0.0\t0\t76068\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1463.9\t0\t35\r\n",
      "4\t3\t366.0\t0\t3\r\n",
      "5\t2\t91.5\t0\t2\r\n",
      "56\t1\t0.0\t0\t1\r\n",
      "60\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12874 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "12800 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t72\t1463.9\t0\t72\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1463.9\t0\t31\r\n",
      "4\t12767\t366.0\t0\t12767\r\n",
      "5\t2\t91.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1735.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1735.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.29 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,303\r\n",
      "Reads with adapters:                    75,818 (94.4%)\r\n",
      "Reads written (passing filters):        80,303 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,394,586 bp\r\n",
      "Total written (filtered):     18,026,006 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62636 times.\r\n",
      "62591 times, it overlapped the 5' end of a read\r\n",
      "45 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t94\t0.0\t0\t94\r\n",
      "21\t62468\t0.0\t0\t62468\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1254.7\t0\t39\r\n",
      "5\t1\t78.4\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 13182 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "13144 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1254.7\t0\t36\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1254.7\t0\t27\r\n",
      "4\t13117\t313.7\t0\t13117\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1785.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1785.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.49 s (28 us/read; 2.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  17,857\r\n",
      "Reads with adapters:                    16,366 (91.7%)\r\n",
      "Reads written (passing filters):        17,857 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     4,987,117 bp\r\n",
      "Total written (filtered):      4,658,255 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 15481 times.\r\n",
      "15462 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t279.0\t0\t1\r\n",
      "12\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t42\t0.0\t0\t42\r\n",
      "21\t15411\t0.0\t0\t15411\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t279.0\t0\t15\r\n",
      "7\t1\t1.1\t0\t1\r\n",
      "57\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 885 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "865 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t279.0\t0\t18\r\n",
      "4\t1\t69.8\t0\t1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t279.0\t0\t7\r\n",
      "4\t858\t69.8\t0\t858\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1785.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1785.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.43 s (24 us/read; 2.51 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  17,857\r\n",
      "Reads with adapters:                     1,181 (6.6%)\r\n",
      "Reads written (passing filters):        17,857 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     4,965,000 bp\r\n",
      "Total written (filtered):      4,950,467 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 670 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "665 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t279.0\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t658\t279.0\t0\t658\r\n",
      "4\t1\t69.8\t0\t1\r\n",
      "5\t5\t17.4\t0\t5\r\n",
      "6\t1\t4.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 511 times.\r\n",
      "460 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t279.0\t0\t3\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t456\t0.0\t1\t0 456\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t279.0\t0\t45\r\n",
      "4\t4\t69.8\t0\t4\r\n",
      "5\t1\t17.4\t0\t1\r\n",
      "6\t1\t4.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1784.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1784.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.18 s (25 us/read; 2.44 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  48,135\r\n",
      "Reads with adapters:                     2,811 (5.8%)\r\n",
      "Reads written (passing filters):        48,135 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,201,961 bp\r\n",
      "Total written (filtered):     13,164,088 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1501 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1496 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t752.1\t0\t3\r\n",
      "4\t2\t188.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1490\t752.1\t0\t1490\r\n",
      "4\t4\t188.0\t0\t4\r\n",
      "5\t2\t47.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1310 times.\r\n",
      "1227 times, it overlapped the 5' end of a read\r\n",
      "83 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t188.0\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1221\t0.0\t1\t0 1221\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t752.1\t0\t73\r\n",
      "4\t6\t188.0\t0\t6\r\n",
      "5\t4\t47.0\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1784.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1784.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.49 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  48,135\r\n",
      "Reads with adapters:                    45,184 (93.9%)\r\n",
      "Reads written (passing filters):        48,135 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,258,538 bp\r\n",
      "Total written (filtered):     12,355,394 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 42427 times.\r\n",
      "42406 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t67\t0.0\t0\t67\r\n",
      "21\t42326\t0.0\t0\t42326\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t752.1\t0\t14\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2757 times.\r\n",
      "27 times, it overlapped the 5' end of a read\r\n",
      "2730 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t752.1\t0\t24\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t752.1\t0\t6\r\n",
      "4\t2724\t188.0\t0\t2724\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1779.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1779.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.08 s (33 us/read; 1.80 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,598\r\n",
      "Reads with adapters:                     3,304 (3.6%)\r\n",
      "Reads written (passing filters):        92,598 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,044,641 bp\r\n",
      "Total written (filtered):     19,997,485 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1619 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1615 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1446.8\t0\t3\r\n",
      "5\t1\t90.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1606\t1446.8\t0\t1606\r\n",
      "4\t5\t361.7\t0\t5\r\n",
      "5\t4\t90.4\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1685 times.\r\n",
      "1554 times, it overlapped the 5' end of a read\r\n",
      "131 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1446.8\t0\t1\r\n",
      "4\t2\t361.7\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1549\t0.0\t1\t0 1549\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t127\t1446.8\t0\t127\r\n",
      "4\t1\t361.7\t0\t1\r\n",
      "5\t3\t90.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1742.R2.fastq.gz.trimmed.fastq.gz /code/sequences/1742.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.72 s (42 us/read; 1.44 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,981\r\n",
      "Reads with adapters:                     4,067 (3.6%)\r\n",
      "Reads written (passing filters):       112,981 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,078,449 bp\r\n",
      "Total written (filtered):     25,019,807 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2000 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1998 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1765.3\t0\t1\r\n",
      "4\t1\t441.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1987\t1765.3\t0\t1987\r\n",
      "4\t8\t441.3\t0\t8\r\n",
      "5\t3\t110.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2067 times.\r\n",
      "1934 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "27\t1933\t0.0\t1\t0 1933\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t126\t1765.3\t0\t126\r\n",
      "4\t2\t441.3\t0\t2\r\n",
      "5\t5\t110.3\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1779.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1779.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.78 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,598\r\n",
      "Reads with adapters:                    87,154 (94.1%)\r\n",
      "Reads written (passing filters):        92,598 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,358,673 bp\r\n",
      "Total written (filtered):     18,904,169 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 65045 times.\r\n",
      "64980 times, it overlapped the 5' end of a read\r\n",
      "65 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t73\t0.0\t0\t73\r\n",
      "21\t64887\t0.0\t0\t64887\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1446.8\t0\t58\r\n",
      "4\t1\t361.7\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 22109 times.\r\n",
      "56 times, it overlapped the 5' end of a read\r\n",
      "22053 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t56\t1446.8\t0\t56\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1446.8\t0\t58\r\n",
      "4\t21994\t361.7\t0\t21994\r\n",
      "5\t1\t90.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/1742.R1.fastq.gz.trimmed.fastq.gz /code/sequences/1742.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.88 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,981\r\n",
      "Reads with adapters:                    88,321 (78.2%)\r\n",
      "Reads written (passing filters):       112,981 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,407,124 bp\r\n",
      "Total written (filtered):     23,954,018 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64714 times.\r\n",
      "64646 times, it overlapped the 5' end of a read\r\n",
      "68 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1765.3\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t101\t0.0\t0\t101\r\n",
      "21\t64526\t0.0\t0\t64526\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1765.3\t0\t62\r\n",
      "4\t1\t441.3\t0\t1\r\n",
      "7\t1\t6.9\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 23607 times.\r\n",
      "72 times, it overlapped the 5' end of a read\r\n",
      "23535 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1765.3\t0\t71\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1765.3\t0\t71\r\n",
      "4\t23464\t441.3\t0\t23464\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2035.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2035.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.34 s (24 us/read; 2.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  56,932\r\n",
      "Reads with adapters:                     1,959 (3.4%)\r\n",
      "Reads written (passing filters):        56,932 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,359,538 bp\r\n",
      "Total written (filtered):     13,332,909 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 993 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "988 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t889.6\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t979\t889.6\t0\t979\r\n",
      "4\t8\t222.4\t0\t8\r\n",
      "5\t1\t55.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 966 times.\r\n",
      "864 times, it overlapped the 5' end of a read\r\n",
      "102 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t864\t0.0\t1\t0 864\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t99\t889.6\t0\t99\r\n",
      "5\t3\t55.6\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2035.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2035.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.31 s (58 us/read; 1.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  56,932\r\n",
      "Reads with adapters:                    51,272 (90.1%)\r\n",
      "Reads written (passing filters):        56,932 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,502,564 bp\r\n",
      "Total written (filtered):     12,600,709 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40949 times.\r\n",
      "40920 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t889.6\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t4\t0.0\t0\t4\r\n",
      "20\t67\t0.0\t0\t67\r\n",
      "21\t40841\t0.0\t0\t40841\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t889.6\t0\t22\r\n",
      "5\t1\t55.6\t0\t1\r\n",
      "7\t1\t3.5\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 10323 times.\r\n",
      "49 times, it overlapped the 5' end of a read\r\n",
      "10274 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t889.6\t0\t45\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t889.6\t0\t28\r\n",
      "4\t10245\t222.4\t0\t10245\r\n",
      "5\t1\t55.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2018.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2018.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.05 s (43 us/read; 1.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 118,563\r\n",
      "Reads with adapters:                     4,190 (3.5%)\r\n",
      "Reads written (passing filters):       118,563 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,071,598 bp\r\n",
      "Total written (filtered):     26,018,109 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2279 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2271 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1852.5\t0\t5\r\n",
      "4\t2\t463.1\t0\t2\r\n",
      "5\t1\t115.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2256\t1852.5\t0\t2256\r\n",
      "4\t8\t463.1\t0\t8\r\n",
      "5\t6\t115.8\t0\t6\r\n",
      "6\t1\t28.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1911 times.\r\n",
      "1695 times, it overlapped the 5' end of a read\r\n",
      "216 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t463.1\t0\t3\r\n",
      "5\t1\t115.8\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1689\t0.0\t1\t0 1689\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t210\t1852.5\t0\t210\r\n",
      "4\t2\t463.1\t0\t2\r\n",
      "5\t3\t115.8\t0\t3\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2033.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2033.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.75 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,661\r\n",
      "Reads with adapters:                     4,819 (4.4%)\r\n",
      "Reads written (passing filters):       108,661 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,866,486 bp\r\n",
      "Total written (filtered):     25,804,853 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2720 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2717 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1697.8\t0\t2\r\n",
      "4\t1\t424.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2700\t1697.8\t0\t2700\r\n",
      "4\t7\t424.5\t0\t7\r\n",
      "5\t10\t106.1\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2099 times.\r\n",
      "1952 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1697.8\t0\t3\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1946\t0.0\t1\t0 1946\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t137\t1697.8\t0\t137\r\n",
      "5\t7\t106.1\t0\t7\r\n",
      "51\t2\t0.0\t1\t0 2\r\n",
      "270\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2056.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2056.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.87 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,650\r\n",
      "Reads with adapters:                   103,818 (95.6%)\r\n",
      "Reads written (passing filters):       108,650 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,103,548 bp\r\n",
      "Total written (filtered):     27,075,675 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 94687 times.\r\n",
      "94652 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t126\t0.0\t0\t126\r\n",
      "21\t94495\t0.0\t0\t94495\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1697.7\t0\t20\r\n",
      "4\t1\t424.4\t0\t1\r\n",
      "6\t1\t26.5\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9131 times.\r\n",
      "105 times, it overlapped the 5' end of a read\r\n",
      "9026 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t104\t1697.7\t0\t104\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1697.7\t0\t27\r\n",
      "4\t8999\t424.4\t0\t8999\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2094.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2094.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.69 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,526\r\n",
      "Reads with adapters:                    64,809 (90.6%)\r\n",
      "Reads written (passing filters):        71,526 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,021,178 bp\r\n",
      "Total written (filtered):     16,815,284 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55688 times.\r\n",
      "55662 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t279.4\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t67\t0.0\t0\t67\r\n",
      "21\t55582\t0.0\t0\t55582\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1117.6\t0\t23\r\n",
      "5\t1\t69.8\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9121 times.\r\n",
      "27 times, it overlapped the 5' end of a read\r\n",
      "9094 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1117.6\t0\t27\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1117.6\t0\t27\r\n",
      "4\t9066\t279.4\t0\t9066\r\n",
      "5\t1\t69.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2018.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2018.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.56 s (55 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 118,563\r\n",
      "Reads with adapters:                   110,792 (93.4%)\r\n",
      "Reads written (passing filters):       118,563 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,390,824 bp\r\n",
      "Total written (filtered):     24,451,703 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 87958 times.\r\n",
      "87890 times, it overlapped the 5' end of a read\r\n",
      "68 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t5\t0.0\t0\t5\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t136\t0.0\t0\t136\r\n",
      "21\t87727\t0.0\t0\t87727\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t1852.5\t0\t57\r\n",
      "4\t3\t463.1\t0\t3\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 22834 times.\r\n",
      "24 times, it overlapped the 5' end of a read\r\n",
      "22810 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1852.5\t0\t22\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1852.5\t0\t54\r\n",
      "4\t22755\t463.1\t0\t22755\r\n",
      "5\t1\t115.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2033.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2033.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.92 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,661\r\n",
      "Reads with adapters:                   101,640 (93.5%)\r\n",
      "Reads written (passing filters):       108,661 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,218,374 bp\r\n",
      "Total written (filtered):     24,377,337 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 84251 times.\r\n",
      "84171 times, it overlapped the 5' end of a read\r\n",
      "80 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t22\t0.0\t0\t22\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t114\t0.0\t0\t114\r\n",
      "21\t84014\t0.0\t0\t84014\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t1697.8\t0\t66\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17389 times.\r\n",
      "139 times, it overlapped the 5' end of a read\r\n",
      "17250 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t133\t1697.8\t0\t133\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1697.8\t0\t45\r\n",
      "4\t17204\t424.5\t0\t17204\r\n",
      "5\t1\t106.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2094.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2094.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.69 s (24 us/read; 2.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,526\r\n",
      "Reads with adapters:                     3,176 (4.4%)\r\n",
      "Reads written (passing filters):        71,526 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,922,930 bp\r\n",
      "Total written (filtered):     17,880,350 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1707 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1704 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1117.6\t0\t1\r\n",
      "4\t2\t279.4\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1696\t1117.6\t0\t1696\r\n",
      "4\t4\t279.4\t0\t4\r\n",
      "5\t3\t69.8\t0\t3\r\n",
      "6\t1\t17.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1469 times.\r\n",
      "1377 times, it overlapped the 5' end of a read\r\n",
      "92 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1117.6\t0\t1\r\n",
      "27\t1376\t0.0\t1\t0 1376\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t1117.6\t0\t85\r\n",
      "4\t3\t279.4\t0\t3\r\n",
      "5\t2\t69.8\t0\t2\r\n",
      "6\t2\t17.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2056.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2056.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.05 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,650\r\n",
      "Reads with adapters:                     4,328 (4.0%)\r\n",
      "Reads written (passing filters):       108,650 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,957,294 bp\r\n",
      "Total written (filtered):     28,890,464 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1941 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1935 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1697.7\t0\t1\r\n",
      "4\t5\t424.4\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1921\t1697.7\t0\t1921\r\n",
      "4\t5\t424.4\t0\t5\r\n",
      "5\t9\t106.1\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2387 times.\r\n",
      "2245 times, it overlapped the 5' end of a read\r\n",
      "142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1697.7\t0\t3\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2239\t0.0\t1\t0 2239\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t132\t1697.7\t0\t132\r\n",
      "4\t7\t424.4\t0\t7\r\n",
      "5\t3\t106.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2216.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2216.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.40 s (43 us/read; 1.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,956\r\n",
      "Reads with adapters:                    90,488 (87.9%)\r\n",
      "Reads written (passing filters):       102,956 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,102,348 bp\r\n",
      "Total written (filtered):     25,374,517 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80267 times.\r\n",
      "80216 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t80102\t0.0\t0\t80102\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1608.7\t0\t40\r\n",
      "7\t1\t6.3\t0\t1\r\n",
      "46\t1\t0.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 10221 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "10203 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1608.7\t0\t18\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1608.7\t0\t37\r\n",
      "4\t10166\t402.2\t0\t10166\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2367.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2367.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.17 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  63,172\r\n",
      "Reads with adapters:                     2,877 (4.6%)\r\n",
      "Reads written (passing filters):        63,172 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,252,392 bp\r\n",
      "Total written (filtered):     16,209,906 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1426 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1425 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t987.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1420\t987.1\t0\t1420\r\n",
      "4\t4\t246.8\t0\t4\r\n",
      "5\t1\t61.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1451 times.\r\n",
      "1398 times, it overlapped the 5' end of a read\r\n",
      "53 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1396\t0.0\t1\t0 1396\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t987.1\t0\t50\r\n",
      "4\t2\t246.8\t0\t2\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2367.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2367.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.73 s (59 us/read; 1.02 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  63,172\r\n",
      "Reads with adapters:                    60,102 (95.1%)\r\n",
      "Reads written (passing filters):        63,172 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,361,099 bp\r\n",
      "Total written (filtered):     15,217,586 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 53094 times.\r\n",
      "53064 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t246.8\t0\t1\r\n",
      "12\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t73\t0.0\t0\t73\r\n",
      "21\t52975\t0.0\t0\t52975\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t987.1\t0\t23\r\n",
      "4\t1\t246.8\t0\t1\r\n",
      "7\t1\t3.9\t0\t1\r\n",
      "58\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7008 times.\r\n",
      "39 times, it overlapped the 5' end of a read\r\n",
      "6969 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t987.1\t0\t39\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t987.1\t0\t23\r\n",
      "4\t6944\t246.8\t0\t6944\r\n",
      "5\t2\t61.7\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2216.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2216.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.06 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,956\r\n",
      "Reads with adapters:                     4,357 (4.2%)\r\n",
      "Reads written (passing filters):       102,956 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,983,942 bp\r\n",
      "Total written (filtered):     26,920,821 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2142 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2139 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1608.7\t0\t2\r\n",
      "4\t1\t402.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2129\t1608.7\t0\t2129\r\n",
      "4\t8\t402.2\t0\t8\r\n",
      "5\t2\t100.5\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2215 times.\r\n",
      "2086 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1608.7\t0\t1\r\n",
      "5\t1\t100.5\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2080\t0.0\t1\t0 2080\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t117\t1608.7\t0\t117\r\n",
      "4\t4\t402.2\t0\t4\r\n",
      "5\t7\t100.5\t0\t7\r\n",
      "6\t1\t25.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2346.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2346.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.95 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,126\r\n",
      "Reads with adapters:                     4,223 (4.0%)\r\n",
      "Reads written (passing filters):       105,126 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,374,953 bp\r\n",
      "Total written (filtered):     27,307,029 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1757 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1753 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1642.6\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1735\t1642.6\t0\t1735\r\n",
      "4\t9\t410.6\t0\t9\r\n",
      "5\t7\t102.7\t0\t7\r\n",
      "7\t2\t6.4\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2466 times.\r\n",
      "2301 times, it overlapped the 5' end of a read\r\n",
      "165 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1642.6\t0\t1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2296\t0.0\t1\t0 2296\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t150\t1642.6\t0\t150\r\n",
      "4\t4\t410.6\t0\t4\r\n",
      "5\t9\t102.7\t0\t9\r\n",
      "6\t2\t25.7\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2346.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2346.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.83 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,126\r\n",
      "Reads with adapters:                    93,089 (88.5%)\r\n",
      "Reads written (passing filters):       105,126 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,563,267 bp\r\n",
      "Total written (filtered):     25,764,283 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 83751 times.\r\n",
      "83695 times, it overlapped the 5' end of a read\r\n",
      "56 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1642.6\t0\t1\r\n",
      "4\t2\t410.6\t0\t2\r\n",
      "18\t15\t0.0\t0\t15\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t124\t0.0\t0\t124\r\n",
      "21\t83540\t0.0\t0\t83540\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1642.6\t0\t38\r\n",
      "4\t3\t410.6\t0\t3\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9338 times.\r\n",
      "69 times, it overlapped the 5' end of a read\r\n",
      "9269 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t68\t1642.6\t0\t68\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1642.6\t0\t42\r\n",
      "4\t9227\t410.6\t0\t9227\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2395.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2395.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.42 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,761\r\n",
      "Reads with adapters:                     4,645 (4.3%)\r\n",
      "Reads written (passing filters):       107,761 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,534,971 bp\r\n",
      "Total written (filtered):     30,462,016 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2081 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "2074 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1683.8\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2064\t1683.8\t0\t2064\r\n",
      "4\t7\t420.9\t0\t7\r\n",
      "5\t3\t105.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2564 times.\r\n",
      "2463 times, it overlapped the 5' end of a read\r\n",
      "101 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1683.8\t0\t4\r\n",
      "4\t1\t420.9\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2455\t0.0\t1\t0 2455\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t92\t1683.8\t0\t92\r\n",
      "4\t3\t420.9\t0\t3\r\n",
      "5\t4\t105.2\t0\t4\r\n",
      "6\t2\t26.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2395.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2395.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.28 s (58 us/read; 1.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,761\r\n",
      "Reads with adapters:                   102,250 (94.9%)\r\n",
      "Reads written (passing filters):       107,761 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,653,224 bp\r\n",
      "Total written (filtered):     28,555,499 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 99284 times.\r\n",
      "99262 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t113\t0.0\t0\t113\r\n",
      "21\t99120\t0.0\t0\t99120\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1683.8\t0\t16\r\n",
      "55\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2966 times.\r\n",
      "98 times, it overlapped the 5' end of a read\r\n",
      "2868 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t95\t1683.8\t0\t95\r\n",
      "4\t1\t420.9\t0\t1\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1683.8\t0\t7\r\n",
      "4\t2861\t420.9\t0\t2861\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2463.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2463.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.72 s (56 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,371\r\n",
      "Reads with adapters:                    93,513 (92.2%)\r\n",
      "Reads written (passing filters):       101,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,906,719 bp\r\n",
      "Total written (filtered):     26,985,919 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90901 times.\r\n",
      "90872 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1583.9\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t122\t0.0\t0\t122\r\n",
      "21\t90723\t0.0\t0\t90723\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1583.9\t0\t21\r\n",
      "51\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2612 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "2545 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1583.9\t0\t62\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1583.9\t0\t18\r\n",
      "4\t2527\t396.0\t0\t2527\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2463.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2463.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.07 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,371\r\n",
      "Reads with adapters:                     4,966 (4.9%)\r\n",
      "Reads written (passing filters):       101,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,819,644 bp\r\n",
      "Total written (filtered):     28,751,087 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2630 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2625 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1583.9\t0\t4\r\n",
      "4\t1\t396.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2616\t1583.9\t0\t2616\r\n",
      "4\t4\t396.0\t0\t4\r\n",
      "5\t5\t99.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2336 times.\r\n",
      "2235 times, it overlapped the 5' end of a read\r\n",
      "101 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2232\t0.0\t1\t0 2232\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t95\t1583.9\t0\t95\r\n",
      "4\t3\t396.0\t0\t3\r\n",
      "5\t3\t99.0\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2464.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2464.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.71 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 120,602\r\n",
      "Reads with adapters:                     5,838 (4.8%)\r\n",
      "Reads written (passing filters):       120,602 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,218,085 bp\r\n",
      "Total written (filtered):     36,133,060 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2901 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2898 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1884.4\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2887\t1884.4\t0\t2887\r\n",
      "4\t5\t471.1\t0\t5\r\n",
      "5\t6\t117.8\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2937 times.\r\n",
      "2817 times, it overlapped the 5' end of a read\r\n",
      "120 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1884.4\t0\t3\r\n",
      "4\t2\t471.1\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t7\t0.0\t1\t0 7\r\n",
      "27\t2804\t0.0\t1\t0 2804\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t111\t1884.4\t0\t111\r\n",
      "4\t4\t471.1\t0\t4\r\n",
      "5\t5\t117.8\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2464.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2464.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.88 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 120,602\r\n",
      "Reads with adapters:                   116,835 (96.9%)\r\n",
      "Reads written (passing filters):       120,602 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,251,953 bp\r\n",
      "Total written (filtered):     33,798,035 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 116729 times.\r\n",
      "116715 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1884.4\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t171\t0.0\t0\t171\r\n",
      "21\t116514\t0.0\t0\t116514\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1884.4\t0\t4\r\n",
      "7\t1\t7.4\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 106 times.\r\n",
      "53 times, it overlapped the 5' end of a read\r\n",
      "53 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1884.4\t0\t50\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1884.4\t0\t3\r\n",
      "4\t49\t471.1\t0\t49\r\n",
      "5\t1\t117.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2484.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2484.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.90 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,351\r\n",
      "Reads with adapters:                     4,995 (5.2%)\r\n",
      "Reads written (passing filters):        95,351 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,024,404 bp\r\n",
      "Total written (filtered):     26,959,311 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2823 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2818 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1489.9\t0\t4\r\n",
      "4\t1\t372.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2809\t1489.9\t0\t2809\r\n",
      "4\t6\t372.5\t0\t6\r\n",
      "5\t3\t93.1\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2172 times.\r\n",
      "2087 times, it overlapped the 5' end of a read\r\n",
      "85 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2083\t0.0\t1\t0 2083\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t81\t1489.9\t0\t81\r\n",
      "4\t1\t372.5\t0\t1\r\n",
      "5\t1\t93.1\t0\t1\r\n",
      "6\t2\t23.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2481.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2481.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.05 s (45 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,281\r\n",
      "Reads with adapters:                   128,217 (96.2%)\r\n",
      "Reads written (passing filters):       133,281 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,534,014 bp\r\n",
      "Total written (filtered):     36,846,124 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 127623 times.\r\n",
      "127590 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t164\t0.0\t0\t164\r\n",
      "21\t127385\t0.0\t0\t127385\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t2082.5\t0\t12\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t20\t0.0\t0\t20\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 594 times.\r\n",
      "37 times, it overlapped the 5' end of a read\r\n",
      "557 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t2082.5\t0\t33\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2082.5\t0\t4\r\n",
      "4\t553\t520.6\t0\t553\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2484.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2484.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.42 s (57 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,351\r\n",
      "Reads with adapters:                    84,584 (88.7%)\r\n",
      "Reads written (passing filters):        95,351 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,090,722 bp\r\n",
      "Total written (filtered):     25,369,536 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 81300 times.\r\n",
      "81271 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1489.9\t0\t3\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t97\t0.0\t0\t97\r\n",
      "21\t81152\t0.0\t0\t81152\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1489.9\t0\t23\r\n",
      "4\t1\t372.5\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3284 times.\r\n",
      "33 times, it overlapped the 5' end of a read\r\n",
      "3251 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1489.9\t0\t32\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1489.9\t0\t13\r\n",
      "4\t3238\t372.5\t0\t3238\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2481.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2481.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.58 s (42 us/read; 1.43 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,281\r\n",
      "Reads with adapters:                     9,863 (7.4%)\r\n",
      "Reads written (passing filters):       133,281 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,476,064 bp\r\n",
      "Total written (filtered):     39,368,847 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 6483 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "6479 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2082.5\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6466\t2082.5\t0\t6466\r\n",
      "4\t6\t520.6\t0\t6\r\n",
      "5\t7\t130.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3380 times.\r\n",
      "3237 times, it overlapped the 5' end of a read\r\n",
      "143 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2082.5\t0\t1\r\n",
      "4\t2\t520.6\t0\t2\r\n",
      "5\t1\t130.2\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3230\t0.0\t1\t0 3230\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t2082.5\t0\t135\r\n",
      "4\t1\t520.6\t0\t1\r\n",
      "5\t7\t130.2\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2486.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2486.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.75 s (38 us/read; 1.59 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,044\r\n",
      "Reads with adapters:                     6,846 (5.4%)\r\n",
      "Reads written (passing filters):       126,044 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    34,406,059 bp\r\n",
      "Total written (filtered):     34,319,662 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3986 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "3978 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1969.4\t0\t8\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3963\t1969.4\t0\t3963\r\n",
      "4\t12\t492.4\t0\t12\r\n",
      "5\t2\t123.1\t0\t2\r\n",
      "6\t1\t30.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2860 times.\r\n",
      "2744 times, it overlapped the 5' end of a read\r\n",
      "116 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t492.4\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2738\t0.0\t1\t0 2738\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1969.4\t0\t108\r\n",
      "4\t4\t492.4\t0\t4\r\n",
      "5\t4\t123.1\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2490.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2490.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.66 s (39 us/read; 1.53 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,544\r\n",
      "Reads with adapters:                     4,182 (4.5%)\r\n",
      "Reads written (passing filters):        93,544 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,917,659 bp\r\n",
      "Total written (filtered):     24,854,516 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1957 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "1946 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1461.6\t0\t10\r\n",
      "4\t1\t365.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1934\t1461.6\t0\t1934\r\n",
      "4\t10\t365.4\t0\t10\r\n",
      "5\t2\t91.4\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2225 times.\r\n",
      "2110 times, it overlapped the 5' end of a read\r\n",
      "115 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1461.6\t0\t3\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2105\t0.0\t1\t0 2105\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t105\t1461.6\t0\t105\r\n",
      "4\t4\t365.4\t0\t4\r\n",
      "5\t6\t91.4\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2490.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2490.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.90 s (42 us/read; 1.44 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,544\r\n",
      "Reads with adapters:                    84,665 (90.5%)\r\n",
      "Reads written (passing filters):        93,544 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,058,150 bp\r\n",
      "Total written (filtered):     23,394,921 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77909 times.\r\n",
      "77881 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1461.6\t0\t3\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t105\t0.0\t0\t105\r\n",
      "21\t77741\t0.0\t0\t77741\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1461.6\t0\t24\r\n",
      "7\t1\t5.7\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6756 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "6694 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1461.6\t0\t60\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1461.6\t0\t18\r\n",
      "4\t6676\t365.4\t0\t6676\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2492.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2492.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.08 s (30 us/read; 2.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,001\r\n",
      "Reads with adapters:                     4,747 (4.6%)\r\n",
      "Reads written (passing filters):       104,001 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,015,777 bp\r\n",
      "Total written (filtered):     30,943,757 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2159 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "2150 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1625.0\t0\t8\r\n",
      "4\t1\t406.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2143\t1625.0\t0\t2143\r\n",
      "4\t7\t406.3\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2588 times.\r\n",
      "2409 times, it overlapped the 5' end of a read\r\n",
      "179 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1625.0\t0\t1\r\n",
      "4\t1\t406.3\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2402\t0.0\t1\t0 2402\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t174\t1625.0\t0\t174\r\n",
      "4\t3\t406.3\t0\t3\r\n",
      "5\t2\t101.6\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2486.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2486.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.25 s (57 us/read; 1.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,044\r\n",
      "Reads with adapters:                   119,942 (95.2%)\r\n",
      "Reads written (passing filters):       126,044 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    34,633,716 bp\r\n",
      "Total written (filtered):     32,220,751 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 113713 times.\r\n",
      "113683 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t144\t0.0\t0\t144\r\n",
      "21\t113512\t0.0\t0\t113512\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1969.4\t0\t25\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6229 times.\r\n",
      "130 times, it overlapped the 5' end of a read\r\n",
      "6099 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t129\t1969.4\t0\t129\r\n",
      "4\t1\t492.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1969.4\t0\t18\r\n",
      "4\t6080\t492.4\t0\t6080\r\n",
      "5\t1\t123.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2487.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2487.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.71 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,701\r\n",
      "Reads with adapters:                   100,833 (96.3%)\r\n",
      "Reads written (passing filters):       104,701 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,537,242 bp\r\n",
      "Total written (filtered):     27,478,461 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97259 times.\r\n",
      "97238 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t135\t0.0\t0\t135\r\n",
      "21\t97073\t0.0\t0\t97073\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1636.0\t0\t10\r\n",
      "5\t2\t102.2\t0\t2\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3574 times.\r\n",
      "111 times, it overlapped the 5' end of a read\r\n",
      "3463 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1636.0\t0\t108\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1636.0\t0\t6\r\n",
      "4\t3457\t409.0\t0\t3457\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2487.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2487.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.23 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,701\r\n",
      "Reads with adapters:                     6,540 (6.2%)\r\n",
      "Reads written (passing filters):       104,701 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,445,712 bp\r\n",
      "Total written (filtered):     29,378,471 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4410 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "4404 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1636.0\t0\t5\r\n",
      "4\t1\t409.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4392\t1636.0\t0\t4392\r\n",
      "4\t4\t409.0\t0\t4\r\n",
      "5\t7\t102.2\t0\t7\r\n",
      "8\t1\t1.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2130 times.\r\n",
      "1984 times, it overlapped the 5' end of a read\r\n",
      "146 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t409.0\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1981\t0.0\t1\t0 1981\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t140\t1636.0\t0\t140\r\n",
      "4\t3\t409.0\t0\t3\r\n",
      "5\t3\t102.2\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2492.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2492.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.20 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,001\r\n",
      "Reads with adapters:                   100,794 (96.9%)\r\n",
      "Reads written (passing filters):       104,001 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,039,910 bp\r\n",
      "Total written (filtered):     28,926,968 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 100532 times.\r\n",
      "100524 times, it overlapped the 5' end of a read\r\n",
      "8 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1625.0\t0\t1\r\n",
      "18\t23\t0.0\t0\t23\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t134\t0.0\t0\t134\r\n",
      "21\t100351\t0.0\t0\t100351\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1625.0\t0\t4\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 262 times.\r\n",
      "94 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t1625.0\t0\t93\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1625.0\t0\t1\r\n",
      "4\t167\t406.3\t0\t167\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2494.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2494.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.43 s (58 us/read; 1.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,384\r\n",
      "Reads with adapters:                    75,465 (80.0%)\r\n",
      "Reads written (passing filters):        94,384 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,084,185 bp\r\n",
      "Total written (filtered):     26,507,315 bp (94.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 75008 times.\r\n",
      "74966 times, it overlapped the 5' end of a read\r\n",
      "42 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t108\t0.0\t0\t108\r\n",
      "21\t74827\t0.0\t0\t74827\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1474.8\t0\t39\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 457 times.\r\n",
      "65 times, it overlapped the 5' end of a read\r\n",
      "392 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t65\t1474.8\t0\t65\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1474.8\t0\t14\r\n",
      "4\t378\t368.7\t0\t378\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2494.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2494.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.21 s (45 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,384\r\n",
      "Reads with adapters:                     5,029 (5.3%)\r\n",
      "Reads written (passing filters):        94,384 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,041,357 bp\r\n",
      "Total written (filtered):     27,971,799 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2692 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2684 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1474.8\t0\t7\r\n",
      "5\t1\t92.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2679\t1474.8\t0\t2679\r\n",
      "4\t4\t368.7\t0\t4\r\n",
      "5\t1\t92.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2337 times.\r\n",
      "2261 times, it overlapped the 5' end of a read\r\n",
      "76 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t368.7\t0\t2\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2254\t0.0\t1\t0 2254\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t72\t1474.8\t0\t72\r\n",
      "4\t2\t368.7\t0\t2\r\n",
      "5\t1\t92.2\t0\t1\r\n",
      "249\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2495.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2495.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.17 s (37 us/read; 1.61 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,562\r\n",
      "Reads with adapters:                     5,338 (4.8%)\r\n",
      "Reads written (passing filters):       111,562 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,729,910 bp\r\n",
      "Total written (filtered):     32,652,553 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2649 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2646 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1743.2\t0\t2\r\n",
      "4\t1\t435.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2623\t1743.2\t0\t2623\r\n",
      "4\t11\t435.8\t0\t11\r\n",
      "5\t12\t108.9\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2689 times.\r\n",
      "2556 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1743.2\t0\t1\r\n",
      "4\t1\t435.8\t0\t1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2550\t0.0\t1\t0 2550\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t125\t1743.2\t0\t125\r\n",
      "4\t3\t435.8\t0\t3\r\n",
      "5\t4\t108.9\t0\t4\r\n",
      "6\t1\t27.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2496.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2496.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.28 s (35 us/read; 1.70 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,401\r\n",
      "Reads with adapters:                     6,095 (5.0%)\r\n",
      "Reads written (passing filters):       121,401 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,849,288 bp\r\n",
      "Total written (filtered):     35,764,525 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3214 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "3210 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1896.9\t0\t3\r\n",
      "4\t1\t474.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3198\t1896.9\t0\t3198\r\n",
      "4\t6\t474.2\t0\t6\r\n",
      "5\t5\t118.6\t0\t5\r\n",
      "6\t1\t29.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2881 times.\r\n",
      "2774 times, it overlapped the 5' end of a read\r\n",
      "107 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1896.9\t0\t4\r\n",
      "4\t1\t474.2\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2765\t0.0\t1\t0 2765\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t103\t1896.9\t0\t103\r\n",
      "4\t2\t474.2\t0\t2\r\n",
      "5\t2\t118.6\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2497.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2497.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.83 s (30 us/read; 1.98 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,306\r\n",
      "Reads with adapters:                     5,447 (4.3%)\r\n",
      "Reads written (passing filters):       126,306 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,315,581 bp\r\n",
      "Total written (filtered):     31,236,586 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2667 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2666 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1973.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2643\t1973.5\t0\t2643\r\n",
      "4\t13\t493.4\t0\t13\r\n",
      "5\t10\t123.3\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2780 times.\r\n",
      "2611 times, it overlapped the 5' end of a read\r\n",
      "169 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1973.5\t0\t1\r\n",
      "4\t1\t493.4\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2606\t0.0\t1\t0 2606\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t164\t1973.5\t0\t164\r\n",
      "4\t3\t493.4\t0\t3\r\n",
      "5\t2\t123.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2495.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2495.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.39 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,562\r\n",
      "Reads with adapters:                   107,761 (96.6%)\r\n",
      "Reads written (passing filters):       111,562 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,770,144 bp\r\n",
      "Total written (filtered):     30,516,977 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 107056 times.\r\n",
      "107042 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1743.2\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t25\t0.0\t0\t25\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t106867\t0.0\t0\t106867\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1743.2\t0\t5\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 705 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "651 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t51\t1743.2\t0\t51\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1743.2\t0\t4\r\n",
      "4\t647\t435.8\t0\t647\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2496.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2496.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.92 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,401\r\n",
      "Reads with adapters:                   115,342 (95.0%)\r\n",
      "Reads written (passing filters):       121,401 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,906,517 bp\r\n",
      "Total written (filtered):     33,494,738 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 114482 times.\r\n",
      "114449 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t149\t0.0\t0\t149\r\n",
      "21\t114263\t0.0\t0\t114263\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1896.9\t0\t14\r\n",
      "4\t1\t474.2\t0\t1\r\n",
      "7\t1\t7.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 860 times.\r\n",
      "25 times, it overlapped the 5' end of a read\r\n",
      "835 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1896.9\t0\t24\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1896.9\t0\t5\r\n",
      "4\t830\t474.2\t0\t830\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2499.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2499.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.24 s (32 us/read; 1.85 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,894\r\n",
      "Reads with adapters:                     4,877 (4.9%)\r\n",
      "Reads written (passing filters):        99,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,949,235 bp\r\n",
      "Total written (filtered):     27,879,372 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2457 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "2450 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1560.8\t0\t6\r\n",
      "4\t1\t390.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2432\t1560.8\t0\t2432\r\n",
      "4\t13\t390.2\t0\t13\r\n",
      "5\t5\t97.6\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2420 times.\r\n",
      "2300 times, it overlapped the 5' end of a read\r\n",
      "120 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2298\t0.0\t1\t0 2298\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1560.8\t0\t114\r\n",
      "4\t2\t390.2\t0\t2\r\n",
      "5\t4\t97.6\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2500.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2500.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.65 s (28 us/read; 2.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  59,725\r\n",
      "Reads with adapters:                     3,081 (5.2%)\r\n",
      "Reads written (passing filters):        59,725 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,602,550 bp\r\n",
      "Total written (filtered):     16,557,071 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1467 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1464 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t933.2\t0\t2\r\n",
      "4\t1\t233.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1450\t933.2\t0\t1450\r\n",
      "4\t11\t233.3\t0\t11\r\n",
      "5\t3\t58.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1614 times.\r\n",
      "1512 times, it overlapped the 5' end of a read\r\n",
      "102 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t933.2\t0\t1\r\n",
      "4\t2\t233.3\t0\t2\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1508\t0.0\t1\t0 1508\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t101\t933.2\t0\t101\r\n",
      "4\t1\t233.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2497.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2497.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.00 s (55 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,306\r\n",
      "Reads with adapters:                   110,870 (87.8%)\r\n",
      "Reads written (passing filters):       126,306 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,472,642 bp\r\n",
      "Total written (filtered):     29,425,581 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 94184 times.\r\n",
      "94115 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t152\t0.0\t0\t152\r\n",
      "21\t93944\t0.0\t0\t93944\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1973.5\t0\t52\r\n",
      "4\t2\t493.4\t0\t2\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "55\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16686 times.\r\n",
      "53 times, it overlapped the 5' end of a read\r\n",
      "16633 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1973.5\t0\t52\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1973.5\t0\t45\r\n",
      "4\t16587\t493.4\t0\t16587\r\n",
      "6\t1\t30.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2500.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2500.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.19 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  59,725\r\n",
      "Reads with adapters:                    39,492 (66.1%)\r\n",
      "Reads written (passing filters):        59,725 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,684,309 bp\r\n",
      "Total written (filtered):     15,888,981 bp (95.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 37519 times.\r\n",
      "37452 times, it overlapped the 5' end of a read\r\n",
      "67 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t4\t0.0\t0\t4\r\n",
      "20\t54\t0.0\t0\t54\r\n",
      "21\t37387\t0.0\t0\t37387\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t933.2\t0\t60\r\n",
      "4\t2\t233.3\t0\t2\r\n",
      "7\t2\t3.6\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1973 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "1865 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t106\t933.2\t0\t106\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t933.2\t0\t15\r\n",
      "4\t1850\t233.3\t0\t1850\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2499.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2499.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.54 s (55 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,894\r\n",
      "Reads with adapters:                    90,610 (90.7%)\r\n",
      "Reads written (passing filters):        99,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,067,885 bp\r\n",
      "Total written (filtered):     26,207,438 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 87957 times.\r\n",
      "87912 times, it overlapped the 5' end of a read\r\n",
      "45 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t21\t0.0\t0\t21\r\n",
      "20\t118\t0.0\t0\t118\r\n",
      "21\t87764\t0.0\t0\t87764\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1560.8\t0\t25\r\n",
      "4\t2\t390.2\t0\t2\r\n",
      "5\t3\t97.6\t0\t3\r\n",
      "7\t2\t6.1\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2653 times.\r\n",
      "118 times, it overlapped the 5' end of a read\r\n",
      "2535 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t118\t1560.8\t0\t118\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1560.8\t0\t16\r\n",
      "4\t2519\t390.2\t0\t2519\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2502.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2502.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.82 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  38,740\r\n",
      "Reads with adapters:                     2,194 (5.7%)\r\n",
      "Reads written (passing filters):        38,740 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,556,169 bp\r\n",
      "Total written (filtered):     11,522,462 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1023 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "1014 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t605.3\t0\t7\r\n",
      "4\t2\t151.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1005\t605.3\t0\t1005\r\n",
      "4\t6\t151.3\t0\t6\r\n",
      "5\t3\t37.8\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1171 times.\r\n",
      "1120 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t605.3\t0\t2\r\n",
      "4\t1\t151.3\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1114\t0.0\t1\t0 1114\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t605.3\t0\t46\r\n",
      "4\t1\t151.3\t0\t1\r\n",
      "5\t2\t37.8\t0\t2\r\n",
      "6\t1\t9.5\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2502.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2502.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.19 s (56 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  38,740\r\n",
      "Reads with adapters:                    36,865 (95.2%)\r\n",
      "Reads written (passing filters):        38,740 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,573,822 bp\r\n",
      "Total written (filtered):     10,801,799 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 36684 times.\r\n",
      "36665 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t605.3\t0\t1\r\n",
      "4\t1\t151.3\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t68\t0.0\t0\t68\r\n",
      "21\t36577\t0.0\t0\t36577\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t605.3\t0\t13\r\n",
      "4\t1\t151.3\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 181 times.\r\n",
      "57 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t56\t605.3\t0\t56\r\n",
      "4\t1\t151.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t605.3\t0\t6\r\n",
      "4\t118\t151.3\t0\t118\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2501.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2501.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.88 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,301\r\n",
      "Reads with adapters:                     4,211 (4.8%)\r\n",
      "Reads written (passing filters):        88,301 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,084,111 bp\r\n",
      "Total written (filtered):     26,020,356 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1981 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1975 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1379.7\t0\t3\r\n",
      "4\t3\t344.9\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1966\t1379.7\t0\t1966\r\n",
      "4\t6\t344.9\t0\t6\r\n",
      "5\t3\t86.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2230 times.\r\n",
      "2133 times, it overlapped the 5' end of a read\r\n",
      "97 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1379.7\t0\t3\r\n",
      "4\t1\t344.9\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2125\t0.0\t1\t0 2125\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t1379.7\t0\t86\r\n",
      "4\t5\t344.9\t0\t5\r\n",
      "5\t6\t86.2\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2501.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2501.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.24 s (59 us/read; 1.01 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,301\r\n",
      "Reads with adapters:                    85,291 (96.6%)\r\n",
      "Reads written (passing filters):        88,301 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,132,018 bp\r\n",
      "Total written (filtered):     24,359,291 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 84179 times.\r\n",
      "84167 times, it overlapped the 5' end of a read\r\n",
      "12 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t111\t0.0\t0\t111\r\n",
      "21\t84024\t0.0\t0\t84024\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1379.7\t0\t9\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1112 times.\r\n",
      "119 times, it overlapped the 5' end of a read\r\n",
      "993 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t1379.7\t0\t112\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1379.7\t0\t4\r\n",
      "4\t989\t344.9\t0\t989\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2504.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2504.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.59 s (36 us/read; 1.69 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,094\r\n",
      "Reads with adapters:                     4,347 (4.3%)\r\n",
      "Reads written (passing filters):       101,094 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,282,122 bp\r\n",
      "Total written (filtered):     30,210,053 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1785 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "1771 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1579.6\t0\t13\r\n",
      "4\t1\t394.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1765\t1579.6\t0\t1765\r\n",
      "4\t4\t394.9\t0\t4\r\n",
      "5\t2\t98.7\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2562 times.\r\n",
      "2459 times, it overlapped the 5' end of a read\r\n",
      "103 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "27\t2458\t0.0\t1\t0 2458\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t100\t1579.6\t0\t100\r\n",
      "4\t1\t394.9\t0\t1\r\n",
      "5\t2\t98.7\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2505.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2505.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.88 s (27 us/read; 2.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,644\r\n",
      "Reads with adapters:                     5,286 (5.0%)\r\n",
      "Reads written (passing filters):       105,644 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,991,327 bp\r\n",
      "Total written (filtered):     29,910,641 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2397 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "2377 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1650.7\t0\t18\r\n",
      "4\t2\t412.7\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2363\t1650.7\t0\t2363\r\n",
      "4\t10\t412.7\t0\t10\r\n",
      "5\t4\t103.2\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2889 times.\r\n",
      "2701 times, it overlapped the 5' end of a read\r\n",
      "188 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t1\t103.2\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2698\t0.0\t1\t0 2698\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t182\t1650.7\t0\t182\r\n",
      "4\t4\t412.7\t0\t4\r\n",
      "5\t2\t103.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2504.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2504.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.41 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,094\r\n",
      "Reads with adapters:                    97,896 (96.8%)\r\n",
      "Reads written (passing filters):       101,094 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,314,087 bp\r\n",
      "Total written (filtered):     28,260,158 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97629 times.\r\n",
      "97613 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t129\t0.0\t0\t129\r\n",
      "21\t97454\t0.0\t0\t97454\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1579.6\t0\t5\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 267 times.\r\n",
      "116 times, it overlapped the 5' end of a read\r\n",
      "151 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t116\t1579.6\t0\t116\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1579.6\t0\t3\r\n",
      "4\t148\t394.9\t0\t148\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2507.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2507.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.29 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,530\r\n",
      "Reads with adapters:                    77,996 (96.9%)\r\n",
      "Reads written (passing filters):        80,530 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,211,553 bp\r\n",
      "Total written (filtered):     22,573,448 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77942 times.\r\n",
      "77933 times, it overlapped the 5' end of a read\r\n",
      "9 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t115\t0.0\t0\t115\r\n",
      "21\t77788\t0.0\t0\t77788\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1258.3\t0\t4\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 54 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1258.3\t0\t22\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1258.3\t0\t5\r\n",
      "4\t27\t314.6\t0\t27\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2506.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2506.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.41 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,859\r\n",
      "Reads with adapters:                     6,109 (5.7%)\r\n",
      "Reads written (passing filters):       107,859 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,382,120 bp\r\n",
      "Total written (filtered):     32,305,384 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3594 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "3581 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1685.3\t0\t12\r\n",
      "4\t1\t421.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3577\t1685.3\t0\t3577\r\n",
      "4\t2\t421.3\t0\t2\r\n",
      "5\t2\t105.3\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2515 times.\r\n",
      "2434 times, it overlapped the 5' end of a read\r\n",
      "81 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t421.3\t0\t1\r\n",
      "27\t2433\t0.0\t1\t0 2433\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t1685.3\t0\t76\r\n",
      "4\t2\t421.3\t0\t2\r\n",
      "5\t2\t105.3\t0\t2\r\n",
      "6\t1\t26.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2505.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2505.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.66 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,644\r\n",
      "Reads with adapters:                    77,379 (73.2%)\r\n",
      "Reads written (passing filters):       105,644 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,120,533 bp\r\n",
      "Total written (filtered):     28,530,253 bp (94.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 75332 times.\r\n",
      "75270 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1650.7\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t114\t0.0\t0\t114\r\n",
      "21\t75125\t0.0\t0\t75125\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t1650.7\t0\t55\r\n",
      "7\t2\t6.4\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2047 times.\r\n",
      "79 times, it overlapped the 5' end of a read\r\n",
      "1968 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1650.7\t0\t75\r\n",
      "4\t1\t412.7\t0\t1\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1650.7\t0\t41\r\n",
      "4\t1927\t412.7\t0\t1927\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2507.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2507.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.98 s (37 us/read; 1.62 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,530\r\n",
      "Reads with adapters:                     3,615 (4.5%)\r\n",
      "Reads written (passing filters):        80,530 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,193,319 bp\r\n",
      "Total written (filtered):     24,131,301 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1423 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "1416 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1258.3\t0\t6\r\n",
      "4\t1\t314.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1410\t1258.3\t0\t1410\r\n",
      "4\t3\t314.6\t0\t3\r\n",
      "5\t2\t78.6\t0\t2\r\n",
      "7\t1\t4.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2192 times.\r\n",
      "2107 times, it overlapped the 5' end of a read\r\n",
      "85 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2103\t0.0\t1\t0 2103\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1258.3\t0\t80\r\n",
      "4\t1\t314.6\t0\t1\r\n",
      "5\t2\t78.6\t0\t2\r\n",
      "300\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2506.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2506.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.24 s (58 us/read; 1.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,859\r\n",
      "Reads with adapters:                   101,797 (94.4%)\r\n",
      "Reads written (passing filters):       107,859 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,412,334 bp\r\n",
      "Total written (filtered):     30,273,934 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 101689 times.\r\n",
      "101676 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t117\t0.0\t0\t117\r\n",
      "21\t101536\t0.0\t0\t101536\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1685.3\t0\t3\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 108 times.\r\n",
      "59 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1685.3\t0\t59\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1685.3\t0\t4\r\n",
      "4\t45\t421.3\t0\t45\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2525.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2525.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.96 s (24 us/read; 2.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,354\r\n",
      "Reads with adapters:                     5,110 (6.1%)\r\n",
      "Reads written (passing filters):        83,354 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,881,955 bp\r\n",
      "Total written (filtered):     24,819,135 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3062 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "3049 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1302.4\t0\t10\r\n",
      "4\t3\t325.6\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3041\t1302.4\t0\t3041\r\n",
      "4\t6\t325.6\t0\t6\r\n",
      "5\t2\t81.4\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2048 times.\r\n",
      "1979 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1302.4\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t1973\t0.0\t1\t0 1973\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1302.4\t0\t63\r\n",
      "4\t1\t325.6\t0\t1\r\n",
      "5\t2\t81.4\t0\t2\r\n",
      "6\t3\t20.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2523.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2523.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.19 s (37 us/read; 1.61 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,578\r\n",
      "Reads with adapters:                     4,901 (4.4%)\r\n",
      "Reads written (passing filters):       112,578 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,825,751 bp\r\n",
      "Total written (filtered):     31,746,075 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2078 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "2067 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1759.0\t0\t8\r\n",
      "4\t2\t439.8\t0\t2\r\n",
      "5\t1\t109.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2052\t1759.0\t0\t2052\r\n",
      "4\t12\t439.8\t0\t12\r\n",
      "5\t3\t109.9\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2823 times.\r\n",
      "2708 times, it overlapped the 5' end of a read\r\n",
      "115 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t439.8\t0\t2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2702\t0.0\t1\t0 2702\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1759.0\t0\t108\r\n",
      "4\t5\t439.8\t0\t5\r\n",
      "5\t2\t109.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2524.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2524.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.89 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,035\r\n",
      "Reads with adapters:                    93,655 (95.5%)\r\n",
      "Reads written (passing filters):        98,035 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,919,185 bp\r\n",
      "Total written (filtered):     24,125,838 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 83375 times.\r\n",
      "83336 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t108\t0.0\t0\t108\r\n",
      "21\t83196\t0.0\t0\t83196\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1531.8\t0\t31\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 10280 times.\r\n",
      "98 times, it overlapped the 5' end of a read\r\n",
      "10182 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t98\t1531.8\t0\t98\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1531.8\t0\t32\r\n",
      "4\t10149\t382.9\t0\t10149\r\n",
      "5\t1\t95.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2525.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2525.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.52 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,354\r\n",
      "Reads with adapters:                    80,763 (96.9%)\r\n",
      "Reads written (passing filters):        83,354 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,903,712 bp\r\n",
      "Total written (filtered):     23,206,497 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80652 times.\r\n",
      "80636 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t21\t0.0\t0\t21\r\n",
      "20\t92\t0.0\t0\t92\r\n",
      "21\t80516\t0.0\t0\t80516\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1302.4\t0\t4\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 111 times.\r\n",
      "33 times, it overlapped the 5' end of a read\r\n",
      "78 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1302.4\t0\t33\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1302.4\t0\t3\r\n",
      "4\t75\t325.6\t0\t75\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2523.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2523.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.96 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,578\r\n",
      "Reads with adapters:                   107,788 (95.7%)\r\n",
      "Reads written (passing filters):       112,578 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,931,830 bp\r\n",
      "Total written (filtered):     29,732,522 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 103936 times.\r\n",
      "103903 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1759.0\t0\t1\r\n",
      "5\t1\t109.9\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t131\t0.0\t0\t131\r\n",
      "21\t103745\t0.0\t0\t103745\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1759.0\t0\t25\r\n",
      "7\t1\t6.9\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3852 times.\r\n",
      "82 times, it overlapped the 5' end of a read\r\n",
      "3770 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t1759.0\t0\t79\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1759.0\t0\t14\r\n",
      "4\t3756\t439.8\t0\t3756\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2524.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2524.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.49 s (46 us/read; 1.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,035\r\n",
      "Reads with adapters:                     3,941 (4.0%)\r\n",
      "Reads written (passing filters):        98,035 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,776,029 bp\r\n",
      "Total written (filtered):     25,715,654 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1786 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1782 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1531.8\t0\t3\r\n",
      "5\t1\t95.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1775\t1531.8\t0\t1775\r\n",
      "4\t6\t382.9\t0\t6\r\n",
      "5\t1\t95.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2155 times.\r\n",
      "2025 times, it overlapped the 5' end of a read\r\n",
      "130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1531.8\t0\t2\r\n",
      "4\t1\t382.9\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2019\t0.0\t1\t0 2019\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t119\t1531.8\t0\t119\r\n",
      "4\t5\t382.9\t0\t5\r\n",
      "5\t6\t95.7\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2526.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2526.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.40 s (30 us/read; 1.97 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,482\r\n",
      "Reads with adapters:                     4,861 (4.4%)\r\n",
      "Reads written (passing filters):       111,482 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,202,505 bp\r\n",
      "Total written (filtered):     29,132,058 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2398 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2395 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1741.9\t0\t2\r\n",
      "5\t1\t108.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2380\t1741.9\t0\t2380\r\n",
      "4\t10\t435.5\t0\t10\r\n",
      "5\t5\t108.9\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2463 times.\r\n",
      "2333 times, it overlapped the 5' end of a read\r\n",
      "130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1741.9\t0\t6\r\n",
      "4\t1\t435.5\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2325\t0.0\t1\t0 2325\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t119\t1741.9\t0\t119\r\n",
      "4\t5\t435.5\t0\t5\r\n",
      "5\t5\t108.9\t0\t5\r\n",
      "6\t1\t27.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2527.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2527.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.88 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,909\r\n",
      "Reads with adapters:                    62,008 (94.1%)\r\n",
      "Reads written (passing filters):        65,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,327,085 bp\r\n",
      "Total written (filtered):     14,202,650 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 51551 times.\r\n",
      "51513 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t68\t0.0\t0\t68\r\n",
      "21\t51424\t0.0\t0\t51424\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1029.8\t0\t35\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 10457 times.\r\n",
      "46 times, it overlapped the 5' end of a read\r\n",
      "10411 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1029.8\t0\t45\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1029.8\t0\t30\r\n",
      "4\t10381\t257.5\t0\t10381\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2526.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2526.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.90 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,482\r\n",
      "Reads with adapters:                    84,155 (75.5%)\r\n",
      "Reads written (passing filters):       111,482 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,397,632 bp\r\n",
      "Total written (filtered):     27,831,659 bp (94.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 72298 times.\r\n",
      "72228 times, it overlapped the 5' end of a read\r\n",
      "70 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t72119\t0.0\t0\t72119\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1741.9\t0\t58\r\n",
      "4\t2\t435.5\t0\t2\r\n",
      "5\t1\t108.9\t0\t1\r\n",
      "6\t2\t27.2\t0\t2\r\n",
      "7\t1\t6.8\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11857 times.\r\n",
      "86 times, it overlapped the 5' end of a read\r\n",
      "11771 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t1741.9\t0\t85\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1741.9\t0\t47\r\n",
      "4\t11723\t435.5\t0\t11723\r\n",
      "5\t1\t108.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2527.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2527.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.03 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,909\r\n",
      "Reads with adapters:                     3,268 (5.0%)\r\n",
      "Reads written (passing filters):        65,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,073,964 bp\r\n",
      "Total written (filtered):     15,033,561 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1902 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "1889 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1029.8\t0\t13\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1880\t1029.8\t0\t1880\r\n",
      "4\t5\t257.5\t0\t5\r\n",
      "5\t4\t64.4\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1366 times.\r\n",
      "1275 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1029.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1273\t0.0\t1\t0 1273\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t84\t1029.8\t0\t84\r\n",
      "4\t3\t257.5\t0\t3\r\n",
      "5\t4\t64.4\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2529.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2529.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.35 s (37 us/read; 1.64 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,229\r\n",
      "Reads with adapters:                     3,379 (3.7%)\r\n",
      "Reads written (passing filters):        91,229 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,578,190 bp\r\n",
      "Total written (filtered):     24,529,318 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1590 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1589 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1425.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1576\t1425.5\t0\t1576\r\n",
      "4\t7\t356.4\t0\t7\r\n",
      "5\t5\t89.1\t0\t5\r\n",
      "6\t1\t22.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1789 times.\r\n",
      "1614 times, it overlapped the 5' end of a read\r\n",
      "175 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t356.4\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t1607\t0.0\t1\t0 1607\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t168\t1425.5\t0\t168\r\n",
      "4\t4\t356.4\t0\t4\r\n",
      "5\t3\t89.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2528.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2528.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.78 s (43 us/read; 1.38 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 110,123\r\n",
      "Reads with adapters:                     4,524 (4.1%)\r\n",
      "Reads written (passing filters):       110,123 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,193,042 bp\r\n",
      "Total written (filtered):     24,134,906 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2503 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2495 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1720.7\t0\t7\r\n",
      "4\t1\t430.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2475\t1720.7\t0\t2475\r\n",
      "4\t11\t430.2\t0\t11\r\n",
      "5\t7\t107.5\t0\t7\r\n",
      "6\t2\t26.9\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2021 times.\r\n",
      "1857 times, it overlapped the 5' end of a read\r\n",
      "164 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1720.7\t0\t2\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1851\t0.0\t1\t0 1851\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t153\t1720.7\t0\t153\r\n",
      "4\t6\t430.2\t0\t6\r\n",
      "5\t4\t107.5\t0\t4\r\n",
      "6\t1\t26.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2529.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2529.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.10 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,229\r\n",
      "Reads with adapters:                    87,554 (96.0%)\r\n",
      "Reads written (passing filters):        91,229 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,648,548 bp\r\n",
      "Total written (filtered):     22,941,548 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 79718 times.\r\n",
      "79687 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1425.5\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t113\t0.0\t0\t113\r\n",
      "21\t79541\t0.0\t0\t79541\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1425.5\t0\t22\r\n",
      "7\t1\t5.6\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7836 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "7807 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1425.5\t0\t28\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1425.5\t0\t30\r\n",
      "4\t7776\t356.4\t0\t7776\r\n",
      "5\t1\t89.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2530.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2530.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.73 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,386\r\n",
      "Reads with adapters:                     3,853 (4.2%)\r\n",
      "Reads written (passing filters):        91,386 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,148,674 bp\r\n",
      "Total written (filtered):     24,092,346 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1837 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1835 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1427.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1825\t1427.9\t0\t1825\r\n",
      "4\t3\t357.0\t0\t3\r\n",
      "5\t7\t89.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2016 times.\r\n",
      "1872 times, it overlapped the 5' end of a read\r\n",
      "144 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1427.9\t0\t5\r\n",
      "4\t3\t357.0\t0\t3\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1862\t0.0\t1\t0 1862\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t136\t1427.9\t0\t136\r\n",
      "4\t3\t357.0\t0\t3\r\n",
      "5\t5\t89.2\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2528.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2528.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.03 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 110,123\r\n",
      "Reads with adapters:                   103,438 (93.9%)\r\n",
      "Reads written (passing filters):       110,123 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,346,887 bp\r\n",
      "Total written (filtered):     22,601,634 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 78309 times.\r\n",
      "78244 times, it overlapped the 5' end of a read\r\n",
      "65 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t104\t0.0\t0\t104\r\n",
      "21\t78121\t0.0\t0\t78121\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1720.7\t0\t58\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "74\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25129 times.\r\n",
      "37 times, it overlapped the 5' end of a read\r\n",
      "25092 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1720.7\t0\t35\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1720.7\t0\t54\r\n",
      "4\t25037\t430.2\t0\t25037\r\n",
      "5\t1\t107.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2531.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2531.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.50 s (30 us/read; 1.98 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 115,732\r\n",
      "Reads with adapters:                     5,459 (4.7%)\r\n",
      "Reads written (passing filters):       115,732 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,778,708 bp\r\n",
      "Total written (filtered):     30,703,400 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2849 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2846 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1808.3\t0\t2\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4\t1\t452.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2833\t1808.3\t0\t2833\r\n",
      "4\t7\t452.1\t0\t7\r\n",
      "5\t6\t113.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2610 times.\r\n",
      "2463 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1808.3\t0\t8\r\n",
      "5\t1\t113.0\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2451\t0.0\t1\t0 2451\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t139\t1808.3\t0\t139\r\n",
      "4\t2\t452.1\t0\t2\r\n",
      "5\t6\t113.0\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2530.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2530.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.22 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,386\r\n",
      "Reads with adapters:                    83,709 (91.6%)\r\n",
      "Reads written (passing filters):        91,386 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,241,225 bp\r\n",
      "Total written (filtered):     22,636,585 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74557 times.\r\n",
      "74516 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t97\t0.0\t0\t97\r\n",
      "21\t74400\t0.0\t0\t74400\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1427.9\t0\t30\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9152 times.\r\n",
      "16 times, it overlapped the 5' end of a read\r\n",
      "9136 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1427.9\t0\t16\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1427.9\t0\t28\r\n",
      "4\t9108\t357.0\t0\t9108\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2531.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2531.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.11 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 115,732\r\n",
      "Reads with adapters:                    90,210 (77.9%)\r\n",
      "Reads written (passing filters):       115,732 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,860,688 bp\r\n",
      "Total written (filtered):     29,094,986 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 82556 times.\r\n",
      "82483 times, it overlapped the 5' end of a read\r\n",
      "73 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t137\t0.0\t0\t137\r\n",
      "21\t82316\t0.0\t0\t82316\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1808.3\t0\t62\r\n",
      "4\t1\t452.1\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7654 times.\r\n",
      "66 times, it overlapped the 5' end of a read\r\n",
      "7588 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1808.3\t0\t63\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1808.3\t0\t39\r\n",
      "4\t7549\t452.1\t0\t7549\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2533.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2533.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.10 s (27 us/read; 2.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 113,846\r\n",
      "Reads with adapters:                     5,423 (4.8%)\r\n",
      "Reads written (passing filters):       113,846 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,836,987 bp\r\n",
      "Total written (filtered):     31,759,908 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2734 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2729 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1778.8\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2711\t1778.8\t0\t2711\r\n",
      "4\t13\t444.7\t0\t13\r\n",
      "5\t5\t111.2\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2689 times.\r\n",
      "2540 times, it overlapped the 5' end of a read\r\n",
      "149 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1778.8\t0\t2\r\n",
      "4\t6\t444.7\t0\t6\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2529\t0.0\t1\t0 2529\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t141\t1778.8\t0\t141\r\n",
      "5\t8\t111.2\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2534.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2534.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.08 s (27 us/read; 2.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 113,405\r\n",
      "Reads with adapters:                     5,536 (4.9%)\r\n",
      "Reads written (passing filters):       113,405 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,334,509 bp\r\n",
      "Total written (filtered):     31,255,605 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2786 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2784 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1772.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2766\t1772.0\t0\t2766\r\n",
      "4\t10\t443.0\t0\t10\r\n",
      "5\t8\t110.7\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2750 times.\r\n",
      "2601 times, it overlapped the 5' end of a read\r\n",
      "149 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1772.0\t0\t1\r\n",
      "4\t6\t443.0\t0\t6\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2592\t0.0\t1\t0 2592\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t142\t1772.0\t0\t142\r\n",
      "4\t2\t443.0\t0\t2\r\n",
      "5\t5\t110.7\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2533.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2533.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.89 s (43 us/read; 1.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 113,846\r\n",
      "Reads with adapters:                   105,675 (92.8%)\r\n",
      "Reads written (passing filters):       113,846 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,925,029 bp\r\n",
      "Total written (filtered):     29,801,501 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 99945 times.\r\n",
      "99911 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1778.8\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t168\t0.0\t0\t168\r\n",
      "21\t99702\t0.0\t0\t99702\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1778.8\t0\t23\r\n",
      "4\t1\t444.7\t0\t1\r\n",
      "5\t1\t111.2\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5730 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "5712 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1778.8\t0\t17\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1778.8\t0\t24\r\n",
      "4\t5687\t444.7\t0\t5687\r\n",
      "5\t1\t111.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2532.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2532.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.60 s (35 us/read; 1.71 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,979\r\n",
      "Reads with adapters:                     3,027 (1.9%)\r\n",
      "Reads written (passing filters):       159,979 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,581,565 bp\r\n",
      "Total written (filtered):     16,533,467 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1100 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "1093 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2499.7\t0\t6\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1068\t2499.7\t0\t1068\r\n",
      "4\t15\t624.9\t0\t15\r\n",
      "5\t10\t156.2\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1927 times.\r\n",
      "1626 times, it overlapped the 5' end of a read\r\n",
      "301 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2499.7\t0\t7\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1617\t0.0\t1\t0 1617\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t267\t2499.7\t0\t267\r\n",
      "4\t18\t624.9\t0\t18\r\n",
      "5\t13\t156.2\t0\t13\r\n",
      "6\t1\t39.1\t0\t1\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2532.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2532.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.55 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,979\r\n",
      "Reads with adapters:                   147,682 (92.3%)\r\n",
      "Reads written (passing filters):       159,979 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,443,374 bp\r\n",
      "Total written (filtered):     15,761,998 bp (90.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64232 times.\r\n",
      "64036 times, it overlapped the 5' end of a read\r\n",
      "196 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t75\t0.0\t0\t75\r\n",
      "21\t63943\t0.0\t0\t63943\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t185\t2499.7\t0\t185\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83450 times.\r\n",
      "61 times, it overlapped the 5' end of a read\r\n",
      "83389 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t2499.7\t0\t55\r\n",
      "7\t1\t9.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t186\t2499.7\t0\t186\r\n",
      "4\t83197\t624.9\t0\t83197\r\n",
      "5\t6\t156.2\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2535.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2535.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.81 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,491\r\n",
      "Reads with adapters:                    66,626 (93.2%)\r\n",
      "Reads written (passing filters):        71,491 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,175,644 bp\r\n",
      "Total written (filtered):     16,919,511 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58127 times.\r\n",
      "58086 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t85\t0.0\t0\t85\r\n",
      "21\t57991\t0.0\t0\t57991\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1117.0\t0\t32\r\n",
      "4\t1\t279.3\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8499 times.\r\n",
      "82 times, it overlapped the 5' end of a read\r\n",
      "8417 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1117.0\t0\t80\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1117.0\t0\t21\r\n",
      "4\t8396\t279.3\t0\t8396\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2535.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2535.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.01 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,491\r\n",
      "Reads with adapters:                     3,590 (5.0%)\r\n",
      "Reads written (passing filters):        71,491 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,945,427 bp\r\n",
      "Total written (filtered):     17,896,000 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1883 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1882 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1117.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1877\t1117.0\t0\t1877\r\n",
      "4\t4\t279.3\t0\t4\r\n",
      "5\t1\t69.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1707 times.\r\n",
      "1613 times, it overlapped the 5' end of a read\r\n",
      "94 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1117.0\t0\t2\r\n",
      "4\t1\t279.3\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1609\t0.0\t1\t0 1609\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t87\t1117.0\t0\t87\r\n",
      "4\t3\t279.3\t0\t3\r\n",
      "5\t4\t69.8\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2534.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2534.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.22 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 113,405\r\n",
      "Reads with adapters:                   108,024 (95.3%)\r\n",
      "Reads written (passing filters):       113,405 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,406,056 bp\r\n",
      "Total written (filtered):     29,222,176 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102949 times.\r\n",
      "102920 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t102767\t0.0\t0\t102767\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1772.0\t0\t20\r\n",
      "4\t1\t443.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5075 times.\r\n",
      "60 times, it overlapped the 5' end of a read\r\n",
      "5015 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t1772.0\t0\t57\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1772.0\t0\t23\r\n",
      "4\t4992\t443.0\t0\t4992\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2537.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2537.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.03 s (25 us/read; 2.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,149\r\n",
      "Reads with adapters:                     3,600 (4.4%)\r\n",
      "Reads written (passing filters):        81,149 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,034,300 bp\r\n",
      "Total written (filtered):     22,981,483 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1721 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1717 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1268.0\t0\t2\r\n",
      "4\t2\t317.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1708\t1268.0\t0\t1708\r\n",
      "4\t5\t317.0\t0\t5\r\n",
      "5\t4\t79.2\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1879 times.\r\n",
      "1741 times, it overlapped the 5' end of a read\r\n",
      "138 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1268.0\t0\t3\r\n",
      "4\t1\t317.0\t0\t1\r\n",
      "27\t1737\t0.0\t1\t0 1737\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t127\t1268.0\t0\t127\r\n",
      "4\t4\t317.0\t0\t4\r\n",
      "5\t6\t79.2\t0\t6\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2539.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2539.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.97 s (25 us/read; 2.38 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  78,191\r\n",
      "Reads with adapters:                     4,159 (5.3%)\r\n",
      "Reads written (passing filters):        78,191 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,147,153 bp\r\n",
      "Total written (filtered):     23,088,652 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2159 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "2149 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1221.7\t0\t10\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2140\t1221.7\t0\t2140\r\n",
      "4\t6\t305.4\t0\t6\r\n",
      "5\t3\t76.4\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2000 times.\r\n",
      "1917 times, it overlapped the 5' end of a read\r\n",
      "83 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t1913\t0.0\t1\t0 1913\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t1221.7\t0\t79\r\n",
      "5\t4\t76.4\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2536.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2536.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.70 s (44 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,811\r\n",
      "Reads with adapters:                     4,265 (4.0%)\r\n",
      "Reads written (passing filters):       105,811 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,674,888 bp\r\n",
      "Total written (filtered):     23,616,434 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2177 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2175 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1653.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2159\t1653.3\t0\t2159\r\n",
      "4\t11\t413.3\t0\t11\r\n",
      "5\t5\t103.3\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2088 times.\r\n",
      "1904 times, it overlapped the 5' end of a read\r\n",
      "184 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1653.3\t0\t3\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1899\t0.0\t1\t0 1899\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t174\t1653.3\t0\t174\r\n",
      "4\t4\t413.3\t0\t4\r\n",
      "5\t6\t103.3\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2537.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2537.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.82 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,149\r\n",
      "Reads with adapters:                    76,850 (94.7%)\r\n",
      "Reads written (passing filters):        81,149 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,085,946 bp\r\n",
      "Total written (filtered):     21,535,859 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 73097 times.\r\n",
      "73077 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t109\t0.0\t0\t109\r\n",
      "21\t72939\t0.0\t0\t72939\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1268.0\t0\t17\r\n",
      "4\t1\t317.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3753 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "3679 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t1268.0\t0\t70\r\n",
      "4\t1\t317.0\t0\t1\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1268.0\t0\t17\r\n",
      "4\t3661\t317.0\t0\t3661\r\n",
      "5\t1\t79.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2539.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2539.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.51 s (32 us/read; 1.87 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  78,191\r\n",
      "Reads with adapters:                    74,745 (95.6%)\r\n",
      "Reads written (passing filters):        78,191 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,195,641 bp\r\n",
      "Total written (filtered):     21,634,045 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74249 times.\r\n",
      "74237 times, it overlapped the 5' end of a read\r\n",
      "12 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1221.7\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t95\t0.0\t0\t95\r\n",
      "21\t74119\t0.0\t0\t74119\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1221.7\t0\t9\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 496 times.\r\n",
      "112 times, it overlapped the 5' end of a read\r\n",
      "384 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t1221.7\t0\t112\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1221.7\t0\t5\r\n",
      "4\t379\t305.4\t0\t379\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2542.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2542.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (26 us/read; 2.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     831\r\n",
      "Reads with adapters:                       734 (88.3%)\r\n",
      "Reads written (passing filters):           831 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       230,449 bp\r\n",
      "Total written (filtered):        216,041 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 677 times.\r\n",
      "675 times, it overlapped the 5' end of a read\r\n",
      "2 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t675\t0.0\t0\t675\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t13.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 57 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "57 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t13.0\t0\t1\r\n",
      "4\t56\t3.2\t0\t56\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2536.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2536.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.61 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,811\r\n",
      "Reads with adapters:                    99,501 (94.0%)\r\n",
      "Reads written (passing filters):       105,811 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,033,174 bp\r\n",
      "Total written (filtered):     22,325,876 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 76949 times.\r\n",
      "76876 times, it overlapped the 5' end of a read\r\n",
      "73 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1653.3\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t116\t0.0\t0\t116\r\n",
      "21\t76727\t0.0\t0\t76727\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1653.3\t0\t62\r\n",
      "4\t1\t413.3\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 22552 times.\r\n",
      "113 times, it overlapped the 5' end of a read\r\n",
      "22439 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1653.3\t0\t108\r\n",
      "27\t5\t0.0\t1\t4 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1653.3\t0\t52\r\n",
      "4\t22386\t413.3\t0\t22386\r\n",
      "5\t1\t103.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2542.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2542.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (23 us/read; 2.58 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     831\r\n",
      "Reads with adapters:                        51 (6.1%)\r\n",
      "Reads written (passing filters):           831 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       224,783 bp\r\n",
      "Total written (filtered):        224,292 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t13.0\t0\t35\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "2 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t14\t0.0\t1\t0 14\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t13.0\t0\t1\r\n",
      "5\t1\t0.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2538.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2538.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.91 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,341\r\n",
      "Reads with adapters:                     3,259 (3.7%)\r\n",
      "Reads written (passing filters):        87,341 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,461,909 bp\r\n",
      "Total written (filtered):     20,418,630 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1717 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "1704 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1364.7\t0\t11\r\n",
      "4\t2\t341.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1690\t1364.7\t0\t1690\r\n",
      "4\t10\t341.2\t0\t10\r\n",
      "5\t4\t85.3\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1542 times.\r\n",
      "1395 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1393\t0.0\t1\t0 1393\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t141\t1364.7\t0\t141\r\n",
      "4\t4\t341.2\t0\t4\r\n",
      "5\t2\t85.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2538.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2538.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.52 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,341\r\n",
      "Reads with adapters:                    81,539 (93.4%)\r\n",
      "Reads written (passing filters):        87,341 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,739,324 bp\r\n",
      "Total written (filtered):     19,292,480 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 65811 times.\r\n",
      "65753 times, it overlapped the 5' end of a read\r\n",
      "58 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t77\t0.0\t0\t77\r\n",
      "21\t65655\t0.0\t0\t65655\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1364.7\t0\t44\r\n",
      "4\t1\t341.2\t0\t1\r\n",
      "5\t1\t85.3\t0\t1\r\n",
      "7\t1\t5.3\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 15728 times.\r\n",
      "79 times, it overlapped the 5' end of a read\r\n",
      "15649 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t72\t1364.7\t0\t72\r\n",
      "27\t7\t0.0\t1\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1364.7\t0\t36\r\n",
      "4\t15611\t341.2\t0\t15611\r\n",
      "5\t2\t85.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2541.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2541.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.48 s (38 us/read; 1.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  64,637\r\n",
      "Reads with adapters:                    49,288 (76.3%)\r\n",
      "Reads written (passing filters):        64,637 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,740,713 bp\r\n",
      "Total written (filtered):     16,770,744 bp (94.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45439 times.\r\n",
      "45417 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1010.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t60\t0.0\t0\t60\r\n",
      "21\t45341\t0.0\t0\t45341\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1010.0\t0\t17\r\n",
      "4\t1\t252.5\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3849 times.\r\n",
      "60 times, it overlapped the 5' end of a read\r\n",
      "3789 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1010.0\t0\t60\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1010.0\t0\t15\r\n",
      "4\t3773\t252.5\t0\t3773\r\n",
      "5\t1\t63.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2540.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2540.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.61 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,122\r\n",
      "Reads with adapters:                    76,164 (87.4%)\r\n",
      "Reads written (passing filters):        87,122 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,863,123 bp\r\n",
      "Total written (filtered):     21,405,450 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67768 times.\r\n",
      "67732 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t67624\t0.0\t0\t67624\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1361.3\t0\t29\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8396 times.\r\n",
      "63 times, it overlapped the 5' end of a read\r\n",
      "8333 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1361.3\t0\t63\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1361.3\t0\t29\r\n",
      "4\t8304\t340.3\t0\t8304\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2541.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2541.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.12 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  64,637\r\n",
      "Reads with adapters:                     2,910 (4.5%)\r\n",
      "Reads written (passing filters):        64,637 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,661,856 bp\r\n",
      "Total written (filtered):     17,617,835 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1380 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1377 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1010.0\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1368\t1010.0\t0\t1368\r\n",
      "4\t4\t252.5\t0\t4\r\n",
      "5\t4\t63.1\t0\t4\r\n",
      "8\t1\t1.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1530 times.\r\n",
      "1458 times, it overlapped the 5' end of a read\r\n",
      "72 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1457\t0.0\t1\t0 1457\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t1010.0\t0\t64\r\n",
      "4\t1\t252.5\t0\t1\r\n",
      "5\t4\t63.1\t0\t4\r\n",
      "6\t2\t15.8\t0\t2\r\n",
      "271\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2540.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2540.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.76 s (43 us/read; 1.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,122\r\n",
      "Reads with adapters:                     4,275 (4.9%)\r\n",
      "Reads written (passing filters):        87,122 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,744,340 bp\r\n",
      "Total written (filtered):     22,685,702 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2273 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "2262 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1361.3\t0\t8\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2258\t1361.3\t0\t2258\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "5\t1\t85.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2002 times.\r\n",
      "1911 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t340.3\t0\t2\r\n",
      "5\t1\t85.1\t0\t1\r\n",
      "27\t1908\t0.0\t1\t0 1908\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t1361.3\t0\t85\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "5\t3\t85.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2543.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2543.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.93 s (40 us/read; 1.52 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 124,738\r\n",
      "Reads with adapters:                     5,372 (4.3%)\r\n",
      "Reads written (passing filters):       124,738 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,642,177 bp\r\n",
      "Total written (filtered):     28,573,079 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2991 times.\r\n",
      "12 times, it overlapped the 5' end of a read\r\n",
      "2979 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1949.0\t0\t12\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2958\t1949.0\t0\t2958\r\n",
      "4\t13\t487.3\t0\t13\r\n",
      "5\t7\t121.8\t0\t7\r\n",
      "7\t1\t7.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2381 times.\r\n",
      "2206 times, it overlapped the 5' end of a read\r\n",
      "175 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2202\t0.0\t1\t0 2202\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t169\t1949.0\t0\t169\r\n",
      "4\t1\t487.3\t0\t1\r\n",
      "5\t5\t121.8\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2543.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2543.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.49 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 124,738\r\n",
      "Reads with adapters:                   109,840 (88.1%)\r\n",
      "Reads written (passing filters):       124,738 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,999,795 bp\r\n",
      "Total written (filtered):     27,113,654 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 84978 times.\r\n",
      "84884 times, it overlapped the 5' end of a read\r\n",
      "94 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t121\t0.0\t0\t121\r\n",
      "21\t84730\t0.0\t0\t84730\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1949.0\t0\t74\r\n",
      "4\t3\t487.3\t0\t3\r\n",
      "5\t2\t121.8\t0\t2\r\n",
      "7\t1\t7.6\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 24862 times.\r\n",
      "122 times, it overlapped the 5' end of a read\r\n",
      "24740 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t119\t1949.0\t0\t119\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1949.0\t0\t71\r\n",
      "4\t24667\t487.3\t0\t24667\r\n",
      "5\t2\t121.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2545.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2545.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.29 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,963\r\n",
      "Reads with adapters:                     2,964 (3.8%)\r\n",
      "Reads written (passing filters):        77,963 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,878,414 bp\r\n",
      "Total written (filtered):     18,832,420 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1285 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1282 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1218.2\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1272\t1218.2\t0\t1272\r\n",
      "4\t2\t304.5\t0\t2\r\n",
      "5\t7\t76.1\t0\t7\r\n",
      "6\t1\t19.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1679 times.\r\n",
      "1545 times, it overlapped the 5' end of a read\r\n",
      "134 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1541\t0.0\t1\t0 1541\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t1218.2\t0\t128\r\n",
      "4\t2\t304.5\t0\t2\r\n",
      "5\t4\t76.1\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2545.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2545.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.18 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,963\r\n",
      "Reads with adapters:                    74,057 (95.0%)\r\n",
      "Reads written (passing filters):        77,963 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,016,631 bp\r\n",
      "Total written (filtered):     17,651,897 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62744 times.\r\n",
      "62709 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t68\t0.0\t0\t68\r\n",
      "21\t62621\t0.0\t0\t62621\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1218.2\t0\t26\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11313 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "11266 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1218.2\t0\t47\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1218.2\t0\t22\r\n",
      "4\t11244\t304.5\t0\t11244\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2546.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2546.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.58 s (32 us/read; 1.88 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 143,703\r\n",
      "Reads with adapters:                     6,574 (4.6%)\r\n",
      "Reads written (passing filters):       143,703 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    40,022,243 bp\r\n",
      "Total written (filtered):     39,925,070 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3210 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "3202 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2245.4\t0\t6\r\n",
      "4\t1\t561.3\t0\t1\r\n",
      "5\t1\t140.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3182\t2245.4\t0\t3182\r\n",
      "4\t16\t561.3\t0\t16\r\n",
      "5\t4\t140.3\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3364 times.\r\n",
      "3217 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2245.4\t0\t2\r\n",
      "4\t2\t561.3\t0\t2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3209\t0.0\t1\t0 3209\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t137\t2245.4\t0\t137\r\n",
      "4\t3\t561.3\t0\t3\r\n",
      "5\t4\t140.3\t0\t4\r\n",
      "6\t2\t35.1\t0\t2\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2547.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2547.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.36 s (36 us/read; 1.64 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,955\r\n",
      "Reads with adapters:                     4,617 (5.0%)\r\n",
      "Reads written (passing filters):        91,955 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,925,050 bp\r\n",
      "Total written (filtered):     24,861,199 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2438 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "2429 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1436.8\t0\t9\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2421\t1436.8\t0\t2421\r\n",
      "4\t7\t359.2\t0\t7\r\n",
      "5\t1\t89.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2179 times.\r\n",
      "2070 times, it overlapped the 5' end of a read\r\n",
      "109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2069\t0.0\t1\t0 2069\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t99\t1436.8\t0\t99\r\n",
      "4\t4\t359.2\t0\t4\r\n",
      "5\t4\t89.8\t0\t4\r\n",
      "6\t1\t22.4\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2544.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2544.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.48 s (35 us/read; 1.70 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 212,406\r\n",
      "Reads with adapters:                     8,684 (4.1%)\r\n",
      "Reads written (passing filters):       212,406 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,916,616 bp\r\n",
      "Total written (filtered):     52,795,930 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4403 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "4384 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t3318.8\t0\t12\r\n",
      "4\t6\t829.7\t0\t6\r\n",
      "5\t1\t207.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4349\t3318.8\t0\t4349\r\n",
      "4\t24\t829.7\t0\t24\r\n",
      "5\t10\t207.4\t0\t10\r\n",
      "6\t1\t51.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4281 times.\r\n",
      "3930 times, it overlapped the 5' end of a read\r\n",
      "351 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t3318.8\t0\t1\r\n",
      "4\t1\t829.7\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3926\t0.0\t1\t0 3926\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t327\t3318.8\t0\t327\r\n",
      "4\t7\t829.7\t0\t7\r\n",
      "5\t16\t207.4\t0\t16\r\n",
      "272\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2544.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2544.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.78 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 212,406\r\n",
      "Reads with adapters:                   201,849 (95.0%)\r\n",
      "Reads written (passing filters):       212,406 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    53,358,782 bp\r\n",
      "Total written (filtered):     49,616,260 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 172494 times.\r\n",
      "172398 times, it overlapped the 5' end of a read\r\n",
      "96 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t20\t0.0\t0\t20\r\n",
      "19\t43\t0.0\t0\t43\r\n",
      "20\t228\t0.0\t0\t228\r\n",
      "21\t172102\t0.0\t0\t172102\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t3318.8\t0\t76\r\n",
      "5\t1\t207.4\t0\t1\r\n",
      "6\t1\t51.9\t0\t1\r\n",
      "7\t1\t13.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 29355 times.\r\n",
      "169 times, it overlapped the 5' end of a read\r\n",
      "29186 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t167\t3318.8\t0\t167\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t3318.8\t0\t80\r\n",
      "4\t29105\t829.7\t0\t29105\r\n",
      "5\t1\t207.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2547.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2547.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.18 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,955\r\n",
      "Reads with adapters:                    88,556 (96.3%)\r\n",
      "Reads written (passing filters):        91,955 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,037,049 bp\r\n",
      "Total written (filtered):     23,286,491 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 82005 times.\r\n",
      "81979 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t101\t0.0\t0\t101\r\n",
      "21\t81860\t0.0\t0\t81860\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1436.8\t0\t16\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6551 times.\r\n",
      "115 times, it overlapped the 5' end of a read\r\n",
      "6436 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1436.8\t0\t114\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1436.8\t0\t15\r\n",
      "4\t6421\t359.2\t0\t6421\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2549.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2549.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.22 s (46 us/read; 1.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,410\r\n",
      "Reads with adapters:                    81,427 (88.1%)\r\n",
      "Reads written (passing filters):        92,410 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,956,022 bp\r\n",
      "Total written (filtered):     20,512,902 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 65716 times.\r\n",
      "65654 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "6\t1\t22.6\t0\t1\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t91\t0.0\t0\t91\r\n",
      "21\t65539\t0.0\t0\t65539\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1443.9\t0\t54\r\n",
      "4\t1\t361.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 15711 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "15668 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1443.9\t0\t42\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1443.9\t0\t35\r\n",
      "4\t15632\t361.0\t0\t15632\r\n",
      "5\t1\t90.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2546.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2546.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.73 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 143,703\r\n",
      "Reads with adapters:                   137,825 (95.9%)\r\n",
      "Reads written (passing filters):       143,703 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    40,193,673 bp\r\n",
      "Total written (filtered):     37,432,015 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 129784 times.\r\n",
      "129744 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t162\t0.0\t0\t162\r\n",
      "21\t129545\t0.0\t0\t129545\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t2245.4\t0\t23\r\n",
      "278\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8041 times.\r\n",
      "117 times, it overlapped the 5' end of a read\r\n",
      "7924 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t2245.4\t0\t112\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t2245.4\t0\t22\r\n",
      "4\t7902\t561.3\t0\t7902\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2549.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2549.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.93 s (43 us/read; 1.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,410\r\n",
      "Reads with adapters:                     4,026 (4.4%)\r\n",
      "Reads written (passing filters):        92,410 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,775,552 bp\r\n",
      "Total written (filtered):     21,721,001 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2106 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "2091 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1443.9\t0\t13\r\n",
      "4\t2\t361.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2080\t1443.9\t0\t2080\r\n",
      "4\t9\t361.0\t0\t9\r\n",
      "5\t2\t90.2\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1920 times.\r\n",
      "1768 times, it overlapped the 5' end of a read\r\n",
      "152 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t361.0\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1765\t0.0\t1\t0 1765\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t1443.9\t0\t146\r\n",
      "4\t2\t361.0\t0\t2\r\n",
      "5\t3\t90.2\t0\t3\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2550.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2550.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.05 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,226\r\n",
      "Reads with adapters:                     5,119 (4.1%)\r\n",
      "Reads written (passing filters):       125,226 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,422,183 bp\r\n",
      "Total written (filtered):     31,349,843 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2615 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "2606 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1956.7\t0\t7\r\n",
      "4\t2\t489.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2599\t1956.7\t0\t2599\r\n",
      "4\t4\t489.2\t0\t4\r\n",
      "5\t3\t122.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2504 times.\r\n",
      "2373 times, it overlapped the 5' end of a read\r\n",
      "131 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2368\t0.0\t1\t0 2368\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t1956.7\t0\t115\r\n",
      "4\t7\t489.2\t0\t7\r\n",
      "5\t9\t122.3\t0\t9\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2550.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2550.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.17 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,226\r\n",
      "Reads with adapters:                   119,772 (95.6%)\r\n",
      "Reads written (passing filters):       125,226 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,579,924 bp\r\n",
      "Total written (filtered):     29,362,701 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102142 times.\r\n",
      "102090 times, it overlapped the 5' end of a read\r\n",
      "52 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t489.2\t0\t2\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t145\t0.0\t0\t145\r\n",
      "21\t101907\t0.0\t0\t101907\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1956.7\t0\t39\r\n",
      "4\t2\t489.2\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "297\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17630 times.\r\n",
      "44 times, it overlapped the 5' end of a read\r\n",
      "17586 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1956.7\t0\t44\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1956.7\t0\t40\r\n",
      "4\t17544\t489.2\t0\t17544\r\n",
      "5\t2\t122.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2552.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2552.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.24 s (36 us/read; 1.68 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 174,535\r\n",
      "Reads with adapters:                     4,913 (2.8%)\r\n",
      "Reads written (passing filters):       174,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,851,067 bp\r\n",
      "Total written (filtered):     23,799,983 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3115 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "3095 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t2727.1\t0\t16\r\n",
      "4\t3\t681.8\t0\t3\r\n",
      "5\t1\t170.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3084\t2727.1\t0\t3084\r\n",
      "4\t3\t681.8\t0\t3\r\n",
      "5\t8\t170.4\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1798 times.\r\n",
      "1511 times, it overlapped the 5' end of a read\r\n",
      "287 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2727.1\t0\t1\r\n",
      "4\t1\t681.8\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1507\t0.0\t1\t0 1507\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t269\t2727.1\t0\t269\r\n",
      "4\t7\t681.8\t0\t7\r\n",
      "5\t8\t170.4\t0\t8\r\n",
      "6\t1\t42.6\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2553.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2553.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.81 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 137,191\r\n",
      "Reads with adapters:                     5,322 (3.9%)\r\n",
      "Reads written (passing filters):       137,191 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,111,610 bp\r\n",
      "Total written (filtered):     33,030,499 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2464 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "2449 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t2143.6\t0\t11\r\n",
      "4\t4\t535.9\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2440\t2143.6\t0\t2440\r\n",
      "4\t5\t535.9\t0\t5\r\n",
      "5\t4\t134.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2858 times.\r\n",
      "2704 times, it overlapped the 5' end of a read\r\n",
      "154 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2143.6\t0\t1\r\n",
      "4\t1\t535.9\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2699\t0.0\t1\t0 2699\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t2143.6\t0\t146\r\n",
      "4\t1\t535.9\t0\t1\r\n",
      "5\t5\t134.0\t0\t5\r\n",
      "6\t1\t33.5\t0\t1\r\n",
      "271\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2553.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2553.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.39 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 137,191\r\n",
      "Reads with adapters:                   124,083 (90.4%)\r\n",
      "Reads written (passing filters):       137,191 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,402,042 bp\r\n",
      "Total written (filtered):     31,131,364 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 104382 times.\r\n",
      "104312 times, it overlapped the 5' end of a read\r\n",
      "70 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2143.6\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t141\t0.0\t0\t141\r\n",
      "21\t104140\t0.0\t0\t104140\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t2143.6\t0\t62\r\n",
      "7\t1\t8.4\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 19701 times.\r\n",
      "31 times, it overlapped the 5' end of a read\r\n",
      "19670 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t2143.6\t0\t26\r\n",
      "5\t1\t134.0\t0\t1\r\n",
      "27\t4\t0.0\t1\t3 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t2143.6\t0\t64\r\n",
      "4\t19604\t535.9\t0\t19604\r\n",
      "5\t2\t134.0\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2552.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2552.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.59 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 174,535\r\n",
      "Reads with adapters:                   158,042 (90.6%)\r\n",
      "Reads written (passing filters):       174,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,466,211 bp\r\n",
      "Total written (filtered):     22,572,376 bp (92.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74412 times.\r\n",
      "74215 times, it overlapped the 5' end of a read\r\n",
      "197 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2727.1\t0\t1\r\n",
      "14\t2\t0.0\t0\t2\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t104\t0.0\t0\t104\r\n",
      "21\t74079\t0.0\t0\t74079\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t187\t2727.1\t0\t187\r\n",
      "4\t1\t681.8\t0\t1\r\n",
      "5\t2\t170.4\t0\t2\r\n",
      "7\t1\t10.7\t0\t1\r\n",
      "54\t2\t0.0\t0\t2\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "61\t2\t0.0\t0\t2\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83630 times.\r\n",
      "36 times, it overlapped the 5' end of a read\r\n",
      "83594 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t2727.1\t0\t34\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t183\t2727.1\t0\t183\r\n",
      "4\t83407\t681.8\t0\t83407\r\n",
      "5\t4\t170.4\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2551.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2551.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.50 s (30 us/read; 1.99 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 281,359\r\n",
      "Reads with adapters:                    10,729 (3.8%)\r\n",
      "Reads written (passing filters):       281,359 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    67,020,370 bp\r\n",
      "Total written (filtered):     66,859,914 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4942 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "4916 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t4396.2\t0\t19\r\n",
      "4\t7\t1099.1\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4864\t4396.2\t0\t4864\r\n",
      "4\t32\t1099.1\t0\t32\r\n",
      "5\t19\t274.8\t0\t19\r\n",
      "6\t1\t68.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5787 times.\r\n",
      "5328 times, it overlapped the 5' end of a read\r\n",
      "459 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t8\t0.0\t1\t0 8\r\n",
      "27\t5318\t0.0\t1\t0 5318\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t440\t4396.2\t0\t440\r\n",
      "4\t5\t1099.1\t0\t5\r\n",
      "5\t12\t274.8\t0\t12\r\n",
      "6\t1\t68.7\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2576.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2576.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.15 s (26 us/read; 2.33 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                     4,967 (4.1%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,730,362 bp\r\n",
      "Total written (filtered):     30,656,892 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2337 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "2327 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1907.5\t0\t10\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2310\t1907.5\t0\t2310\r\n",
      "4\t9\t476.9\t0\t9\r\n",
      "5\t8\t119.2\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2630 times.\r\n",
      "2442 times, it overlapped the 5' end of a read\r\n",
      "188 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1907.5\t0\t1\r\n",
      "4\t2\t476.9\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2437\t0.0\t1\t0 2437\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t182\t1907.5\t0\t182\r\n",
      "4\t4\t476.9\t0\t4\r\n",
      "5\t2\t119.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2577.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2577.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.73 s (38 us/read; 1.57 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,403\r\n",
      "Reads with adapters:                    68,404 (95.8%)\r\n",
      "Reads written (passing filters):        71,403 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,304,006 bp\r\n",
      "Total written (filtered):     19,869,107 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 68214 times.\r\n",
      "68203 times, it overlapped the 5' end of a read\r\n",
      "11 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t104\t0.0\t0\t104\r\n",
      "21\t68085\t0.0\t0\t68085\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1115.7\t0\t4\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 190 times.\r\n",
      "122 times, it overlapped the 5' end of a read\r\n",
      "68 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t121\t1115.7\t0\t121\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1115.7\t0\t1\r\n",
      "4\t67\t278.9\t0\t67\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2577.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2577.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.19 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,403\r\n",
      "Reads with adapters:                     3,886 (5.4%)\r\n",
      "Reads written (passing filters):        71,403 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,281,617 bp\r\n",
      "Total written (filtered):     21,227,582 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2030 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "2021 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1115.7\t0\t9\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2011\t1115.7\t0\t2011\r\n",
      "4\t9\t278.9\t0\t9\r\n",
      "5\t1\t69.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1856 times.\r\n",
      "1765 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1763\t0.0\t1\t0 1763\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t1115.7\t0\t86\r\n",
      "4\t2\t278.9\t0\t2\r\n",
      "5\t3\t69.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2578.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2578.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.91 s (34 us/read; 1.74 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,432\r\n",
      "Reads with adapters:                    80,653 (95.5%)\r\n",
      "Reads written (passing filters):        84,432 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,275,985 bp\r\n",
      "Total written (filtered):     23,584,102 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80479 times.\r\n",
      "80468 times, it overlapped the 5' end of a read\r\n",
      "11 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t116\t0.0\t0\t116\r\n",
      "21\t80325\t0.0\t0\t80325\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1319.2\t0\t6\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 174 times.\r\n",
      "65 times, it overlapped the 5' end of a read\r\n",
      "109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1319.2\t0\t60\r\n",
      "4\t1\t329.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1319.2\t0\t7\r\n",
      "4\t101\t329.8\t0\t101\r\n",
      "5\t1\t82.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2576.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2576.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.24 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                   117,064 (95.9%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,894,711 bp\r\n",
      "Total written (filtered):     28,671,861 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 103215 times.\r\n",
      "103159 times, it overlapped the 5' end of a read\r\n",
      "56 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t144\t0.0\t0\t144\r\n",
      "21\t102976\t0.0\t0\t102976\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t49\t1907.5\t0\t49\r\n",
      "5\t3\t119.2\t0\t3\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 13849 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "13804 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1907.5\t0\t42\r\n",
      "27\t3\t0.0\t1\t1 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1907.5\t0\t33\r\n",
      "4\t13771\t476.9\t0\t13771\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2578.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2578.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.59 s (31 us/read; 1.96 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,432\r\n",
      "Reads with adapters:                     4,178 (4.9%)\r\n",
      "Reads written (passing filters):        84,432 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,243,540 bp\r\n",
      "Total written (filtered):     25,184,245 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2162 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "2155 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1319.2\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2142\t1319.2\t0\t2142\r\n",
      "4\t7\t329.8\t0\t7\r\n",
      "5\t6\t82.5\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2016 times.\r\n",
      "1935 times, it overlapped the 5' end of a read\r\n",
      "81 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "27\t1934\t0.0\t1\t0 1934\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1319.2\t0\t75\r\n",
      "4\t2\t329.8\t0\t2\r\n",
      "5\t3\t82.5\t0\t3\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2551.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2551.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 14.43 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 281,359\r\n",
      "Reads with adapters:                   261,134 (92.8%)\r\n",
      "Reads written (passing filters):       281,359 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    67,585,599 bp\r\n",
      "Total written (filtered):     62,798,987 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 220127 times.\r\n",
      "220015 times, it overlapped the 5' end of a read\r\n",
      "112 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t24\t0.0\t0\t24\r\n",
      "19\t44\t0.0\t0\t44\r\n",
      "20\t304\t0.0\t0\t304\r\n",
      "21\t219640\t0.0\t0\t219640\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t99\t4396.2\t0\t99\r\n",
      "4\t1\t1099.1\t0\t1\r\n",
      "5\t1\t274.8\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "56\t1\t0.0\t0\t1\r\n",
      "255\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 41007 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "40933 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t4396.2\t0\t70\r\n",
      "27\t4\t0.0\t1\t3 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t94\t4396.2\t0\t94\r\n",
      "4\t40835\t1099.1\t0\t40835\r\n",
      "5\t4\t274.8\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2579.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2579.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.92 s (32 us/read; 1.87 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                     6,509 (5.3%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,520,467 bp\r\n",
      "Total written (filtered):     36,433,341 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3573 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "3572 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1907.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3564\t1907.5\t0\t3564\r\n",
      "4\t4\t476.9\t0\t4\r\n",
      "5\t4\t119.2\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2936 times.\r\n",
      "2823 times, it overlapped the 5' end of a read\r\n",
      "113 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1907.5\t0\t4\r\n",
      "4\t2\t476.9\t0\t2\r\n",
      "5\t1\t119.2\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2812\t0.0\t1\t0 2812\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1907.5\t0\t108\r\n",
      "4\t3\t476.9\t0\t3\r\n",
      "5\t2\t119.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2580.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2580.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.12 s (42 us/read; 1.43 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,268\r\n",
      "Reads with adapters:                   115,177 (94.2%)\r\n",
      "Reads written (passing filters):       122,268 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,293,918 bp\r\n",
      "Total written (filtered):     33,882,411 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 114622 times.\r\n",
      "114600 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t173\t0.0\t0\t173\r\n",
      "21\t114381\t0.0\t0\t114381\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1910.4\t0\t11\r\n",
      "4\t1\t477.6\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 555 times.\r\n",
      "81 times, it overlapped the 5' end of a read\r\n",
      "474 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1910.4\t0\t80\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1910.4\t0\t5\r\n",
      "4\t469\t477.6\t0\t469\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2579.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2579.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.29 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                   106,203 (87.0%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,563,591 bp\r\n",
      "Total written (filtered):     34,335,356 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 105945 times.\r\n",
      "105912 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t133\t0.0\t0\t133\r\n",
      "21\t105740\t0.0\t0\t105740\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1907.5\t0\t22\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 258 times.\r\n",
      "118 times, it overlapped the 5' end of a read\r\n",
      "140 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t116\t1907.5\t0\t116\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1907.5\t0\t9\r\n",
      "4\t131\t476.9\t0\t131\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2580.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2580.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.93 s (40 us/read; 1.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,268\r\n",
      "Reads with adapters:                     6,692 (5.5%)\r\n",
      "Reads written (passing filters):       122,268 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,251,580 bp\r\n",
      "Total written (filtered):     36,162,213 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3661 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3656 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1910.4\t0\t3\r\n",
      "4\t2\t477.6\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3636\t1910.4\t0\t3636\r\n",
      "4\t6\t477.6\t0\t6\r\n",
      "5\t14\t119.4\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3031 times.\r\n",
      "2889 times, it overlapped the 5' end of a read\r\n",
      "142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1910.4\t0\t2\r\n",
      "4\t2\t477.6\t0\t2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2881\t0.0\t1\t0 2881\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t133\t1910.4\t0\t133\r\n",
      "4\t1\t477.6\t0\t1\r\n",
      "5\t8\t119.4\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2581.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2581.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.09 s (28 us/read; 2.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,366\r\n",
      "Reads with adapters:                     6,636 (4.6%)\r\n",
      "Reads written (passing filters):       145,366 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,092,394 bp\r\n",
      "Total written (filtered):     42,995,619 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3269 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3264 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2271.3\t0\t4\r\n",
      "4\t1\t567.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3250\t2271.3\t0\t3250\r\n",
      "4\t8\t567.8\t0\t8\r\n",
      "5\t6\t142.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3367 times.\r\n",
      "3192 times, it overlapped the 5' end of a read\r\n",
      "175 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2271.3\t0\t1\r\n",
      "4\t1\t567.8\t0\t1\r\n",
      "27\t3190\t0.0\t1\t0 3190\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t165\t2271.3\t0\t165\r\n",
      "4\t3\t567.8\t0\t3\r\n",
      "5\t5\t142.0\t0\t5\r\n",
      "6\t1\t35.5\t0\t1\r\n",
      "272\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2581.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2581.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.77 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,366\r\n",
      "Reads with adapters:                   119,743 (82.4%)\r\n",
      "Reads written (passing filters):       145,366 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,148,117 bp\r\n",
      "Total written (filtered):     40,642,471 bp (94.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 119063 times.\r\n",
      "119022 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t27\t0.0\t0\t27\r\n",
      "20\t137\t0.0\t0\t137\r\n",
      "21\t118851\t0.0\t0\t118851\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t2271.3\t0\t29\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 680 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "613 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t2271.3\t0\t64\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t2271.3\t0\t13\r\n",
      "4\t600\t567.8\t0\t600\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2583.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2583.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.54 s (32 us/read; 1.86 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 140,909\r\n",
      "Reads with adapters:                     6,698 (4.8%)\r\n",
      "Reads written (passing filters):       140,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,837,516 bp\r\n",
      "Total written (filtered):     41,738,361 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3277 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "3274 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2201.7\t0\t1\r\n",
      "4\t2\t550.4\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3257\t2201.7\t0\t3257\r\n",
      "4\t11\t550.4\t0\t11\r\n",
      "5\t6\t137.6\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3421 times.\r\n",
      "3295 times, it overlapped the 5' end of a read\r\n",
      "126 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2201.7\t0\t1\r\n",
      "4\t1\t550.4\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "27\t3292\t0.0\t1\t0 3292\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t121\t2201.7\t0\t121\r\n",
      "4\t3\t550.4\t0\t3\r\n",
      "5\t2\t137.6\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2584.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2584.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.28 s (35 us/read; 1.71 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,196\r\n",
      "Reads with adapters:                     5,939 (4.9%)\r\n",
      "Reads written (passing filters):       122,196 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,437,582 bp\r\n",
      "Total written (filtered):     36,352,519 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3005 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "3004 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1909.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2989\t1909.3\t0\t2989\r\n",
      "4\t7\t477.3\t0\t7\r\n",
      "5\t8\t119.3\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2934 times.\r\n",
      "2767 times, it overlapped the 5' end of a read\r\n",
      "167 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1909.3\t0\t2\r\n",
      "4\t1\t477.3\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2761\t0.0\t1\t0 2761\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t1909.3\t0\t160\r\n",
      "4\t3\t477.3\t0\t3\r\n",
      "5\t1\t119.3\t0\t1\r\n",
      "300\t3\t0.0\t1\t0 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2583.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2583.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.79 s (41 us/read; 1.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 140,909\r\n",
      "Reads with adapters:                   136,478 (96.9%)\r\n",
      "Reads written (passing filters):       140,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,890,783 bp\r\n",
      "Total written (filtered):     39,028,735 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 136108 times.\r\n",
      "136089 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t35\t0.0\t0\t35\r\n",
      "20\t170\t0.0\t0\t170\r\n",
      "21\t135872\t0.0\t0\t135872\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t2201.7\t0\t9\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 370 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "327 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t2201.7\t0\t43\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2201.7\t0\t6\r\n",
      "4\t321\t550.4\t0\t321\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2582.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2582.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.07 s (41 us/read; 1.45 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 171,077\r\n",
      "Reads with adapters:                     7,955 (4.6%)\r\n",
      "Reads written (passing filters):       171,077 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,858,397 bp\r\n",
      "Total written (filtered):     43,745,237 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4013 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "3991 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t2673.1\t0\t15\r\n",
      "4\t6\t668.3\t0\t6\r\n",
      "5\t1\t167.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3967\t2673.1\t0\t3967\r\n",
      "4\t18\t668.3\t0\t18\r\n",
      "5\t6\t167.1\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3942 times.\r\n",
      "3720 times, it overlapped the 5' end of a read\r\n",
      "222 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t668.3\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3714\t0.0\t1\t0 3714\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t206\t2673.1\t0\t206\r\n",
      "4\t4\t668.3\t0\t4\r\n",
      "5\t11\t167.1\t0\t11\r\n",
      "6\t1\t41.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2582.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2582.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.80 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 171,077\r\n",
      "Reads with adapters:                   153,224 (89.6%)\r\n",
      "Reads written (passing filters):       171,077 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    44,234,621 bp\r\n",
      "Total written (filtered):     41,343,038 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 133958 times.\r\n",
      "133876 times, it overlapped the 5' end of a read\r\n",
      "82 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2673.1\t0\t2\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t27\t0.0\t0\t27\r\n",
      "20\t186\t0.0\t0\t186\r\n",
      "21\t133646\t0.0\t0\t133646\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t2673.1\t0\t66\r\n",
      "4\t2\t668.3\t0\t2\r\n",
      "5\t1\t167.1\t0\t1\r\n",
      "7\t1\t10.4\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 19266 times.\r\n",
      "176 times, it overlapped the 5' end of a read\r\n",
      "19090 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t172\t2673.1\t0\t172\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t2673.1\t0\t52\r\n",
      "4\t19037\t668.3\t0\t19037\r\n",
      "5\t1\t167.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2584.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2584.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.86 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,196\r\n",
      "Reads with adapters:                   103,492 (84.7%)\r\n",
      "Reads written (passing filters):       122,196 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,473,042 bp\r\n",
      "Total written (filtered):     34,304,267 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 103137 times.\r\n",
      "103104 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t146\t0.0\t0\t146\r\n",
      "21\t102927\t0.0\t0\t102927\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1909.3\t0\t25\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 355 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "281 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1909.3\t0\t74\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1909.3\t0\t13\r\n",
      "4\t267\t477.3\t0\t267\r\n",
      "5\t1\t119.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2586.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2586.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.37 s (59 us/read; 1.01 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,255\r\n",
      "Reads with adapters:                   100,887 (94.1%)\r\n",
      "Reads written (passing filters):       107,255 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,118,218 bp\r\n",
      "Total written (filtered):     30,002,927 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 100549 times.\r\n",
      "100529 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t131\t0.0\t0\t131\r\n",
      "21\t100366\t0.0\t0\t100366\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1675.9\t0\t9\r\n",
      "5\t1\t104.7\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 338 times.\r\n",
      "155 times, it overlapped the 5' end of a read\r\n",
      "183 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t148\t1675.9\t0\t148\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t6\t0.0\t1\t3 3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1675.9\t0\t5\r\n",
      "4\t178\t419.0\t0\t178\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2590.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2590.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.07 s (34 us/read; 1.79 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   2,174\r\n",
      "Reads with adapters:                     1,981 (91.1%)\r\n",
      "Reads written (passing filters):         2,174 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       635,404 bp\r\n",
      "Total written (filtered):        594,764 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1928 times.\r\n",
      "1925 times, it overlapped the 5' end of a read\r\n",
      "3 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t1925\t0.0\t0\t1925\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t34.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 53 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t34.0\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t34.0\t0\t1\r\n",
      "4\t47\t8.5\t0\t47\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2590.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2590.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.05 s (24 us/read; 2.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   2,174\r\n",
      "Reads with adapters:                       146 (6.7%)\r\n",
      "Reads written (passing filters):         2,174 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       631,553 bp\r\n",
      "Total written (filtered):        629,939 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 95 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "95 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t95\t34.0\t0\t95\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 51 times.\r\n",
      "49 times, it overlapped the 5' end of a read\r\n",
      "2 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t49\t0.0\t1\t0 49\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t34.0\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2586.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2586.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.45 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,255\r\n",
      "Reads with adapters:                     4,658 (4.3%)\r\n",
      "Reads written (passing filters):       107,255 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,082,507 bp\r\n",
      "Total written (filtered):     32,013,792 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2172 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2166 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1675.9\t0\t5\r\n",
      "4\t1\t419.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2156\t1675.9\t0\t2156\r\n",
      "4\t9\t419.0\t0\t9\r\n",
      "5\t1\t104.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2486 times.\r\n",
      "2283 times, it overlapped the 5' end of a read\r\n",
      "203 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1675.9\t0\t1\r\n",
      "4\t2\t419.0\t0\t2\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2275\t0.0\t1\t0 2275\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t196\t1675.9\t0\t196\r\n",
      "4\t1\t419.0\t0\t1\r\n",
      "5\t6\t104.7\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2588.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2588.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.51 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,341\r\n",
      "Reads with adapters:                    65,234 (78.3%)\r\n",
      "Reads written (passing filters):        83,341 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,529,473 bp\r\n",
      "Total written (filtered):     15,523,460 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 43797 times.\r\n",
      "43708 times, it overlapped the 5' end of a read\r\n",
      "89 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1302.2\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t72\t0.0\t0\t72\r\n",
      "21\t43619\t0.0\t0\t43619\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1302.2\t0\t75\r\n",
      "5\t1\t81.4\t0\t1\r\n",
      "7\t1\t5.1\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "61\t3\t0.0\t0\t3\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 21437 times.\r\n",
      "77 times, it overlapped the 5' end of a read\r\n",
      "21360 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1302.2\t0\t75\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t1302.2\t0\t64\r\n",
      "4\t21294\t325.6\t0\t21294\r\n",
      "5\t2\t81.4\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2588.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2588.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.84 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,341\r\n",
      "Reads with adapters:                     3,083 (3.7%)\r\n",
      "Reads written (passing filters):        83,341 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,172,839 bp\r\n",
      "Total written (filtered):     16,125,625 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1364 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "1351 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1302.2\t0\t12\r\n",
      "4\t1\t325.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1344\t1302.2\t0\t1344\r\n",
      "4\t6\t325.6\t0\t6\r\n",
      "6\t1\t20.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1719 times.\r\n",
      "1580 times, it overlapped the 5' end of a read\r\n",
      "139 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1302.2\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1576\t0.0\t1\t0 1576\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t1302.2\t0\t135\r\n",
      "4\t1\t325.6\t0\t1\r\n",
      "5\t1\t81.4\t0\t1\r\n",
      "6\t1\t20.3\t0\t1\r\n",
      "61\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2587.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2587.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.10 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,785\r\n",
      "Reads with adapters:                     4,416 (4.4%)\r\n",
      "Reads written (passing filters):        99,785 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,248,571 bp\r\n",
      "Total written (filtered):     29,177,507 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1860 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1855 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1559.1\t0\t4\r\n",
      "4\t1\t389.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1844\t1559.1\t0\t1844\r\n",
      "4\t3\t389.8\t0\t3\r\n",
      "5\t8\t97.4\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2556 times.\r\n",
      "2409 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t389.8\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2403\t0.0\t1\t0 2403\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t138\t1559.1\t0\t138\r\n",
      "4\t5\t389.8\t0\t5\r\n",
      "5\t4\t97.4\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2587.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2587.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.60 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,785\r\n",
      "Reads with adapters:                    96,664 (96.9%)\r\n",
      "Reads written (passing filters):        99,785 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,298,918 bp\r\n",
      "Total written (filtered):     27,285,239 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 95602 times.\r\n",
      "95585 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t95431\t0.0\t0\t95431\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1559.1\t0\t9\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1062 times.\r\n",
      "103 times, it overlapped the 5' end of a read\r\n",
      "959 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t1559.1\t0\t102\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1559.1\t0\t7\r\n",
      "4\t952\t389.8\t0\t952\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2589.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2589.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.59 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,752\r\n",
      "Reads with adapters:                     4,399 (5.2%)\r\n",
      "Reads written (passing filters):        84,752 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,268,056 bp\r\n",
      "Total written (filtered):     25,201,981 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2093 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "2093 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2080\t1324.2\t0\t2080\r\n",
      "4\t6\t331.1\t0\t6\r\n",
      "5\t7\t82.8\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2306 times.\r\n",
      "2206 times, it overlapped the 5' end of a read\r\n",
      "100 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1324.2\t0\t3\r\n",
      "4\t1\t331.1\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2199\t0.0\t1\t0 2199\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t1324.2\t0\t93\r\n",
      "4\t1\t331.1\t0\t1\r\n",
      "5\t5\t82.8\t0\t5\r\n",
      "6\t1\t20.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2589.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2589.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.47 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,752\r\n",
      "Reads with adapters:                    45,996 (54.3%)\r\n",
      "Reads written (passing filters):        84,752 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,305,503 bp\r\n",
      "Total written (filtered):     24,344,681 bp (96.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45698 times.\r\n",
      "45628 times, it overlapped the 5' end of a read\r\n",
      "70 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1324.2\t0\t3\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t63\t0.0\t0\t63\r\n",
      "21\t45550\t0.0\t0\t45550\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t1324.2\t0\t61\r\n",
      "4\t1\t331.1\t0\t1\r\n",
      "5\t1\t82.8\t0\t1\r\n",
      "7\t2\t5.2\t0\t2\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 298 times.\r\n",
      "82 times, it overlapped the 5' end of a read\r\n",
      "216 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t82\t1324.2\t0\t82\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1324.2\t0\t35\r\n",
      "4\t181\t331.1\t0\t181\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2592.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2592.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.20 s (29 us/read; 2.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,982\r\n",
      "Reads with adapters:                     4,898 (4.4%)\r\n",
      "Reads written (passing filters):       111,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,542,600 bp\r\n",
      "Total written (filtered):     33,467,397 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2250 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "2246 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1749.7\t0\t3\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2237\t1749.7\t0\t2237\r\n",
      "4\t4\t437.4\t0\t4\r\n",
      "5\t5\t109.4\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2648 times.\r\n",
      "2521 times, it overlapped the 5' end of a read\r\n",
      "127 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "27\t2520\t0.0\t1\t0 2520\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t119\t1749.7\t0\t119\r\n",
      "4\t3\t437.4\t0\t3\r\n",
      "5\t5\t109.4\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2591.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2591.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.06 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 129,918\r\n",
      "Reads with adapters:                     6,485 (5.0%)\r\n",
      "Reads written (passing filters):       129,918 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    38,724,435 bp\r\n",
      "Total written (filtered):     38,633,730 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3435 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "3431 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2030.0\t0\t1\r\n",
      "4\t2\t507.5\t0\t2\r\n",
      "5\t1\t126.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3428\t2030.0\t0\t3428\r\n",
      "4\t1\t507.5\t0\t1\r\n",
      "5\t2\t126.9\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3050 times.\r\n",
      "2968 times, it overlapped the 5' end of a read\r\n",
      "82 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2964\t0.0\t1\t0 2964\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t2030.0\t0\t74\r\n",
      "4\t3\t507.5\t0\t3\r\n",
      "5\t5\t126.9\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2594.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2594.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.90 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,123\r\n",
      "Reads with adapters:                     3,509 (4.4%)\r\n",
      "Reads written (passing filters):        80,123 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,176,998 bp\r\n",
      "Total written (filtered):     22,124,312 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1675 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1669 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1251.9\t0\t4\r\n",
      "4\t1\t313.0\t0\t1\r\n",
      "5\t1\t78.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1665\t1251.9\t0\t1665\r\n",
      "4\t1\t313.0\t0\t1\r\n",
      "5\t3\t78.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1834 times.\r\n",
      "1756 times, it overlapped the 5' end of a read\r\n",
      "78 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1755\t0.0\t1\t0 1755\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1251.9\t0\t75\r\n",
      "5\t3\t78.2\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2593.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2593.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.61 s (36 us/read; 1.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,836\r\n",
      "Reads with adapters:                     4,771 (3.8%)\r\n",
      "Reads written (passing filters):       126,836 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,303,051 bp\r\n",
      "Total written (filtered):     29,235,837 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2400 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2394 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1981.8\t0\t5\r\n",
      "4\t1\t495.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2377\t1981.8\t0\t2377\r\n",
      "4\t13\t495.5\t0\t13\r\n",
      "5\t4\t123.9\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2371 times.\r\n",
      "2203 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2198\t0.0\t1\t0 2198\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1981.8\t0\t161\r\n",
      "4\t3\t495.5\t0\t3\r\n",
      "5\t3\t123.9\t0\t3\r\n",
      "6\t1\t31.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2594.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2594.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.46 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,123\r\n",
      "Reads with adapters:                    77,044 (96.2%)\r\n",
      "Reads written (passing filters):        80,123 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,275,214 bp\r\n",
      "Total written (filtered):     20,743,545 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 71897 times.\r\n",
      "71876 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t85\t0.0\t0\t85\r\n",
      "21\t71772\t0.0\t0\t71772\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1251.9\t0\t15\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5147 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "5085 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1251.9\t0\t60\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1251.9\t0\t15\r\n",
      "4\t5070\t313.0\t0\t5070\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2591.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2591.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.33 s (56 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 129,918\r\n",
      "Reads with adapters:                   125,593 (96.7%)\r\n",
      "Reads written (passing filters):       129,918 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    38,782,247 bp\r\n",
      "Total written (filtered):     36,148,398 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 125165 times.\r\n",
      "125140 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t141\t0.0\t0\t141\r\n",
      "21\t124964\t0.0\t0\t124964\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2030.0\t0\t10\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 428 times.\r\n",
      "98 times, it overlapped the 5' end of a read\r\n",
      "330 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t98\t2030.0\t0\t98\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2030.0\t0\t4\r\n",
      "4\t326\t507.5\t0\t326\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2592.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2592.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.18 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,982\r\n",
      "Reads with adapters:                   108,500 (96.9%)\r\n",
      "Reads written (passing filters):       111,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,576,097 bp\r\n",
      "Total written (filtered):     31,298,691 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 108325 times.\r\n",
      "108309 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t131\t0.0\t0\t131\r\n",
      "21\t108152\t0.0\t0\t108152\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1749.7\t0\t7\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 175 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "121 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1749.7\t0\t52\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1749.7\t0\t2\r\n",
      "4\t119\t437.4\t0\t119\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2593.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2593.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.95 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,836\r\n",
      "Reads with adapters:                   120,525 (95.0%)\r\n",
      "Reads written (passing filters):       126,836 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,607,504 bp\r\n",
      "Total written (filtered):     27,503,256 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 95477 times.\r\n",
      "95401 times, it overlapped the 5' end of a read\r\n",
      "76 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1981.8\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t154\t0.0\t0\t154\r\n",
      "21\t95225\t0.0\t0\t95225\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t1981.8\t0\t70\r\n",
      "4\t1\t495.5\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25048 times.\r\n",
      "77 times, it overlapped the 5' end of a read\r\n",
      "24971 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t1981.8\t0\t76\r\n",
      "26\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t1981.8\t0\t64\r\n",
      "4\t24907\t495.5\t0\t24907\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2596.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2596.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.61 s (38 us/read; 1.57 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,188\r\n",
      "Reads with adapters:                     3,936 (4.2%)\r\n",
      "Reads written (passing filters):        94,188 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,998,384 bp\r\n",
      "Total written (filtered):     22,947,631 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2101 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2098 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1471.7\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2091\t1471.7\t0\t2091\r\n",
      "4\t5\t367.9\t0\t5\r\n",
      "5\t2\t92.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1835 times.\r\n",
      "1626 times, it overlapped the 5' end of a read\r\n",
      "209 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t4\t367.9\t0\t4\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1620\t0.0\t1\t0 1620\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t204\t1471.7\t0\t204\r\n",
      "4\t3\t367.9\t0\t3\r\n",
      "5\t2\t92.0\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2598.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2598.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.37 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  60,696\r\n",
      "Reads with adapters:                    58,654 (96.6%)\r\n",
      "Reads written (passing filters):        60,696 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,217,576 bp\r\n",
      "Total written (filtered):     16,036,173 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55602 times.\r\n",
      "55580 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t84\t0.0\t0\t84\r\n",
      "21\t55480\t0.0\t0\t55480\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t948.4\t0\t14\r\n",
      "4\t1\t237.1\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3052 times.\r\n",
      "23 times, it overlapped the 5' end of a read\r\n",
      "3029 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t948.4\t0\t22\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t948.4\t0\t7\r\n",
      "4\t3022\t237.1\t0\t3022\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2598.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2598.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.62 s (27 us/read; 2.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  60,696\r\n",
      "Reads with adapters:                     2,560 (4.2%)\r\n",
      "Reads written (passing filters):        60,696 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,178,888 bp\r\n",
      "Total written (filtered):     17,142,155 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1290 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "1283 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t948.4\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1277\t948.4\t0\t1277\r\n",
      "4\t5\t237.1\t0\t5\r\n",
      "5\t1\t59.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1270 times.\r\n",
      "1210 times, it overlapped the 5' end of a read\r\n",
      "60 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "27\t1209\t0.0\t1\t0 1209\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t948.4\t0\t54\r\n",
      "4\t2\t237.1\t0\t2\r\n",
      "5\t4\t59.3\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2596.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2596.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.14 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,188\r\n",
      "Reads with adapters:                    89,516 (95.0%)\r\n",
      "Reads written (passing filters):        94,188 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,187,550 bp\r\n",
      "Total written (filtered):     21,553,165 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 75062 times.\r\n",
      "75021 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t1\t92.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t120\t0.0\t0\t120\r\n",
      "21\t74880\t0.0\t0\t74880\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1471.7\t0\t34\r\n",
      "4\t2\t367.9\t0\t2\r\n",
      "7\t1\t5.7\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 14454 times.\r\n",
      "40 times, it overlapped the 5' end of a read\r\n",
      "14414 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1471.7\t0\t37\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1471.7\t0\t32\r\n",
      "4\t14382\t367.9\t0\t14382\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2595.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2595.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.73 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,183\r\n",
      "Reads with adapters:                   120,709 (84.9%)\r\n",
      "Reads written (passing filters):       142,183 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,198,595 bp\r\n",
      "Total written (filtered):     33,973,349 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102525 times.\r\n",
      "102446 times, it overlapped the 5' end of a read\r\n",
      "79 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t131\t0.0\t0\t131\r\n",
      "21\t102282\t0.0\t0\t102282\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t2221.6\t0\t73\r\n",
      "7\t2\t8.7\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 18184 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "18137 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t2221.6\t0\t45\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t2221.6\t0\t71\r\n",
      "4\t18066\t555.4\t0\t18066\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2595.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2595.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.63 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,183\r\n",
      "Reads with adapters:                     5,910 (4.2%)\r\n",
      "Reads written (passing filters):       142,183 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,001,204 bp\r\n",
      "Total written (filtered):     35,918,612 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3016 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "3010 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2221.6\t0\t5\r\n",
      "4\t1\t555.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2997\t2221.6\t0\t2997\r\n",
      "4\t8\t555.4\t0\t8\r\n",
      "5\t5\t138.9\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2894 times.\r\n",
      "2709 times, it overlapped the 5' end of a read\r\n",
      "185 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2221.6\t0\t5\r\n",
      "4\t3\t555.4\t0\t3\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2698\t0.0\t1\t0 2698\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t174\t2221.6\t0\t174\r\n",
      "4\t4\t555.4\t0\t4\r\n",
      "5\t6\t138.9\t0\t6\r\n",
      "6\t1\t34.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2600.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2600.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.20 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,064\r\n",
      "Reads with adapters:                     3,089 (4.2%)\r\n",
      "Reads written (passing filters):        73,064 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,783,062 bp\r\n",
      "Total written (filtered):     17,743,081 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1676 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1673 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1141.6\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1665\t1141.6\t0\t1665\r\n",
      "4\t7\t285.4\t0\t7\r\n",
      "5\t1\t71.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1413 times.\r\n",
      "1279 times, it overlapped the 5' end of a read\r\n",
      "134 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t1279\t0.0\t1\t0 1279\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t129\t1141.6\t0\t129\r\n",
      "4\t2\t285.4\t0\t2\r\n",
      "5\t2\t71.4\t0\t2\r\n",
      "6\t1\t17.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2600.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2600.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.85 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,064\r\n",
      "Reads with adapters:                    68,541 (93.8%)\r\n",
      "Reads written (passing filters):        73,064 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,942,015 bp\r\n",
      "Total written (filtered):     16,696,725 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 57114 times.\r\n",
      "57068 times, it overlapped the 5' end of a read\r\n",
      "46 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1141.6\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t76\t0.0\t0\t76\r\n",
      "21\t56975\t0.0\t0\t56975\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1141.6\t0\t40\r\n",
      "4\t2\t285.4\t0\t2\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11427 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "11373 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1141.6\t0\t52\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1141.6\t0\t36\r\n",
      "4\t11337\t285.4\t0\t11337\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2601.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2601.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.63 s (51 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,697\r\n",
      "Reads with adapters:                    87,433 (95.3%)\r\n",
      "Reads written (passing filters):        91,697 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,107,399 bp\r\n",
      "Total written (filtered):     22,437,746 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77575 times.\r\n",
      "77547 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t91\t0.0\t0\t91\r\n",
      "21\t77427\t0.0\t0\t77427\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1432.8\t0\t22\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9858 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "9840 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1432.8\t0\t15\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1432.8\t0\t20\r\n",
      "4\t9818\t358.2\t0\t9818\r\n",
      "5\t2\t89.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2601.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2601.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.55 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,697\r\n",
      "Reads with adapters:                     4,081 (4.5%)\r\n",
      "Reads written (passing filters):        91,697 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,987,479 bp\r\n",
      "Total written (filtered):     23,928,389 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2039 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2034 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1432.8\t0\t4\r\n",
      "4\t1\t358.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2024\t1432.8\t0\t2024\r\n",
      "4\t8\t358.2\t0\t8\r\n",
      "5\t2\t89.5\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2042 times.\r\n",
      "1951 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1949\t0.0\t1\t0 1949\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1432.8\t0\t83\r\n",
      "4\t2\t358.2\t0\t2\r\n",
      "5\t6\t89.5\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2602.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2602.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.78 s (36 us/read; 1.68 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,075\r\n",
      "Reads with adapters:                     4,282 (4.0%)\r\n",
      "Reads written (passing filters):       106,075 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,306,746 bp\r\n",
      "Total written (filtered):     25,251,182 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2313 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "2309 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1657.4\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2288\t1657.4\t0\t2288\r\n",
      "4\t8\t414.4\t0\t8\r\n",
      "5\t12\t103.6\t0\t12\r\n",
      "6\t1\t25.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1969 times.\r\n",
      "1778 times, it overlapped the 5' end of a read\r\n",
      "191 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1775\t0.0\t1\t0 1775\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t1657.4\t0\t180\r\n",
      "4\t8\t414.4\t0\t8\r\n",
      "5\t3\t103.6\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2602.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2602.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.42 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,075\r\n",
      "Reads with adapters:                    83,801 (79.0%)\r\n",
      "Reads written (passing filters):       106,075 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,470,288 bp\r\n",
      "Total written (filtered):     24,023,071 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 65494 times.\r\n",
      "65428 times, it overlapped the 5' end of a read\r\n",
      "66 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t117\t0.0\t0\t117\r\n",
      "21\t65292\t0.0\t0\t65292\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1657.4\t0\t63\r\n",
      "5\t1\t103.6\t0\t1\r\n",
      "6\t1\t25.9\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 18307 times.\r\n",
      "51 times, it overlapped the 5' end of a read\r\n",
      "18256 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1657.4\t0\t50\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t56\t1657.4\t0\t56\r\n",
      "4\t18199\t414.4\t0\t18199\r\n",
      "5\t1\t103.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2597.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2597.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 10.63 s (37 us/read; 1.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 288,244\r\n",
      "Reads with adapters:                    12,183 (4.2%)\r\n",
      "Reads written (passing filters):       288,244 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    70,749,847 bp\r\n",
      "Total written (filtered):     70,575,034 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5985 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "5965 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t4503.8\t0\t15\r\n",
      "4\t5\t1126.0\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5915\t4503.8\t0\t5915\r\n",
      "4\t35\t1126.0\t0\t35\r\n",
      "5\t15\t281.5\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6198 times.\r\n",
      "5762 times, it overlapped the 5' end of a read\r\n",
      "436 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t6\t1126.0\t0\t6\r\n",
      "26\t7\t0.0\t1\t0 7\r\n",
      "27\t5749\t0.0\t1\t0 5749\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t407\t4503.8\t0\t407\r\n",
      "4\t11\t1126.0\t0\t11\r\n",
      "5\t16\t281.5\t0\t16\r\n",
      "6\t1\t70.4\t0\t1\r\n",
      "8\t1\t4.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2604.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2604.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.89 s (31 us/read; 1.93 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,869\r\n",
      "Reads with adapters:                     3,407 (3.7%)\r\n",
      "Reads written (passing filters):        92,869 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,337,278 bp\r\n",
      "Total written (filtered):     24,298,902 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2013 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "2009 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1451.1\t0\t1\r\n",
      "4\t3\t362.8\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2000\t1451.1\t0\t2000\r\n",
      "4\t7\t362.8\t0\t7\r\n",
      "5\t2\t90.7\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1394 times.\r\n",
      "1175 times, it overlapped the 5' end of a read\r\n",
      "219 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t362.8\t0\t3\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1170\t0.0\t1\t0 1170\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t213\t1451.1\t0\t213\r\n",
      "4\t3\t362.8\t0\t3\r\n",
      "5\t2\t90.7\t0\t2\r\n",
      "6\t1\t22.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2603.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2603.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.62 s (45 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,891\r\n",
      "Reads with adapters:                     4,326 (4.2%)\r\n",
      "Reads written (passing filters):       101,891 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,501,300 bp\r\n",
      "Total written (filtered):     26,436,074 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2018 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "2008 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1592.0\t0\t8\r\n",
      "4\t2\t398.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1997\t1592.0\t0\t1997\r\n",
      "4\t10\t398.0\t0\t10\r\n",
      "5\t1\t99.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2308 times.\r\n",
      "2179 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t398.0\t0\t3\r\n",
      "23\t2\t0.0\t1\t0 2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2172\t0.0\t1\t0 2172\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t118\t1592.0\t0\t118\r\n",
      "4\t5\t398.0\t0\t5\r\n",
      "5\t6\t99.5\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2603.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2603.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.25 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,891\r\n",
      "Reads with adapters:                    94,388 (92.6%)\r\n",
      "Reads written (passing filters):       101,891 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,649,069 bp\r\n",
      "Total written (filtered):     24,876,163 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 82053 times.\r\n",
      "82010 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1592.0\t0\t2\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t118\t0.0\t0\t118\r\n",
      "21\t81854\t0.0\t0\t81854\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1592.0\t0\t37\r\n",
      "4\t1\t398.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12335 times.\r\n",
      "53 times, it overlapped the 5' end of a read\r\n",
      "12282 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1592.0\t0\t50\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1592.0\t0\t31\r\n",
      "4\t12251\t398.0\t0\t12251\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2604.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2604.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.73 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,869\r\n",
      "Reads with adapters:                    86,493 (93.1%)\r\n",
      "Reads written (passing filters):        92,869 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,471,417 bp\r\n",
      "Total written (filtered):     22,794,987 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 78224 times.\r\n",
      "78189 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t105\t0.0\t0\t105\r\n",
      "21\t78058\t0.0\t0\t78058\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1451.1\t0\t29\r\n",
      "7\t1\t5.7\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8269 times.\r\n",
      "80 times, it overlapped the 5' end of a read\r\n",
      "8189 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t77\t1451.1\t0\t77\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1451.1\t0\t28\r\n",
      "4\t8159\t362.8\t0\t8159\r\n",
      "5\t2\t90.7\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/265.R1.fastq.gz.trimmed.fastq.gz /code/sequences/265.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.47 s (43 us/read; 1.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,378\r\n",
      "Reads with adapters:                    76,490 (95.2%)\r\n",
      "Reads written (passing filters):        80,378 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,917,092 bp\r\n",
      "Total written (filtered):     15,600,777 bp (92.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 59439 times.\r\n",
      "59401 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t56\t0.0\t0\t56\r\n",
      "21\t59332\t0.0\t0\t59332\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1255.9\t0\t35\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17051 times.\r\n",
      "30 times, it overlapped the 5' end of a read\r\n",
      "17021 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1255.9\t0\t27\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1255.9\t0\t32\r\n",
      "4\t16988\t314.0\t0\t16988\r\n",
      "5\t1\t78.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/265.R2.fastq.gz.trimmed.fastq.gz /code/sequences/265.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.56 s (44 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,378\r\n",
      "Reads with adapters:                     2,982 (3.7%)\r\n",
      "Reads written (passing filters):        80,378 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,774,161 bp\r\n",
      "Total written (filtered):     16,727,650 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1326 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1322 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1255.9\t0\t2\r\n",
      "4\t2\t314.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1312\t1255.9\t0\t1312\r\n",
      "4\t6\t314.0\t0\t6\r\n",
      "5\t3\t78.5\t0\t3\r\n",
      "6\t1\t19.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1656 times.\r\n",
      "1502 times, it overlapped the 5' end of a read\r\n",
      "154 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1255.9\t0\t4\r\n",
      "4\t1\t314.0\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1495\t0.0\t1\t0 1495\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t118\t1255.9\t0\t118\r\n",
      "4\t4\t314.0\t0\t4\r\n",
      "5\t6\t78.5\t0\t6\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "57\t24\t0.0\t1\t5 19\r\n",
      "269\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/282.R2.fastq.gz.trimmed.fastq.gz /code/sequences/282.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.24 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,537\r\n",
      "Reads with adapters:                     3,773 (4.5%)\r\n",
      "Reads written (passing filters):        83,537 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,319,022 bp\r\n",
      "Total written (filtered):     20,269,198 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2086 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "2082 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1305.3\t0\t3\r\n",
      "4\t1\t326.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2081\t1305.3\t0\t2081\r\n",
      "4\t1\t326.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1687 times.\r\n",
      "1604 times, it overlapped the 5' end of a read\r\n",
      "83 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1603\t0.0\t1\t0 1603\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t78\t1305.3\t0\t78\r\n",
      "4\t2\t326.3\t0\t2\r\n",
      "5\t3\t81.6\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/282.R1.fastq.gz.trimmed.fastq.gz /code/sequences/282.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.24 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,537\r\n",
      "Reads with adapters:                    79,695 (95.4%)\r\n",
      "Reads written (passing filters):        83,537 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,519,724 bp\r\n",
      "Total written (filtered):     19,048,623 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67745 times.\r\n",
      "67707 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t80\t0.0\t0\t80\r\n",
      "21\t67605\t0.0\t0\t67605\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1305.3\t0\t28\r\n",
      "4\t1\t326.3\t0\t1\r\n",
      "6\t2\t20.4\t0\t2\r\n",
      "7\t1\t5.1\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11950 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "11905 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1305.3\t0\t45\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1305.3\t0\t25\r\n",
      "4\t11880\t326.3\t0\t11880\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2597.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2597.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 16.46 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 288,244\r\n",
      "Reads with adapters:                   269,597 (93.5%)\r\n",
      "Reads written (passing filters):       288,244 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    71,395,158 bp\r\n",
      "Total written (filtered):     66,368,608 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 232065 times.\r\n",
      "231937 times, it overlapped the 5' end of a read\r\n",
      "128 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t4503.8\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t5\t0.0\t0\t5\r\n",
      "18\t23\t0.0\t0\t23\r\n",
      "19\t36\t0.0\t0\t36\r\n",
      "20\t349\t0.0\t0\t349\r\n",
      "21\t231522\t0.0\t0\t231522\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t100\t4503.8\t0\t100\r\n",
      "4\t2\t1126.0\t0\t2\r\n",
      "7\t2\t17.6\t0\t2\r\n",
      "47\t3\t0.0\t0\t3\r\n",
      "61\t2\t0.0\t0\t2\r\n",
      "270\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t17\t0.0\t0\t17\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 37532 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "37470 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t4503.8\t0\t54\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t104\t4503.8\t0\t104\r\n",
      "4\t37365\t1126.0\t0\t37365\r\n",
      "5\t1\t281.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2870.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2870.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.30 s (29 us/read; 2.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,631\r\n",
      "Reads with adapters:                     6,901 (4.7%)\r\n",
      "Reads written (passing filters):       145,631 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,856,293 bp\r\n",
      "Total written (filtered):     41,751,068 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3206 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "3200 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2275.5\t0\t5\r\n",
      "4\t1\t568.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3181\t2275.5\t0\t3181\r\n",
      "4\t14\t568.9\t0\t14\r\n",
      "5\t5\t142.2\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3695 times.\r\n",
      "3525 times, it overlapped the 5' end of a read\r\n",
      "170 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t5\t568.9\t0\t5\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t3515\t0.0\t1\t0 3515\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t2275.5\t0\t160\r\n",
      "4\t3\t568.9\t0\t3\r\n",
      "5\t7\t142.2\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2871.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2871.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.31 s (36 us/read; 1.66 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,453\r\n",
      "Reads with adapters:                     5,830 (4.9%)\r\n",
      "Reads written (passing filters):       119,453 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    34,109,072 bp\r\n",
      "Total written (filtered):     34,025,439 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2941 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2940 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1866.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2925\t1866.5\t0\t2925\r\n",
      "4\t10\t466.6\t0\t10\r\n",
      "5\t5\t116.7\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2889 times.\r\n",
      "2744 times, it overlapped the 5' end of a read\r\n",
      "145 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1866.5\t0\t1\r\n",
      "4\t1\t466.6\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2739\t0.0\t1\t0 2739\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t134\t1866.5\t0\t134\r\n",
      "4\t1\t466.6\t0\t1\r\n",
      "5\t8\t116.7\t0\t8\r\n",
      "6\t1\t29.2\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2872.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2872.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.02 s (29 us/read; 2.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,951\r\n",
      "Reads with adapters:                     5,295 (5.1%)\r\n",
      "Reads written (passing filters):       103,951 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,941,785 bp\r\n",
      "Total written (filtered):     29,862,325 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2522 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2521 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2500\t1624.2\t0\t2500\r\n",
      "4\t8\t406.1\t0\t8\r\n",
      "5\t12\t101.5\t0\t12\r\n",
      "6\t1\t25.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2773 times.\r\n",
      "2649 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1624.2\t0\t1\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2645\t0.0\t1\t0 2645\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1624.2\t0\t114\r\n",
      "4\t8\t406.1\t0\t8\r\n",
      "5\t2\t101.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2872.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2872.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.93 s (38 us/read; 1.59 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,951\r\n",
      "Reads with adapters:                    76,158 (73.3%)\r\n",
      "Reads written (passing filters):       103,951 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,029,127 bp\r\n",
      "Total written (filtered):     28,466,353 bp (94.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 73826 times.\r\n",
      "73756 times, it overlapped the 5' end of a read\r\n",
      "70 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1624.2\t0\t2\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t96\t0.0\t0\t96\r\n",
      "21\t73631\t0.0\t0\t73631\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1624.2\t0\t52\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "7\t1\t6.3\t0\t1\r\n",
      "266\t2\t0.0\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2332 times.\r\n",
      "129 times, it overlapped the 5' end of a read\r\n",
      "2203 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t1624.2\t0\t128\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1624.2\t0\t21\r\n",
      "4\t2181\t406.1\t0\t2181\r\n",
      "5\t1\t101.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2873.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2873.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.62 s (25 us/read; 2.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,259\r\n",
      "Reads with adapters:                     5,391 (5.2%)\r\n",
      "Reads written (passing filters):       104,259 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,869,039 bp\r\n",
      "Total written (filtered):     29,793,157 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2801 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2799 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1629.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2787\t1629.0\t0\t2787\r\n",
      "4\t8\t407.3\t0\t8\r\n",
      "5\t4\t101.8\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2590 times.\r\n",
      "2491 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1629.0\t0\t1\r\n",
      "4\t3\t407.3\t0\t3\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2484\t0.0\t1\t0 2484\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t95\t1629.0\t0\t95\r\n",
      "4\t3\t407.3\t0\t3\r\n",
      "5\t1\t101.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2870.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2870.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.76 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,631\r\n",
      "Reads with adapters:                   119,877 (82.3%)\r\n",
      "Reads written (passing filters):       145,631 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,965,994 bp\r\n",
      "Total written (filtered):     39,504,609 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 116466 times.\r\n",
      "116402 times, it overlapped the 5' end of a read\r\n",
      "64 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2275.5\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t161\t0.0\t0\t161\r\n",
      "21\t116211\t0.0\t0\t116211\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t2275.5\t0\t48\r\n",
      "4\t1\t568.9\t0\t1\r\n",
      "7\t3\t8.9\t0\t3\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "298\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3411 times.\r\n",
      "49 times, it overlapped the 5' end of a read\r\n",
      "3362 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t2275.5\t0\t45\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t2275.5\t0\t16\r\n",
      "4\t3345\t568.9\t0\t3345\r\n",
      "5\t1\t142.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2871.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2871.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.36 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,453\r\n",
      "Reads with adapters:                   114,274 (95.7%)\r\n",
      "Reads written (passing filters):       119,453 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    34,179,792 bp\r\n",
      "Total written (filtered):     31,836,619 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 110888 times.\r\n",
      "110869 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t137\t0.0\t0\t137\r\n",
      "21\t110697\t0.0\t0\t110697\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1866.5\t0\t12\r\n",
      "4\t1\t466.6\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3386 times.\r\n",
      "65 times, it overlapped the 5' end of a read\r\n",
      "3321 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t1866.5\t0\t61\r\n",
      "27\t4\t0.0\t1\t2 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1866.5\t0\t11\r\n",
      "4\t3310\t466.6\t0\t3310\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2873.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2873.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.51 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,259\r\n",
      "Reads with adapters:                    86,109 (82.6%)\r\n",
      "Reads written (passing filters):       104,259 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,979,873 bp\r\n",
      "Total written (filtered):     28,195,997 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 84581 times.\r\n",
      "84547 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t101\t0.0\t0\t101\r\n",
      "21\t84420\t0.0\t0\t84420\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1629.0\t0\t23\r\n",
      "4\t2\t407.3\t0\t2\r\n",
      "5\t1\t101.8\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1528 times.\r\n",
      "137 times, it overlapped the 5' end of a read\r\n",
      "1391 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t132\t1629.0\t0\t132\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1629.0\t0\t24\r\n",
      "4\t1367\t407.3\t0\t1367\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2874.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2874.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.34 s (34 us/read; 1.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,478\r\n",
      "Reads with adapters:                     4,867 (4.9%)\r\n",
      "Reads written (passing filters):        98,478 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,392,836 bp\r\n",
      "Total written (filtered):     28,322,974 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2435 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2432 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1538.7\t0\t2\r\n",
      "4\t1\t384.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2417\t1538.7\t0\t2417\r\n",
      "4\t7\t384.7\t0\t7\r\n",
      "5\t8\t96.2\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2432 times.\r\n",
      "2302 times, it overlapped the 5' end of a read\r\n",
      "130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t384.7\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2298\t0.0\t1\t0 2298\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t121\t1538.7\t0\t121\r\n",
      "4\t3\t384.7\t0\t3\r\n",
      "5\t6\t96.2\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2874.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2874.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.96 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,478\r\n",
      "Reads with adapters:                    60,292 (61.2%)\r\n",
      "Reads written (passing filters):        98,478 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,472,098 bp\r\n",
      "Total written (filtered):     27,236,766 bp (95.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58481 times.\r\n",
      "58425 times, it overlapped the 5' end of a read\r\n",
      "56 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1538.7\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t70\t0.0\t0\t70\r\n",
      "21\t58335\t0.0\t0\t58335\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1538.7\t0\t48\r\n",
      "5\t1\t96.2\t0\t1\r\n",
      "7\t3\t6.0\t0\t3\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1811 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "1744 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t1538.7\t0\t66\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1538.7\t0\t26\r\n",
      "4\t1718\t384.7\t0\t1718\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2876.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2876.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.80 s (27 us/read; 2.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,277\r\n",
      "Reads with adapters:                     7,299 (5.1%)\r\n",
      "Reads written (passing filters):       142,277 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,654,832 bp\r\n",
      "Total written (filtered):     41,553,599 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3818 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "3807 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t2223.1\t0\t9\r\n",
      "4\t2\t555.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3789\t2223.1\t0\t3789\r\n",
      "4\t11\t555.8\t0\t11\r\n",
      "5\t7\t138.9\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3481 times.\r\n",
      "3314 times, it overlapped the 5' end of a read\r\n",
      "167 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2223.1\t0\t8\r\n",
      "4\t2\t555.8\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3302\t0.0\t1\t0 3302\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t157\t2223.1\t0\t157\r\n",
      "4\t7\t555.8\t0\t7\r\n",
      "5\t3\t138.9\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2880.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2880.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.48 s (24 us/read; 2.52 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,326\r\n",
      "Reads with adapters:                     5,389 (5.2%)\r\n",
      "Reads written (passing filters):       104,326 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,038,532 bp\r\n",
      "Total written (filtered):     30,961,627 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2738 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2737 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1630.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2720\t1630.1\t0\t2720\r\n",
      "4\t11\t407.5\t0\t11\r\n",
      "5\t6\t101.9\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2651 times.\r\n",
      "2527 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1630.1\t0\t9\r\n",
      "4\t1\t407.5\t0\t1\r\n",
      "26\t6\t0.0\t1\t0 6\r\n",
      "27\t2511\t0.0\t1\t0 2511\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t1630.1\t0\t115\r\n",
      "4\t1\t407.5\t0\t1\r\n",
      "5\t7\t101.9\t0\t7\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2880.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2880.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.79 s (36 us/read; 1.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 104,326\r\n",
      "Reads with adapters:                    90,162 (86.4%)\r\n",
      "Reads written (passing filters):       104,326 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,071,371 bp\r\n",
      "Total written (filtered):     29,182,423 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 89800 times.\r\n",
      "89776 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1630.1\t0\t2\r\n",
      "15\t2\t0.0\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t120\t0.0\t0\t120\r\n",
      "21\t89624\t0.0\t0\t89624\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1630.1\t0\t15\r\n",
      "7\t1\t6.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 362 times.\r\n",
      "23 times, it overlapped the 5' end of a read\r\n",
      "339 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1630.1\t0\t22\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1630.1\t0\t10\r\n",
      "4\t329\t407.5\t0\t329\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2877.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2877.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.53 s (29 us/read; 2.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 155,034\r\n",
      "Reads with adapters:                     7,019 (4.5%)\r\n",
      "Reads written (passing filters):       155,034 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    46,562,762 bp\r\n",
      "Total written (filtered):     46,456,251 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3279 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3274 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2422.4\t0\t3\r\n",
      "4\t2\t605.6\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3265\t2422.4\t0\t3265\r\n",
      "4\t6\t605.6\t0\t6\r\n",
      "5\t3\t151.4\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3740 times.\r\n",
      "3567 times, it overlapped the 5' end of a read\r\n",
      "173 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2422.4\t0\t8\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3554\t0.0\t1\t0 3554\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t155\t2422.4\t0\t155\r\n",
      "4\t5\t605.6\t0\t5\r\n",
      "5\t13\t151.4\t0\t13\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2876.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2876.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.11 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,277\r\n",
      "Reads with adapters:                   133,188 (93.6%)\r\n",
      "Reads written (passing filters):       142,277 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,728,624 bp\r\n",
      "Total written (filtered):     38,952,488 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 131774 times.\r\n",
      "131740 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2223.1\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t39\t0.0\t0\t39\r\n",
      "20\t139\t0.0\t0\t139\r\n",
      "21\t131544\t0.0\t0\t131544\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t2223.1\t0\t19\r\n",
      "4\t1\t555.8\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1414 times.\r\n",
      "96 times, it overlapped the 5' end of a read\r\n",
      "1318 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t2223.1\t0\t93\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2223.1\t0\t8\r\n",
      "4\t1310\t555.8\t0\t1310\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2877.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2877.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.90 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 155,034\r\n",
      "Reads with adapters:                   150,277 (96.9%)\r\n",
      "Reads written (passing filters):       155,034 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    46,606,572 bp\r\n",
      "Total written (filtered):     43,449,311 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 150144 times.\r\n",
      "150120 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t226\t0.0\t0\t226\r\n",
      "21\t149854\t0.0\t0\t149854\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t2422.4\t0\t9\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 133 times.\r\n",
      "85 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t81\t2422.4\t0\t81\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2422.4\t0\t2\r\n",
      "4\t45\t605.6\t0\t45\r\n",
      "5\t1\t151.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2881.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2881.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.85 s (28 us/read; 2.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,535\r\n",
      "Reads with adapters:                     4,960 (4.9%)\r\n",
      "Reads written (passing filters):       101,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,416,410 bp\r\n",
      "Total written (filtered):     30,342,646 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2372 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2369 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1586.5\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2357\t1586.5\t0\t2357\r\n",
      "4\t5\t396.6\t0\t5\r\n",
      "5\t7\t99.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2588 times.\r\n",
      "2464 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1586.5\t0\t11\r\n",
      "4\t1\t396.6\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2451\t0.0\t1\t0 2451\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1586.5\t0\t114\r\n",
      "4\t3\t396.6\t0\t3\r\n",
      "5\t7\t99.2\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2881.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2881.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.47 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,535\r\n",
      "Reads with adapters:                    97,596 (96.1%)\r\n",
      "Reads written (passing filters):       101,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,446,922 bp\r\n",
      "Total written (filtered):     28,397,709 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97421 times.\r\n",
      "97403 times, it overlapped the 5' end of a read\r\n",
      "18 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t396.6\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t27\t0.0\t0\t27\r\n",
      "20\t141\t0.0\t0\t141\r\n",
      "21\t97222\t0.0\t0\t97222\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1586.5\t0\t7\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 175 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "128 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1586.5\t0\t45\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1586.5\t0\t5\r\n",
      "4\t123\t396.6\t0\t123\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2875.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2875.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.50 s (28 us/read; 2.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 340,357\r\n",
      "Reads with adapters:                    16,228 (4.8%)\r\n",
      "Reads written (passing filters):       340,357 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    99,071,476 bp\r\n",
      "Total written (filtered):     98,834,234 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 8017 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "7999 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t5318.1\t0\t12\r\n",
      "4\t5\t1329.5\t0\t5\r\n",
      "5\t1\t332.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7962\t5318.1\t0\t7962\r\n",
      "4\t21\t1329.5\t0\t21\r\n",
      "5\t16\t332.4\t0\t16\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8211 times.\r\n",
      "7873 times, it overlapped the 5' end of a read\r\n",
      "338 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t5318.1\t0\t20\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t8\t0.0\t1\t0 8\r\n",
      "27\t7842\t0.0\t1\t0 7842\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t317\t5318.1\t0\t317\r\n",
      "4\t4\t1329.5\t0\t4\r\n",
      "5\t16\t332.4\t0\t16\r\n",
      "6\t1\t83.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2926.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2926.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.46 s (35 us/read; 1.71 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,593\r\n",
      "Reads with adapters:                     4,350 (4.4%)\r\n",
      "Reads written (passing filters):        98,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,206,424 bp\r\n",
      "Total written (filtered):     28,147,941 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2279 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2274 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1540.5\t0\t3\r\n",
      "4\t2\t385.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2259\t1540.5\t0\t2259\r\n",
      "4\t8\t385.1\t0\t8\r\n",
      "5\t7\t96.3\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2071 times.\r\n",
      "1903 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1540.5\t0\t11\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1888\t0.0\t1\t0 1888\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1540.5\t0\t161\r\n",
      "4\t4\t385.1\t0\t4\r\n",
      "5\t3\t96.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2926.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2926.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.68 s (47 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,593\r\n",
      "Reads with adapters:                    81,672 (82.8%)\r\n",
      "Reads written (passing filters):        98,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,299,238 bp\r\n",
      "Total written (filtered):     26,625,771 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 79171 times.\r\n",
      "79135 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t126\t0.0\t0\t126\r\n",
      "21\t78979\t0.0\t0\t78979\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1540.5\t0\t29\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2501 times.\r\n",
      "116 times, it overlapped the 5' end of a read\r\n",
      "2385 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t1540.5\t0\t115\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1540.5\t0\t18\r\n",
      "4\t2367\t385.1\t0\t2367\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2940.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2940.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.32 s (28 us/read; 2.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,480\r\n",
      "Reads with adapters:                     3,512 (4.2%)\r\n",
      "Reads written (passing filters):        83,480 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,861,218 bp\r\n",
      "Total written (filtered):     24,815,408 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1852 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1849 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1304.4\t0\t2\r\n",
      "4\t1\t326.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1839\t1304.4\t0\t1839\r\n",
      "4\t9\t326.1\t0\t9\r\n",
      "5\t1\t81.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1660 times.\r\n",
      "1482 times, it overlapped the 5' end of a read\r\n",
      "178 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1304.4\t0\t13\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1468\t0.0\t1\t0 1468\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t173\t1304.4\t0\t173\r\n",
      "4\t3\t326.1\t0\t3\r\n",
      "5\t2\t81.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2943.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2943.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.27 s (28 us/read; 2.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,593\r\n",
      "Reads with adapters:                     3,843 (4.8%)\r\n",
      "Reads written (passing filters):        80,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,974,674 bp\r\n",
      "Total written (filtered):     23,918,043 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1866 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1861 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1259.3\t0\t4\r\n",
      "4\t1\t314.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1848\t1259.3\t0\t1848\r\n",
      "4\t9\t314.8\t0\t9\r\n",
      "5\t4\t78.7\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1977 times.\r\n",
      "1884 times, it overlapped the 5' end of a read\r\n",
      "93 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1259.3\t0\t6\r\n",
      "27\t1878\t0.0\t1\t0 1878\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t87\t1259.3\t0\t87\r\n",
      "4\t1\t314.8\t0\t1\r\n",
      "5\t4\t78.7\t0\t4\r\n",
      "6\t1\t19.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2940.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2940.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.21 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,480\r\n",
      "Reads with adapters:                    79,297 (95.0%)\r\n",
      "Reads written (passing filters):        83,480 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,898,611 bp\r\n",
      "Total written (filtered):     23,238,597 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 78925 times.\r\n",
      "78916 times, it overlapped the 5' end of a read\r\n",
      "9 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1304.4\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t88\t0.0\t0\t88\r\n",
      "21\t78798\t0.0\t0\t78798\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1304.4\t0\t4\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 372 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "264 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t105\t1304.4\t0\t105\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1304.4\t0\t4\r\n",
      "4\t260\t326.1\t0\t260\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2943.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2943.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.48 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,593\r\n",
      "Reads with adapters:                    75,930 (94.2%)\r\n",
      "Reads written (passing filters):        80,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,001,403 bp\r\n",
      "Total written (filtered):     22,409,622 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 75695 times.\r\n",
      "75678 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t314.8\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t95\t0.0\t0\t95\r\n",
      "21\t75560\t0.0\t0\t75560\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1259.3\t0\t11\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 235 times.\r\n",
      "64 times, it overlapped the 5' end of a read\r\n",
      "171 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1259.3\t0\t63\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1259.3\t0\t12\r\n",
      "4\t159\t314.8\t0\t159\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2977.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2977.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.89 s (28 us/read; 2.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 173,134\r\n",
      "Reads with adapters:                     8,506 (4.9%)\r\n",
      "Reads written (passing filters):       173,134 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    50,692,088 bp\r\n",
      "Total written (filtered):     50,576,304 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4551 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "4550 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2705.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4519\t2705.2\t0\t4519\r\n",
      "4\t20\t676.3\t0\t20\r\n",
      "5\t11\t169.1\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3955 times.\r\n",
      "3750 times, it overlapped the 5' end of a read\r\n",
      "205 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2705.2\t0\t3\r\n",
      "4\t1\t676.3\t0\t1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3742\t0.0\t1\t0 3742\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t188\t2705.2\t0\t188\r\n",
      "4\t6\t676.3\t0\t6\r\n",
      "5\t10\t169.1\t0\t10\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2875.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2875.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 18.88 s (55 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 340,357\r\n",
      "Reads with adapters:                   283,266 (83.2%)\r\n",
      "Reads written (passing filters):       340,357 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    99,275,741 bp\r\n",
      "Total written (filtered):     93,400,607 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 278690 times.\r\n",
      "278603 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t5318.1\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t36\t0.0\t0\t36\r\n",
      "19\t52\t0.0\t0\t52\r\n",
      "20\t361\t0.0\t0\t361\r\n",
      "21\t278148\t0.0\t0\t278148\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t5318.1\t0\t60\r\n",
      "4\t1\t1329.5\t0\t1\r\n",
      "7\t3\t20.8\t0\t3\r\n",
      "300\t23\t0.0\t0\t23\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4576 times.\r\n",
      "340 times, it overlapped the 5' end of a read\r\n",
      "4236 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t338\t5318.1\t0\t338\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t5318.1\t0\t34\r\n",
      "4\t4201\t1329.5\t0\t4201\r\n",
      "5\t1\t332.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2976.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2976.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.95 s (41 us/read; 1.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 192,952\r\n",
      "Reads with adapters:                     8,668 (4.5%)\r\n",
      "Reads written (passing filters):       192,952 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    55,215,250 bp\r\n",
      "Total written (filtered):     55,090,156 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4296 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "4288 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t3014.9\t0\t7\r\n",
      "4\t1\t753.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4258\t3014.9\t0\t4258\r\n",
      "4\t20\t753.7\t0\t20\r\n",
      "5\t10\t188.4\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4372 times.\r\n",
      "4116 times, it overlapped the 5' end of a read\r\n",
      "256 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t3014.9\t0\t1\r\n",
      "4\t1\t753.7\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t4113\t0.0\t1\t0 4113\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t244\t3014.9\t0\t244\r\n",
      "4\t6\t753.7\t0\t6\r\n",
      "5\t5\t188.4\t0\t5\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2977.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2977.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.74 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 173,134\r\n",
      "Reads with adapters:                   154,428 (89.2%)\r\n",
      "Reads written (passing filters):       173,134 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    50,789,731 bp\r\n",
      "Total written (filtered):     47,577,164 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 152566 times.\r\n",
      "152512 times, it overlapped the 5' end of a read\r\n",
      "54 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2705.2\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t231\t0.0\t0\t231\r\n",
      "21\t152233\t0.0\t0\t152233\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t2705.2\t0\t42\r\n",
      "4\t3\t676.3\t0\t3\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1862 times.\r\n",
      "128 times, it overlapped the 5' end of a read\r\n",
      "1734 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t126\t2705.2\t0\t126\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t2705.2\t0\t22\r\n",
      "4\t1712\t676.3\t0\t1712\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2978.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2978.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.17 s (45 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 183,511\r\n",
      "Reads with adapters:                     9,537 (5.2%)\r\n",
      "Reads written (passing filters):       183,511 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,087,985 bp\r\n",
      "Total written (filtered):     51,964,975 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5300 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "5292 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2867.4\t0\t6\r\n",
      "4\t2\t716.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5265\t2867.4\t0\t5265\r\n",
      "4\t20\t716.8\t0\t20\r\n",
      "5\t6\t179.2\t0\t6\r\n",
      "6\t1\t44.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4237 times.\r\n",
      "3934 times, it overlapped the 5' end of a read\r\n",
      "303 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2867.4\t0\t3\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3929\t0.0\t1\t0 3929\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t291\t2867.4\t0\t291\r\n",
      "4\t4\t716.8\t0\t4\r\n",
      "5\t8\t179.2\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2976.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2976.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 10.30 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 192,952\r\n",
      "Reads with adapters:                   179,977 (93.3%)\r\n",
      "Reads written (passing filters):       192,952 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    55,352,796 bp\r\n",
      "Total written (filtered):     51,637,891 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 176151 times.\r\n",
      "176112 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t3014.9\t0\t1\r\n",
      "11\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t22\t0.0\t0\t22\r\n",
      "19\t33\t0.0\t0\t33\r\n",
      "20\t230\t0.0\t0\t230\r\n",
      "21\t175823\t0.0\t0\t175823\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t3014.9\t0\t29\r\n",
      "4\t1\t753.7\t0\t1\r\n",
      "5\t2\t188.4\t0\t2\r\n",
      "7\t1\t11.8\t0\t1\r\n",
      "240\t1\t0.0\t0\t1\r\n",
      "298\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3826 times.\r\n",
      "201 times, it overlapped the 5' end of a read\r\n",
      "3625 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t200\t3014.9\t0\t200\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t3014.9\t0\t22\r\n",
      "4\t3603\t753.7\t0\t3603\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2979.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2979.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.34 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 130,695\r\n",
      "Reads with adapters:                   126,307 (96.6%)\r\n",
      "Reads written (passing filters):       130,695 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    38,390,595 bp\r\n",
      "Total written (filtered):     35,751,231 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 125357 times.\r\n",
      "125329 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2042.1\t0\t1\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t29\t0.0\t0\t29\r\n",
      "20\t210\t0.0\t0\t210\r\n",
      "21\t125078\t0.0\t0\t125078\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t2042.1\t0\t12\r\n",
      "4\t1\t510.5\t0\t1\r\n",
      "7\t1\t8.0\t0\t1\r\n",
      "76\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t5\t0.0\t0\t5\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 950 times.\r\n",
      "30 times, it overlapped the 5' end of a read\r\n",
      "920 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t2042.1\t0\t30\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2042.1\t0\t6\r\n",
      "4\t914\t510.5\t0\t914\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2978.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2978.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 10.24 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 183,511\r\n",
      "Reads with adapters:                   177,377 (96.7%)\r\n",
      "Reads written (passing filters):       183,511 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,200,698 bp\r\n",
      "Total written (filtered):     48,545,596 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 173187 times.\r\n",
      "173162 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t200\t0.0\t0\t200\r\n",
      "21\t172920\t0.0\t0\t172920\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t2867.4\t0\t16\r\n",
      "4\t1\t716.8\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4190 times.\r\n",
      "88 times, it overlapped the 5' end of a read\r\n",
      "4102 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t82\t2867.4\t0\t82\r\n",
      "27\t6\t0.0\t1\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t2867.4\t0\t12\r\n",
      "4\t4090\t716.8\t0\t4090\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2979.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2979.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.55 s (42 us/read; 1.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 130,695\r\n",
      "Reads with adapters:                     7,256 (5.6%)\r\n",
      "Reads written (passing filters):       130,695 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    38,355,413 bp\r\n",
      "Total written (filtered):     38,269,135 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4381 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "4380 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2042.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4359\t2042.1\t0\t4359\r\n",
      "4\t2\t510.5\t0\t2\r\n",
      "5\t19\t127.6\t0\t19\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2875 times.\r\n",
      "2690 times, it overlapped the 5' end of a read\r\n",
      "185 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2042.1\t0\t3\r\n",
      "4\t1\t510.5\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2683\t0.0\t1\t0 2683\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t2042.1\t0\t180\r\n",
      "4\t3\t510.5\t0\t3\r\n",
      "5\t1\t127.6\t0\t1\r\n",
      "6\t1\t31.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2980.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2980.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.74 s (33 us/read; 1.81 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,982\r\n",
      "Reads with adapters:                     5,514 (4.9%)\r\n",
      "Reads written (passing filters):       112,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,701,755 bp\r\n",
      "Total written (filtered):     32,622,895 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2700 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2698 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1765.3\t0\t1\r\n",
      "4\t1\t441.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2645\t1765.3\t0\t2645\r\n",
      "4\t19\t441.3\t0\t19\r\n",
      "5\t34\t110.3\t0\t34\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2814 times.\r\n",
      "2593 times, it overlapped the 5' end of a read\r\n",
      "221 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t441.3\t0\t1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2588\t0.0\t1\t0 2588\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t207\t1765.3\t0\t207\r\n",
      "4\t4\t441.3\t0\t4\r\n",
      "5\t9\t110.3\t0\t9\r\n",
      "6\t1\t27.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2982.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2982.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.17 s (26 us/read; 2.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,540\r\n",
      "Reads with adapters:                     6,240 (5.1%)\r\n",
      "Reads written (passing filters):       121,540 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,556,265 bp\r\n",
      "Total written (filtered):     35,469,337 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3253 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "3249 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1899.1\t0\t2\r\n",
      "4\t2\t474.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3232\t1899.1\t0\t3232\r\n",
      "4\t12\t474.8\t0\t12\r\n",
      "5\t4\t118.7\t0\t4\r\n",
      "8\t1\t1.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2987 times.\r\n",
      "2842 times, it overlapped the 5' end of a read\r\n",
      "145 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1899.1\t0\t4\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2833\t0.0\t1\t0 2833\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t1899.1\t0\t135\r\n",
      "4\t4\t474.8\t0\t4\r\n",
      "5\t5\t118.7\t0\t5\r\n",
      "61\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2980.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2980.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.29 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 112,982\r\n",
      "Reads with adapters:                   106,621 (94.4%)\r\n",
      "Reads written (passing filters):       112,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,740,019 bp\r\n",
      "Total written (filtered):     30,548,373 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 103770 times.\r\n",
      "103747 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t24\t0.0\t0\t24\r\n",
      "20\t140\t0.0\t0\t140\r\n",
      "21\t103574\t0.0\t0\t103574\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1765.3\t0\t12\r\n",
      "6\t1\t27.6\t0\t1\r\n",
      "7\t1\t6.9\t0\t1\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "67\t2\t0.0\t0\t2\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2851 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "2806 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1765.3\t0\t43\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1765.3\t0\t9\r\n",
      "4\t2797\t441.3\t0\t2797\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2981.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2981.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.97 s (29 us/read; 2.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 134,877\r\n",
      "Reads with adapters:                     5,857 (4.3%)\r\n",
      "Reads written (passing filters):       134,877 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,407,621 bp\r\n",
      "Total written (filtered):     39,324,633 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2895 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2892 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2107.5\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2858\t2107.5\t0\t2858\r\n",
      "4\t16\t526.9\t0\t16\r\n",
      "5\t16\t131.7\t0\t16\r\n",
      "8\t2\t2.1\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2962 times.\r\n",
      "2729 times, it overlapped the 5' end of a read\r\n",
      "233 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2107.5\t0\t5\r\n",
      "4\t1\t526.9\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t7\t0.0\t1\t0 7\r\n",
      "27\t2715\t0.0\t1\t0 2715\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t223\t2107.5\t0\t223\r\n",
      "4\t4\t526.9\t0\t4\r\n",
      "5\t6\t131.7\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2983.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2983.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.68 s (25 us/read; 2.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,834\r\n",
      "Reads with adapters:                     5,945 (5.6%)\r\n",
      "Reads written (passing filters):       106,834 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,695,751 bp\r\n",
      "Total written (filtered):     31,614,149 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3143 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3138 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1669.3\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3126\t1669.3\t0\t3126\r\n",
      "4\t7\t417.3\t0\t7\r\n",
      "5\t4\t104.3\t0\t4\r\n",
      "6\t1\t26.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2802 times.\r\n",
      "2660 times, it overlapped the 5' end of a read\r\n",
      "142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1669.3\t0\t2\r\n",
      "4\t2\t417.3\t0\t2\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2653\t0.0\t1\t0 2653\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t139\t1669.3\t0\t139\r\n",
      "5\t3\t104.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2982.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2982.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.67 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,540\r\n",
      "Reads with adapters:                    94,924 (78.1%)\r\n",
      "Reads written (passing filters):       121,540 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,644,163 bp\r\n",
      "Total written (filtered):     33,673,573 bp (94.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 93506 times.\r\n",
      "93465 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t24\t0.0\t0\t24\r\n",
      "20\t119\t0.0\t0\t119\r\n",
      "21\t93316\t0.0\t0\t93316\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1899.1\t0\t31\r\n",
      "4\t2\t474.8\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1418 times.\r\n",
      "159 times, it overlapped the 5' end of a read\r\n",
      "1259 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t158\t1899.1\t0\t158\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1899.1\t0\t22\r\n",
      "4\t1237\t474.8\t0\t1237\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2981.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2981.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.86 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 134,877\r\n",
      "Reads with adapters:                   123,821 (91.8%)\r\n",
      "Reads written (passing filters):       134,877 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,460,247 bp\r\n",
      "Total written (filtered):     36,888,623 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 121989 times.\r\n",
      "121953 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2107.5\t0\t2\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t164\t0.0\t0\t164\r\n",
      "21\t121746\t0.0\t0\t121746\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t2107.5\t0\t24\r\n",
      "291\t1\t0.0\t0\t1\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1832 times.\r\n",
      "61 times, it overlapped the 5' end of a read\r\n",
      "1771 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t2107.5\t0\t61\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t2107.5\t0\t19\r\n",
      "4\t1752\t526.9\t0\t1752\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2984.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2984.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.06 s (25 us/read; 2.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,249\r\n",
      "Reads with adapters:                     6,392 (5.2%)\r\n",
      "Reads written (passing filters):       122,249 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,046,008 bp\r\n",
      "Total written (filtered):     35,959,037 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3419 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "3418 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1910.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3402\t1910.1\t0\t3402\r\n",
      "4\t12\t477.5\t0\t12\r\n",
      "5\t3\t119.4\t0\t3\r\n",
      "6\t1\t29.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2973 times.\r\n",
      "2815 times, it overlapped the 5' end of a read\r\n",
      "158 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1910.1\t0\t4\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2809\t0.0\t1\t0 2809\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t148\t1910.1\t0\t148\r\n",
      "4\t5\t477.5\t0\t5\r\n",
      "5\t4\t119.4\t0\t4\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2983.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2983.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.06 s (57 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,834\r\n",
      "Reads with adapters:                   103,420 (96.8%)\r\n",
      "Reads written (passing filters):       106,834 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,740,487 bp\r\n",
      "Total written (filtered):     29,577,874 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102753 times.\r\n",
      "102738 times, it overlapped the 5' end of a read\r\n",
      "15 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t102585\t0.0\t0\t102585\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1669.3\t0\t6\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 667 times.\r\n",
      "125 times, it overlapped the 5' end of a read\r\n",
      "542 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t123\t1669.3\t0\t123\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1669.3\t0\t2\r\n",
      "4\t540\t417.3\t0\t540\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2984.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2984.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.65 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,249\r\n",
      "Reads with adapters:                   116,609 (95.4%)\r\n",
      "Reads written (passing filters):       122,249 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,107,363 bp\r\n",
      "Total written (filtered):     33,674,884 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 115486 times.\r\n",
      "115461 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1910.1\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t137\t0.0\t0\t137\r\n",
      "21\t115285\t0.0\t0\t115285\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1910.1\t0\t11\r\n",
      "4\t1\t477.5\t0\t1\r\n",
      "7\t1\t7.5\t0\t1\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1123 times.\r\n",
      "86 times, it overlapped the 5' end of a read\r\n",
      "1037 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t1910.1\t0\t85\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1910.1\t0\t10\r\n",
      "4\t1027\t477.5\t0\t1027\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2985.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2985.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.15 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,360\r\n",
      "Reads with adapters:                     6,812 (5.1%)\r\n",
      "Reads written (passing filters):       133,360 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,827,676 bp\r\n",
      "Total written (filtered):     39,738,912 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3825 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "3823 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2083.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3798\t2083.8\t0\t3798\r\n",
      "4\t14\t520.9\t0\t14\r\n",
      "5\t11\t130.2\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2987 times.\r\n",
      "2846 times, it overlapped the 5' end of a read\r\n",
      "141 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t520.9\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2844\t0.0\t1\t0 2844\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t134\t2083.8\t0\t134\r\n",
      "4\t2\t520.9\t0\t2\r\n",
      "5\t5\t130.2\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2985.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2985.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.35 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,360\r\n",
      "Reads with adapters:                   129,327 (97.0%)\r\n",
      "Reads written (passing filters):       133,360 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,882,716 bp\r\n",
      "Total written (filtered):     37,167,480 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 129123 times.\r\n",
      "129104 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t30\t0.0\t0\t30\r\n",
      "20\t149\t0.0\t0\t149\r\n",
      "21\t128909\t0.0\t0\t128909\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2083.8\t0\t7\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 204 times.\r\n",
      "111 times, it overlapped the 5' end of a read\r\n",
      "93 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t111\t2083.8\t0\t111\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2083.8\t0\t2\r\n",
      "4\t91\t520.9\t0\t91\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2987.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2987.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.08 s (27 us/read; 2.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 148,349\r\n",
      "Reads with adapters:                     7,070 (4.8%)\r\n",
      "Reads written (passing filters):       148,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,467,797 bp\r\n",
      "Total written (filtered):     43,367,567 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3590 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "3588 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2318.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3570\t2318.0\t0\t3570\r\n",
      "4\t10\t579.5\t0\t10\r\n",
      "5\t8\t144.9\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3480 times.\r\n",
      "3291 times, it overlapped the 5' end of a read\r\n",
      "189 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3288\t0.0\t1\t0 3288\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t181\t2318.0\t0\t181\r\n",
      "4\t1\t579.5\t0\t1\r\n",
      "5\t7\t144.9\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2986.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2986.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.20 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 160,375\r\n",
      "Reads with adapters:                     7,414 (4.6%)\r\n",
      "Reads written (passing filters):       160,375 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    47,889,470 bp\r\n",
      "Total written (filtered):     47,779,159 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3568 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "3566 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2505.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3532\t2505.9\t0\t3532\r\n",
      "4\t26\t626.5\t0\t26\r\n",
      "5\t7\t156.6\t0\t7\r\n",
      "6\t1\t39.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3846 times.\r\n",
      "3669 times, it overlapped the 5' end of a read\r\n",
      "177 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t626.5\t0\t2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3663\t0.0\t1\t0 3663\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t164\t2505.9\t0\t164\r\n",
      "4\t6\t626.5\t0\t6\r\n",
      "5\t7\t156.6\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2987.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2987.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.40 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 148,349\r\n",
      "Reads with adapters:                   143,102 (96.5%)\r\n",
      "Reads written (passing filters):       148,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,529,358 bp\r\n",
      "Total written (filtered):     40,544,190 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 141567 times.\r\n",
      "141537 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t24\t0.0\t0\t24\r\n",
      "20\t202\t0.0\t0\t202\r\n",
      "21\t141301\t0.0\t0\t141301\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2318.0\t0\t6\r\n",
      "4\t1\t579.5\t0\t1\r\n",
      "300\t23\t0.0\t0\t23\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1535 times.\r\n",
      "78 times, it overlapped the 5' end of a read\r\n",
      "1457 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t2318.0\t0\t70\r\n",
      "27\t8\t0.0\t1\t8\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2318.0\t0\t1\r\n",
      "4\t1456\t579.5\t0\t1456\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2986.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2986.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.59 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 160,375\r\n",
      "Reads with adapters:                   150,808 (94.0%)\r\n",
      "Reads written (passing filters):       160,375 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    47,946,774 bp\r\n",
      "Total written (filtered):     44,782,000 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 150562 times.\r\n",
      "150541 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t20\t0.0\t0\t20\r\n",
      "19\t24\t0.0\t0\t24\r\n",
      "20\t171\t0.0\t0\t171\r\n",
      "21\t150324\t0.0\t0\t150324\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2505.9\t0\t10\r\n",
      "4\t1\t626.5\t0\t1\r\n",
      "5\t1\t156.6\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 246 times.\r\n",
      "104 times, it overlapped the 5' end of a read\r\n",
      "142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t100\t2505.9\t0\t100\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t2505.9\t0\t14\r\n",
      "4\t128\t626.5\t0\t128\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2988.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2988.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.80 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,261\r\n",
      "Reads with adapters:                   116,914 (93.3%)\r\n",
      "Reads written (passing filters):       125,261 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    37,292,783 bp\r\n",
      "Total written (filtered):     34,840,997 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 116492 times.\r\n",
      "116465 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t159\t0.0\t0\t159\r\n",
      "21\t116271\t0.0\t0\t116271\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1957.2\t0\t12\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t4\t0.0\t0\t4\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 422 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "402 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1957.2\t0\t17\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1957.2\t0\t7\r\n",
      "4\t395\t489.3\t0\t395\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2988.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2988.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.24 s (34 us/read; 1.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,261\r\n",
      "Reads with adapters:                     7,006 (5.6%)\r\n",
      "Reads written (passing filters):       125,261 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    37,263,506 bp\r\n",
      "Total written (filtered):     37,170,432 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3883 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "3876 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1957.2\t0\t5\r\n",
      "4\t2\t489.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3862\t1957.2\t0\t3862\r\n",
      "4\t9\t489.3\t0\t9\r\n",
      "5\t5\t122.3\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3123 times.\r\n",
      "2990 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1957.2\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2986\t0.0\t1\t0 2986\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t1957.2\t0\t128\r\n",
      "5\t4\t122.3\t0\t4\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2992.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2992.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.05 s (43 us/read; 1.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,754\r\n",
      "Reads with adapters:                    90,737 (96.8%)\r\n",
      "Reads written (passing filters):        93,754 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,685,581 bp\r\n",
      "Total written (filtered):     25,787,085 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90218 times.\r\n",
      "90202 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t100\t0.0\t0\t100\r\n",
      "21\t90068\t0.0\t0\t90068\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1464.9\t0\t8\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 519 times.\r\n",
      "128 times, it overlapped the 5' end of a read\r\n",
      "391 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t125\t1464.9\t0\t125\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1464.9\t0\t4\r\n",
      "4\t387\t366.2\t0\t387\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2993.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2993.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.19 s (30 us/read; 1.98 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,383\r\n",
      "Reads with adapters:                     5,168 (7.1%)\r\n",
      "Reads written (passing filters):        72,383 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,881,178 bp\r\n",
      "Total written (filtered):     20,825,739 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3415 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3410 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1131.0\t0\t3\r\n",
      "4\t2\t282.7\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3401\t1131.0\t0\t3401\r\n",
      "5\t5\t70.7\t0\t5\r\n",
      "6\t4\t17.7\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1753 times.\r\n",
      "1663 times, it overlapped the 5' end of a read\r\n",
      "90 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1661\t0.0\t1\t0 1661\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t1131.0\t0\t86\r\n",
      "4\t3\t282.7\t0\t3\r\n",
      "5\t1\t70.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2990.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2990.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.27 s (30 us/read; 2.02 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 143,962\r\n",
      "Reads with adapters:                     6,842 (4.8%)\r\n",
      "Reads written (passing filters):       143,962 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,807,830 bp\r\n",
      "Total written (filtered):     42,702,071 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3116 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "3109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2249.4\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3083\t2249.4\t0\t3083\r\n",
      "4\t17\t562.4\t0\t17\r\n",
      "5\t9\t140.6\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3726 times.\r\n",
      "3550 times, it overlapped the 5' end of a read\r\n",
      "176 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2249.4\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3544\t0.0\t1\t0 3544\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t2249.4\t0\t160\r\n",
      "4\t4\t562.4\t0\t4\r\n",
      "5\t12\t140.6\t0\t12\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2992.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2992.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.33 s (36 us/read; 1.69 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,754\r\n",
      "Reads with adapters:                     5,117 (5.5%)\r\n",
      "Reads written (passing filters):        93,754 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,638,480 bp\r\n",
      "Total written (filtered):     27,570,347 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2830 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2828 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1464.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2814\t1464.9\t0\t2814\r\n",
      "4\t10\t366.2\t0\t10\r\n",
      "5\t4\t91.6\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2287 times.\r\n",
      "2200 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1464.9\t0\t1\r\n",
      "4\t1\t366.2\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2196\t0.0\t1\t0 2196\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1464.9\t0\t80\r\n",
      "4\t2\t366.2\t0\t2\r\n",
      "5\t4\t91.6\t0\t4\r\n",
      "6\t1\t22.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2993.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2993.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.84 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,383\r\n",
      "Reads with adapters:                    36,401 (50.3%)\r\n",
      "Reads written (passing filters):        72,383 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,935,556 bp\r\n",
      "Total written (filtered):     20,201,930 bp (96.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 34540 times.\r\n",
      "34495 times, it overlapped the 5' end of a read\r\n",
      "45 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1131.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t6\t0.0\t0\t6\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t34439\t0.0\t0\t34439\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1131.0\t0\t37\r\n",
      "4\t1\t282.7\t0\t1\r\n",
      "6\t1\t17.7\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1861 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "1818 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1131.0\t0\t42\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1131.0\t0\t23\r\n",
      "4\t1795\t282.7\t0\t1795\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2990.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2990.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.39 s (58 us/read; 1.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 143,962\r\n",
      "Reads with adapters:                   137,570 (95.6%)\r\n",
      "Reads written (passing filters):       143,962 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,867,736 bp\r\n",
      "Total written (filtered):     39,985,948 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 136992 times.\r\n",
      "136965 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2249.4\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t195\t0.0\t0\t195\r\n",
      "21\t136732\t0.0\t0\t136732\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t2249.4\t0\t14\r\n",
      "4\t1\t562.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 578 times.\r\n",
      "158 times, it overlapped the 5' end of a read\r\n",
      "420 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t156\t2249.4\t0\t156\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2249.4\t0\t10\r\n",
      "4\t410\t562.4\t0\t410\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2994.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2994.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.65 s (28 us/read; 2.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 164,877\r\n",
      "Reads with adapters:                     7,746 (4.7%)\r\n",
      "Reads written (passing filters):       164,877 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    48,515,949 bp\r\n",
      "Total written (filtered):     48,400,476 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3749 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "3739 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2576.2\t0\t8\r\n",
      "4\t2\t644.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3713\t2576.2\t0\t3713\r\n",
      "4\t17\t644.1\t0\t17\r\n",
      "5\t9\t161.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3997 times.\r\n",
      "3821 times, it overlapped the 5' end of a read\r\n",
      "176 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2576.2\t0\t5\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3812\t0.0\t1\t0 3812\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t158\t2576.2\t0\t158\r\n",
      "4\t5\t644.1\t0\t5\r\n",
      "5\t11\t161.0\t0\t11\r\n",
      "297\t1\t0.0\t1\t0 1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2995.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2995.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.73 s (30 us/read; 2.02 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 193,363\r\n",
      "Reads with adapters:                     8,000 (4.1%)\r\n",
      "Reads written (passing filters):       193,363 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    45,977,015 bp\r\n",
      "Total written (filtered):     45,865,348 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4056 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "4041 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t3021.3\t0\t11\r\n",
      "4\t4\t755.3\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4011\t3021.3\t0\t4011\r\n",
      "4\t19\t755.3\t0\t19\r\n",
      "5\t10\t188.8\t0\t10\r\n",
      "8\t1\t3.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3944 times.\r\n",
      "3648 times, it overlapped the 5' end of a read\r\n",
      "296 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3646\t0.0\t1\t0 3646\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t279\t3021.3\t0\t279\r\n",
      "4\t7\t755.3\t0\t7\r\n",
      "5\t9\t188.8\t0\t9\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2994.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2994.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.93 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 164,877\r\n",
      "Reads with adapters:                   158,896 (96.4%)\r\n",
      "Reads written (passing filters):       164,877 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    48,617,109 bp\r\n",
      "Total written (filtered):     45,297,435 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 157776 times.\r\n",
      "157753 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t189\t0.0\t0\t189\r\n",
      "21\t157529\t0.0\t0\t157529\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t2576.2\t0\t14\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1120 times.\r\n",
      "170 times, it overlapped the 5' end of a read\r\n",
      "950 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t166\t2576.2\t0\t166\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t2576.2\t0\t11\r\n",
      "4\t939\t644.1\t0\t939\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2996.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2996.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.21 s (37 us/read; 1.62 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 168,093\r\n",
      "Reads with adapters:                     9,320 (5.5%)\r\n",
      "Reads written (passing filters):       168,093 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    50,217,283 bp\r\n",
      "Total written (filtered):     50,100,259 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5435 times.\r\n",
      "21 times, it overlapped the 5' end of a read\r\n",
      "5414 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t2626.5\t0\t18\r\n",
      "4\t3\t656.6\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5384\t2626.5\t0\t5384\r\n",
      "4\t21\t656.6\t0\t21\r\n",
      "5\t9\t164.2\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3885 times.\r\n",
      "3711 times, it overlapped the 5' end of a read\r\n",
      "174 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2626.5\t0\t2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3703\t0.0\t1\t0 3703\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t166\t2626.5\t0\t166\r\n",
      "4\t3\t656.6\t0\t3\r\n",
      "5\t5\t164.2\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2997.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2997.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.68 s (32 us/read; 1.85 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 174,977\r\n",
      "Reads with adapters:                     9,721 (5.6%)\r\n",
      "Reads written (passing filters):       174,977 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    51,374,453 bp\r\n",
      "Total written (filtered):     51,252,142 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5700 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "5685 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t2734.0\t0\t12\r\n",
      "4\t1\t683.5\t0\t1\r\n",
      "5\t2\t170.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5676\t2734.0\t0\t5676\r\n",
      "4\t4\t683.5\t0\t4\r\n",
      "5\t5\t170.9\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4021 times.\r\n",
      "3874 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2734.0\t0\t4\r\n",
      "4\t1\t683.5\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3866\t0.0\t1\t0 3866\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t138\t2734.0\t0\t138\r\n",
      "4\t5\t683.5\t0\t5\r\n",
      "5\t3\t170.9\t0\t3\r\n",
      "271\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2995.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2995.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.41 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 193,363\r\n",
      "Reads with adapters:                   159,214 (82.3%)\r\n",
      "Reads written (passing filters):       193,363 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    46,490,038 bp\r\n",
      "Total written (filtered):     43,661,174 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 128853 times.\r\n",
      "128725 times, it overlapped the 5' end of a read\r\n",
      "128 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t31\t0.0\t0\t31\r\n",
      "20\t163\t0.0\t0\t163\r\n",
      "21\t128516\t0.0\t0\t128516\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t103\t3021.3\t0\t103\r\n",
      "4\t5\t755.3\t0\t5\r\n",
      "5\t2\t188.8\t0\t2\r\n",
      "6\t1\t47.2\t0\t1\r\n",
      "7\t2\t11.8\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 30361 times.\r\n",
      "130 times, it overlapped the 5' end of a read\r\n",
      "30231 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t127\t3021.3\t0\t127\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t3021.3\t0\t79\r\n",
      "4\t30151\t755.3\t0\t30151\r\n",
      "5\t1\t188.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2996.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2996.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.34 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 168,093\r\n",
      "Reads with adapters:                   163,166 (97.1%)\r\n",
      "Reads written (passing filters):       168,093 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    50,276,513 bp\r\n",
      "Total written (filtered):     46,850,772 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 162891 times.\r\n",
      "162867 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2626.5\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t214\t0.0\t0\t214\r\n",
      "21\t162603\t0.0\t0\t162603\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2626.5\t0\t8\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 275 times.\r\n",
      "73 times, it overlapped the 5' end of a read\r\n",
      "202 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t2626.5\t0\t71\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2626.5\t0\t5\r\n",
      "4\t197\t656.6\t0\t197\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2998.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2998.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.64 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 128,067\r\n",
      "Reads with adapters:                   115,829 (90.4%)\r\n",
      "Reads written (passing filters):       128,067 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,274,527 bp\r\n",
      "Total written (filtered):     32,908,929 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 111798 times.\r\n",
      "111759 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t20\t0.0\t0\t20\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t151\t0.0\t0\t151\r\n",
      "21\t111567\t0.0\t0\t111567\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t2001.0\t0\t29\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4031 times.\r\n",
      "66 times, it overlapped the 5' end of a read\r\n",
      "3965 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t2001.0\t0\t64\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t2001.0\t0\t24\r\n",
      "4\t3941\t500.3\t0\t3941\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2998.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2998.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.19 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 128,067\r\n",
      "Reads with adapters:                     6,194 (4.8%)\r\n",
      "Reads written (passing filters):       128,067 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,162,008 bp\r\n",
      "Total written (filtered):     35,076,655 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3257 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "3237 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t2001.0\t0\t18\r\n",
      "4\t2\t500.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3196\t2001.0\t0\t3196\r\n",
      "4\t15\t500.3\t0\t15\r\n",
      "5\t25\t125.1\t0\t25\r\n",
      "8\t1\t2.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2937 times.\r\n",
      "2730 times, it overlapped the 5' end of a read\r\n",
      "207 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2001.0\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2726\t0.0\t1\t0 2726\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t193\t2001.0\t0\t193\r\n",
      "4\t2\t500.3\t0\t2\r\n",
      "5\t8\t125.1\t0\t8\r\n",
      "300\t4\t0.0\t1\t0 4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2997.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2997.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.97 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 174,977\r\n",
      "Reads with adapters:                   169,287 (96.7%)\r\n",
      "Reads written (passing filters):       174,977 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    51,478,884 bp\r\n",
      "Total written (filtered):     47,943,988 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 168022 times.\r\n",
      "167999 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t683.5\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t41\t0.0\t0\t41\r\n",
      "20\t194\t0.0\t0\t194\r\n",
      "21\t167745\t0.0\t0\t167745\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t2734.0\t0\t16\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1265 times.\r\n",
      "34 times, it overlapped the 5' end of a read\r\n",
      "1231 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t2734.0\t0\t30\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2734.0\t0\t5\r\n",
      "4\t1226\t683.5\t0\t1226\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3000.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3000.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.72 s (37 us/read; 1.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,683\r\n",
      "Reads with adapters:                    69,039 (93.7%)\r\n",
      "Reads written (passing filters):        73,683 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,864,143 bp\r\n",
      "Total written (filtered):     16,630,666 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 56220 times.\r\n",
      "56184 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t56078\t0.0\t0\t56078\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1151.3\t0\t28\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12819 times.\r\n",
      "79 times, it overlapped the 5' end of a read\r\n",
      "12740 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t1151.3\t0\t75\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1151.3\t0\t34\r\n",
      "4\t12704\t287.8\t0\t12704\r\n",
      "5\t2\t72.0\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3000.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3000.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.36 s (46 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,683\r\n",
      "Reads with adapters:                     2,961 (4.0%)\r\n",
      "Reads written (passing filters):        73,683 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,705,420 bp\r\n",
      "Total written (filtered):     17,662,413 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1451 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1445 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1151.3\t0\t5\r\n",
      "4\t1\t287.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1440\t1151.3\t0\t1440\r\n",
      "4\t3\t287.8\t0\t3\r\n",
      "5\t2\t72.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1510 times.\r\n",
      "1421 times, it overlapped the 5' end of a read\r\n",
      "89 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1420\t0.0\t1\t0 1420\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1151.3\t0\t80\r\n",
      "4\t5\t287.8\t0\t5\r\n",
      "5\t4\t72.0\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2999.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2999.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.89 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 134,349\r\n",
      "Reads with adapters:                     5,726 (4.3%)\r\n",
      "Reads written (passing filters):       134,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,297,496 bp\r\n",
      "Total written (filtered):     32,213,670 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2791 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2783 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2099.2\t0\t8\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2768\t2099.2\t0\t2768\r\n",
      "4\t7\t524.8\t0\t7\r\n",
      "5\t8\t131.2\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2935 times.\r\n",
      "2764 times, it overlapped the 5' end of a read\r\n",
      "171 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t524.8\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2761\t0.0\t1\t0 2761\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t2099.2\t0\t160\r\n",
      "4\t4\t524.8\t0\t4\r\n",
      "5\t5\t131.2\t0\t5\r\n",
      "6\t1\t32.8\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2999.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2999.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.44 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 134,349\r\n",
      "Reads with adapters:                   123,311 (91.8%)\r\n",
      "Reads written (passing filters):       134,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,750,384 bp\r\n",
      "Total written (filtered):     30,521,973 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 101981 times.\r\n",
      "101899 times, it overlapped the 5' end of a read\r\n",
      "82 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t15\t0.0\t0\t15\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t146\t0.0\t0\t146\r\n",
      "21\t101718\t0.0\t0\t101718\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t68\t2099.2\t0\t68\r\n",
      "7\t1\t8.2\t0\t1\r\n",
      "48\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 21330 times.\r\n",
      "138 times, it overlapped the 5' end of a read\r\n",
      "21192 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t137\t2099.2\t0\t137\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t2099.2\t0\t64\r\n",
      "4\t21127\t524.8\t0\t21127\r\n",
      "5\t1\t131.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3002.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3002.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.49 s (34 us/read; 1.76 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,525\r\n",
      "Reads with adapters:                     3,821 (3.7%)\r\n",
      "Reads written (passing filters):       102,525 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,230,717 bp\r\n",
      "Total written (filtered):     23,175,519 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1851 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1843 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1602.0\t0\t6\r\n",
      "4\t2\t400.5\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1833\t1602.0\t0\t1833\r\n",
      "4\t7\t400.5\t0\t7\r\n",
      "5\t3\t100.1\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1970 times.\r\n",
      "1824 times, it overlapped the 5' end of a read\r\n",
      "146 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1602.0\t0\t1\r\n",
      "4\t1\t400.5\t0\t1\r\n",
      "5\t1\t100.1\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t1816\t0.0\t1\t0 1816\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t136\t1602.0\t0\t136\r\n",
      "4\t2\t400.5\t0\t2\r\n",
      "5\t8\t100.1\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3003.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3003.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.38 s (40 us/read; 1.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  59,108\r\n",
      "Reads with adapters:                     2,344 (4.0%)\r\n",
      "Reads written (passing filters):        59,108 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,354,333 bp\r\n",
      "Total written (filtered):     12,315,356 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 914 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "901 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t923.6\t0\t10\r\n",
      "4\t2\t230.9\t0\t2\r\n",
      "6\t1\t14.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t893\t923.6\t0\t893\r\n",
      "4\t2\t230.9\t0\t2\r\n",
      "5\t6\t57.7\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1430 times.\r\n",
      "1330 times, it overlapped the 5' end of a read\r\n",
      "100 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1329\t0.0\t1\t0 1329\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t96\t923.6\t0\t96\r\n",
      "4\t1\t230.9\t0\t1\r\n",
      "5\t3\t57.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3003.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3003.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.22 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  59,108\r\n",
      "Reads with adapters:                    52,103 (88.1%)\r\n",
      "Reads written (passing filters):        59,108 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,756,529 bp\r\n",
      "Total written (filtered):     11,882,998 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 39144 times.\r\n",
      "39066 times, it overlapped the 5' end of a read\r\n",
      "78 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t923.6\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t70\t0.0\t0\t70\r\n",
      "21\t38979\t0.0\t0\t38979\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t67\t923.6\t0\t67\r\n",
      "4\t1\t230.9\t0\t1\r\n",
      "5\t2\t57.7\t0\t2\r\n",
      "6\t1\t14.4\t0\t1\r\n",
      "7\t1\t3.6\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12959 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "12892 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t923.6\t0\t63\r\n",
      "4\t2\t230.9\t0\t2\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t923.6\t0\t60\r\n",
      "4\t12829\t230.9\t0\t12829\r\n",
      "5\t3\t57.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3002.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3002.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.15 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,525\r\n",
      "Reads with adapters:                    96,355 (94.0%)\r\n",
      "Reads written (passing filters):       102,525 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,486,791 bp\r\n",
      "Total written (filtered):     21,832,674 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74509 times.\r\n",
      "74435 times, it overlapped the 5' end of a read\r\n",
      "74 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1602.0\t0\t1\r\n",
      "4\t1\t400.5\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t122\t0.0\t0\t122\r\n",
      "21\t74292\t0.0\t0\t74292\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1602.0\t0\t60\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "58\t1\t0.0\t0\t1\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 21846 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "21792 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1602.0\t0\t50\r\n",
      "4\t1\t400.5\t0\t1\r\n",
      "5\t1\t100.1\t0\t1\r\n",
      "8\t1\t1.6\t0\t1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1602.0\t0\t62\r\n",
      "4\t21729\t400.5\t0\t21729\r\n",
      "5\t1\t100.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3005.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3005.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.99 s (42 us/read; 1.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,033\r\n",
      "Reads with adapters:                     4,118 (4.4%)\r\n",
      "Reads written (passing filters):        94,033 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,592,532 bp\r\n",
      "Total written (filtered):     27,526,125 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1731 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1730 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1469.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1718\t1469.3\t0\t1718\r\n",
      "4\t6\t367.3\t0\t6\r\n",
      "5\t6\t91.8\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2387 times.\r\n",
      "2255 times, it overlapped the 5' end of a read\r\n",
      "132 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1469.3\t0\t1\r\n",
      "4\t3\t367.3\t0\t3\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t2246\t0.0\t1\t0 2246\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t122\t1469.3\t0\t122\r\n",
      "4\t5\t367.3\t0\t5\r\n",
      "5\t5\t91.8\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3129.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3129.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.04 s (28 us/read; 2.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   1,556\r\n",
      "Reads with adapters:                     1,392 (89.5%)\r\n",
      "Reads written (passing filters):         1,556 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       453,734 bp\r\n",
      "Total written (filtered):        424,993 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1362 times.\r\n",
      "1362 times, it overlapped the 5' end of a read\r\n",
      "0 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t1362\t0.0\t0\t1362\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 30 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t24.3\t0\t3\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t26\t6.1\t0\t26\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3005.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3005.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.81 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,033\r\n",
      "Reads with adapters:                    90,894 (96.7%)\r\n",
      "Reads written (passing filters):        94,033 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,632,460 bp\r\n",
      "Total written (filtered):     25,745,526 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 89484 times.\r\n",
      "89472 times, it overlapped the 5' end of a read\r\n",
      "12 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t109\t0.0\t0\t109\r\n",
      "21\t89327\t0.0\t0\t89327\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1469.3\t0\t3\r\n",
      "111\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1410 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "1390 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1469.3\t0\t19\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1469.3\t0\t3\r\n",
      "4\t1386\t367.3\t0\t1386\r\n",
      "67\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3129.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3129.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.04 s (23 us/read; 2.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   1,556\r\n",
      "Reads with adapters:                       108 (6.9%)\r\n",
      "Reads written (passing filters):         1,556 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       447,034 bp\r\n",
      "Total written (filtered):        445,936 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "71 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t24.3\t0\t1\r\n",
      "4\t2\t6.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t69\t24.3\t0\t69\r\n",
      "4\t1\t6.1\t0\t1\r\n",
      "5\t1\t1.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 34 times.\r\n",
      "33 times, it overlapped the 5' end of a read\r\n",
      "1 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t6.1\t0\t1\r\n",
      "27\t32\t0.0\t1\t0 32\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t24.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3004.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3004.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.17 s (40 us/read; 1.51 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 129,622\r\n",
      "Reads with adapters:                     6,141 (4.7%)\r\n",
      "Reads written (passing filters):       129,622 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,712,923 bp\r\n",
      "Total written (filtered):     35,631,612 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3387 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "3385 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2025.3\t0\t1\r\n",
      "4\t1\t506.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3376\t2025.3\t0\t3376\r\n",
      "4\t7\t506.3\t0\t7\r\n",
      "5\t2\t126.6\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2754 times.\r\n",
      "2622 times, it overlapped the 5' end of a read\r\n",
      "132 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2025.3\t0\t2\r\n",
      "4\t1\t506.3\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2616\t0.0\t1\t0 2616\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t2025.3\t0\t115\r\n",
      "4\t10\t506.3\t0\t10\r\n",
      "5\t7\t126.6\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3004.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3004.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.32 s (56 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 129,622\r\n",
      "Reads with adapters:                   122,339 (94.4%)\r\n",
      "Reads written (passing filters):       129,622 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,808,313 bp\r\n",
      "Total written (filtered):     33,374,967 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 114280 times.\r\n",
      "114242 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2025.3\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t138\t0.0\t0\t138\r\n",
      "21\t114068\t0.0\t0\t114068\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t2025.3\t0\t28\r\n",
      "4\t2\t506.3\t0\t2\r\n",
      "121\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8059 times.\r\n",
      "48 times, it overlapped the 5' end of a read\r\n",
      "8011 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t2025.3\t0\t46\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t2025.3\t0\t22\r\n",
      "4\t7989\t506.3\t0\t7989\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3128.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3128.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.42 s (28 us/read; 2.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,773\r\n",
      "Reads with adapters:                     5,551 (4.6%)\r\n",
      "Reads written (passing filters):       121,773 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,896,797 bp\r\n",
      "Total written (filtered):     33,815,984 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2754 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "2744 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1902.7\t0\t9\r\n",
      "4\t1\t475.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2726\t1902.7\t0\t2726\r\n",
      "4\t11\t475.7\t0\t11\r\n",
      "5\t7\t118.9\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2797 times.\r\n",
      "2649 times, it overlapped the 5' end of a read\r\n",
      "148 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1902.7\t0\t1\r\n",
      "4\t1\t475.7\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2644\t0.0\t1\t0 2644\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t138\t1902.7\t0\t138\r\n",
      "4\t2\t475.7\t0\t2\r\n",
      "5\t6\t118.9\t0\t6\r\n",
      "300\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3127.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3127.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.94 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 181,429\r\n",
      "Reads with adapters:                     9,115 (5.0%)\r\n",
      "Reads written (passing filters):       181,429 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,605,019 bp\r\n",
      "Total written (filtered):     52,468,543 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4320 times.\r\n",
      "24 times, it overlapped the 5' end of a read\r\n",
      "4296 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t2834.8\t0\t21\r\n",
      "4\t3\t708.7\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4251\t2834.8\t0\t4251\r\n",
      "4\t30\t708.7\t0\t30\r\n",
      "5\t15\t177.2\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4795 times.\r\n",
      "4533 times, it overlapped the 5' end of a read\r\n",
      "262 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2834.8\t0\t2\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t4524\t0.0\t1\t0 4524\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t241\t2834.8\t0\t241\r\n",
      "4\t4\t708.7\t0\t4\r\n",
      "5\t16\t177.2\t0\t16\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3128.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3128.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.26 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,773\r\n",
      "Reads with adapters:                   110,785 (91.0%)\r\n",
      "Reads written (passing filters):       121,773 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    34,048,975 bp\r\n",
      "Total written (filtered):     31,772,852 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 107683 times.\r\n",
      "107650 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "6\t1\t29.7\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t17\t0.0\t0\t17\r\n",
      "20\t143\t0.0\t0\t143\r\n",
      "21\t107476\t0.0\t0\t107476\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1902.7\t0\t20\r\n",
      "4\t1\t475.7\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3102 times.\r\n",
      "131 times, it overlapped the 5' end of a read\r\n",
      "2971 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t129\t1902.7\t0\t129\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1902.7\t0\t23\r\n",
      "4\t2948\t475.7\t0\t2948\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3130.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3130.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.72 s (31 us/read; 1.96 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,650\r\n",
      "Reads with adapters:                     6,258 (5.1%)\r\n",
      "Reads written (passing filters):       121,650 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,555,625 bp\r\n",
      "Total written (filtered):     35,469,735 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3290 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "3287 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1900.8\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3269\t1900.8\t0\t3269\r\n",
      "4\t12\t475.2\t0\t12\r\n",
      "5\t6\t118.8\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2968 times.\r\n",
      "2804 times, it overlapped the 5' end of a read\r\n",
      "164 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t9\t475.2\t0\t9\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "27\t2794\t0.0\t1\t0 2794\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t1900.8\t0\t160\r\n",
      "4\t3\t475.2\t0\t3\r\n",
      "5\t1\t118.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3131.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3131.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.33 s (29 us/read; 2.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 114,741\r\n",
      "Reads with adapters:                     5,340 (4.7%)\r\n",
      "Reads written (passing filters):       114,741 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,663,546 bp\r\n",
      "Total written (filtered):     32,585,991 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2648 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "2648 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2631\t1792.8\t0\t2631\r\n",
      "4\t10\t448.2\t0\t10\r\n",
      "5\t6\t112.1\t0\t6\r\n",
      "6\t1\t28.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2692 times.\r\n",
      "2570 times, it overlapped the 5' end of a read\r\n",
      "122 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t8\t448.2\t0\t8\r\n",
      "27\t2562\t0.0\t1\t0 2562\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t1792.8\t0\t112\r\n",
      "4\t6\t448.2\t0\t6\r\n",
      "5\t4\t112.1\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3134.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3134.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (27 us/read; 2.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     880\r\n",
      "Reads with adapters:                       791 (89.9%)\r\n",
      "Reads written (passing filters):           880 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       249,531 bp\r\n",
      "Total written (filtered):        233,483 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 760 times.\r\n",
      "758 times, it overlapped the 5' end of a read\r\n",
      "2 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t758\t0.0\t0\t758\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t13.8\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 31 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t31\t3.4\t0\t31\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3127.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3127.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.02 s (50 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 181,429\r\n",
      "Reads with adapters:                   172,799 (95.2%)\r\n",
      "Reads written (passing filters):       181,429 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,795,430 bp\r\n",
      "Total written (filtered):     49,200,148 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 170612 times.\r\n",
      "170569 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t29\t0.0\t0\t29\r\n",
      "19\t34\t0.0\t0\t34\r\n",
      "20\t230\t0.0\t0\t230\r\n",
      "21\t170272\t0.0\t0\t170272\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t2834.8\t0\t24\r\n",
      "7\t2\t11.1\t0\t2\r\n",
      "300\t17\t0.0\t0\t17\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2187 times.\r\n",
      "246 times, it overlapped the 5' end of a read\r\n",
      "1941 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t243\t2834.8\t0\t243\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t2834.8\t0\t22\r\n",
      "4\t1919\t708.7\t0\t1919\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3134.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3134.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (24 us/read; 2.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     880\r\n",
      "Reads with adapters:                        52 (5.9%)\r\n",
      "Reads written (passing filters):           880 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       245,948 bp\r\n",
      "Total written (filtered):        245,479 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 36 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t13.8\t0\t36\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "3 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t13\t0.0\t1\t0 13\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t13.8\t0\t2\r\n",
      "4\t1\t3.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3132.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3132.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.55 s (25 us/read; 2.38 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 141,146\r\n",
      "Reads with adapters:                     6,180 (4.4%)\r\n",
      "Reads written (passing filters):       141,146 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,355,103 bp\r\n",
      "Total written (filtered):     41,260,395 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2848 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2842 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2205.4\t0\t3\r\n",
      "4\t3\t551.4\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2794\t2205.4\t0\t2794\r\n",
      "4\t11\t551.4\t0\t11\r\n",
      "5\t36\t137.8\t0\t36\r\n",
      "8\t1\t2.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3332 times.\r\n",
      "3139 times, it overlapped the 5' end of a read\r\n",
      "193 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2205.4\t0\t1\r\n",
      "4\t5\t551.4\t0\t5\r\n",
      "7\t1\t8.6\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3129\t0.0\t1\t0 3129\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t183\t2205.4\t0\t183\r\n",
      "4\t2\t551.4\t0\t2\r\n",
      "5\t5\t137.8\t0\t5\r\n",
      "300\t3\t0.0\t1\t0 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3130.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3130.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.33 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,650\r\n",
      "Reads with adapters:                   105,797 (87.0%)\r\n",
      "Reads written (passing filters):       121,650 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,639,215 bp\r\n",
      "Total written (filtered):     33,440,278 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 104334 times.\r\n",
      "104301 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t123\t0.0\t0\t123\r\n",
      "21\t104156\t0.0\t0\t104156\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1900.8\t0\t21\r\n",
      "4\t2\t475.2\t0\t2\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1463 times.\r\n",
      "143 times, it overlapped the 5' end of a read\r\n",
      "1320 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t142\t1900.8\t0\t142\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1900.8\t0\t9\r\n",
      "4\t1310\t475.2\t0\t1310\r\n",
      "5\t1\t118.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3131.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3131.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.05 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 114,741\r\n",
      "Reads with adapters:                   105,317 (91.8%)\r\n",
      "Reads written (passing filters):       114,741 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,779,971 bp\r\n",
      "Total written (filtered):     30,604,340 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 103037 times.\r\n",
      "103006 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t29\t0.0\t0\t29\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t102833\t0.0\t0\t102833\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1792.8\t0\t17\r\n",
      "7\t1\t7.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "297\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2280 times.\r\n",
      "93 times, it overlapped the 5' end of a read\r\n",
      "2187 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t91\t1792.8\t0\t91\r\n",
      "4\t1\t448.2\t0\t1\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1792.8\t0\t12\r\n",
      "4\t2174\t448.2\t0\t2174\r\n",
      "5\t1\t112.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3136.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3136.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (23 us/read; 2.64 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     954\r\n",
      "Reads with adapters:                        47 (4.9%)\r\n",
      "Reads written (passing filters):           954 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       237,983 bp\r\n",
      "Total written (filtered):        237,410 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 29 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t14.9\t0\t29\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 18 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "0 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t18\t0.0\t1\t0 18\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3136.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3136.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.02 s (26 us/read; 2.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                     954\r\n",
      "Reads with adapters:                       854 (89.5%)\r\n",
      "Reads written (passing filters):           954 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       248,151 bp\r\n",
      "Total written (filtered):        230,865 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 816 times.\r\n",
      "816 times, it overlapped the 5' end of a read\r\n",
      "0 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t815\t0.0\t0\t815\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 38 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "37 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t14.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t37\t3.7\t0\t37\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3132.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3132.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.19 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 141,146\r\n",
      "Reads with adapters:                   135,393 (95.9%)\r\n",
      "Reads written (passing filters):       141,146 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,392,084 bp\r\n",
      "Total written (filtered):     38,566,330 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 134135 times.\r\n",
      "134103 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t21\t0.0\t0\t21\r\n",
      "19\t30\t0.0\t0\t30\r\n",
      "20\t180\t0.0\t0\t180\r\n",
      "21\t133869\t0.0\t0\t133869\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t2205.4\t0\t11\r\n",
      "4\t5\t551.4\t0\t5\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1258 times.\r\n",
      "61 times, it overlapped the 5' end of a read\r\n",
      "1197 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t2205.4\t0\t57\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2205.4\t0\t7\r\n",
      "4\t1190\t551.4\t0\t1190\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3133.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3133.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.06 s (32 us/read; 1.89 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,522\r\n",
      "Reads with adapters:                     7,264 (4.6%)\r\n",
      "Reads written (passing filters):       159,522 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,933,669 bp\r\n",
      "Total written (filtered):     42,826,244 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3340 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "3334 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2492.5\t0\t4\r\n",
      "4\t2\t623.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3300\t2492.5\t0\t3300\r\n",
      "4\t18\t623.1\t0\t18\r\n",
      "5\t16\t155.8\t0\t16\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3924 times.\r\n",
      "3565 times, it overlapped the 5' end of a read\r\n",
      "359 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t13\t623.1\t0\t13\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3547\t0.0\t1\t0 3547\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t340\t2492.5\t0\t340\r\n",
      "4\t6\t623.1\t0\t6\r\n",
      "5\t10\t155.8\t0\t10\r\n",
      "6\t2\t38.9\t0\t2\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3137.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3137.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.68 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  89,002\r\n",
      "Reads with adapters:                    76,653 (86.1%)\r\n",
      "Reads written (passing filters):        89,002 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,367,402 bp\r\n",
      "Total written (filtered):     24,765,563 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 76105 times.\r\n",
      "76082 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1390.7\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t102\t0.0\t0\t102\r\n",
      "21\t75953\t0.0\t0\t75953\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1390.7\t0\t13\r\n",
      "4\t3\t347.7\t0\t3\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 548 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "486 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1390.7\t0\t60\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1390.7\t0\t8\r\n",
      "4\t478\t347.7\t0\t478\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3137.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3137.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.06 s (46 us/read; 1.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  89,002\r\n",
      "Reads with adapters:                     4,886 (5.5%)\r\n",
      "Reads written (passing filters):        89,002 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,324,402 bp\r\n",
      "Total written (filtered):     26,258,215 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2625 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2622 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1390.7\t0\t2\r\n",
      "4\t1\t347.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2614\t1390.7\t0\t2614\r\n",
      "4\t5\t347.7\t0\t5\r\n",
      "5\t3\t86.9\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2261 times.\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2152 times, it overlapped the 5' end of a read\r\n",
      "109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t6\t347.7\t0\t6\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "27\t2145\t0.0\t1\t0 2145\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t1390.7\t0\t102\r\n",
      "4\t4\t347.7\t0\t4\r\n",
      "5\t2\t86.9\t0\t2\r\n",
      "6\t1\t21.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3135.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3135.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.95 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,426\r\n",
      "Reads with adapters:                     7,747 (5.3%)\r\n",
      "Reads written (passing filters):       145,426 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,687,990 bp\r\n",
      "Total written (filtered):     42,574,326 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3820 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "3816 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2272.3\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3805\t2272.3\t0\t3805\r\n",
      "4\t7\t568.1\t0\t7\r\n",
      "5\t4\t142.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3927 times.\r\n",
      "3774 times, it overlapped the 5' end of a read\r\n",
      "153 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t7\t568.1\t0\t7\r\n",
      "5\t1\t142.0\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3761\t0.0\t1\t0 3761\r\n",
      "28\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t142\t2272.3\t0\t142\r\n",
      "4\t4\t568.1\t0\t4\r\n",
      "5\t7\t142.0\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3133.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3133.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.22 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,522\r\n",
      "Reads with adapters:                   149,150 (93.5%)\r\n",
      "Reads written (passing filters):       159,522 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,147,504 bp\r\n",
      "Total written (filtered):     40,115,488 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 143028 times.\r\n",
      "142977 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t193\t0.0\t0\t193\r\n",
      "21\t142749\t0.0\t0\t142749\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t2492.5\t0\t29\r\n",
      "4\t3\t623.1\t0\t3\r\n",
      "5\t1\t155.8\t0\t1\r\n",
      "7\t1\t9.7\t0\t1\r\n",
      "299\t5\t0.0\t0\t5\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6122 times.\r\n",
      "44 times, it overlapped the 5' end of a read\r\n",
      "6078 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t2492.5\t0\t37\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t2492.5\t0\t28\r\n",
      "4\t6050\t623.1\t0\t6050\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3135.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3135.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.43 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 145,426\r\n",
      "Reads with adapters:                   139,480 (95.9%)\r\n",
      "Reads written (passing filters):       145,426 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,793,207 bp\r\n",
      "Total written (filtered):     39,873,191 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 138742 times.\r\n",
      "138714 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t568.1\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t25\t0.0\t0\t25\r\n",
      "20\t184\t0.0\t0\t184\r\n",
      "21\t138484\t0.0\t0\t138484\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t2272.3\t0\t13\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 738 times.\r\n",
      "165 times, it overlapped the 5' end of a read\r\n",
      "573 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t164\t2272.3\t0\t164\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2272.3\t0\t5\r\n",
      "4\t568\t568.1\t0\t568\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3138.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3138.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.71 s (41 us/read; 1.48 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 165,338\r\n",
      "Reads with adapters:                     7,870 (4.8%)\r\n",
      "Reads written (passing filters):       165,338 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    47,175,784 bp\r\n",
      "Total written (filtered):     47,066,858 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4070 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "4056 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t2583.4\t0\t13\r\n",
      "4\t1\t645.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4025\t2583.4\t0\t4025\r\n",
      "4\t18\t645.9\t0\t18\r\n",
      "5\t13\t161.5\t0\t13\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3800 times.\r\n",
      "3541 times, it overlapped the 5' end of a read\r\n",
      "259 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2583.4\t0\t1\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3538\t0.0\t1\t0 3538\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t247\t2583.4\t0\t247\r\n",
      "4\t4\t645.9\t0\t4\r\n",
      "5\t7\t161.5\t0\t7\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3138.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3138.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.53 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 165,338\r\n",
      "Reads with adapters:                   160,062 (96.8%)\r\n",
      "Reads written (passing filters):       165,338 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    47,339,918 bp\r\n",
      "Total written (filtered):     44,030,943 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 156899 times.\r\n",
      "156867 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2583.4\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t20\t0.0\t0\t20\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t215\t0.0\t0\t215\r\n",
      "21\t156609\t0.0\t0\t156609\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t2583.4\t0\t20\r\n",
      "4\t1\t645.9\t0\t1\r\n",
      "7\t1\t10.1\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3163 times.\r\n",
      "151 times, it overlapped the 5' end of a read\r\n",
      "3012 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t149\t2583.4\t0\t149\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2583.4\t0\t7\r\n",
      "4\t3005\t645.9\t0\t3005\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.53 s (41 us/read; 1.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                    81,198 (94.5%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,568,415 bp\r\n",
      "Total written (filtered):     23,862,680 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80995 times.\r\n",
      "80966 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t136\t0.0\t0\t136\r\n",
      "21\t80808\t0.0\t0\t80808\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1342.8\t0\t13\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 203 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "174 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1342.8\t0\t28\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1342.8\t0\t6\r\n",
      "4\t168\t335.7\t0\t168\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.78 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                     5,105 (5.9%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,518,505 bp\r\n",
      "Total written (filtered):     25,444,522 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2561 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2553 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1342.8\t0\t7\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2543\t1342.8\t0\t2543\r\n",
      "4\t9\t335.7\t0\t9\r\n",
      "5\t1\t83.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2544 times.\r\n",
      "2445 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1342.8\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2439\t0.0\t1\t0 2439\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t96\t1342.8\t0\t96\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "5\t2\t83.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3140.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3140.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.28 s (26 us/read; 2.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 162,528\r\n",
      "Reads with adapters:                     7,427 (4.6%)\r\n",
      "Reads written (passing filters):       162,528 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,530,912 bp\r\n",
      "Total written (filtered):     42,424,857 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3681 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "3670 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2539.5\t0\t10\r\n",
      "4\t1\t634.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3640\t2539.5\t0\t3640\r\n",
      "4\t15\t634.9\t0\t15\r\n",
      "5\t14\t158.7\t0\t14\r\n",
      "6\t1\t39.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3746 times.\r\n",
      "3489 times, it overlapped the 5' end of a read\r\n",
      "257 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2539.5\t0\t3\r\n",
      "4\t1\t634.9\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3482\t0.0\t1\t0 3482\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t237\t2539.5\t0\t237\r\n",
      "4\t7\t634.9\t0\t7\r\n",
      "5\t12\t158.7\t0\t12\r\n",
      "62\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3139.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3139.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.60 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 164,170\r\n",
      "Reads with adapters:                     8,090 (4.9%)\r\n",
      "Reads written (passing filters):       164,170 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    48,235,363 bp\r\n",
      "Total written (filtered):     48,116,719 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4018 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "3998 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t2565.2\t0\t16\r\n",
      "4\t4\t641.3\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3987\t2565.2\t0\t3987\r\n",
      "4\t8\t641.3\t0\t8\r\n",
      "5\t3\t160.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4072 times.\r\n",
      "3910 times, it overlapped the 5' end of a read\r\n",
      "162 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2565.2\t0\t4\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3900\t0.0\t1\t0 3900\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t144\t2565.2\t0\t144\r\n",
      "4\t7\t641.3\t0\t7\r\n",
      "5\t8\t160.3\t0\t8\r\n",
      "6\t1\t40.1\t0\t1\r\n",
      "300\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3140.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3140.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.98 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 162,528\r\n",
      "Reads with adapters:                   153,561 (94.5%)\r\n",
      "Reads written (passing filters):       162,528 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,762,919 bp\r\n",
      "Total written (filtered):     39,655,276 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 146559 times.\r\n",
      "146518 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "12\t3\t0.0\t0\t3\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t25\t0.0\t0\t25\r\n",
      "20\t212\t0.0\t0\t212\r\n",
      "21\t146259\t0.0\t0\t146259\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t2539.5\t0\t29\r\n",
      "7\t1\t9.9\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7002 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "6940 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t2539.5\t0\t61\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t2539.5\t0\t17\r\n",
      "4\t6918\t634.9\t0\t6918\r\n",
      "5\t2\t158.7\t0\t2\r\n",
      "6\t3\t39.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3142.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3142.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.46 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 140,662\r\n",
      "Reads with adapters:                     6,378 (4.5%)\r\n",
      "Reads written (passing filters):       140,662 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,071,911 bp\r\n",
      "Total written (filtered):     41,971,926 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2835 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "2816 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t2197.8\t0\t14\r\n",
      "4\t5\t549.5\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2800\t2197.8\t0\t2800\r\n",
      "4\t9\t549.5\t0\t9\r\n",
      "5\t7\t137.4\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3543 times.\r\n",
      "3371 times, it overlapped the 5' end of a read\r\n",
      "172 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2197.8\t0\t4\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3363\t0.0\t1\t0 3363\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t2197.8\t0\t160\r\n",
      "4\t4\t549.5\t0\t4\r\n",
      "5\t8\t137.4\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3139.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3139.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.13 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 164,170\r\n",
      "Reads with adapters:                   158,781 (96.7%)\r\n",
      "Reads written (passing filters):       164,170 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    48,333,237 bp\r\n",
      "Total written (filtered):     45,009,932 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 157929 times.\r\n",
      "157903 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t25\t0.0\t0\t25\r\n",
      "19\t20\t0.0\t0\t20\r\n",
      "20\t148\t0.0\t0\t148\r\n",
      "21\t157706\t0.0\t0\t157706\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2565.2\t0\t10\r\n",
      "5\t1\t160.3\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t13\t0.0\t0\t13\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 852 times.\r\n",
      "133 times, it overlapped the 5' end of a read\r\n",
      "719 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t130\t2565.2\t0\t130\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2565.2\t0\t5\r\n",
      "4\t714\t641.3\t0\t714\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3142.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3142.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.10 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 140,662\r\n",
      "Reads with adapters:                   135,207 (96.1%)\r\n",
      "Reads written (passing filters):       140,662 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,111,879 bp\r\n",
      "Total written (filtered):     39,274,642 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 134902 times.\r\n",
      "134874 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t26\t0.0\t0\t26\r\n",
      "19\t29\t0.0\t0\t29\r\n",
      "20\t149\t0.0\t0\t149\r\n",
      "21\t134667\t0.0\t0\t134667\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t2197.8\t0\t15\r\n",
      "300\t13\t0.0\t0\t13\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 305 times.\r\n",
      "75 times, it overlapped the 5' end of a read\r\n",
      "230 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t2197.8\t0\t71\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t2197.8\t0\t5\r\n",
      "4\t225\t549.5\t0\t225\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3143.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3143.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.40 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,315\r\n",
      "Reads with adapters:                   115,121 (96.5%)\r\n",
      "Reads written (passing filters):       119,315 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,648,503 bp\r\n",
      "Total written (filtered):     33,236,209 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 114749 times.\r\n",
      "114730 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1864.3\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t33\t0.0\t0\t33\r\n",
      "20\t146\t0.0\t0\t146\r\n",
      "21\t114538\t0.0\t0\t114538\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1864.3\t0\t12\r\n",
      "6\t1\t29.1\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 372 times.\r\n",
      "164 times, it overlapped the 5' end of a read\r\n",
      "208 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1864.3\t0\t161\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1864.3\t0\t3\r\n",
      "4\t205\t466.1\t0\t205\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3143.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3143.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.59 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,315\r\n",
      "Reads with adapters:                     5,899 (4.9%)\r\n",
      "Reads written (passing filters):       119,315 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,602,624 bp\r\n",
      "Total written (filtered):     35,519,594 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3049 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "3039 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1864.3\t0\t7\r\n",
      "4\t3\t466.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3014\t1864.3\t0\t3014\r\n",
      "4\t17\t466.1\t0\t17\r\n",
      "5\t8\t116.5\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2850 times.\r\n",
      "2721 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t466.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2718\t0.0\t1\t0 2718\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t1864.3\t0\t120\r\n",
      "4\t1\t466.1\t0\t1\r\n",
      "5\t7\t116.5\t0\t7\r\n",
      "6\t1\t29.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3145.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3145.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.71 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 123,448\r\n",
      "Reads with adapters:                   119,258 (96.6%)\r\n",
      "Reads written (passing filters):       123,448 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,604,729 bp\r\n",
      "Total written (filtered):     34,106,515 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 118703 times.\r\n",
      "118686 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t144\t0.0\t0\t144\r\n",
      "21\t118519\t0.0\t0\t118519\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1928.9\t0\t3\r\n",
      "5\t1\t120.6\t0\t1\r\n",
      "300\t13\t0.0\t0\t13\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 555 times.\r\n",
      "148 times, it overlapped the 5' end of a read\r\n",
      "407 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t147\t1928.9\t0\t147\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1928.9\t0\t5\r\n",
      "4\t402\t482.2\t0\t402\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3145.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3145.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.51 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 123,448\r\n",
      "Reads with adapters:                     6,904 (5.6%)\r\n",
      "Reads written (passing filters):       123,448 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,548,213 bp\r\n",
      "Total written (filtered):     36,459,544 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3929 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "3920 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1928.9\t0\t7\r\n",
      "4\t2\t482.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3916\t1928.9\t0\t3916\r\n",
      "4\t3\t482.2\t0\t3\r\n",
      "5\t1\t120.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2975 times.\r\n",
      "2835 times, it overlapped the 5' end of a read\r\n",
      "140 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1928.9\t0\t4\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "27\t2828\t0.0\t1\t0 2828\r\n",
      "28\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t1928.9\t0\t135\r\n",
      "4\t4\t482.2\t0\t4\r\n",
      "5\t1\t120.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3148.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3148.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.71 s (34 us/read; 1.74 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,687\r\n",
      "Reads with adapters:                     4,145 (3.8%)\r\n",
      "Reads written (passing filters):       107,687 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,564,266 bp\r\n",
      "Total written (filtered):     25,506,303 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2075 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2067 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1682.6\t0\t7\r\n",
      "4\t1\t420.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2042\t1682.6\t0\t2042\r\n",
      "4\t15\t420.7\t0\t15\r\n",
      "5\t10\t105.2\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2070 times.\r\n",
      "1897 times, it overlapped the 5' end of a read\r\n",
      "173 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t420.7\t0\t2\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1891\t0.0\t1\t0 1891\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t163\t1682.6\t0\t163\r\n",
      "4\t2\t420.7\t0\t2\r\n",
      "5\t8\t105.2\t0\t8\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3148.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3148.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.09 s (57 us/read; 1.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,687\r\n",
      "Reads with adapters:                    92,944 (86.3%)\r\n",
      "Reads written (passing filters):       107,687 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,971,216 bp\r\n",
      "Total written (filtered):     24,295,968 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 76751 times.\r\n",
      "76679 times, it overlapped the 5' end of a read\r\n",
      "72 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1682.6\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t105\t0.0\t0\t105\r\n",
      "21\t76549\t0.0\t0\t76549\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t65\t1682.6\t0\t65\r\n",
      "5\t2\t105.2\t0\t2\r\n",
      "7\t3\t6.6\t0\t3\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16193 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "16085 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1682.6\t0\t108\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1682.6\t0\t62\r\n",
      "4\t16023\t420.7\t0\t16023\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3146.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3146.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 9.40 s (40 us/read; 1.52 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 237,589\r\n",
      "Reads with adapters:                    10,075 (4.2%)\r\n",
      "Reads written (passing filters):       237,589 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    64,592,138 bp\r\n",
      "Total written (filtered):     64,454,396 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5274 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "5265 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t3712.3\t0\t9\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5215\t3712.3\t0\t5215\r\n",
      "4\t26\t928.1\t0\t26\r\n",
      "5\t24\t232.0\t0\t24\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4801 times.\r\n",
      "4479 times, it overlapped the 5' end of a read\r\n",
      "322 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t4\t928.1\t0\t4\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t4472\t0.0\t1\t0 4472\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t297\t3712.3\t0\t297\r\n",
      "4\t8\t928.1\t0\t8\r\n",
      "5\t17\t232.0\t0\t17\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3147.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3147.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.45 s (26 us/read; 2.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 251,269\r\n",
      "Reads with adapters:                     9,928 (4.0%)\r\n",
      "Reads written (passing filters):       251,269 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    70,830,724 bp\r\n",
      "Total written (filtered):     70,706,249 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5503 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "5497 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t3926.1\t0\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5456\t3926.1\t0\t5456\r\n",
      "4\t23\t981.5\t0\t23\r\n",
      "5\t17\t245.4\t0\t17\r\n",
      "6\t1\t61.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4425 times.\r\n",
      "3949 times, it overlapped the 5' end of a read\r\n",
      "476 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t3926.1\t0\t2\r\n",
      "4\t5\t981.5\t0\t5\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3938\t0.0\t1\t0 3938\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t460\t3926.1\t0\t460\r\n",
      "4\t7\t981.5\t0\t7\r\n",
      "5\t9\t245.4\t0\t9\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3146.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3146.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 10.25 s (43 us/read; 1.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 237,589\r\n",
      "Reads with adapters:                   217,638 (91.6%)\r\n",
      "Reads written (passing filters):       237,589 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    64,887,697 bp\r\n",
      "Total written (filtered):     60,516,161 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 205813 times.\r\n",
      "205726 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t28\t0.0\t0\t28\r\n",
      "19\t38\t0.0\t0\t38\r\n",
      "20\t280\t0.0\t0\t280\r\n",
      "21\t205377\t0.0\t0\t205377\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t3712.3\t0\t61\r\n",
      "4\t9\t928.1\t0\t9\r\n",
      "5\t1\t232.0\t0\t1\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "74\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11825 times.\r\n",
      "217 times, it overlapped the 5' end of a read\r\n",
      "11608 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t3712.3\t0\t211\r\n",
      "27\t6\t0.0\t1\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t3712.3\t0\t34\r\n",
      "4\t11573\t928.1\t0\t11573\r\n",
      "5\t1\t232.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/458.R2.fastq.gz.trimmed.fastq.gz /code/sequences/458.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.86 s (37 us/read; 1.61 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,942\r\n",
      "Reads with adapters:                     2,338 (2.2%)\r\n",
      "Reads written (passing filters):       103,942 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,437,935 bp\r\n",
      "Total written (filtered):     14,401,542 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 947 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "937 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1624.1\t0\t8\r\n",
      "4\t2\t406.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t927\t1624.1\t0\t927\r\n",
      "4\t6\t406.0\t0\t6\r\n",
      "5\t4\t101.5\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1391 times.\r\n",
      "1225 times, it overlapped the 5' end of a read\r\n",
      "166 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1624.1\t0\t1\r\n",
      "4\t1\t406.0\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1220\t0.0\t1\t0 1220\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t157\t1624.1\t0\t157\r\n",
      "4\t3\t406.0\t0\t3\r\n",
      "5\t5\t101.5\t0\t5\r\n",
      "6\t1\t25.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3150.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3150.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.57 s (27 us/read; 2.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 320,550\r\n",
      "Reads with adapters:                    15,695 (4.9%)\r\n",
      "Reads written (passing filters):       320,550 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    91,541,030 bp\r\n",
      "Total written (filtered):     91,322,187 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 8036 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "8016 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t5008.6\t0\t16\r\n",
      "4\t4\t1252.1\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7943\t5008.6\t0\t7943\r\n",
      "4\t44\t1252.1\t0\t44\r\n",
      "5\t29\t313.0\t0\t29\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7659 times.\r\n",
      "7133 times, it overlapped the 5' end of a read\r\n",
      "526 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t5008.6\t0\t1\r\n",
      "4\t3\t1252.1\t0\t3\r\n",
      "23\t4\t0.0\t1\t0 4\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t7120\t0.0\t1\t0 7120\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t496\t5008.6\t0\t496\r\n",
      "4\t9\t1252.1\t0\t9\r\n",
      "5\t19\t313.0\t0\t19\r\n",
      "267\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/458.R1.fastq.gz.trimmed.fastq.gz /code/sequences/458.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.26 s (51 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,942\r\n",
      "Reads with adapters:                    94,668 (91.1%)\r\n",
      "Reads written (passing filters):       103,942 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,047,583 bp\r\n",
      "Total written (filtered):     13,881,139 bp (92.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 46412 times.\r\n",
      "46287 times, it overlapped the 5' end of a read\r\n",
      "125 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1624.1\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t64\t0.0\t0\t64\r\n",
      "21\t46206\t0.0\t0\t46206\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t118\t1624.1\t0\t118\r\n",
      "5\t3\t101.5\t0\t3\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 48256 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "48209 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1624.1\t0\t42\r\n",
      "4\t1\t406.0\t0\t1\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t109\t1624.1\t0\t109\r\n",
      "4\t48093\t406.0\t0\t48093\r\n",
      "5\t7\t101.5\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3147.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3147.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 13.61 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 251,269\r\n",
      "Reads with adapters:                   234,954 (93.5%)\r\n",
      "Reads written (passing filters):       251,269 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    71,044,019 bp\r\n",
      "Total written (filtered):     66,219,807 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 228360 times.\r\n",
      "228303 times, it overlapped the 5' end of a read\r\n",
      "57 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t3926.1\t0\t1\r\n",
      "4\t1\t981.5\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t22\t0.0\t0\t22\r\n",
      "19\t45\t0.0\t0\t45\r\n",
      "20\t310\t0.0\t0\t310\r\n",
      "21\t227919\t0.0\t0\t227919\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t3926.1\t0\t37\r\n",
      "4\t2\t981.5\t0\t2\r\n",
      "7\t1\t15.3\t0\t1\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "63\t1\t0.0\t0\t1\r\n",
      "219\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6594 times.\r\n",
      "126 times, it overlapped the 5' end of a read\r\n",
      "6468 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t126\t3926.1\t0\t126\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t3926.1\t0\t28\r\n",
      "4\t6440\t981.5\t0\t6440\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/472.R1.fastq.gz.trimmed.fastq.gz /code/sequences/472.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.32 s (37 us/read; 1.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 117,427\r\n",
      "Reads with adapters:                   107,955 (91.9%)\r\n",
      "Reads written (passing filters):       117,427 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,787,615 bp\r\n",
      "Total written (filtered):     15,467,750 bp (92.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 52392 times.\r\n",
      "52240 times, it overlapped the 5' end of a read\r\n",
      "152 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1834.8\t0\t1\r\n",
      "4\t1\t458.7\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t75\t0.0\t0\t75\r\n",
      "21\t52146\t0.0\t0\t52146\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t150\t1834.8\t0\t150\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 55563 times.\r\n",
      "41 times, it overlapped the 5' end of a read\r\n",
      "55522 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1834.8\t0\t37\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t133\t1834.8\t0\t133\r\n",
      "4\t55386\t458.7\t0\t55386\r\n",
      "5\t3\t114.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3151.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3151.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.13 s (36 us/read; 1.64 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 168,141\r\n",
      "Reads with adapters:                     7,931 (4.7%)\r\n",
      "Reads written (passing filters):       168,141 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    46,320,710 bp\r\n",
      "Total written (filtered):     46,208,577 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4052 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "4042 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2627.2\t0\t7\r\n",
      "4\t3\t656.8\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4027\t2627.2\t0\t4027\r\n",
      "4\t8\t656.8\t0\t8\r\n",
      "5\t7\t164.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3879 times.\r\n",
      "3685 times, it overlapped the 5' end of a read\r\n",
      "194 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2627.2\t0\t2\r\n",
      "4\t4\t656.8\t0\t4\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t3675\t0.0\t1\t0 3675\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t184\t2627.2\t0\t184\r\n",
      "4\t2\t656.8\t0\t2\r\n",
      "5\t7\t164.2\t0\t7\r\n",
      "6\t1\t41.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3151.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3151.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.38 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 168,141\r\n",
      "Reads with adapters:                   107,676 (64.0%)\r\n",
      "Reads written (passing filters):       168,141 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    46,538,082 bp\r\n",
      "Total written (filtered):     44,362,370 bp (95.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102499 times.\r\n",
      "102398 times, it overlapped the 5' end of a read\r\n",
      "101 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t21\t0.0\t0\t21\r\n",
      "20\t137\t0.0\t0\t137\r\n",
      "21\t102229\t0.0\t0\t102229\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t2627.2\t0\t79\r\n",
      "5\t3\t164.2\t0\t3\r\n",
      "7\t2\t10.3\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "105\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5177 times.\r\n",
      "34 times, it overlapped the 5' end of a read\r\n",
      "5143 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t2627.2\t0\t33\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t2627.2\t0\t39\r\n",
      "4\t5103\t656.8\t0\t5103\r\n",
      "5\t1\t164.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/481.R2.fastq.gz.trimmed.fastq.gz /code/sequences/481.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.24 s (40 us/read; 1.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  55,949\r\n",
      "Reads with adapters:                     2,254 (4.0%)\r\n",
      "Reads written (passing filters):        55,949 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,004,590 bp\r\n",
      "Total written (filtered):     11,969,442 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 971 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "968 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t874.2\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t960\t874.2\t0\t960\r\n",
      "4\t1\t218.6\t0\t1\r\n",
      "5\t5\t54.6\t0\t5\r\n",
      "6\t1\t13.7\t0\t1\r\n",
      "9\t1\t0.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1283 times.\r\n",
      "1184 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t874.2\t0\t3\r\n",
      "27\t1181\t0.0\t1\t0 1181\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t87\t874.2\t0\t87\r\n",
      "4\t2\t218.6\t0\t2\r\n",
      "5\t10\t54.6\t0\t10\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/481.R1.fastq.gz.trimmed.fastq.gz /code/sequences/481.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.70 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  55,949\r\n",
      "Reads with adapters:                    52,336 (93.5%)\r\n",
      "Reads written (passing filters):        55,949 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,336,783 bp\r\n",
      "Total written (filtered):     11,435,732 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40637 times.\r\n",
      "40579 times, it overlapped the 5' end of a read\r\n",
      "58 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t874.2\t0\t1\r\n",
      "4\t1\t218.6\t0\t1\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t66\t0.0\t0\t66\r\n",
      "21\t40502\t0.0\t0\t40502\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t49\t874.2\t0\t49\r\n",
      "4\t1\t218.6\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11699 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "11625 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t874.2\t0\t73\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t874.2\t0\t45\r\n",
      "4\t11579\t218.6\t0\t11579\r\n",
      "5\t1\t54.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/509.R2.fastq.gz.trimmed.fastq.gz /code/sequences/509.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.29 s (21 us/read; 2.85 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,616\r\n",
      "Reads with adapters:                     2,691 (2.5%)\r\n",
      "Reads written (passing filters):       108,616 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,394,865 bp\r\n",
      "Total written (filtered):     16,354,338 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1143 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1137 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1697.1\t0\t4\r\n",
      "4\t2\t424.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1122\t1697.1\t0\t1122\r\n",
      "4\t10\t424.3\t0\t10\r\n",
      "5\t5\t106.1\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1548 times.\r\n",
      "1355 times, it overlapped the 5' end of a read\r\n",
      "193 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1697.1\t0\t2\r\n",
      "4\t2\t424.3\t0\t2\r\n",
      "22\t3\t0.0\t1\t0 3\r\n",
      "27\t1348\t0.0\t1\t0 1348\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t181\t1697.1\t0\t181\r\n",
      "4\t3\t424.3\t0\t3\r\n",
      "5\t9\t106.1\t0\t9\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/472.R2.fastq.gz.trimmed.fastq.gz /code/sequences/472.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.95 s (34 us/read; 1.78 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 117,427\r\n",
      "Reads with adapters:                     2,807 (2.4%)\r\n",
      "Reads written (passing filters):       117,427 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,075,334 bp\r\n",
      "Total written (filtered):     16,033,309 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1217 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "1208 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1834.8\t0\t5\r\n",
      "4\t4\t458.7\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1198\t1834.8\t0\t1198\r\n",
      "4\t4\t458.7\t0\t4\r\n",
      "5\t5\t114.7\t0\t5\r\n",
      "7\t1\t7.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1590 times.\r\n",
      "1323 times, it overlapped the 5' end of a read\r\n",
      "267 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1834.8\t0\t2\r\n",
      "4\t1\t458.7\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1319\t0.0\t1\t0 1319\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t215\t1834.8\t0\t215\r\n",
      "4\t7\t458.7\t0\t7\r\n",
      "5\t11\t114.7\t0\t11\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "51\t26\t0.0\t1\t5 21\r\n",
      "52\t2\t0.0\t1\t0 2\r\n",
      "53\t2\t0.0\t1\t0 2\r\n",
      "67\t2\t0.0\t1\t0 2\r\n",
      "264\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/523.R2.fastq.gz.trimmed.fastq.gz /code/sequences/523.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.31 s (37 us/read; 1.64 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,242\r\n",
      "Reads with adapters:                     2,878 (3.2%)\r\n",
      "Reads written (passing filters):        90,242 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,027,519 bp\r\n",
      "Total written (filtered):     17,982,944 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1271 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1266 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1410.0\t0\t1\r\n",
      "4\t4\t352.5\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1263\t1410.0\t0\t1263\r\n",
      "4\t1\t352.5\t0\t1\r\n",
      "5\t2\t88.1\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1607 times.\r\n",
      "1487 times, it overlapped the 5' end of a read\r\n",
      "120 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1410.0\t0\t1\r\n",
      "4\t1\t352.5\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "27\t1483\t0.0\t1\t0 1483\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t106\t1410.0\t0\t106\r\n",
      "4\t3\t352.5\t0\t3\r\n",
      "5\t7\t88.1\t0\t7\r\n",
      "6\t3\t22.0\t0\t3\r\n",
      "274\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3150.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3150.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 14.85 s (46 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 320,550\r\n",
      "Reads with adapters:                   269,139 (84.0%)\r\n",
      "Reads written (passing filters):       320,550 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    91,623,409 bp\r\n",
      "Total written (filtered):     86,076,169 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 262704 times.\r\n",
      "262589 times, it overlapped the 5' end of a read\r\n",
      "115 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t38\t0.0\t0\t38\r\n",
      "19\t56\t0.0\t0\t56\r\n",
      "20\t349\t0.0\t0\t349\r\n",
      "21\t262145\t0.0\t0\t262145\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t82\t5008.6\t0\t82\r\n",
      "4\t3\t1252.1\t0\t3\r\n",
      "6\t3\t78.3\t0\t3\r\n",
      "7\t2\t19.6\t0\t2\r\n",
      "299\t6\t0.0\t0\t6\r\n",
      "300\t19\t0.0\t0\t19\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6435 times.\r\n",
      "132 times, it overlapped the 5' end of a read\r\n",
      "6303 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t5008.6\t0\t128\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t5008.6\t0\t39\r\n",
      "4\t6263\t1252.1\t0\t6263\r\n",
      "5\t1\t313.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/523.R1.fastq.gz.trimmed.fastq.gz /code/sequences/523.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.84 s (43 us/read; 1.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,242\r\n",
      "Reads with adapters:                    84,788 (94.0%)\r\n",
      "Reads written (passing filters):        90,242 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,282,878 bp\r\n",
      "Total written (filtered):     16,947,624 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58660 times.\r\n",
      "58586 times, it overlapped the 5' end of a read\r\n",
      "74 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t72\t0.0\t0\t72\r\n",
      "21\t58493\t0.0\t0\t58493\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1410.0\t0\t71\r\n",
      "4\t1\t352.5\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 26128 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "26115 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1410.0\t0\t11\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t1410.0\t0\t57\r\n",
      "4\t26057\t352.5\t0\t26057\r\n",
      "5\t1\t88.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/509.R1.fastq.gz.trimmed.fastq.gz /code/sequences/509.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.09 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,616\r\n",
      "Reads with adapters:                   100,638 (92.7%)\r\n",
      "Reads written (passing filters):       108,616 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,882,623 bp\r\n",
      "Total written (filtered):     15,503,726 bp (91.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 57525 times.\r\n",
      "57428 times, it overlapped the 5' end of a read\r\n",
      "97 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1697.1\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t86\t0.0\t0\t86\r\n",
      "21\t57323\t0.0\t0\t57323\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t94\t1697.1\t0\t94\r\n",
      "4\t1\t424.3\t0\t1\r\n",
      "56\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 43113 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "43059 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t49\t1697.1\t0\t49\r\n",
      "5\t1\t106.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1697.1\t0\t83\r\n",
      "4\t42974\t424.3\t0\t42974\r\n",
      "5\t2\t106.1\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/526.R2.fastq.gz.trimmed.fastq.gz /code/sequences/526.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.32 s (36 us/read; 1.68 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,631\r\n",
      "Reads with adapters:                     3,028 (3.3%)\r\n",
      "Reads written (passing filters):        92,631 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,607,982 bp\r\n",
      "Total written (filtered):     17,561,931 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1356 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1352 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1447.4\t0\t3\r\n",
      "4\t1\t361.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1337\t1447.4\t0\t1337\r\n",
      "4\t12\t361.8\t0\t12\r\n",
      "5\t2\t90.5\t0\t2\r\n",
      "6\t1\t22.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1672 times.\r\n",
      "1542 times, it overlapped the 5' end of a read\r\n",
      "130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1447.4\t0\t1\r\n",
      "4\t2\t361.8\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1537\t0.0\t1\t0 1537\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t124\t1447.4\t0\t124\r\n",
      "4\t1\t361.8\t0\t1\r\n",
      "5\t5\t90.5\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/526.R1.fastq.gz.trimmed.fastq.gz /code/sequences/526.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.14 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,631\r\n",
      "Reads with adapters:                    87,139 (94.1%)\r\n",
      "Reads written (passing filters):        92,631 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,712,163 bp\r\n",
      "Total written (filtered):     16,315,591 bp (92.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61651 times.\r\n",
      "61582 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t91\t0.0\t0\t91\r\n",
      "21\t61470\t0.0\t0\t61470\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1447.4\t0\t62\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "61\t2\t0.0\t0\t2\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25488 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "25469 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1447.4\t0\t17\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1447.4\t0\t44\r\n",
      "4\t25425\t361.8\t0\t25425\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/528.R2.fastq.gz.trimmed.fastq.gz /code/sequences/528.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.38 s (38 us/read; 1.60 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 116,569\r\n",
      "Reads with adapters:                     3,341 (2.9%)\r\n",
      "Reads written (passing filters):       116,569 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,103,743 bp\r\n",
      "Total written (filtered):     21,054,509 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1540 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "1531 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1821.4\t0\t8\r\n",
      "5\t1\t113.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1525\t1821.4\t0\t1525\r\n",
      "4\t3\t455.3\t0\t3\r\n",
      "5\t3\t113.8\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1801 times.\r\n",
      "1601 times, it overlapped the 5' end of a read\r\n",
      "200 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1821.4\t0\t2\r\n",
      "4\t1\t455.3\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1596\t0.0\t1\t0 1596\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t185\t1821.4\t0\t185\r\n",
      "4\t6\t455.3\t0\t6\r\n",
      "5\t6\t113.8\t0\t6\r\n",
      "270\t2\t0.0\t1\t0 2\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/842.R2.fastq.gz.trimmed.fastq.gz /code/sequences/842.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.06 s (30 us/read; 2.01 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 135,539\r\n",
      "Reads with adapters:                     4,494 (3.3%)\r\n",
      "Reads written (passing filters):       135,539 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,214,401 bp\r\n",
      "Total written (filtered):     26,145,485 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2012 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "2002 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2117.8\t0\t8\r\n",
      "4\t2\t529.4\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1996\t2117.8\t0\t1996\r\n",
      "4\t2\t529.4\t0\t2\r\n",
      "5\t4\t132.4\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2482 times.\r\n",
      "2311 times, it overlapped the 5' end of a read\r\n",
      "171 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2117.8\t0\t2\r\n",
      "4\t1\t529.4\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2306\t0.0\t1\t0 2306\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t151\t2117.8\t0\t151\r\n",
      "4\t10\t529.4\t0\t10\r\n",
      "5\t9\t132.4\t0\t9\r\n",
      "6\t1\t33.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/EB.R1.fastq.gz.trimmed.fastq.gz /code/sequences/EB.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.47 s (42 us/read; 1.43 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  11,190\r\n",
      "Reads with adapters:                    10,415 (93.1%)\r\n",
      "Reads written (passing filters):        11,190 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     1,031,169 bp\r\n",
      "Total written (filtered):        897,963 bp (87.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5398 times.\r\n",
      "5387 times, it overlapped the 5' end of a read\r\n",
      "11 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t3\t0.0\t0\t3\r\n",
      "21\t5383\t0.0\t0\t5383\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t174.8\t0\t10\r\n",
      "5\t1\t10.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5017 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "5016 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t174.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t174.8\t0\t18\r\n",
      "4\t4998\t43.7\t0\t4998\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/EB.R2.fastq.gz.trimmed.fastq.gz /code/sequences/EB.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.36 s (32 us/read; 1.85 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  11,190\r\n",
      "Reads with adapters:                       228 (2.0%)\r\n",
      "Reads written (passing filters):        11,190 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       963,284 bp\r\n",
      "Total written (filtered):        959,047 bp (99.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t174.8\t0\t62\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 166 times.\r\n",
      "148 times, it overlapped the 5' end of a read\r\n",
      "18 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "27\t148\t0.0\t1\t0 148\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t174.8\t0\t17\r\n",
      "4\t1\t43.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/528.R1.fastq.gz.trimmed.fastq.gz /code/sequences/528.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.89 s (51 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 116,569\r\n",
      "Reads with adapters:                   108,923 (93.4%)\r\n",
      "Reads written (passing filters):       116,569 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,449,670 bp\r\n",
      "Total written (filtered):     19,872,303 bp (92.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67119 times.\r\n",
      "67001 times, it overlapped the 5' end of a read\r\n",
      "118 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t455.3\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t99\t0.0\t0\t99\r\n",
      "21\t66876\t0.0\t0\t66876\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t103\t1821.4\t0\t103\r\n",
      "4\t1\t455.3\t0\t1\r\n",
      "7\t2\t7.1\t0\t2\r\n",
      "51\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 41804 times.\r\n",
      "40 times, it overlapped the 5' end of a read\r\n",
      "41764 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1821.4\t0\t39\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t117\t1821.4\t0\t117\r\n",
      "4\t41645\t455.3\t0\t41645\r\n",
      "5\t2\t113.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/EB2.R1.fastq.gz.trimmed.fastq.gz /code/sequences/EB2.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.25 s (40 us/read; 1.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   6,094\r\n",
      "Reads with adapters:                     5,717 (93.8%)\r\n",
      "Reads written (passing filters):         6,094 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       769,943 bp\r\n",
      "Total written (filtered):        674,942 bp (87.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4250 times.\r\n",
      "4244 times, it overlapped the 5' end of a read\r\n",
      "6 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "20\t5\t0.0\t0\t5\r\n",
      "21\t4239\t0.0\t0\t4239\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t95.2\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1467 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1466 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t95.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t95.2\t0\t3\r\n",
      "4\t1463\t23.8\t0\t1463\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/928.R2.fastq.gz.trimmed.fastq.gz /code/sequences/928.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.77 s (31 us/read; 1.94 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  89,236\r\n",
      "Reads with adapters:                     3,632 (4.1%)\r\n",
      "Reads written (passing filters):        89,236 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,581,868 bp\r\n",
      "Total written (filtered):     20,528,168 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1755 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1749 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1394.3\t0\t5\r\n",
      "4\t1\t348.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1737\t1394.3\t0\t1737\r\n",
      "4\t8\t348.6\t0\t8\r\n",
      "5\t3\t87.1\t0\t3\r\n",
      "6\t1\t21.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1877 times.\r\n",
      "1778 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1394.3\t0\t4\r\n",
      "4\t4\t348.6\t0\t4\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1768\t0.0\t1\t0 1768\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t1394.3\t0\t93\r\n",
      "4\t2\t348.6\t0\t2\r\n",
      "5\t2\t87.1\t0\t2\r\n",
      "6\t1\t21.8\t0\t1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/EB2.R2.fastq.gz.trimmed.fastq.gz /code/sequences/EB2.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.23 s (37 us/read; 1.61 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   6,094\r\n",
      "Reads with adapters:                       193 (3.2%)\r\n",
      "Reads written (passing filters):         6,094 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:       769,002 bp\r\n",
      "Total written (filtered):        765,850 bp (99.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 69 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t67\t95.2\t0\t67\r\n",
      "4\t2\t23.8\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 124 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t23.8\t0\t1\r\n",
      "27\t107\t0.0\t1\t0 107\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t95.2\t0\t15\r\n",
      "5\t1\t6.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/PCRneg.R1.fastq.gz.trimmed.fastq.gz /code/sequences/PCRneg.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.62 s (39 us/read; 1.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  15,986\r\n",
      "Reads with adapters:                    14,591 (91.3%)\r\n",
      "Reads written (passing filters):        15,986 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     1,409,369 bp\r\n",
      "Total written (filtered):      1,249,445 bp (88.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5993 times.\r\n",
      "5969 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t5\t0.0\t0\t5\r\n",
      "21\t5962\t0.0\t0\t5962\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t249.8\t0\t21\r\n",
      "5\t1\t15.6\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "62\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8598 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "8596 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t249.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t249.8\t0\t20\r\n",
      "4\t8575\t62.4\t0\t8575\r\n",
      "5\t1\t15.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/PCRneg.R2.fastq.gz.trimmed.fastq.gz /code/sequences/PCRneg.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.53 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  15,986\r\n",
      "Reads with adapters:                       278 (1.7%)\r\n",
      "Reads written (passing filters):        15,986 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     1,239,707 bp\r\n",
      "Total written (filtered):      1,234,958 bp (99.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 89 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "88 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t62.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t88\t249.8\t0\t88\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 189 times.\r\n",
      "164 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t62.4\t0\t1\r\n",
      "27\t163\t0.0\t1\t0 163\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t249.8\t0\t24\r\n",
      "4\t1\t62.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/928.R1.fastq.gz.trimmed.fastq.gz /code/sequences/928.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.96 s (44 us/read; 1.35 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  89,236\r\n",
      "Reads with adapters:                    84,703 (94.9%)\r\n",
      "Reads written (passing filters):        89,236 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,877,782 bp\r\n",
      "Total written (filtered):     19,353,531 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 69593 times.\r\n",
      "69542 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1394.3\t0\t3\r\n",
      "8\t1\t1.4\t0\t1\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t100\t0.0\t0\t100\r\n",
      "21\t69415\t0.0\t0\t69415\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1394.3\t0\t37\r\n",
      "5\t1\t87.1\t0\t1\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "267\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 15110 times.\r\n",
      "85 times, it overlapped the 5' end of a read\r\n",
      "15025 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1394.3\t0\t83\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1394.3\t0\t38\r\n",
      "4\t14987\t348.6\t0\t14987\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/PCRneg2.R1.fastq.gz.trimmed.fastq.gz /code/sequences/PCRneg2.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.42 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   7,348\r\n",
      "Reads with adapters:                     6,810 (92.7%)\r\n",
      "Reads written (passing filters):         7,348 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     1,271,567 bp\r\n",
      "Total written (filtered):      1,151,086 bp (90.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5494 times.\r\n",
      "5486 times, it overlapped the 5' end of a read\r\n",
      "8 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t5485\t0.0\t0\t5485\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t114.8\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1316 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1314 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t114.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t114.8\t0\t10\r\n",
      "4\t1304\t28.7\t0\t1304\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/PCRneg2.R2.fastq.gz.trimmed.fastq.gz /code/sequences/PCRneg2.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.18 s (24 us/read; 2.52 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                   7,348\r\n",
      "Reads with adapters:                       293 (4.0%)\r\n",
      "Reads written (passing filters):         7,348 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     1,207,871 bp\r\n",
      "Total written (filtered):      1,204,058 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 156 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "156 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t153\t114.8\t0\t153\r\n",
      "4\t1\t28.7\t0\t1\r\n",
      "5\t2\t7.2\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 137 times.\r\n",
      "123 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t28.7\t0\t1\r\n",
      "27\t122\t0.0\t1\t0 122\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t114.8\t0\t14\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/842.R1.fastq.gz.trimmed.fastq.gz /code/sequences/842.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.58 s (49 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 135,539\r\n",
      "Reads with adapters:                   127,214 (93.9%)\r\n",
      "Reads written (passing filters):       135,539 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,674,553 bp\r\n",
      "Total written (filtered):     24,681,525 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 87250 times.\r\n",
      "87131 times, it overlapped the 5' end of a read\r\n",
      "119 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t529.4\t0\t2\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t112\t0.0\t0\t112\r\n",
      "21\t86981\t0.0\t0\t86981\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t105\t2117.8\t0\t105\r\n",
      "5\t1\t132.4\t0\t1\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 39964 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "39935 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t2117.8\t0\t28\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t2117.8\t0\t86\r\n",
      "4\t39844\t529.4\t0\t39844\r\n",
      "5\t5\t132.4\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/864.R2.fastq.gz.trimmed.fastq.gz /code/sequences/864.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.36 s (34 us/read; 1.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 157,857\r\n",
      "Reads with adapters:                     2,828 (1.8%)\r\n",
      "Reads written (passing filters):       157,857 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,471,438 bp\r\n",
      "Total written (filtered):     16,426,914 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1073 times.\r\n",
      "12 times, it overlapped the 5' end of a read\r\n",
      "1061 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t2466.5\t0\t9\r\n",
      "4\t3\t616.6\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1051\t2466.5\t0\t1051\r\n",
      "4\t5\t616.6\t0\t5\r\n",
      "5\t3\t154.2\t0\t3\r\n",
      "6\t2\t38.5\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1755 times.\r\n",
      "1502 times, it overlapped the 5' end of a read\r\n",
      "253 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2466.5\t0\t1\r\n",
      "4\t2\t616.6\t0\t2\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1495\t0.0\t1\t0 1495\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t223\t2466.5\t0\t223\r\n",
      "4\t12\t616.6\t0\t12\r\n",
      "5\t16\t154.2\t0\t16\r\n",
      "6\t2\t38.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/864.R1.fastq.gz.trimmed.fastq.gz /code/sequences/864.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.75 s (43 us/read; 1.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 157,857\r\n",
      "Reads with adapters:                   142,015 (90.0%)\r\n",
      "Reads written (passing filters):       157,857 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,521,269 bp\r\n",
      "Total written (filtered):     16,016,142 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55280 times.\r\n",
      "55061 times, it overlapped the 5' end of a read\r\n",
      "219 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2466.5\t0\t2\r\n",
      "4\t4\t616.6\t0\t4\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t54953\t0.0\t0\t54953\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t210\t2466.5\t0\t210\r\n",
      "5\t1\t154.2\t0\t1\r\n",
      "7\t2\t9.6\t0\t2\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 86735 times.\r\n",
      "69 times, it overlapped the 5' end of a read\r\n",
      "86666 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t2466.5\t0\t60\r\n",
      "4\t1\t616.6\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t6\t0.0\t1\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t191\t2466.5\t0\t191\r\n",
      "4\t86471\t616.6\t0\t86471\r\n",
      "5\t4\t154.2\t0\t4\r\n",
      "\r\n",
      "\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 176648 (88324 pairs)\n",
      "FastQ single records kept: 14210 (from PE1: 12110, from PE2: 2100)\n",
      "FastQ paired records discarded: 2816 (1408 pairs)\n",
      "FastQ single records discarded: 14210 (from PE1: 2100, from PE2: 12110)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 300182 (150091 pairs)\n",
      "FastQ single records kept: 8456 (from PE1: 6296, from PE2: 2160)\n",
      "FastQ paired records discarded: 1950 (975 pairs)\n",
      "FastQ single records discarded: 8456 (from PE1: 2160, from PE2: 6296)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 150902 (75451 pairs)\n",
      "FastQ single records kept: 5037 (from PE1: 4286, from PE2: 751)\n",
      "FastQ paired records discarded: 916 (458 pairs)\n",
      "FastQ single records discarded: 5037 (from PE1: 751, from PE2: 4286)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 90788 (45394 pairs)\n",
      "FastQ single records kept: 2528 (from PE1: 2066, from PE2: 462)\n",
      "FastQ paired records discarded: 426 (213 pairs)\n",
      "FastQ single records discarded: 2528 (from PE1: 462, from PE2: 2066)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 31866 (15933 pairs)\n",
      "FastQ single records kept: 1729 (from PE1: 1411, from PE2: 318)\n",
      "FastQ paired records discarded: 390 (195 pairs)\n",
      "FastQ single records discarded: 1729 (from PE1: 318, from PE2: 1411)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 133710 (66855 pairs)\n",
      "FastQ single records kept: 4305 (from PE1: 3560, from PE2: 745)\n",
      "FastQ paired records discarded: 732 (366 pairs)\n",
      "FastQ single records discarded: 4305 (from PE1: 745, from PE2: 3560)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 162534 (81267 pairs)\n",
      "FastQ single records kept: 1973 (from PE1: 1729, from PE2: 244)\n",
      "FastQ paired records discarded: 228 (114 pairs)\n",
      "FastQ single records discarded: 1973 (from PE1: 244, from PE2: 1729)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 184976 (92488 pairs)\n",
      "FastQ single records kept: 5114 (from PE1: 4313, from PE2: 801)\n",
      "FastQ paired records discarded: 866 (433 pairs)\n",
      "FastQ single records discarded: 5114 (from PE1: 801, from PE2: 4313)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 119292 (59646 pairs)\n",
      "FastQ single records kept: 5660 (from PE1: 4337, from PE2: 1323)\n",
      "FastQ paired records discarded: 1206 (603 pairs)\n",
      "FastQ single records discarded: 5660 (from PE1: 1323, from PE2: 4337)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 210570 (105285 pairs)\n",
      "FastQ single records kept: 5743 (from PE1: 4850, from PE2: 893)\n",
      "FastQ paired records discarded: 908 (454 pairs)\n",
      "FastQ single records discarded: 5743 (from PE1: 893, from PE2: 4850)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 214622 (107311 pairs)\n",
      "FastQ single records kept: 4838 (from PE1: 4102, from PE2: 736)\n",
      "FastQ paired records discarded: 858 (429 pairs)\n",
      "FastQ single records discarded: 4838 (from PE1: 736, from PE2: 4102)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 239582 (119791 pairs)\n",
      "FastQ single records kept: 5533 (from PE1: 4073, from PE2: 1460)\n",
      "FastQ paired records discarded: 1440 (720 pairs)\n",
      "FastQ single records discarded: 5533 (from PE1: 1460, from PE2: 4073)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 200858 (100429 pairs)\n",
      "FastQ single records kept: 3894 (from PE1: 3075, from PE2: 819)\n",
      "FastQ paired records discarded: 756 (378 pairs)\n",
      "FastQ single records discarded: 3894 (from PE1: 819, from PE2: 3075)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 183778 (91889 pairs)\n",
      "FastQ single records kept: 3180 (from PE1: 2548, from PE2: 632)\n",
      "FastQ paired records discarded: 564 (282 pairs)\n",
      "FastQ single records discarded: 3180 (from PE1: 632, from PE2: 2548)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 168194 (84097 pairs)\n",
      "FastQ single records kept: 7791 (from PE1: 6528, from PE2: 1263)\n",
      "FastQ paired records discarded: 1420 (710 pairs)\n",
      "FastQ single records discarded: 7791 (from PE1: 1263, from PE2: 6528)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 200356 (100178 pairs)\n",
      "FastQ single records kept: 9172 (from PE1: 7827, from PE2: 1345)\n",
      "FastQ paired records discarded: 1546 (773 pairs)\n",
      "FastQ single records discarded: 9172 (from PE1: 1345, from PE2: 7827)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 259174 (129587 pairs)\n",
      "FastQ single records kept: 3455 (from PE1: 2879, from PE2: 576)\n",
      "FastQ paired records discarded: 478 (239 pairs)\n",
      "FastQ single records discarded: 3455 (from PE1: 576, from PE2: 2879)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 214406 (107203 pairs)\n",
      "FastQ single records kept: 10241 (from PE1: 8095, from PE2: 2146)\n",
      "FastQ paired records discarded: 2238 (1119 pairs)\n",
      "FastQ single records discarded: 10241 (from PE1: 2146, from PE2: 8095)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 154426 (77213 pairs)\n",
      "FastQ single records kept: 5815 (from PE1: 4843, from PE2: 972)\n",
      "FastQ paired records discarded: 1018 (509 pairs)\n",
      "FastQ single records discarded: 5815 (from PE1: 972, from PE2: 4843)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 175740 (87870 pairs)\n",
      "FastQ single records kept: 5390 (from PE1: 4576, from PE2: 814)\n",
      "FastQ paired records discarded: 860 (430 pairs)\n",
      "FastQ single records discarded: 5390 (from PE1: 814, from PE2: 4576)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 235796 (117898 pairs)\n",
      "FastQ single records kept: 2562 (from PE1: 2321, from PE2: 241)\n",
      "FastQ paired records discarded: 284 (142 pairs)\n",
      "FastQ single records discarded: 2562 (from PE1: 241, from PE2: 2321)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 190610 (95305 pairs)\n",
      "FastQ single records kept: 4142 (from PE1: 2982, from PE2: 1160)\n",
      "FastQ paired records discarded: 894 (447 pairs)\n",
      "FastQ single records discarded: 4142 (from PE1: 1160, from PE2: 2982)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 193950 (96975 pairs)\n",
      "FastQ single records kept: 4011 (from PE1: 3164, from PE2: 847)\n",
      "FastQ paired records discarded: 770 (385 pairs)\n",
      "FastQ single records discarded: 4011 (from PE1: 847, from PE2: 3164)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 281162 (140581 pairs)\n",
      "FastQ single records kept: 4469 (from PE1: 3447, from PE2: 1022)\n",
      "FastQ paired records discarded: 752 (376 pairs)\n",
      "FastQ single records discarded: 4469 (from PE1: 1022, from PE2: 3447)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 1388 (694 pairs)\n",
      "FastQ single records kept: 165 (from PE1: 131, from PE2: 34)\n",
      "FastQ paired records discarded: 42 (21 pairs)\n",
      "FastQ single records discarded: 165 (from PE1: 34, from PE2: 131)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 172136 (86068 pairs)\n",
      "FastQ single records kept: 2745 (from PE1: 2429, from PE2: 316)\n",
      "FastQ paired records discarded: 378 (189 pairs)\n",
      "FastQ single records discarded: 2745 (from PE1: 316, from PE2: 2429)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 1444 (722 pairs)\n",
      "FastQ single records kept: 193 (from PE1: 121, from PE2: 72)\n",
      "FastQ paired records discarded: 78 (39 pairs)\n",
      "FastQ single records discarded: 193 (from PE1: 72, from PE2: 121)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 219834 (109917 pairs)\n",
      "FastQ single records kept: 4419 (from PE1: 3491, from PE2: 928)\n",
      "FastQ paired records discarded: 810 (405 pairs)\n",
      "FastQ single records discarded: 4419 (from PE1: 928, from PE2: 3491)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 234454 (117227 pairs)\n",
      "FastQ single records kept: 4121 (from PE1: 3454, from PE2: 667)\n",
      "FastQ paired records discarded: 604 (302 pairs)\n",
      "FastQ single records discarded: 4121 (from PE1: 667, from PE2: 3454)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 143168 (71584 pairs)\n",
      "FastQ single records kept: 7954 (from PE1: 6469, from PE2: 1485)\n",
      "FastQ paired records discarded: 1680 (840 pairs)\n",
      "FastQ single records discarded: 7954 (from PE1: 1485, from PE2: 6469)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 272844 (136422 pairs)\n",
      "FastQ single records kept: 4335 (from PE1: 3668, from PE2: 667)\n",
      "FastQ paired records discarded: 778 (389 pairs)\n",
      "FastQ single records discarded: 4335 (from PE1: 667, from PE2: 3668)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 317512 (158756 pairs)\n",
      "FastQ single records kept: 5003 (from PE1: 4072, from PE2: 931)\n",
      "FastQ paired records discarded: 822 (411 pairs)\n",
      "FastQ single records discarded: 5003 (from PE1: 931, from PE2: 4072)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 315994 (157997 pairs)\n",
      "FastQ single records kept: 6718 (from PE1: 5433, from PE2: 1285)\n",
      "FastQ paired records discarded: 1246 (623 pairs)\n",
      "FastQ single records discarded: 6718 (from PE1: 1285, from PE2: 5433)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 235050 (117525 pairs)\n",
      "FastQ single records kept: 7143 (from PE1: 6012, from PE2: 1131)\n",
      "FastQ paired records discarded: 1116 (558 pairs)\n",
      "FastQ single records discarded: 7143 (from PE1: 1131, from PE2: 6012)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 517504 (258752 pairs)\n",
      "FastQ single records kept: 20450 (from PE1: 16130, from PE2: 4320)\n",
      "FastQ paired records discarded: 4314 (2157 pairs)\n",
      "FastQ single records discarded: 20450 (from PE1: 4320, from PE2: 16130)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 296904 (148452 pairs)\n",
      "FastQ single records kept: 23913 (from PE1: 20252, from PE2: 3661)\n",
      "FastQ paired records discarded: 4340 (2170 pairs)\n",
      "FastQ single records discarded: 23913 (from PE1: 3661, from PE2: 20252)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 254180 (127090 pairs)\n",
      "FastQ single records kept: 9249 (from PE1: 7482, from PE2: 1767)\n",
      "FastQ paired records discarded: 1704 (852 pairs)\n",
      "FastQ single records discarded: 9249 (from PE1: 1767, from PE2: 7482)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 243418 (121709 pairs)\n",
      "FastQ single records kept: 3339 (from PE1: 2898, from PE2: 441)\n",
      "FastQ paired records discarded: 426 (213 pairs)\n",
      "FastQ single records discarded: 3339 (from PE1: 441, from PE2: 2898)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 279648 (139824 pairs)\n",
      "FastQ single records kept: 3884 (from PE1: 3295, from PE2: 589)\n",
      "FastQ paired records discarded: 508 (254 pairs)\n",
      "FastQ single records discarded: 3884 (from PE1: 589, from PE2: 3295)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 175182 (87591 pairs)\n",
      "FastQ single records kept: 12575 (from PE1: 10105, from PE2: 2470)\n",
      "FastQ paired records discarded: 2778 (1389 pairs)\n",
      "FastQ single records discarded: 12575 (from PE1: 2470, from PE2: 10105)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 181838 (90919 pairs)\n",
      "FastQ single records kept: 2635 (from PE1: 2202, from PE2: 433)\n",
      "FastQ paired records discarded: 400 (200 pairs)\n",
      "FastQ single records discarded: 2635 (from PE1: 433, from PE2: 2202)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 202090 (101045 pairs)\n",
      "FastQ single records kept: 2758 (from PE1: 2434, from PE2: 324)\n",
      "FastQ paired records discarded: 396 (198 pairs)\n",
      "FastQ single records discarded: 2758 (from PE1: 324, from PE2: 2434)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 217024 (108512 pairs)\n",
      "FastQ single records kept: 2847 (from PE1: 2355, from PE2: 492)\n",
      "FastQ paired records discarded: 406 (203 pairs)\n",
      "FastQ single records discarded: 2847 (from PE1: 492, from PE2: 2355)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 183732 (91866 pairs)\n",
      "FastQ single records kept: 2348 (from PE1: 2019, from PE2: 329)\n",
      "FastQ paired records discarded: 340 (170 pairs)\n",
      "FastQ single records discarded: 2348 (from PE1: 329, from PE2: 2019)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 236500 (118250 pairs)\n",
      "FastQ single records kept: 7395 (from PE1: 6073, from PE2: 1322)\n",
      "FastQ paired records discarded: 1322 (661 pairs)\n",
      "FastQ single records discarded: 7395 (from PE1: 1322, from PE2: 6073)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 235612 (117806 pairs)\n",
      "FastQ single records kept: 3383 (from PE1: 2977, from PE2: 406)\n",
      "FastQ paired records discarded: 424 (212 pairs)\n",
      "FastQ single records discarded: 3383 (from PE1: 406, from PE2: 2977)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 163190 (81595 pairs)\n",
      "FastQ single records kept: 6967 (from PE1: 5553, from PE2: 1414)\n",
      "FastQ paired records discarded: 1348 (674 pairs)\n",
      "FastQ single records discarded: 6967 (from PE1: 1414, from PE2: 5553)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 205716 (102858 pairs)\n",
      "FastQ single records kept: 4482 (from PE1: 3646, from PE2: 836)\n",
      "FastQ paired records discarded: 842 (421 pairs)\n",
      "FastQ single records discarded: 4482 (from PE1: 836, from PE2: 3646)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 176046 (88023 pairs)\n",
      "FastQ single records kept: 4992 (from PE1: 4056, from PE2: 936)\n",
      "FastQ paired records discarded: 1058 (529 pairs)\n",
      "FastQ single records discarded: 4992 (from PE1: 936, from PE2: 4056)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 242094 (121047 pairs)\n",
      "FastQ single records kept: 6287 (from PE1: 4781, from PE2: 1506)\n",
      "FastQ paired records discarded: 1466 (733 pairs)\n",
      "FastQ single records discarded: 6287 (from PE1: 1506, from PE2: 4781)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 114280 (57140 pairs)\n",
      "FastQ single records kept: 2362 (from PE1: 1694, from PE2: 668)\n",
      "FastQ paired records discarded: 446 (223 pairs)\n",
      "FastQ single records discarded: 2362 (from PE1: 668, from PE2: 1694)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 183552 (91776 pairs)\n",
      "FastQ single records kept: 11560 (from PE1: 9850, from PE2: 1710)\n",
      "FastQ paired records discarded: 1998 (999 pairs)\n",
      "FastQ single records discarded: 11560 (from PE1: 1710, from PE2: 9850)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 139610 (69805 pairs)\n",
      "FastQ single records kept: 2390 (from PE1: 2072, from PE2: 318)\n",
      "FastQ paired records discarded: 376 (188 pairs)\n",
      "FastQ single records discarded: 2390 (from PE1: 318, from PE2: 2072)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 11814 (5907 pairs)\n",
      "FastQ single records kept: 1226 (from PE1: 783, from PE2: 443)\n",
      "FastQ paired records discarded: 430 (215 pairs)\n",
      "FastQ single records discarded: 1226 (from PE1: 443, from PE2: 783)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 317758 (158879 pairs)\n",
      "FastQ single records kept: 5579 (from PE1: 4785, from PE2: 794)\n",
      "FastQ paired records discarded: 838 (419 pairs)\n",
      "FastQ single records discarded: 5579 (from PE1: 794, from PE2: 4785)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 356310 (178155 pairs)\n",
      "FastQ single records kept: 13875 (from PE1: 11166, from PE2: 2709)\n",
      "FastQ paired records discarded: 2666 (1333 pairs)\n",
      "FastQ single records discarded: 13875 (from PE1: 2709, from PE2: 11166)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 304714 (152357 pairs)\n",
      "FastQ single records kept: 9043 (from PE1: 6456, from PE2: 2587)\n",
      "FastQ paired records discarded: 2256 (1128 pairs)\n",
      "FastQ single records discarded: 9043 (from PE1: 2587, from PE2: 6456)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 166882 (83441 pairs)\n",
      "FastQ single records kept: 2272 (from PE1: 1696, from PE2: 576)\n",
      "FastQ paired records discarded: 448 (224 pairs)\n",
      "FastQ single records discarded: 2272 (from PE1: 576, from PE2: 1696)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 274858 (137429 pairs)\n",
      "FastQ single records kept: 3070 (from PE1: 2702, from PE2: 368)\n",
      "FastQ paired records discarded: 326 (163 pairs)\n",
      "FastQ single records discarded: 3070 (from PE1: 368, from PE2: 2702)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 232102 (116051 pairs)\n",
      "FastQ single records kept: 3036 (from PE1: 2495, from PE2: 541)\n",
      "FastQ paired records discarded: 456 (228 pairs)\n",
      "FastQ single records discarded: 3036 (from PE1: 541, from PE2: 2495)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 327508 (163754 pairs)\n",
      "FastQ single records kept: 4087 (from PE1: 3611, from PE2: 476)\n",
      "FastQ paired records discarded: 504 (252 pairs)\n",
      "FastQ single records discarded: 4087 (from PE1: 476, from PE2: 3611)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 449386 (224693 pairs)\n",
      "FastQ single records kept: 11685 (from PE1: 9260, from PE2: 2425)\n",
      "FastQ paired records discarded: 2422 (1211 pairs)\n",
      "FastQ single records discarded: 11685 (from PE1: 2425, from PE2: 9260)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 249544 (124772 pairs)\n",
      "FastQ single records kept: 8751 (from PE1: 7011, from PE2: 1740)\n",
      "FastQ paired records discarded: 1652 (826 pairs)\n",
      "FastQ single records discarded: 8751 (from PE1: 1740, from PE2: 7011)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 197510 (98755 pairs)\n",
      "FastQ single records kept: 7995 (from PE1: 6135, from PE2: 1860)\n",
      "FastQ paired records discarded: 1874 (937 pairs)\n",
      "FastQ single records discarded: 7995 (from PE1: 1860, from PE2: 6135)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 337298 (168649 pairs)\n",
      "FastQ single records kept: 5862 (from PE1: 4966, from PE2: 896)\n",
      "FastQ paired records discarded: 932 (466 pairs)\n",
      "FastQ single records discarded: 5862 (from PE1: 896, from PE2: 4966)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 229848 (114924 pairs)\n",
      "FastQ single records kept: 9027 (from PE1: 7413, from PE2: 1614)\n",
      "FastQ paired records discarded: 1574 (787 pairs)\n",
      "FastQ single records discarded: 9027 (from PE1: 1614, from PE2: 7413)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 1240 (620 pairs)\n",
      "FastQ single records kept: 185 (from PE1: 136, from PE2: 49)\n",
      "FastQ paired records discarded: 52 (26 pairs)\n",
      "FastQ single records discarded: 185 (from PE1: 49, from PE2: 136)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 123014 (61507 pairs)\n",
      "FastQ single records kept: 2871 (from PE1: 2303, from PE2: 568)\n",
      "FastQ paired records discarded: 518 (259 pairs)\n",
      "FastQ single records discarded: 2871 (from PE1: 568, from PE2: 2303)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 163426 (81713 pairs)\n",
      "FastQ single records kept: 4939 (from PE1: 3992, from PE2: 947)\n",
      "FastQ paired records discarded: 940 (470 pairs)\n",
      "FastQ single records discarded: 4939 (from PE1: 947, from PE2: 3992)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 175324 (87662 pairs)\n",
      "FastQ single records kept: 3941 (from PE1: 3184, from PE2: 757)\n",
      "FastQ paired records discarded: 704 (352 pairs)\n",
      "FastQ single records discarded: 3941 (from PE1: 757, from PE2: 3184)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 273402 (136701 pairs)\n",
      "FastQ single records kept: 6488 (from PE1: 5481, from PE2: 1007)\n",
      "FastQ paired records discarded: 1028 (514 pairs)\n",
      "FastQ single records discarded: 6488 (from PE1: 1007, from PE2: 5481)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 144014 (72007 pairs)\n",
      "FastQ single records kept: 5475 (from PE1: 4515, from PE2: 960)\n",
      "FastQ paired records discarded: 962 (481 pairs)\n",
      "FastQ single records discarded: 5475 (from PE1: 960, from PE2: 4515)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 395418 (197709 pairs)\n",
      "FastQ single records kept: 13485 (from PE1: 11320, from PE2: 2165)\n",
      "FastQ paired records discarded: 2424 (1212 pairs)\n",
      "FastQ single records discarded: 13485 (from PE1: 2165, from PE2: 11320)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 207568 (103784 pairs)\n",
      "FastQ single records kept: 2841 (from PE1: 2423, from PE2: 418)\n",
      "FastQ paired records discarded: 418 (209 pairs)\n",
      "FastQ single records discarded: 2841 (from PE1: 418, from PE2: 2423)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 235324 (117662 pairs)\n",
      "FastQ single records kept: 3570 (from PE1: 2826, from PE2: 744)\n",
      "FastQ paired records discarded: 616 (308 pairs)\n",
      "FastQ single records discarded: 3570 (from PE1: 744, from PE2: 2826)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 170484 (85242 pairs)\n",
      "FastQ single records kept: 6519 (from PE1: 5418, from PE2: 1101)\n",
      "FastQ paired records discarded: 1298 (649 pairs)\n",
      "FastQ single records discarded: 6519 (from PE1: 1101, from PE2: 5418)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 218236 (109118 pairs)\n",
      "FastQ single records kept: 3583 (from PE1: 3048, from PE2: 535)\n",
      "FastQ paired records discarded: 562 (281 pairs)\n",
      "FastQ single records discarded: 3583 (from PE1: 535, from PE2: 3048)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 288006 (144003 pairs)\n",
      "FastQ single records kept: 4040 (from PE1: 3440, from PE2: 600)\n",
      "FastQ paired records discarded: 612 (306 pairs)\n",
      "FastQ single records discarded: 4040 (from PE1: 600, from PE2: 3440)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 311834 (155917 pairs)\n",
      "FastQ single records kept: 4247 (from PE1: 3774, from PE2: 473)\n",
      "FastQ paired records discarded: 422 (211 pairs)\n",
      "FastQ single records discarded: 4247 (from PE1: 473, from PE2: 3774)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 258654 (129327 pairs)\n",
      "FastQ single records kept: 3826 (from PE1: 3426, from PE2: 400)\n",
      "FastQ paired records discarded: 414 (207 pairs)\n",
      "FastQ single records discarded: 3826 (from PE1: 400, from PE2: 3426)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 235958 (117979 pairs)\n",
      "FastQ single records kept: 4008 (from PE1: 3495, from PE2: 513)\n",
      "FastQ paired records discarded: 524 (262 pairs)\n",
      "FastQ single records discarded: 4008 (from PE1: 513, from PE2: 3495)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 137016 (68508 pairs)\n",
      "FastQ single records kept: 5975 (from PE1: 4670, from PE2: 1305)\n",
      "FastQ paired records discarded: 1182 (591 pairs)\n",
      "FastQ single records discarded: 5975 (from PE1: 1305, from PE2: 4670)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 162208 (81104 pairs)\n",
      "FastQ single records kept: 10415 (from PE1: 8586, from PE2: 1829)\n",
      "FastQ paired records discarded: 2224 (1112 pairs)\n",
      "FastQ single records discarded: 10415 (from PE1: 1829, from PE2: 8586)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 228144 (114072 pairs)\n",
      "FastQ single records kept: 8748 (from PE1: 6970, from PE2: 1778)\n",
      "FastQ paired records discarded: 1714 (857 pairs)\n",
      "FastQ single records discarded: 8748 (from PE1: 1778, from PE2: 6970)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 160912 (80456 pairs)\n",
      "FastQ single records kept: 8936 (from PE1: 7693, from PE2: 1243)\n",
      "FastQ paired records discarded: 1700 (850 pairs)\n",
      "FastQ single records discarded: 8936 (from PE1: 1243, from PE2: 7693)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 205224 (102612 pairs)\n",
      "FastQ single records kept: 12879 (from PE1: 11133, from PE2: 1746)\n",
      "FastQ paired records discarded: 2156 (1078 pairs)\n",
      "FastQ single records discarded: 12879 (from PE1: 1746, from PE2: 11133)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 173870 (86935 pairs)\n",
      "FastQ single records kept: 5397 (from PE1: 4385, from PE2: 1012)\n",
      "FastQ paired records discarded: 1074 (537 pairs)\n",
      "FastQ single records discarded: 5397 (from PE1: 1012, from PE2: 4385)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 135914 (67957 pairs)\n",
      "FastQ single records kept: 4659 (from PE1: 3940, from PE2: 719)\n",
      "FastQ paired records discarded: 896 (448 pairs)\n",
      "FastQ single records discarded: 4659 (from PE1: 719, from PE2: 3940)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 172982 (86491 pairs)\n",
      "FastQ single records kept: 4804 (from PE1: 4049, from PE2: 755)\n",
      "FastQ paired records discarded: 804 (402 pairs)\n",
      "FastQ single records discarded: 4804 (from PE1: 755, from PE2: 4049)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 196836 (98418 pairs)\n",
      "FastQ single records kept: 7062 (from PE1: 5899, from PE2: 1163)\n",
      "FastQ paired records discarded: 1190 (595 pairs)\n",
      "FastQ single records discarded: 7062 (from PE1: 1163, from PE2: 5899)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 192586 (96293 pairs)\n",
      "FastQ single records kept: 5185 (from PE1: 4396, from PE2: 789)\n",
      "FastQ paired records discarded: 826 (413 pairs)\n",
      "FastQ single records discarded: 5185 (from PE1: 789, from PE2: 4396)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 151330 (75665 pairs)\n",
      "FastQ single records kept: 4690 (from PE1: 3800, from PE2: 890)\n",
      "FastQ paired records discarded: 834 (417 pairs)\n",
      "FastQ single records discarded: 4690 (from PE1: 890, from PE2: 3800)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 231516 (115758 pairs)\n",
      "FastQ single records kept: 5450 (from PE1: 4102, from PE2: 1348)\n",
      "FastQ paired records discarded: 1130 (565 pairs)\n",
      "FastQ single records discarded: 5450 (from PE1: 1348, from PE2: 4102)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 25414 (12707 pairs)\n",
      "FastQ single records kept: 2969 (from PE1: 2202, from PE2: 767)\n",
      "FastQ paired records discarded: 620 (310 pairs)\n",
      "FastQ single records discarded: 2969 (from PE1: 767, from PE2: 2202)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 17036 (8518 pairs)\n",
      "FastQ single records kept: 2356 (from PE1: 1730, from PE2: 626)\n",
      "FastQ paired records discarded: 632 (316 pairs)\n",
      "FastQ single records discarded: 2356 (from PE1: 626, from PE2: 1730)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 320216 (160108 pairs)\n",
      "FastQ single records kept: 7293 (from PE1: 5720, from PE2: 1573)\n",
      "FastQ paired records discarded: 1480 (740 pairs)\n",
      "FastQ single records discarded: 7293 (from PE1: 1573, from PE2: 5720)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 2562 (1281 pairs)\n",
      "FastQ single records kept: 239 (from PE1: 183, from PE2: 56)\n",
      "FastQ paired records discarded: 72 (36 pairs)\n",
      "FastQ single records discarded: 239 (from PE1: 56, from PE2: 183)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 239912 (119956 pairs)\n",
      "FastQ single records kept: 14169 (from PE1: 11839, from PE2: 2330)\n",
      "FastQ paired records discarded: 2828 (1414 pairs)\n",
      "FastQ single records discarded: 14169 (from PE1: 2330, from PE2: 11839)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 164138 (82069 pairs)\n",
      "FastQ single records kept: 2236 (from PE1: 2002, from PE2: 234)\n",
      "FastQ paired records discarded: 254 (127 pairs)\n",
      "FastQ single records discarded: 2236 (from PE1: 234, from PE2: 2002)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 237372 (118686 pairs)\n",
      "FastQ single records kept: 3246 (from PE1: 2938, from PE2: 308)\n",
      "FastQ paired records discarded: 292 (146 pairs)\n",
      "FastQ single records discarded: 3246 (from PE1: 308, from PE2: 2938)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 228530 (114265 pairs)\n",
      "FastQ single records kept: 7121 (from PE1: 5644, from PE2: 1477)\n",
      "FastQ paired records discarded: 1384 (692 pairs)\n",
      "FastQ single records discarded: 7121 (from PE1: 1477, from PE2: 5644)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 139226 (69613 pairs)\n",
      "FastQ single records kept: 1655 (from PE1: 1307, from PE2: 348)\n",
      "FastQ paired records discarded: 270 (135 pairs)\n",
      "FastQ single records discarded: 1655 (from PE1: 348, from PE2: 1307)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 349520 (174760 pairs)\n",
      "FastQ single records kept: 6093 (from PE1: 4614, from PE2: 1479)\n",
      "FastQ paired records discarded: 1152 (576 pairs)\n",
      "FastQ single records discarded: 6093 (from PE1: 1479, from PE2: 4614)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 171116 (85558 pairs)\n",
      "FastQ single records kept: 7667 (from PE1: 6477, from PE2: 1190)\n",
      "FastQ paired records discarded: 1384 (692 pairs)\n",
      "FastQ single records discarded: 7667 (from PE1: 1190, from PE2: 6477)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 615986 (307993 pairs)\n",
      "FastQ single records kept: 11632 (from PE1: 9430, from PE2: 2202)\n",
      "FastQ paired records discarded: 1850 (925 pairs)\n",
      "FastQ single records discarded: 11632 (from PE1: 2202, from PE2: 9430)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 230998 (115499 pairs)\n",
      "FastQ single records kept: 6624 (from PE1: 5637, from PE2: 987)\n",
      "FastQ paired records discarded: 1072 (536 pairs)\n",
      "FastQ single records discarded: 6624 (from PE1: 987, from PE2: 5637)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 153040 (76520 pairs)\n",
      "FastQ single records kept: 3321 (from PE1: 2806, from PE2: 515)\n",
      "FastQ paired records discarded: 564 (282 pairs)\n",
      "FastQ single records discarded: 3321 (from PE1: 515, from PE2: 2806)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 266490 (133245 pairs)\n",
      "FastQ single records kept: 8203 (from PE1: 6866, from PE2: 1337)\n",
      "FastQ paired records discarded: 1470 (735 pairs)\n",
      "FastQ single records discarded: 8203 (from PE1: 1337, from PE2: 6866)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 174904 (87452 pairs)\n",
      "FastQ single records kept: 6158 (from PE1: 5064, from PE2: 1094)\n",
      "FastQ paired records discarded: 1156 (578 pairs)\n",
      "FastQ single records discarded: 6158 (from PE1: 1094, from PE2: 5064)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 532664 (266332 pairs)\n",
      "FastQ single records kept: 19859 (from PE1: 15968, from PE2: 3891)\n",
      "FastQ paired records discarded: 4106 (2053 pairs)\n",
      "FastQ single records discarded: 19859 (from PE1: 3891, from PE2: 15968)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 3724 (1862 pairs)\n",
      "FastQ single records kept: 285 (from PE1: 204, from PE2: 81)\n",
      "FastQ paired records discarded: 54 (27 pairs)\n",
      "FastQ single records discarded: 285 (from PE1: 81, from PE2: 204)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 251830 (125915 pairs)\n",
      "FastQ single records kept: 3759 (from PE1: 3308, from PE2: 451)\n",
      "FastQ paired records discarded: 488 (244 pairs)\n",
      "FastQ single records discarded: 3759 (from PE1: 451, from PE2: 3308)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 218462 (109231 pairs)\n",
      "FastQ single records kept: 2627 (from PE1: 2328, from PE2: 299)\n",
      "FastQ paired records discarded: 248 (124 pairs)\n",
      "FastQ single records discarded: 2627 (from PE1: 299, from PE2: 2328)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 233770 (116885 pairs)\n",
      "FastQ single records kept: 9166 (from PE1: 7866, from PE2: 1300)\n",
      "FastQ paired records discarded: 1570 (785 pairs)\n",
      "FastQ single records discarded: 9166 (from PE1: 1300, from PE2: 7866)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 173604 (86802 pairs)\n",
      "FastQ single records kept: 4108 (from PE1: 3504, from PE2: 604)\n",
      "FastQ paired records discarded: 638 (319 pairs)\n",
      "FastQ single records discarded: 4108 (from PE1: 604, from PE2: 3504)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 147078 (73539 pairs)\n",
      "FastQ single records kept: 4584 (from PE1: 3815, from PE2: 769)\n",
      "FastQ paired records discarded: 814 (407 pairs)\n",
      "FastQ single records discarded: 4584 (from PE1: 769, from PE2: 3815)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 116786 (58393 pairs)\n",
      "FastQ single records kept: 2159 (from PE1: 1922, from PE2: 237)\n",
      "FastQ paired records discarded: 288 (144 pairs)\n",
      "FastQ single records discarded: 2159 (from PE1: 237, from PE2: 1922)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 189316 (94658 pairs)\n",
      "FastQ single records kept: 3626 (from PE1: 2842, from PE2: 784)\n",
      "FastQ paired records discarded: 618 (309 pairs)\n",
      "FastQ single records discarded: 3626 (from PE1: 784, from PE2: 2842)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 181978 (90989 pairs)\n",
      "FastQ single records kept: 2861 (from PE1: 2465, from PE2: 396)\n",
      "FastQ paired records discarded: 366 (183 pairs)\n",
      "FastQ single records discarded: 2861 (from PE1: 396, from PE2: 2465)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 248264 (124132 pairs)\n",
      "FastQ single records kept: 5134 (from PE1: 4296, from PE2: 838)\n",
      "FastQ paired records discarded: 712 (356 pairs)\n",
      "FastQ single records discarded: 5134 (from PE1: 838, from PE2: 4296)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 137726 (68863 pairs)\n",
      "FastQ single records kept: 4414 (from PE1: 3703, from PE2: 711)\n",
      "FastQ paired records discarded: 812 (406 pairs)\n",
      "FastQ single records discarded: 4414 (from PE1: 711, from PE2: 3703)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 106568 (53284 pairs)\n",
      "FastQ single records kept: 5304 (from PE1: 4138, from PE2: 1166)\n",
      "FastQ paired records discarded: 1040 (520 pairs)\n",
      "FastQ single records discarded: 5304 (from PE1: 1166, from PE2: 4138)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 187976 (93988 pairs)\n",
      "FastQ single records kept: 7834 (from PE1: 6812, from PE2: 1022)\n",
      "FastQ paired records discarded: 1406 (703 pairs)\n",
      "FastQ single records discarded: 7834 (from PE1: 1022, from PE2: 6812)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 202380 (101190 pairs)\n",
      "FastQ single records kept: 2964 (from PE1: 2598, from PE2: 366)\n",
      "FastQ paired records discarded: 344 (172 pairs)\n",
      "FastQ single records discarded: 2964 (from PE1: 366, from PE2: 2598)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 198282 (99141 pairs)\n",
      "FastQ single records kept: 2267 (from PE1: 2041, from PE2: 226)\n",
      "FastQ paired records discarded: 254 (127 pairs)\n",
      "FastQ single records discarded: 2267 (from PE1: 226, from PE2: 2041)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 206278 (103139 pairs)\n",
      "FastQ single records kept: 5097 (from PE1: 4238, from PE2: 859)\n",
      "FastQ paired records discarded: 828 (414 pairs)\n",
      "FastQ single records discarded: 5097 (from PE1: 859, from PE2: 4238)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148056 (74028 pairs)\n",
      "FastQ single records kept: 5760 (from PE1: 4875, from PE2: 885)\n",
      "FastQ paired records discarded: 1030 (515 pairs)\n",
      "FastQ single records discarded: 5760 (from PE1: 885, from PE2: 4875)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 261574 (130787 pairs)\n",
      "FastQ single records kept: 3801 (from PE1: 3244, from PE2: 557)\n",
      "FastQ paired records discarded: 578 (289 pairs)\n",
      "FastQ single records discarded: 3801 (from PE1: 557, from PE2: 3244)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 119008 (59504 pairs)\n",
      "FastQ single records kept: 3393 (from PE1: 2789, from PE2: 604)\n",
      "FastQ paired records discarded: 550 (275 pairs)\n",
      "FastQ single records discarded: 3393 (from PE1: 604, from PE2: 2789)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 264074 (132037 pairs)\n",
      "FastQ single records kept: 11025 (from PE1: 9073, from PE2: 1952)\n",
      "FastQ paired records discarded: 2088 (1044 pairs)\n",
      "FastQ single records discarded: 11025 (from PE1: 1952, from PE2: 9073)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 193702 (96851 pairs)\n",
      "FastQ single records kept: 2719 (from PE1: 2252, from PE2: 467)\n",
      "FastQ paired records discarded: 430 (215 pairs)\n",
      "FastQ single records discarded: 2719 (from PE1: 467, from PE2: 2252)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 209048 (104524 pairs)\n",
      "FastQ single records kept: 2541 (from PE1: 2143, from PE2: 398)\n",
      "FastQ paired records discarded: 380 (190 pairs)\n",
      "FastQ single records discarded: 2541 (from PE1: 398, from PE2: 2143)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 238904 (119452 pairs)\n",
      "FastQ single records kept: 2605 (from PE1: 2320, from PE2: 285)\n",
      "FastQ paired records discarded: 278 (139 pairs)\n",
      "FastQ single records discarded: 2605 (from PE1: 285, from PE2: 2320)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 273938 (136969 pairs)\n",
      "FastQ single records kept: 3694 (from PE1: 3196, from PE2: 498)\n",
      "FastQ paired records discarded: 492 (246 pairs)\n",
      "FastQ single records discarded: 3694 (from PE1: 498, from PE2: 3196)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 321674 (160837 pairs)\n",
      "FastQ single records kept: 9405 (from PE1: 7609, from PE2: 1796)\n",
      "FastQ paired records discarded: 1670 (835 pairs)\n",
      "FastQ single records discarded: 9405 (from PE1: 1796, from PE2: 7609)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 283408 (141704 pairs)\n",
      "FastQ single records kept: 3507 (from PE1: 3057, from PE2: 450)\n",
      "FastQ paired records discarded: 310 (155 pairs)\n",
      "FastQ single records discarded: 3507 (from PE1: 450, from PE2: 3057)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 237344 (118672 pairs)\n",
      "FastQ single records kept: 3383 (from PE1: 2865, from PE2: 518)\n",
      "FastQ paired records discarded: 426 (213 pairs)\n",
      "FastQ single records discarded: 3383 (from PE1: 518, from PE2: 2865)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 184880 (92440 pairs)\n",
      "FastQ single records kept: 14553 (from PE1: 11988, from PE2: 2565)\n",
      "FastQ paired records discarded: 3246 (1623 pairs)\n",
      "FastQ single records discarded: 14553 (from PE1: 2565, from PE2: 11988)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 164702 (82351 pairs)\n",
      "FastQ single records kept: 2266 (from PE1: 2019, from PE2: 247)\n",
      "FastQ paired records discarded: 270 (135 pairs)\n",
      "FastQ single records discarded: 2266 (from PE1: 247, from PE2: 2019)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148986 (74493 pairs)\n",
      "FastQ single records kept: 8016 (from PE1: 6368, from PE2: 1648)\n",
      "FastQ paired records discarded: 1664 (832 pairs)\n",
      "FastQ single records discarded: 8016 (from PE1: 1648, from PE2: 6368)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 194878 (97439 pairs)\n",
      "FastQ single records kept: 5084 (from PE1: 4299, from PE2: 785)\n",
      "FastQ paired records discarded: 866 (433 pairs)\n",
      "FastQ single records discarded: 5084 (from PE1: 785, from PE2: 4299)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 259426 (129713 pairs)\n",
      "FastQ single records kept: 25638 (from PE1: 21459, from PE2: 4179)\n",
      "FastQ paired records discarded: 5012 (2506 pairs)\n",
      "FastQ single records discarded: 25638 (from PE1: 4179, from PE2: 21459)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 206616 (103308 pairs)\n",
      "FastQ single records kept: 8865 (from PE1: 7436, from PE2: 1429)\n",
      "FastQ paired records discarded: 1616 (808 pairs)\n",
      "FastQ single records discarded: 8865 (from PE1: 1429, from PE2: 7436)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 240230 (120115 pairs)\n",
      "FastQ single records kept: 3107 (from PE1: 2613, from PE2: 494)\n",
      "FastQ paired records discarded: 452 (226 pairs)\n",
      "FastQ single records discarded: 3107 (from PE1: 494, from PE2: 2613)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 480336 (240168 pairs)\n",
      "FastQ single records kept: 10148 (from PE1: 7849, from PE2: 2299)\n",
      "FastQ paired records discarded: 1906 (953 pairs)\n",
      "FastQ single records discarded: 10148 (from PE1: 2299, from PE2: 7849)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 9214 (4607 pairs)\n",
      "FastQ single records kept: 1297 (from PE1: 883, from PE2: 414)\n",
      "FastQ paired records discarded: 380 (190 pairs)\n",
      "FastQ single records discarded: 1297 (from PE1: 414, from PE2: 883)\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./ampliconQC.py --data sequences --forward ATGCGTTGGAGAGARCGTTTC --reverse GATCACCTTCTAATTTACCWACAACTG --threads 8 --histograms --qiime"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. Generate sequence counts file used in final step to produce reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!for file in sequences/*.passedQC.fastq; \\\n",
    "do \\\n",
    "  awk 'NR%4==2{sum+=1}END{print FILENAME,sum}' $file >> sequences/diatomSequenceCounts.txt; \\\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4. Assign similar sequences to OTUs using user-defined similarity threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!pick_otus.py -i sequences/readyForQiime.allsamples.fasta -o sequences/picked_otus_97"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  5. Pick a representative set of sequences. For each OTU, one sequence will be used in subsequent analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!pick_rep_set.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -f sequences/readyForQiime.allsamples.fasta \\\n",
    "  -o sequences/repset.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Query BLAST database with OTU representatives"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!blastn -db diatoms -query sequences/repset.fasta \\\n",
    "  -out sequences/repset.diatoms.blastn \\\n",
    "  -task blastn -max_target_seqs 1 -num_threads 8 -outfmt 6 -evalue 0.01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir sequences/assigned_taxonomy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Create taxonomy assignments from BLAST outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!python ./create_taxonomy_assignments_from_blast.py --taxonomy diatoms.taxonomy.FINAL2017.txt \\\n",
    "  --percid 95.0 --blast sequences/repset.diatoms.blastn --output sequences/assigned_taxonomy/repset.taxonomy.txt "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Reports how often an OTU is found in each sample and adds the taxonomic predictions for each OTU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!make_otu_table.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -t sequences/assigned_taxonomy/repset.taxonomy.txt \\\n",
    "  -o sequences/otu_table.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 9. Filters an OTU table based on taxonomic metadata excluding specific taxa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!filter_taxa_from_otu_table.py -i sequences/otu_table.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom \\\n",
    "  -n MARINE,NOT_DIATOM,Yellow_green_Algae,None"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 10. Sort OTU table by sample id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!sort_otu_table.py -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 11. Summary information of the representation of taxonomic groups within each sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!summarize_taxa.py -L 1 \\\n",
    "  -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/visualised_taxonomy -a"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 12. Produce Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!python ./produceDiatomReports.py --folder sequences --lookup lookuptable.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 13. Inspect producted Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.read_csv('sequences/Abundances.fail.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.read_csv('sequences/Abundances.pass.csv')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
