{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Diatom Analysis Pipeline - OTU\n",
    "\n",
    "#### A notebook  for the diatom analysis pipeline using OTU classification."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1. Create BLAST reference database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 06/04/2019 13:06:08\n",
      "New DB name:   /code/diatoms\n",
      "New DB title:  diatoms.sequences.FINAL2017.fasta\n",
      "Sequence type: Nucleotide\n",
      "Keep Linkouts: T\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2701 sequences in 0.198415 seconds.\n",
      "CPU times: user 30 ms, sys: 20 ms, total: 50 ms\n",
      "Wall time: 2.4 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!makeblastdb -in diatoms.sequences.FINAL2017.fasta -out diatoms -dbtype nucl"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Quality Control\n",
    "* *Cutadapt:* Trim primers from sequences\n",
    "* *SicklePE:* Trim off bad quality 3' bases\n",
    "* *Pear:* Merge R1 and R2 reads\n",
    "* *SickleSE:* Remove post-merging bad quality sequences\n",
    "* *Histogram Generation:* Plots sequence length against number of sequences\n",
    "* *QIIME Prep:* Pepare passed QC files for QIIME processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2377.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2377.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.73 s (25 us/read; 2.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  68,270\r\n",
      "Reads with adapters:                     1,448 (2.1%)\r\n",
      "Reads written (passing filters):        68,270 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,434,171 bp\r\n",
      "Total written (filtered):     19,429,790 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1372 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1364 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1066.7\t0\t7\r\n",
      "5\t1\t66.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1346\t1066.7\t0\t1346\r\n",
      "4\t8\t266.7\t0\t8\r\n",
      "5\t7\t66.7\t0\t7\r\n",
      "6\t3\t16.7\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 76 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "73 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1066.7\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t1066.7\t0\t70\r\n",
      "4\t2\t266.7\t0\t2\r\n",
      "5\t1\t66.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2609.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2609.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.03 s (25 us/read; 2.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,996\r\n",
      "Reads with adapters:                     1,399 (1.7%)\r\n",
      "Reads written (passing filters):        79,996 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,108,280 bp\r\n",
      "Total written (filtered):     19,104,047 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1272 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "1250 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1249.9\t0\t17\r\n",
      "4\t5\t312.5\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1235\t1249.9\t0\t1235\r\n",
      "4\t9\t312.5\t0\t9\r\n",
      "5\t6\t78.1\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 127 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "127 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t1249.9\t0\t120\r\n",
      "4\t4\t312.5\t0\t4\r\n",
      "5\t3\t78.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2379.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2379.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.25 s (27 us/read; 2.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,003\r\n",
      "Reads with adapters:                    72,554 (88.5%)\r\n",
      "Reads written (passing filters):        82,003 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,565,013 bp\r\n",
      "Total written (filtered):     17,613,574 bp (90.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 32 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t320.3\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1281.3\t0\t26\r\n",
      "4\t1\t320.3\t0\t1\r\n",
      "5\t1\t80.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 72522 times.\r\n",
      "72472 times, it overlapped the 5' end of a read\r\n",
      "50 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1281.3\t0\t2\r\n",
      "4\t1\t320.3\t0\t1\r\n",
      "20\t6\t0.0\t0\t6\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t2\t0.0\t1\t0 2\r\n",
      "24\t19\t0.0\t1\t12 7\r\n",
      "25\t12\t0.0\t1\t8 4\r\n",
      "26\t7931\t0.0\t1\t44 7887\r\n",
      "27\t63921\t0.0\t1\t57523 6398\r\n",
      "28\t576\t0.0\t1\t0 576\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1281.3\t0\t35\r\n",
      "4\t2\t320.3\t0\t2\r\n",
      "5\t2\t80.1\t0\t2\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "51\t1\t0.0\t1\t1\r\n",
      "59\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "300\t6\t0.0\t1\t5 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2377.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2377.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.60 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  68,270\r\n",
      "Reads with adapters:                    58,169 (85.2%)\r\n",
      "Reads written (passing filters):        68,270 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,445,140 bp\r\n",
      "Total written (filtered):     18,265,737 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55861 times.\r\n",
      "55677 times, it overlapped the 5' end of a read\r\n",
      "184 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t77\t0.0\t0\t77\r\n",
      "21\t55588\t0.0\t0\t55588\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t1066.7\t0\t180\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "48\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2308 times.\r\n",
      "178 times, it overlapped the 5' end of a read\r\n",
      "2130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t176\t1066.7\t0\t176\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1066.7\t0\t45\r\n",
      "4\t2085\t266.7\t0\t2085\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2379.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2379.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.46 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,003\r\n",
      "Reads with adapters:                    68,293 (83.3%)\r\n",
      "Reads written (passing filters):        82,003 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,727,664 bp\r\n",
      "Total written (filtered):     18,514,394 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55410 times.\r\n",
      "55257 times, it overlapped the 5' end of a read\r\n",
      "153 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1281.3\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t55212\t0.0\t0\t55212\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t1281.3\t0\t146\r\n",
      "4\t2\t320.3\t0\t2\r\n",
      "5\t1\t80.1\t0\t1\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12883 times.\r\n",
      "40 times, it overlapped the 5' end of a read\r\n",
      "12843 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1281.3\t0\t34\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1281.3\t0\t34\r\n",
      "4\t12806\t320.3\t0\t12806\r\n",
      "5\t3\t80.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2609.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2609.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.43 s (55 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,996\r\n",
      "Reads with adapters:                    71,517 (89.4%)\r\n",
      "Reads written (passing filters):        79,996 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,007,979 bp\r\n",
      "Total written (filtered):     17,660,003 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62600 times.\r\n",
      "62431 times, it overlapped the 5' end of a read\r\n",
      "169 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t88\t0.0\t0\t88\r\n",
      "21\t62323\t0.0\t0\t62323\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t165\t1249.9\t0\t165\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8917 times.\r\n",
      "69 times, it overlapped the 5' end of a read\r\n",
      "8848 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t1249.9\t0\t66\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1249.9\t0\t58\r\n",
      "4\t8789\t312.5\t0\t8789\r\n",
      "5\t1\t78.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2587.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2587.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.37 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,785\r\n",
      "Reads with adapters:                     4,416 (4.4%)\r\n",
      "Reads written (passing filters):        99,785 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,248,571 bp\r\n",
      "Total written (filtered):     29,177,507 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1860 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1855 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1559.1\t0\t4\r\n",
      "4\t1\t389.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1844\t1559.1\t0\t1844\r\n",
      "4\t3\t389.8\t0\t3\r\n",
      "5\t8\t97.4\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2556 times.\r\n",
      "2409 times, it overlapped the 5' end of a read\r\n",
      "147 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t389.8\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2403\t0.0\t1\t0 2403\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t138\t1559.1\t0\t138\r\n",
      "4\t5\t389.8\t0\t5\r\n",
      "5\t4\t97.4\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2733.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2733.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.29 s (39 us/read; 1.52 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,161\r\n",
      "Reads with adapters:                       735 (1.3%)\r\n",
      "Reads written (passing filters):        58,161 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,045,715 bp\r\n",
      "Total written (filtered):      9,043,226 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 649 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "638 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t908.8\t0\t9\r\n",
      "4\t2\t227.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t621\t908.8\t0\t621\r\n",
      "4\t1\t227.2\t0\t1\r\n",
      "5\t13\t56.8\t0\t13\r\n",
      "6\t2\t14.2\t0\t2\r\n",
      "7\t1\t3.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 86 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "85 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t908.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t908.8\t0\t80\r\n",
      "4\t2\t227.2\t0\t2\r\n",
      "5\t2\t56.8\t0\t2\r\n",
      "242\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2733.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2733.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.63 s (45 us/read; 1.33 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,161\r\n",
      "Reads with adapters:                    21,442 (36.9%)\r\n",
      "Reads written (passing filters):        58,161 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,246,310 bp\r\n",
      "Total written (filtered):      8,903,471 bp (96.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 15353 times.\r\n",
      "15142 times, it overlapped the 5' end of a read\r\n",
      "211 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t46\t0.0\t0\t46\r\n",
      "21\t15089\t0.0\t0\t15089\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t205\t908.8\t0\t205\r\n",
      "4\t4\t227.2\t0\t4\r\n",
      "5\t1\t56.8\t0\t1\r\n",
      "67\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6089 times.\r\n",
      "79 times, it overlapped the 5' end of a read\r\n",
      "6010 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t908.8\t0\t76\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t132\t908.8\t0\t132\r\n",
      "4\t5877\t227.2\t0\t5877\r\n",
      "5\t1\t56.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2587.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2587.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.71 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,785\r\n",
      "Reads with adapters:                    96,664 (96.9%)\r\n",
      "Reads written (passing filters):        99,785 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,298,918 bp\r\n",
      "Total written (filtered):     27,285,239 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 95602 times.\r\n",
      "95585 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t95431\t0.0\t0\t95431\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1559.1\t0\t9\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1062 times.\r\n",
      "103 times, it overlapped the 5' end of a read\r\n",
      "959 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t1559.1\t0\t102\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1559.1\t0\t7\r\n",
      "4\t952\t389.8\t0\t952\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2745.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2745.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.63 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,821\r\n",
      "Reads with adapters:                    88,020 (90.0%)\r\n",
      "Reads written (passing filters):        97,821 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,108,499 bp\r\n",
      "Total written (filtered):     27,268,454 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 87542 times.\r\n",
      "87494 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t55\t0.0\t0\t55\r\n",
      "21\t87434\t0.0\t0\t87434\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1528.5\t0\t44\r\n",
      "5\t1\t95.5\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 478 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "452 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1528.5\t0\t26\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t181\t1528.5\t0\t181\r\n",
      "4\t271\t382.1\t0\t271\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2745.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2745.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.26 s (33 us/read; 1.80 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,821\r\n",
      "Reads with adapters:                    95,941 (98.1%)\r\n",
      "Reads written (passing filters):        97,821 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,077,805 bp\r\n",
      "Total written (filtered):     26,487,290 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 39 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1528.5\t0\t5\r\n",
      "4\t3\t382.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1528.5\t0\t29\r\n",
      "4\t2\t382.1\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 95902 times.\r\n",
      "95875 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1528.5\t0\t2\r\n",
      "4\t3\t382.1\t0\t3\r\n",
      "5\t1\t95.5\t0\t1\r\n",
      "23\t6\t0.0\t1\t4 2\r\n",
      "24\t6\t0.0\t1\t5 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1154\t0.0\t1\t56 1098\r\n",
      "27\t94664\t0.0\t1\t89424 5240\r\n",
      "28\t38\t0.0\t1\t0 38\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1528.5\t0\t9\r\n",
      "5\t8\t95.5\t0\t8\r\n",
      "300\t10\t0.0\t1\t6 4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2790.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2790.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.25 s (27 us/read; 2.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,781\r\n",
      "Reads with adapters:                    80,816 (96.5%)\r\n",
      "Reads written (passing filters):        83,781 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,040,397 bp\r\n",
      "Total written (filtered):     22,856,420 bp (91.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 28 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1309.1\t0\t22\r\n",
      "4\t2\t327.3\t0\t2\r\n",
      "5\t4\t81.8\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 80788 times.\r\n",
      "80758 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1309.1\t0\t1\r\n",
      "5\t1\t81.8\t0\t1\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t5\t0.0\t1\t4 1\r\n",
      "23\t4\t0.0\t1\t0 4\r\n",
      "24\t12\t0.0\t1\t5 7\r\n",
      "25\t11\t0.0\t1\t4 7\r\n",
      "26\t685\t0.0\t1\t64 621\r\n",
      "27\t80023\t0.0\t1\t73394 6629\r\n",
      "28\t13\t0.0\t1\t0 13\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1309.1\t0\t13\r\n",
      "4\t1\t327.3\t0\t1\r\n",
      "5\t2\t81.8\t0\t2\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t13\t0.0\t1\t11 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2790.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2790.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.01 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,781\r\n",
      "Reads with adapters:                    69,448 (82.9%)\r\n",
      "Reads written (passing filters):        83,781 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,045,183 bp\r\n",
      "Total written (filtered):     23,597,581 bp (94.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 68919 times.\r\n",
      "68824 times, it overlapped the 5' end of a read\r\n",
      "95 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t51\t0.0\t0\t51\r\n",
      "21\t68769\t0.0\t0\t68769\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t89\t1309.1\t0\t89\r\n",
      "4\t2\t327.3\t0\t2\r\n",
      "5\t2\t81.8\t0\t2\r\n",
      "7\t1\t5.1\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 529 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "475 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1309.1\t0\t54\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t293\t1309.1\t0\t293\r\n",
      "4\t182\t327.3\t0\t182\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2933.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2933.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.77 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,794\r\n",
      "Reads with adapters:                    66,220 (94.9%)\r\n",
      "Reads written (passing filters):        69,794 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,006,884 bp\r\n",
      "Total written (filtered):     14,806,631 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55037 times.\r\n",
      "54997 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t26\t0.0\t0\t26\r\n",
      "21\t54970\t0.0\t0\t54970\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1090.5\t0\t39\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11183 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "11157 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1090.5\t0\t25\r\n",
      "5\t1\t68.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1090.5\t0\t23\r\n",
      "4\t11133\t272.6\t0\t11133\r\n",
      "248\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2933.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2933.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.38 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,794\r\n",
      "Reads with adapters:                    62,305 (89.3%)\r\n",
      "Reads written (passing filters):        69,794 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,807,847 bp\r\n",
      "Total written (filtered):     14,131,802 bp (89.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 29 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1090.5\t0\t2\r\n",
      "4\t1\t272.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1090.5\t0\t21\r\n",
      "4\t1\t272.6\t0\t1\r\n",
      "5\t3\t68.2\t0\t3\r\n",
      "6\t1\t17.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 62276 times.\r\n",
      "62250 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t6\t0.0\t1\t5 1\r\n",
      "25\t6\t0.0\t1\t4 2\r\n",
      "26\t7028\t0.0\t1\t26 7002\r\n",
      "27\t55099\t0.0\t1\t51797 3302\r\n",
      "28\t108\t0.0\t1\t0 108\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1090.5\t0\t15\r\n",
      "4\t2\t272.6\t0\t2\r\n",
      "5\t2\t68.2\t0\t2\r\n",
      "300\t7\t0.0\t1\t5 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2828.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2828.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.46 s (49 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,297\r\n",
      "Reads with adapters:                    87,605 (97.0%)\r\n",
      "Reads written (passing filters):        90,297 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,304,670 bp\r\n",
      "Total written (filtered):     20,941,606 bp (89.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 36 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1410.9\t0\t2\r\n",
      "4\t3\t352.7\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1410.9\t0\t24\r\n",
      "4\t3\t352.7\t0\t3\r\n",
      "5\t3\t88.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 87569 times.\r\n",
      "87533 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1410.9\t0\t2\r\n",
      "4\t1\t352.7\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "19\t2\t0.0\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t4\t0.0\t1\t1 3\r\n",
      "23\t10\t0.0\t1\t3 7\r\n",
      "24\t8\t0.0\t1\t5 3\r\n",
      "25\t7\t0.0\t1\t2 5\r\n",
      "26\t4852\t0.0\t1\t37 4815\r\n",
      "27\t82611\t0.0\t1\t73680 8931\r\n",
      "28\t32\t0.0\t1\t0 32\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1410.9\t0\t14\r\n",
      "5\t3\t88.2\t0\t3\r\n",
      "54\t1\t0.0\t1\t1\r\n",
      "61\t2\t0.0\t1\t2\r\n",
      "68\t1\t0.0\t1\t1\r\n",
      "75\t1\t0.0\t1\t0 1\r\n",
      "300\t14\t0.0\t1\t8 6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2828.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2828.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.91 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,297\r\n",
      "Reads with adapters:                    75,395 (83.5%)\r\n",
      "Reads written (passing filters):        90,297 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,494,403 bp\r\n",
      "Total written (filtered):     22,014,454 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 69402 times.\r\n",
      "69279 times, it overlapped the 5' end of a read\r\n",
      "123 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "20\t47\t0.0\t0\t47\r\n",
      "21\t69230\t0.0\t0\t69230\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t113\t1410.9\t0\t113\r\n",
      "5\t2\t88.2\t0\t2\r\n",
      "6\t1\t22.0\t0\t1\r\n",
      "7\t3\t5.5\t0\t3\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5993 times.\r\n",
      "41 times, it overlapped the 5' end of a read\r\n",
      "5952 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1410.9\t0\t40\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t387\t1410.9\t0\t387\r\n",
      "4\t5565\t352.7\t0\t5565\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3008.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3008.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.76 s (29 us/read; 2.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,001\r\n",
      "Reads with adapters:                    60,091 (98.5%)\r\n",
      "Reads written (passing filters):        61,001 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,213,856 bp\r\n",
      "Total written (filtered):     16,590,800 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 14 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t953.1\t0\t11\r\n",
      "4\t2\t238.3\t0\t2\r\n",
      "5\t1\t59.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 60077 times.\r\n",
      "60065 times, it overlapped the 5' end of a read\r\n",
      "12 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t953.1\t0\t2\r\n",
      "22\t3\t0.0\t1\t0 3\r\n",
      "23\t1\t0.0\t1\t1\r\n",
      "24\t5\t0.0\t1\t4 1\r\n",
      "25\t7\t0.0\t1\t6 1\r\n",
      "26\t503\t0.0\t1\t29 474\r\n",
      "27\t59510\t0.0\t1\t57393 2117\r\n",
      "28\t34\t0.0\t1\t0 34\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t953.1\t0\t5\r\n",
      "5\t1\t59.6\t0\t1\r\n",
      "300\t6\t0.0\t1\t5 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3008.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3008.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.17 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,001\r\n",
      "Reads with adapters:                    43,410 (71.2%)\r\n",
      "Reads written (passing filters):        61,001 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,236,694 bp\r\n",
      "Total written (filtered):     17,337,558 bp (95.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 42775 times.\r\n",
      "42631 times, it overlapped the 5' end of a read\r\n",
      "144 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "20\t13\t0.0\t0\t13\r\n",
      "21\t42615\t0.0\t0\t42615\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t953.1\t0\t135\r\n",
      "5\t2\t59.6\t0\t2\r\n",
      "7\t2\t3.7\t0\t2\r\n",
      "296\t2\t0.0\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 635 times.\r\n",
      "40 times, it overlapped the 5' end of a read\r\n",
      "595 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t953.1\t0\t40\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t512\t953.1\t0\t512\r\n",
      "4\t83\t238.3\t0\t83\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3068.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3068.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.30 s (30 us/read; 2.00 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  76,778\r\n",
      "Reads with adapters:                     1,937 (2.5%)\r\n",
      "Reads written (passing filters):        76,778 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,944,370 bp\r\n",
      "Total written (filtered):     20,938,476 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1808 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "1799 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1199.7\t0\t4\r\n",
      "4\t4\t299.9\t0\t4\r\n",
      "5\t1\t75.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1762\t1199.7\t0\t1762\r\n",
      "4\t13\t299.9\t0\t13\r\n",
      "5\t21\t75.0\t0\t21\r\n",
      "6\t2\t18.7\t0\t2\r\n",
      "7\t1\t4.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 129 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "125 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t4\t299.9\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t1199.7\t0\t120\r\n",
      "4\t3\t299.9\t0\t3\r\n",
      "5\t1\t75.0\t0\t1\r\n",
      "6\t1\t18.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3068.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3068.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.86 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  76,778\r\n",
      "Reads with adapters:                    65,146 (84.8%)\r\n",
      "Reads written (passing filters):        76,778 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,049,888 bp\r\n",
      "Total written (filtered):     19,747,145 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61563 times.\r\n",
      "61296 times, it overlapped the 5' end of a read\r\n",
      "267 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t76\t0.0\t0\t76\r\n",
      "21\t61199\t0.0\t0\t61199\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t261\t1199.7\t0\t261\r\n",
      "5\t2\t75.0\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3583 times.\r\n",
      "339 times, it overlapped the 5' end of a read\r\n",
      "3244 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t339\t1199.7\t0\t339\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t1199.7\t0\t79\r\n",
      "4\t3164\t299.9\t0\t3164\r\n",
      "5\t1\t75.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3140.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3140.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.37 s (33 us/read; 1.82 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 162,528\r\n",
      "Reads with adapters:                     7,427 (4.6%)\r\n",
      "Reads written (passing filters):       162,528 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,530,912 bp\r\n",
      "Total written (filtered):     42,424,857 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3681 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "3670 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t2539.5\t0\t10\r\n",
      "4\t1\t634.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3640\t2539.5\t0\t3640\r\n",
      "4\t15\t634.9\t0\t15\r\n",
      "5\t14\t158.7\t0\t14\r\n",
      "6\t1\t39.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3746 times.\r\n",
      "3489 times, it overlapped the 5' end of a read\r\n",
      "257 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2539.5\t0\t3\r\n",
      "4\t1\t634.9\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3482\t0.0\t1\t0 3482\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t237\t2539.5\t0\t237\r\n",
      "4\t7\t634.9\t0\t7\r\n",
      "5\t12\t158.7\t0\t12\r\n",
      "62\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3140.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3140.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.66 s (41 us/read; 1.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 162,528\r\n",
      "Reads with adapters:                   153,561 (94.5%)\r\n",
      "Reads written (passing filters):       162,528 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    42,762,919 bp\r\n",
      "Total written (filtered):     39,655,276 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 146559 times.\r\n",
      "146518 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "12\t3\t0.0\t0\t3\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t25\t0.0\t0\t25\r\n",
      "20\t212\t0.0\t0\t212\r\n",
      "21\t146259\t0.0\t0\t146259\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t2539.5\t0\t29\r\n",
      "7\t1\t9.9\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7002 times.\r\n",
      "62 times, it overlapped the 5' end of a read\r\n",
      "6940 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t2539.5\t0\t61\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t2539.5\t0\t17\r\n",
      "4\t6918\t634.9\t0\t6918\r\n",
      "5\t2\t158.7\t0\t2\r\n",
      "6\t3\t39.7\t0\t3\r\n",
      "\r\n",
      "\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 193702 (96851 pairs)\n",
      "FastQ single records kept: 2719 (from PE1: 2252, from PE2: 467)\n",
      "FastQ paired records discarded: 430 (215 pairs)\n",
      "FastQ single records discarded: 2719 (from PE1: 467, from PE2: 2252)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 134916 (67458 pairs)\n",
      "FastQ single records kept: 8258 (from PE1: 7317, from PE2: 941)\n",
      "FastQ paired records discarded: 2124 (1062 pairs)\n",
      "FastQ single records discarded: 8258 (from PE1: 941, from PE2: 7317)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 51332 (25666 pairs)\n",
      "FastQ single records kept: 9827 (from PE1: 8750, from PE2: 1077)\n",
      "FastQ paired records discarded: 45336 (22668 pairs)\n",
      "FastQ single records discarded: 9827 (from PE1: 1077, from PE2: 8750)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 123904 (61952 pairs)\n",
      "FastQ single records kept: 16116 (from PE1: 15064, from PE2: 1052)\n",
      "FastQ paired records discarded: 3856 (1928 pairs)\n",
      "FastQ single records discarded: 16116 (from PE1: 1052, from PE2: 15064)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 121272 (60636 pairs)\n",
      "FastQ single records kept: 8363 (from PE1: 7429, from PE2: 934)\n",
      "FastQ paired records discarded: 1590 (795 pairs)\n",
      "FastQ single records discarded: 8363 (from PE1: 934, from PE2: 7429)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 304714 (152357 pairs)\n",
      "FastQ single records kept: 9043 (from PE1: 6456, from PE2: 2587)\n",
      "FastQ paired records discarded: 2256 (1128 pairs)\n",
      "FastQ single records discarded: 9043 (from PE1: 2587, from PE2: 6456)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 192496 (96248 pairs)\n",
      "FastQ single records kept: 1400 (from PE1: 1110, from PE2: 290)\n",
      "FastQ paired records discarded: 346 (173 pairs)\n",
      "FastQ single records discarded: 1400 (from PE1: 290, from PE2: 1110)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 143818 (71909 pairs)\n",
      "FastQ single records kept: 8960 (from PE1: 8026, from PE2: 934)\n",
      "FastQ paired records discarded: 2268 (1134 pairs)\n",
      "FastQ single records discarded: 8960 (from PE1: 934, from PE2: 8026)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 166864 (83432 pairs)\n",
      "FastQ single records kept: 6155 (from PE1: 5385, from PE2: 770)\n",
      "FastQ paired records discarded: 1420 (710 pairs)\n",
      "FastQ single records discarded: 6155 (from PE1: 770, from PE2: 5385)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 164954 (82477 pairs)\n",
      "FastQ single records kept: 1157 (from PE1: 869, from PE2: 288)\n",
      "FastQ paired records discarded: 294 (147 pairs)\n",
      "FastQ single records discarded: 1157 (from PE1: 288, from PE2: 869)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 124952 (62476 pairs)\n",
      "FastQ single records kept: 5235 (from PE1: 4747, from PE2: 488)\n",
      "FastQ paired records discarded: 1118 (559 pairs)\n",
      "FastQ single records discarded: 5235 (from PE1: 488, from PE2: 4747)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 120232 (60116 pairs)\n",
      "FastQ single records kept: 787 (from PE1: 576, from PE2: 211)\n",
      "FastQ paired records discarded: 196 (98 pairs)\n",
      "FastQ single records discarded: 787 (from PE1: 211, from PE2: 576)\n",
      "\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2587.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2587.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 84%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 92,280 / 96,851 (95.280%)\n",
      "Discarded reads ...................: 0 / 96,851 (0.000%)\n",
      "Not assembled reads ...............: 4,571 / 96,851 (4.720%)\n",
      "Assembled reads file...............: /code/sequences/2587.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2587.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2587.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2587.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3068.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3068.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 62,636 / 67,458 (92.852%)\n",
      "Discarded reads ...................: 0 / 67,458 (0.000%)\n",
      "Not assembled reads ...............: 4,822 / 67,458 (7.148%)\n",
      "Assembled reads file...............: /code/sequences/3068.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3068.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3068.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3068.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2733.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2733.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,125 / 25,666 (86.204%)\n",
      "Discarded reads ...................: 0 / 25,666 (0.000%)\n",
      "Not assembled reads ...............: 3,541 / 25,666 (13.796%)\n",
      "Assembled reads file...............: /code/sequences/2733.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2733.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2733.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2733.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2609.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2609.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 54,952 / 61,952 (88.701%)\n",
      "Discarded reads ...................: 0 / 61,952 (0.000%)\n",
      "Not assembled reads ...............: 7,000 / 61,952 (11.299%)\n",
      "Assembled reads file...............: /code/sequences/2609.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2609.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2609.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2609.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2933.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2933.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 42,875 / 60,636 (70.709%)\n",
      "Discarded reads ...................: 0 / 60,636 (0.000%)\n",
      "Not assembled reads ...............: 17,761 / 60,636 (29.291%)\n",
      "Assembled reads file...............: /code/sequences/2933.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2933.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2933.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2933.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3140.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3140.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 59%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 131,384 / 152,357 (86.234%)\n",
      "Discarded reads ...................: 0 / 152,357 (0.000%)\n",
      "Not assembled reads ...............: 20,973 / 152,357 (13.766%)\n",
      "Assembled reads file...............: /code/sequences/3140.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3140.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3140.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3140.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2745.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2745.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 95%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 83,603 / 96,248 (86.862%)\n",
      "Discarded reads ...................: 0 / 96,248 (0.000%)\n",
      "Not assembled reads ...............: 12,645 / 96,248 (13.138%)\n",
      "Assembled reads file...............: /code/sequences/2745.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2745.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2745.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2745.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2379.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2379.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 53,186 / 71,909 (73.963%)\n",
      "Discarded reads ...................: 0 / 71,909 (0.000%)\n",
      "Not assembled reads ...............: 18,723 / 71,909 (26.037%)\n",
      "Assembled reads file...............: /code/sequences/2379.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2379.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2379.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2379.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2828.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2828.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61,884 / 83,432 (74.173%)\n",
      "Discarded reads ...................: 0 / 83,432 (0.000%)\n",
      "Not assembled reads ...............: 21,548 / 83,432 (25.827%)\n",
      "Assembled reads file...............: /code/sequences/2828.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2828.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2828.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2828.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2790.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2790.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 69,544 / 82,477 (84.319%)\n",
      "Discarded reads ...................: 0 / 82,477 (0.000%)\n",
      "Not assembled reads ...............: 12,933 / 82,477 (15.681%)\n",
      "Assembled reads file...............: /code/sequences/2790.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2790.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2790.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2790.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2377.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2377.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 59,008 / 62,476 (94.449%)\n",
      "Discarded reads ...................: 0 / 62,476 (0.000%)\n",
      "Not assembled reads ...............: 3,468 / 62,476 (5.551%)\n",
      "Assembled reads file...............: /code/sequences/2377.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2377.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2377.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2377.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3008.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3008.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,718 / 60,116 (84.367%)\n",
      "Discarded reads ...................: 0 / 60,116 (0.000%)\n",
      "Not assembled reads ...............: 9,398 / 60,116 (15.633%)\n",
      "Assembled reads file...............: /code/sequences/3008.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3008.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3008.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3008.unassembled.reverse.fastq\n",
      "\n",
      "\n",
      "FastQ records kept: 18733\n",
      "FastQ records discarded: 3392\n",
      "\n",
      "\n",
      "FastQ records kept: 39797\n",
      "FastQ records discarded: 13389\n",
      "\n",
      "\n",
      "FastQ records kept: 48403\n",
      "FastQ records discarded: 6549\n",
      "\n",
      "\n",
      "FastQ records kept: 53122\n",
      "FastQ records discarded: 5886\n",
      "\n",
      "\n",
      "FastQ records kept: 47962\n",
      "FastQ records discarded: 13922\n",
      "\n",
      "\n",
      "FastQ records kept: 32831\n",
      "FastQ records discarded: 10044\n",
      "\n",
      "\n",
      "FastQ records kept: 52919\n",
      "FastQ records discarded: 16625\n",
      "\n",
      "\n",
      "FastQ records kept: 66275\n",
      "FastQ records discarded: 17328\n",
      "\n",
      "\n",
      "FastQ records kept: 90222\n",
      "FastQ records discarded: 2058\n",
      "\n",
      "\n",
      "FastQ records kept: 37453\n",
      "FastQ records discarded: 13265\n",
      "\n",
      "\n",
      "FastQ records kept: 55805\n",
      "FastQ records discarded: 6831\n",
      "\n",
      "\n",
      "FastQ records kept: 125649\n",
      "FastQ records discarded: 5735\n",
      "\n",
      "\n",
      "Histogram /code/sequences/readyForQiime.allsamples.fasta.histogram.svg finished\n",
      "Histogram /code/sequences/2377.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2379.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2587.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2609.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2733.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2745.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2790.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2828.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2933.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3008.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3068.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3140.passedQC.fastq.histogram.svg finished\n",
      "FINISHED Amplicon quality control!\n",
      "CPU times: user 10.1 s, sys: 1.77 s, total: 11.9 s\n",
      "Wall time: 11min 26s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./ampliconQC.py --data sequences --forward ATGCGTTGGAGAGARCGTTTC --reverse GATCACCTTCTAATTTACCWACAACTG --threads 8 --histograms --qiime"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. Generate sequence counts file used in final step to produce reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 40 ms, sys: 40 ms, total: 80 ms\n",
      "Wall time: 4.96 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!for file in sequences/*.passedQC.fastq; \\\n",
    "do \\\n",
    "  awk 'NR%4==2{sum+=1}END{print FILENAME,sum}' $file >> sequences/diatomSequenceCounts.txt; \\\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4. Assign similar sequences to OTUs using user-defined similarity threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/miniconda/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.\n",
      "  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')\n",
      "CPU times: user 2.78 s, sys: 420 ms, total: 3.2 s\n",
      "Wall time: 3min 25s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!pick_otus.py -i sequences/readyForQiime.allsamples.fasta -o sequences/picked_otus_97"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  5. Pick a representative set of sequences. For each OTU, one sequence will be used in subsequent analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 160 ms, sys: 90 ms, total: 250 ms\n",
      "Wall time: 15.5 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!pick_rep_set.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -f sequences/readyForQiime.allsamples.fasta \\\n",
    "  -o sequences/repset.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Query BLAST database with OTU representatives"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 30.2 s, sys: 5.74 s, total: 35.9 s\n",
      "Wall time: 39min 20s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!blastn -db diatoms -query sequences/repset.fasta \\\n",
    "  -out sequences/repset.diatoms.blastn \\\n",
    "  -task blastn -max_target_seqs 1 -num_threads 8 -outfmt 6 -evalue 0.01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir sequences/assigned_taxonomy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Create taxonomy assignments from BLAST outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 0 ns, sys: 30 ms, total: 30 ms\n",
      "Wall time: 2.3 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./create_taxonomy_assignments_from_blast.py --taxonomy diatoms.taxonomy.FINAL2017.txt \\\n",
    "  --percid 95.0 --blast sequences/repset.diatoms.blastn --output sequences/assigned_taxonomy/repset.taxonomy.txt "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Reports how often an OTU is found in each sample and adds the taxonomic predictions for each OTU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 70 ms, sys: 40 ms, total: 110 ms\n",
      "Wall time: 7.27 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!make_otu_table.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -t sequences/assigned_taxonomy/repset.taxonomy.txt \\\n",
    "  -o sequences/otu_table.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 9. Filters an OTU table based on taxonomic metadata excluding specific taxa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 140 ms, sys: 0 ns, total: 140 ms\n",
      "Wall time: 8.55 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!filter_taxa_from_otu_table.py -i sequences/otu_table.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom \\\n",
    "  -n MARINE,NOT_DIATOM,Yellow_green_Algae,None"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 10. Sort OTU table by sample id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 140 ms, sys: 0 ns, total: 140 ms\n",
      "Wall time: 8.88 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!sort_otu_table.py -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 11. Summary information of the representation of taxonomic groups within each sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 130 ms, sys: 30 ms, total: 160 ms\n",
      "Wall time: 11 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!summarize_taxa.py -L 1 \\\n",
    "  -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/visualised_taxonomy -a"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 12. Produce Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reports completed\n",
      "CPU times: user 30 ms, sys: 10 ms, total: 40 ms\n",
      "Wall time: 2.61 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./produceDiatomReports.py --folder sequences --lookup lookuptable.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 13. Inspect producted Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>101210740</th>\n",
       "      <th>101527498</th>\n",
       "      <th>101403272</th>\n",
       "      <th>101563446</th>\n",
       "      <th>101611446</th>\n",
       "      <th>101799933</th>\n",
       "      <th>101353981</th>\n",
       "      <th>101522698</th>\n",
       "      <th>101496665</th>\n",
       "      <th>20560447</th>\n",
       "      <th>101353095</th>\n",
       "      <th>101318072</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [Taxonomy, 101210740, 101527498, 101403272, 101563446, 101611446, 101799933, 101353981, 101522698, 101496665, 20560447, 101353095, 101318072]\n",
       "Index: []"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('sequences/Abundances.fail.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>101210740</th>\n",
       "      <th>101527498</th>\n",
       "      <th>101403272</th>\n",
       "      <th>101563446</th>\n",
       "      <th>101611446</th>\n",
       "      <th>101799933</th>\n",
       "      <th>101353981</th>\n",
       "      <th>101522698</th>\n",
       "      <th>101496665</th>\n",
       "      <th>20560447</th>\n",
       "      <th>101353095</th>\n",
       "      <th>101318072</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [Taxonomy, 101210740, 101527498, 101403272, 101563446, 101611446, 101799933, 101353981, 101522698, 101496665, 20560447, 101353095, 101318072]\n",
       "Index: []"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('sequences/Abundances.pass.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 14. List sequence files used"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2377.R1.fastq.gz  2609.R1.fastq.gz  2790.R1.fastq.gz  3008.R1.fastq.gz\r\n",
      "2377.R2.fastq.gz  2609.R2.fastq.gz  2790.R2.fastq.gz  3008.R2.fastq.gz\r\n",
      "2379.R1.fastq.gz  2733.R1.fastq.gz  2828.R1.fastq.gz  3068.R1.fastq.gz\r\n",
      "2379.R2.fastq.gz  2733.R2.fastq.gz  2828.R2.fastq.gz  3068.R2.fastq.gz\r\n",
      "2587.R1.fastq.gz  2745.R1.fastq.gz  2933.R1.fastq.gz  3140.R1.fastq.gz\r\n",
      "2587.R2.fastq.gz  2745.R2.fastq.gz  2933.R2.fastq.gz  3140.R2.fastq.gz\r\n"
     ]
    }
   ],
   "source": [
    "!ls sequences/raw_data"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
