{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Diatom Analysis Pipeline - OTU\n",
    "\n",
    "#### A notebook  for the diatom analysis pipeline using OTU classification."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1. Create sub-selection of samples for local use"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### All 2016 samples that meet selection criteria"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>FERAID</th>\n",
       "      <th>NGS Run Number</th>\n",
       "      <th>BIOSYS site ID</th>\n",
       "      <th>Season</th>\n",
       "      <th>Dataset</th>\n",
       "      <th>SPT_Name</th>\n",
       "      <th>Conductivity</th>\n",
       "      <th>N_NH4</th>\n",
       "      <th>N_NO3</th>\n",
       "      <th>P_PO4</th>\n",
       "      <th>Alkalinity</th>\n",
       "      <th>pH</th>\n",
       "      <th>TotalRead</th>\n",
       "      <th>NonPlankRead</th>\n",
       "      <th>NoBlast Hit</th>\n",
       "      <th>% No Blast Hit of total count</th>\n",
       "      <th>NGS TDI</th>\n",
       "      <th>Northing</th>\n",
       "      <th>Easting</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2662</td>\n",
       "      <td>Diatom2</td>\n",
       "      <td>53</td>\n",
       "      <td>Spr</td>\n",
       "      <td>2016</td>\n",
       "      <td>MASHAM</td>\n",
       "      <td>175.611222</td>\n",
       "      <td>0.030000</td>\n",
       "      <td>1.022745</td>\n",
       "      <td>0.020203</td>\n",
       "      <td>50.416667</td>\n",
       "      <td>8.047222</td>\n",
       "      <td>30710</td>\n",
       "      <td>30706</td>\n",
       "      <td>12770</td>\n",
       "      <td>41.587963</td>\n",
       "      <td>40.267399</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2854</td>\n",
       "      <td>Diatom4</td>\n",
       "      <td>53</td>\n",
       "      <td>Aut</td>\n",
       "      <td>2016</td>\n",
       "      <td>MASHAM</td>\n",
       "      <td>175.611222</td>\n",
       "      <td>0.030000</td>\n",
       "      <td>1.022745</td>\n",
       "      <td>0.020203</td>\n",
       "      <td>50.416667</td>\n",
       "      <td>8.047222</td>\n",
       "      <td>49744</td>\n",
       "      <td>49696</td>\n",
       "      <td>18357</td>\n",
       "      <td>36.938587</td>\n",
       "      <td>59.602807</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2632</td>\n",
       "      <td>Diatom2</td>\n",
       "      <td>83</td>\n",
       "      <td>Spr</td>\n",
       "      <td>2016</td>\n",
       "      <td>COLNEBRIDGE</td>\n",
       "      <td>323.866253</td>\n",
       "      <td>0.109224</td>\n",
       "      <td>2.365707</td>\n",
       "      <td>0.091403</td>\n",
       "      <td>44.250000</td>\n",
       "      <td>7.882188</td>\n",
       "      <td>22553</td>\n",
       "      <td>21006</td>\n",
       "      <td>1771</td>\n",
       "      <td>8.430924</td>\n",
       "      <td>48.468433</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2700</td>\n",
       "      <td>Diatom3</td>\n",
       "      <td>83</td>\n",
       "      <td>Aut</td>\n",
       "      <td>2016</td>\n",
       "      <td>COLNEBRIDGE</td>\n",
       "      <td>323.866253</td>\n",
       "      <td>0.109224</td>\n",
       "      <td>2.365707</td>\n",
       "      <td>0.091403</td>\n",
       "      <td>44.250000</td>\n",
       "      <td>7.882188</td>\n",
       "      <td>57772</td>\n",
       "      <td>45069</td>\n",
       "      <td>12879</td>\n",
       "      <td>28.576183</td>\n",
       "      <td>71.478087</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2987</td>\n",
       "      <td>Diatom1</td>\n",
       "      <td>110</td>\n",
       "      <td>Spr</td>\n",
       "      <td>2016</td>\n",
       "      <td>HARTLINGTON BRIDGE</td>\n",
       "      <td>113.406089</td>\n",
       "      <td>0.015710</td>\n",
       "      <td>0.523256</td>\n",
       "      <td>0.014135</td>\n",
       "      <td>37.500000</td>\n",
       "      <td>7.857308</td>\n",
       "      <td>27403</td>\n",
       "      <td>27403</td>\n",
       "      <td>15705</td>\n",
       "      <td>57.311243</td>\n",
       "      <td>39.948032</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   FERAID NGS Run Number  BIOSYS site ID Season  Dataset            SPT_Name  \\\n",
       "0    2662        Diatom2              53    Spr     2016              MASHAM   \n",
       "1    2854        Diatom4              53    Aut     2016              MASHAM   \n",
       "2    2632        Diatom2              83    Spr     2016         COLNEBRIDGE   \n",
       "3    2700        Diatom3              83    Aut     2016         COLNEBRIDGE   \n",
       "4    2987        Diatom1             110    Spr     2016  HARTLINGTON BRIDGE   \n",
       "\n",
       "   Conductivity     N_NH4     N_NO3     P_PO4  Alkalinity        pH  \\\n",
       "0    175.611222  0.030000  1.022745  0.020203   50.416667  8.047222   \n",
       "1    175.611222  0.030000  1.022745  0.020203   50.416667  8.047222   \n",
       "2    323.866253  0.109224  2.365707  0.091403   44.250000  7.882188   \n",
       "3    323.866253  0.109224  2.365707  0.091403   44.250000  7.882188   \n",
       "4    113.406089  0.015710  0.523256  0.014135   37.500000  7.857308   \n",
       "\n",
       "   TotalRead  NonPlankRead  NoBlast Hit  % No Blast Hit of total count  \\\n",
       "0      30710         30706        12770                      41.587963   \n",
       "1      49744         49696        18357                      36.938587   \n",
       "2      22553         21006         1771                       8.430924   \n",
       "3      57772         45069        12879                      28.576183   \n",
       "4      27403         27403        15705                      57.311243   \n",
       "\n",
       "     NGS TDI  Northing  Easting  \n",
       "0  40.267399       NaN      NaN  \n",
       "1  59.602807       NaN      NaN  \n",
       "2  48.468433       NaN      NaN  \n",
       "3  71.478087       NaN      NaN  \n",
       "4  39.948032       NaN      NaN  "
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "df = pd.read_csv('2016 Data.csv')\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Sample across the TDI range \n",
    "#### (Actually this is wrong, as we really want to create n samples of sites (each with 4 sequence files)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### We want to have representative set of samples across TDI range. Group continuous TDI values into 10 bins."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "binned = pd.cut(df['NGS TDI'], 10)\n",
    "df['Binned'] = pd.cut(df['NGS TDI'], 10)\n",
    "grouped = df.groupby(df['Binned'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Select n samples from each bin\n",
    "##### (n * 10 * 2 sequence files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "sub_selection = grouped.apply(lambda x: x.sample(n=2, replace=True))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Copy selected forward and reverse read files into working directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('sequences/complete_set/3096.R1.fastq.gz', 'sequences/3096.R1.fastq.gz')\n",
      "('sequences/complete_set/3096.R2.fastq.gz', 'sequences/3096.R2.fastq.gz')\n",
      "('sequences/complete_set/2988.R1.fastq.gz', 'sequences/2988.R1.fastq.gz')\n",
      "('sequences/complete_set/2988.R2.fastq.gz', 'sequences/2988.R2.fastq.gz')\n",
      "('sequences/complete_set/3088.R1.fastq.gz', 'sequences/3088.R1.fastq.gz')\n",
      "('sequences/complete_set/3088.R2.fastq.gz', 'sequences/3088.R2.fastq.gz')\n",
      "('sequences/complete_set/3059.R1.fastq.gz', 'sequences/3059.R1.fastq.gz')\n",
      "('sequences/complete_set/3059.R2.fastq.gz', 'sequences/3059.R2.fastq.gz')\n",
      "('sequences/complete_set/2947.R1.fastq.gz', 'sequences/2947.R1.fastq.gz')\n",
      "('sequences/complete_set/2947.R2.fastq.gz', 'sequences/2947.R2.fastq.gz')\n",
      "('sequences/complete_set/2866.R1.fastq.gz', 'sequences/2866.R1.fastq.gz')\n",
      "('sequences/complete_set/2866.R2.fastq.gz', 'sequences/2866.R2.fastq.gz')\n",
      "('sequences/complete_set/3141.R1.fastq.gz', 'sequences/3141.R1.fastq.gz')\n",
      "('sequences/complete_set/3141.R2.fastq.gz', 'sequences/3141.R2.fastq.gz')\n",
      "('sequences/complete_set/2532.R1.fastq.gz', 'sequences/2532.R1.fastq.gz')\n",
      "('sequences/complete_set/2532.R2.fastq.gz', 'sequences/2532.R2.fastq.gz')\n",
      "('sequences/complete_set/3143.R1.fastq.gz', 'sequences/3143.R1.fastq.gz')\n",
      "('sequences/complete_set/3143.R2.fastq.gz', 'sequences/3143.R2.fastq.gz')\n",
      "('sequences/complete_set/2628.R1.fastq.gz', 'sequences/2628.R1.fastq.gz')\n",
      "('sequences/complete_set/2628.R2.fastq.gz', 'sequences/2628.R2.fastq.gz')\n",
      "('sequences/complete_set/2966.R1.fastq.gz', 'sequences/2966.R1.fastq.gz')\n",
      "('sequences/complete_set/2966.R2.fastq.gz', 'sequences/2966.R2.fastq.gz')\n",
      "('sequences/complete_set/2622.R1.fastq.gz', 'sequences/2622.R1.fastq.gz')\n",
      "('sequences/complete_set/2622.R2.fastq.gz', 'sequences/2622.R2.fastq.gz')\n",
      "('sequences/complete_set/2823.R1.fastq.gz', 'sequences/2823.R1.fastq.gz')\n",
      "('sequences/complete_set/2823.R2.fastq.gz', 'sequences/2823.R2.fastq.gz')\n",
      "('sequences/complete_set/3208.R1.fastq.gz', 'sequences/3208.R1.fastq.gz')\n",
      "('sequences/complete_set/3208.R2.fastq.gz', 'sequences/3208.R2.fastq.gz')\n",
      "('sequences/complete_set/2478.R1.fastq.gz', 'sequences/2478.R1.fastq.gz')\n",
      "('sequences/complete_set/2478.R2.fastq.gz', 'sequences/2478.R2.fastq.gz')\n",
      "('sequences/complete_set/2743.R1.fastq.gz', 'sequences/2743.R1.fastq.gz')\n",
      "('sequences/complete_set/2743.R2.fastq.gz', 'sequences/2743.R2.fastq.gz')\n",
      "('sequences/complete_set/2973.R1.fastq.gz', 'sequences/2973.R1.fastq.gz')\n",
      "('sequences/complete_set/2973.R2.fastq.gz', 'sequences/2973.R2.fastq.gz')\n",
      "('sequences/complete_set/2753.R1.fastq.gz', 'sequences/2753.R1.fastq.gz')\n",
      "('sequences/complete_set/2753.R2.fastq.gz', 'sequences/2753.R2.fastq.gz')\n",
      "('sequences/complete_set/2385.R1.fastq.gz', 'sequences/2385.R1.fastq.gz')\n",
      "('sequences/complete_set/2385.R2.fastq.gz', 'sequences/2385.R2.fastq.gz')\n",
      "('sequences/complete_set/3089.R1.fastq.gz', 'sequences/3089.R1.fastq.gz')\n",
      "('sequences/complete_set/3089.R2.fastq.gz', 'sequences/3089.R2.fastq.gz')\n"
     ]
    }
   ],
   "source": [
    "from shutil import copyfile\n",
    "for index, row in sub_selection.iterrows():\n",
    "    for n in range(1,3):\n",
    "        from_file = \"sequences/complete_set/{}.R{}.fastq.gz\".format(row['FERAID'],n)\n",
    "        to_file = \"sequences/{}.R{}.fastq.gz\".format(row['FERAID'],n)\n",
    "        print(from_file, to_file)\n",
    "        copyfile(from_file, to_file)    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Create BLAST reference database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 06/07/2019 13:53:23\n",
      "New DB name:   /code/diatoms\n",
      "New DB title:  diatoms.sequences.FINAL2017.fasta\n",
      "Sequence type: Nucleotide\n",
      "Keep Linkouts: T\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2701 sequences in 0.1822 seconds.\n",
      "CPU times: user 0 ns, sys: 20 ms, total: 20 ms\n",
      "Wall time: 1.03 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!makeblastdb -in diatoms.sequences.FINAL2017.fasta -out diatoms -dbtype nucl"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. Quality Control\n",
    "* *Cutadapt:* Trim primers from sequences\n",
    "* *SicklePE:* Trim off bad quality 3' bases\n",
    "* *Pear:* Merge R1 and R2 reads\n",
    "* *SickleSE:* Remove post-merging bad quality sequences\n",
    "* *Histogram Generation:* Plots sequence length against number of sequences\n",
    "* *QIIME Prep:* Pepare passed QC files for QIIME processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2385.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2385.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.05 s (23 us/read; 2.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  46,371\r\n",
      "Reads with adapters:                    40,033 (86.3%)\r\n",
      "Reads written (passing filters):        46,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,647,633 bp\r\n",
      "Total written (filtered):     11,568,466 bp (91.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t724.5\t0\t3\r\n",
      "4\t3\t181.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t49\t724.5\t0\t49\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 39978 times.\r\n",
      "39964 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t724.5\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t8\t0.0\t1\t2 6\r\n",
      "25\t9\t0.0\t1\t3 6\r\n",
      "26\t1071\t0.0\t1\t35 1036\r\n",
      "27\t38738\t0.0\t1\t31508 7230\r\n",
      "28\t135\t0.0\t1\t0 135\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t724.5\t0\t7\r\n",
      "4\t2\t181.1\t0\t2\r\n",
      "6\t1\t11.3\t0\t1\r\n",
      "48\t1\t0.0\t1\t0 1\r\n",
      "300\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2385.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2385.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.33 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  46,371\r\n",
      "Reads with adapters:                    38,381 (82.8%)\r\n",
      "Reads written (passing filters):        46,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,695,240 bp\r\n",
      "Total written (filtered):     11,904,660 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 37454 times.\r\n",
      "37445 times, it overlapped the 5' end of a read\r\n",
      "9 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t13\t0.0\t0\t13\r\n",
      "21\t37420\t0.0\t0\t37420\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t724.5\t0\t6\r\n",
      "5\t1\t45.3\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 927 times.\r\n",
      "32 times, it overlapped the 5' end of a read\r\n",
      "895 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t724.5\t0\t32\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t724.5\t0\t22\r\n",
      "4\t873\t181.1\t0\t873\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2478.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2478.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.99 s (20 us/read; 3.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,195\r\n",
      "Reads with adapters:                     2,178 (2.2%)\r\n",
      "Reads written (passing filters):       101,195 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,458,580 bp\r\n",
      "Total written (filtered):     25,451,987 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2044 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2038 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1581.2\t0\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2010\t1581.2\t0\t2010\r\n",
      "4\t11\t395.3\t0\t11\r\n",
      "5\t11\t98.8\t0\t11\r\n",
      "6\t6\t24.7\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 134 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "131 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1581.2\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t126\t1581.2\t0\t126\r\n",
      "4\t3\t395.3\t0\t3\r\n",
      "5\t1\t98.8\t0\t1\r\n",
      "6\t1\t24.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2622.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2622.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.20 s (23 us/read; 2.58 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,727\r\n",
      "Reads with adapters:                    83,833 (88.5%)\r\n",
      "Reads written (passing filters):        94,727 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,919,343 bp\r\n",
      "Total written (filtered):     24,266,030 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77459 times.\r\n",
      "77335 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t54\t0.0\t0\t54\r\n",
      "21\t77276\t0.0\t0\t77276\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t112\t1480.1\t0\t112\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6374 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "6329 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1480.1\t0\t45\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1480.1\t0\t41\r\n",
      "4\t6288\t370.0\t0\t6288\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2622.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2622.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.45 s (26 us/read; 2.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,727\r\n",
      "Reads with adapters:                    90,747 (95.8%)\r\n",
      "Reads written (passing filters):        94,727 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,849,322 bp\r\n",
      "Total written (filtered):     23,399,984 bp (90.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 19 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "18 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1480.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1480.1\t0\t17\r\n",
      "4\t1\t370.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 90728 times.\r\n",
      "90686 times, it overlapped the 5' end of a read\r\n",
      "42 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t370.0\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t3\t0.0\t1\t0 3\r\n",
      "24\t17\t0.0\t1\t11 6\r\n",
      "25\t24\t0.0\t1\t13 11\r\n",
      "26\t4173\t0.0\t1\t73 4100\r\n",
      "27\t86238\t0.0\t1\t81155 5083\r\n",
      "28\t224\t0.0\t1\t0 224\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1480.1\t0\t26\r\n",
      "297\t1\t0.0\t1\t0 1\r\n",
      "300\t15\t0.0\t1\t12 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2478.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2478.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.45 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,195\r\n",
      "Reads with adapters:                    60,143 (59.4%)\r\n",
      "Reads written (passing filters):       101,195 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,484,960 bp\r\n",
      "Total written (filtered):     24,382,981 bp (95.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 51049 times.\r\n",
      "50575 times, it overlapped the 5' end of a read\r\n",
      "474 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t4\t0.0\t0\t4\r\n",
      "20\t58\t0.0\t0\t58\r\n",
      "21\t50508\t0.0\t0\t50508\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t462\t1581.2\t0\t462\r\n",
      "4\t1\t395.3\t0\t1\r\n",
      "7\t1\t6.2\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9094 times.\r\n",
      "120 times, it overlapped the 5' end of a read\r\n",
      "8974 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t1581.2\t0\t120\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t168\t1581.2\t0\t168\r\n",
      "4\t8804\t395.3\t0\t8804\r\n",
      "5\t2\t98.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2628.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2628.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.10 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  63,113\r\n",
      "Reads with adapters:                    57,417 (91.0%)\r\n",
      "Reads written (passing filters):        63,113 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,930,101 bp\r\n",
      "Total written (filtered):     12,009,397 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40693 times.\r\n",
      "40636 times, it overlapped the 5' end of a read\r\n",
      "57 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t3\t0.0\t0\t3\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "20\t22\t0.0\t0\t22\r\n",
      "21\t40604\t0.0\t0\t40604\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t986.1\t0\t55\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16724 times.\r\n",
      "18 times, it overlapped the 5' end of a read\r\n",
      "16706 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t986.1\t0\t15\r\n",
      "7\t1\t3.9\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t986.1\t0\t45\r\n",
      "4\t16661\t246.5\t0\t16661\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2532.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2532.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.73 s (30 us/read; 2.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,979\r\n",
      "Reads with adapters:                     3,027 (1.9%)\r\n",
      "Reads written (passing filters):       159,979 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,581,565 bp\r\n",
      "Total written (filtered):     16,533,467 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1100 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "1093 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2499.7\t0\t6\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1068\t2499.7\t0\t1068\r\n",
      "4\t15\t624.9\t0\t15\r\n",
      "5\t10\t156.2\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1927 times.\r\n",
      "1626 times, it overlapped the 5' end of a read\r\n",
      "301 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t2499.7\t0\t7\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1617\t0.0\t1\t0 1617\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t267\t2499.7\t0\t267\r\n",
      "4\t18\t624.9\t0\t18\r\n",
      "5\t13\t156.2\t0\t13\r\n",
      "6\t1\t39.1\t0\t1\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2628.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2628.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.58 s (41 us/read; 1.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  63,113\r\n",
      "Reads with adapters:                    55,404 (87.8%)\r\n",
      "Reads written (passing filters):        63,113 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,644,393 bp\r\n",
      "Total written (filtered):     11,160,410 bp (88.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 29 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t986.1\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t986.1\t0\t23\r\n",
      "4\t1\t246.5\t0\t1\r\n",
      "5\t2\t61.6\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 55375 times.\r\n",
      "55339 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t3\t0.0\t0\t2 1\r\n",
      "22\t3\t0.0\t1\t1 2\r\n",
      "23\t4\t0.0\t1\t0 4\r\n",
      "24\t5\t0.0\t1\t3 2\r\n",
      "25\t22\t0.0\t1\t5 17\r\n",
      "26\t12393\t0.0\t1\t38 12355\r\n",
      "27\t42707\t0.0\t1\t40777 1930\r\n",
      "28\t201\t0.0\t1\t0 201\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t986.1\t0\t21\r\n",
      "4\t3\t246.5\t0\t3\r\n",
      "5\t3\t61.6\t0\t3\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "57\t1\t0.0\t1\t1\r\n",
      "298\t1\t0.0\t1\t0 1\r\n",
      "300\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2866.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2866.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.37 s (24 us/read; 2.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,207\r\n",
      "Reads with adapters:                    47,247 (82.6%)\r\n",
      "Reads written (passing filters):        57,207 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,234,156 bp\r\n",
      "Total written (filtered):     15,279,531 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45137 times.\r\n",
      "44950 times, it overlapped the 5' end of a read\r\n",
      "187 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t81\t0.0\t0\t81\r\n",
      "21\t44857\t0.0\t0\t44857\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t178\t893.9\t0\t178\r\n",
      "4\t2\t223.5\t0\t2\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2110 times.\r\n",
      "259 times, it overlapped the 5' end of a read\r\n",
      "1851 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t259\t893.9\t0\t259\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t893.9\t0\t35\r\n",
      "4\t1816\t223.5\t0\t1816\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2532.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2532.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.77 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 159,979\r\n",
      "Reads with adapters:                   147,682 (92.3%)\r\n",
      "Reads written (passing filters):       159,979 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,443,374 bp\r\n",
      "Total written (filtered):     15,761,998 bp (90.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64232 times.\r\n",
      "64036 times, it overlapped the 5' end of a read\r\n",
      "196 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t75\t0.0\t0\t75\r\n",
      "21\t63943\t0.0\t0\t63943\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t185\t2499.7\t0\t185\r\n",
      "4\t1\t624.9\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83450 times.\r\n",
      "61 times, it overlapped the 5' end of a read\r\n",
      "83389 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t2499.7\t0\t55\r\n",
      "7\t1\t9.8\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t186\t2499.7\t0\t186\r\n",
      "4\t83197\t624.9\t0\t83197\r\n",
      "5\t6\t156.2\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2743.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2743.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.09 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,256\r\n",
      "Reads with adapters:                    83,575 (98.0%)\r\n",
      "Reads written (passing filters):        85,256 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,064,822 bp\r\n",
      "Total written (filtered):     22,807,418 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 42 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1332.1\t0\t5\r\n",
      "4\t1\t333.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1332.1\t0\t31\r\n",
      "4\t3\t333.0\t0\t3\r\n",
      "5\t1\t83.3\t0\t1\r\n",
      "6\t1\t20.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83533 times.\r\n",
      "83513 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1332.1\t0\t2\r\n",
      "23\t2\t0.0\t1\t2\r\n",
      "24\t22\t0.0\t1\t12 10\r\n",
      "25\t5\t0.0\t1\t4 1\r\n",
      "26\t1106\t0.0\t1\t46 1060\r\n",
      "27\t82346\t0.0\t1\t78886 3460\r\n",
      "28\t30\t0.0\t1\t0 30\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1332.1\t0\t7\r\n",
      "5\t1\t83.3\t0\t1\r\n",
      "300\t12\t0.0\t1\t6 6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2743.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2743.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.19 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,256\r\n",
      "Reads with adapters:                    69,365 (81.4%)\r\n",
      "Reads written (passing filters):        85,256 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,094,682 bp\r\n",
      "Total written (filtered):     23,659,021 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 68189 times.\r\n",
      "68090 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "20\t36\t0.0\t0\t36\r\n",
      "21\t68052\t0.0\t0\t68052\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t90\t1332.1\t0\t90\r\n",
      "5\t1\t83.3\t0\t1\r\n",
      "7\t4\t5.2\t0\t4\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1176 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "1131 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1332.1\t0\t45\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t349\t1332.1\t0\t349\r\n",
      "4\t782\t333.0\t0\t782\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2823.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2823.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.43 s (24 us/read; 2.53 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,378\r\n",
      "Reads with adapters:                   100,559 (98.2%)\r\n",
      "Reads written (passing filters):       102,378 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,701,062 bp\r\n",
      "Total written (filtered):     27,986,125 bp (91.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 27 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1599.7\t0\t2\r\n",
      "4\t1\t399.9\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1599.7\t0\t20\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 100532 times.\r\n",
      "100504 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t13\t399.9\t0\t13\r\n",
      "7\t1\t6.2\t0\t1\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t3\t0.0\t1\t2 1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t19\t0.0\t1\t12 7\r\n",
      "25\t17\t0.0\t1\t10 7\r\n",
      "26\t869\t0.0\t1\t82 787\r\n",
      "27\t99550\t0.0\t1\t94527 5023\r\n",
      "28\t28\t0.0\t1\t0 28\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1599.7\t0\t17\r\n",
      "4\t1\t399.9\t0\t1\r\n",
      "5\t2\t100.0\t0\t2\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "300\t7\t0.0\t1\t6 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2753.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2753.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.67 s (25 us/read; 2.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,485\r\n",
      "Reads with adapters:                   103,976 (98.6%)\r\n",
      "Reads written (passing filters):       105,485 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,571,757 bp\r\n",
      "Total written (filtered):     28,762,946 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 26 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1648.2\t0\t1\r\n",
      "4\t2\t412.1\t0\t2\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1648.2\t0\t16\r\n",
      "4\t2\t412.1\t0\t2\r\n",
      "5\t1\t103.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 103950 times.\r\n",
      "103921 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1648.2\t0\t1\r\n",
      "4\t2\t412.1\t0\t2\r\n",
      "20\t3\t0.0\t0\t3\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t7\t0.0\t1\t3 4\r\n",
      "24\t7\t0.0\t1\t4 3\r\n",
      "25\t22\t0.0\t1\t13 9\r\n",
      "26\t982\t0.0\t1\t66 916\r\n",
      "27\t102851\t0.0\t1\t98479 4372\r\n",
      "28\t40\t0.0\t1\t0 40\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1648.2\t0\t14\r\n",
      "4\t1\t412.1\t0\t1\r\n",
      "5\t1\t103.0\t0\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t11\t0.0\t1\t10 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2753.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2753.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.78 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,485\r\n",
      "Reads with adapters:                    97,847 (92.8%)\r\n",
      "Reads written (passing filters):       105,485 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,583,264 bp\r\n",
      "Total written (filtered):     29,533,735 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97549 times.\r\n",
      "97496 times, it overlapped the 5' end of a read\r\n",
      "53 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "20\t34\t0.0\t0\t34\r\n",
      "21\t97454\t0.0\t0\t97454\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1648.2\t0\t47\r\n",
      "4\t1\t412.1\t0\t1\r\n",
      "5\t1\t103.0\t0\t1\r\n",
      "7\t1\t6.4\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 298 times.\r\n",
      "36 times, it overlapped the 5' end of a read\r\n",
      "262 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1648.2\t0\t32\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t139\t1648.2\t0\t139\r\n",
      "4\t122\t412.1\t0\t122\r\n",
      "5\t1\t103.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2866.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2866.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.47 s (26 us/read; 2.33 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,207\r\n",
      "Reads with adapters:                     1,619 (2.8%)\r\n",
      "Reads written (passing filters):        57,207 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,200,674 bp\r\n",
      "Total written (filtered):     16,195,758 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1528 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1520 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t893.9\t0\t7\r\n",
      "5\t1\t55.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1490\t893.9\t0\t1490\r\n",
      "4\t8\t223.5\t0\t8\r\n",
      "5\t20\t55.9\t0\t20\r\n",
      "6\t2\t14.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 91 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "90 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t893.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t88\t893.9\t0\t88\r\n",
      "4\t1\t223.5\t0\t1\r\n",
      "5\t1\t55.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2823.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2823.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.18 s (51 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,378\r\n",
      "Reads with adapters:                    78,165 (76.3%)\r\n",
      "Reads written (passing filters):       102,378 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,711,255 bp\r\n",
      "Total written (filtered):     29,081,621 bp (94.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77573 times.\r\n",
      "77429 times, it overlapped the 5' end of a read\r\n",
      "144 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t77385\t0.0\t0\t77385\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t129\t1599.7\t0\t129\r\n",
      "4\t2\t399.9\t0\t2\r\n",
      "7\t9\t6.2\t0\t9\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 592 times.\r\n",
      "50 times, it overlapped the 5' end of a read\r\n",
      "542 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1599.7\t0\t43\r\n",
      "27\t7\t0.0\t1\t6 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t472\t1599.7\t0\t472\r\n",
      "4\t70\t399.9\t0\t70\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2966.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2966.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.95 s (36 us/read; 1.67 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,101\r\n",
      "Reads with adapters:                    63,779 (77.7%)\r\n",
      "Reads written (passing filters):        82,101 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,412,060 bp\r\n",
      "Total written (filtered):     19,183,788 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 57409 times.\r\n",
      "57277 times, it overlapped the 5' end of a read\r\n",
      "132 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "20\t33\t0.0\t0\t33\r\n",
      "21\t57237\t0.0\t0\t57237\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t124\t1282.8\t0\t124\r\n",
      "4\t2\t320.7\t0\t2\r\n",
      "5\t1\t80.2\t0\t1\r\n",
      "6\t1\t20.0\t0\t1\r\n",
      "7\t3\t5.0\t0\t3\r\n",
      "8\t1\t1.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6370 times.\r\n",
      "25 times, it overlapped the 5' end of a read\r\n",
      "6345 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1282.8\t0\t25\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t367\t1282.8\t0\t367\r\n",
      "4\t5977\t320.7\t0\t5977\r\n",
      "5\t1\t80.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2947.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2947.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.59 s (37 us/read; 1.60 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,765\r\n",
      "Reads with adapters:                    86,795 (90.6%)\r\n",
      "Reads written (passing filters):        95,765 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,246,110 bp\r\n",
      "Total written (filtered):     16,908,268 bp (87.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1496.3\t0\t1\r\n",
      "4\t3\t374.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1496.3\t0\t27\r\n",
      "4\t3\t374.1\t0\t3\r\n",
      "5\t1\t93.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 86760 times.\r\n",
      "86717 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t374.1\t0\t2\r\n",
      "7\t1\t5.8\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "23\t5\t0.0\t1\t1 4\r\n",
      "24\t4\t0.0\t1\t2 2\r\n",
      "25\t13\t0.0\t1\t5 8\r\n",
      "26\t7475\t0.0\t1\t43 7432\r\n",
      "27\t77970\t0.0\t1\t73395 4575\r\n",
      "28\t1245\t0.0\t1\t0 1245\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1496.3\t0\t23\r\n",
      "4\t2\t374.1\t0\t2\r\n",
      "5\t5\t93.5\t0\t5\r\n",
      "47\t3\t0.0\t1\t3\r\n",
      "51\t1\t0.0\t1\t0 1\r\n",
      "52\t1\t0.0\t1\t1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2966.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2966.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.73 s (33 us/read; 1.80 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,101\r\n",
      "Reads with adapters:                    71,894 (87.6%)\r\n",
      "Reads written (passing filters):        82,101 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,247,492 bp\r\n",
      "Total written (filtered):     18,311,128 bp (90.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 44 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t1\t80.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1282.8\t0\t38\r\n",
      "4\t2\t320.7\t0\t2\r\n",
      "5\t3\t80.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 71850 times.\r\n",
      "71779 times, it overlapped the 5' end of a read\r\n",
      "71 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t1\t80.2\t0\t1\r\n",
      "22\t1\t0.0\t1\t1\r\n",
      "23\t5\t0.0\t1\t3 2\r\n",
      "24\t5\t0.0\t1\t2 3\r\n",
      "25\t21\t0.0\t1\t4 17\r\n",
      "26\t4227\t0.0\t1\t39 4188\r\n",
      "27\t66789\t0.0\t1\t56436 10353\r\n",
      "28\t730\t0.0\t1\t0 730\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1282.8\t0\t47\r\n",
      "4\t4\t320.7\t0\t4\r\n",
      "5\t13\t80.2\t0\t13\r\n",
      "6\t1\t20.0\t0\t1\r\n",
      "62\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3059.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3059.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.46 s (42 us/read; 1.44 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  34,911\r\n",
      "Reads with adapters:                    29,574 (84.7%)\r\n",
      "Reads written (passing filters):        34,911 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,155,927 bp\r\n",
      "Total written (filtered):      9,545,643 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 29042 times.\r\n",
      "28921 times, it overlapped the 5' end of a read\r\n",
      "121 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t545.5\t0\t3\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t6\t0.0\t0\t6\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t28860\t0.0\t0\t28860\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t545.5\t0\t115\r\n",
      "4\t3\t136.4\t0\t3\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 532 times.\r\n",
      "78 times, it overlapped the 5' end of a read\r\n",
      "454 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t545.5\t0\t76\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t545.5\t0\t21\r\n",
      "4\t433\t136.4\t0\t433\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2947.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2947.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.89 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,765\r\n",
      "Reads with adapters:                    89,915 (93.9%)\r\n",
      "Reads written (passing filters):        95,765 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,780,738 bp\r\n",
      "Total written (filtered):     18,124,521 bp (91.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 76343 times.\r\n",
      "76266 times, it overlapped the 5' end of a read\r\n",
      "77 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t76214\t0.0\t0\t76214\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1496.3\t0\t74\r\n",
      "5\t1\t93.5\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 13572 times.\r\n",
      "48 times, it overlapped the 5' end of a read\r\n",
      "13524 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1496.3\t0\t48\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t110\t1496.3\t0\t110\r\n",
      "4\t13411\t374.1\t0\t13411\r\n",
      "5\t3\t93.5\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3059.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3059.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.15 s (33 us/read; 1.82 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  34,911\r\n",
      "Reads with adapters:                       813 (2.3%)\r\n",
      "Reads written (passing filters):        34,911 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,137,164 bp\r\n",
      "Total written (filtered):     10,134,702 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 725 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "717 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t545.5\t0\t7\r\n",
      "4\t1\t136.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t704\t545.5\t0\t704\r\n",
      "4\t7\t136.4\t0\t7\r\n",
      "5\t6\t34.1\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 88 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "86 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t545.5\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t84\t545.5\t0\t84\r\n",
      "4\t1\t136.4\t0\t1\r\n",
      "5\t1\t34.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2973.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2973.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.92 s (37 us/read; 1.62 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,814\r\n",
      "Reads with adapters:                   101,492 (95.9%)\r\n",
      "Reads written (passing filters):       105,814 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,246,410 bp\r\n",
      "Total written (filtered):     28,506,794 bp (91.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 54 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "44 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1653.3\t0\t7\r\n",
      "4\t2\t413.3\t0\t2\r\n",
      "5\t1\t103.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1653.3\t0\t38\r\n",
      "4\t3\t413.3\t0\t3\r\n",
      "5\t2\t103.3\t0\t2\r\n",
      "6\t1\t25.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 101438 times.\r\n",
      "101393 times, it overlapped the 5' end of a read\r\n",
      "45 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1653.3\t0\t2\r\n",
      "4\t2\t413.3\t0\t2\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t2\t0.0\t1\t1 1\r\n",
      "23\t4\t0.0\t1\t2 2\r\n",
      "24\t9\t0.0\t1\t2 7\r\n",
      "25\t25\t0.0\t1\t16 9\r\n",
      "26\t1214\t0.0\t1\t88 1126\r\n",
      "27\t100005\t0.0\t1\t93091 6914\r\n",
      "28\t128\t0.0\t1\t0 128\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1653.3\t0\t26\r\n",
      "5\t8\t103.3\t0\t8\r\n",
      "7\t1\t6.5\t0\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t9\t0.0\t1\t9\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3088.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3088.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.02 s (27 us/read; 2.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,940\r\n",
      "Reads with adapters:                    45,167 (61.1%)\r\n",
      "Reads written (passing filters):        73,940 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,024,332 bp\r\n",
      "Total written (filtered):     18,167,395 bp (95.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40029 times.\r\n",
      "39793 times, it overlapped the 5' end of a read\r\n",
      "236 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1155.3\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t25\t0.0\t0\t25\r\n",
      "21\t39765\t0.0\t0\t39765\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t222\t1155.3\t0\t222\r\n",
      "4\t1\t288.8\t0\t1\r\n",
      "5\t1\t72.2\t0\t1\r\n",
      "7\t9\t4.5\t0\t9\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5138 times.\r\n",
      "41 times, it overlapped the 5' end of a read\r\n",
      "5097 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1155.3\t0\t40\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t839\t1155.3\t0\t839\r\n",
      "4\t4258\t288.8\t0\t4258\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2973.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2973.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.45 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 105,814\r\n",
      "Reads with adapters:                    94,873 (89.7%)\r\n",
      "Reads written (passing filters):       105,814 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,270,808 bp\r\n",
      "Total written (filtered):     29,290,710 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 94110 times.\r\n",
      "94038 times, it overlapped the 5' end of a read\r\n",
      "72 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t41\t0.0\t0\t41\r\n",
      "21\t93980\t0.0\t0\t93980\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t60\t1653.3\t0\t60\r\n",
      "4\t1\t413.3\t0\t1\r\n",
      "7\t3\t6.5\t0\t3\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 763 times.\r\n",
      "44 times, it overlapped the 5' end of a read\r\n",
      "719 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1653.3\t0\t44\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t216\t1653.3\t0\t216\r\n",
      "4\t503\t413.3\t0\t503\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3088.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3088.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.98 s (27 us/read; 2.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,940\r\n",
      "Reads with adapters:                    70,602 (95.5%)\r\n",
      "Reads written (passing filters):        73,940 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,800,845 bp\r\n",
      "Total written (filtered):     16,896,247 bp (89.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 23 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1155.3\t0\t7\r\n",
      "4\t2\t288.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1155.3\t0\t12\r\n",
      "4\t1\t288.8\t0\t1\r\n",
      "5\t1\t72.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 70579 times.\r\n",
      "70563 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "22\t3\t0.0\t1\t1 2\r\n",
      "23\t4\t0.0\t1\t1 3\r\n",
      "24\t11\t0.0\t1\t4 7\r\n",
      "25\t11\t0.0\t1\t6 5\r\n",
      "26\t2972\t0.0\t1\t53 2919\r\n",
      "27\t67074\t0.0\t1\t63960 3114\r\n",
      "28\t488\t0.0\t1\t0 488\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1155.3\t0\t6\r\n",
      "4\t1\t288.8\t0\t1\r\n",
      "5\t2\t72.2\t0\t2\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "300\t6\t0.0\t1\t4 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3089.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3089.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.82 s (22 us/read; 2.69 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,370\r\n",
      "Reads with adapters:                    61,082 (75.1%)\r\n",
      "Reads written (passing filters):        81,370 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,115,972 bp\r\n",
      "Total written (filtered):     17,468,560 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 72 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "55 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1271.4\t0\t10\r\n",
      "4\t6\t317.9\t0\t6\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1271.4\t0\t45\r\n",
      "4\t4\t317.9\t0\t4\r\n",
      "5\t5\t79.5\t0\t5\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 61010 times.\r\n",
      "60971 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1271.4\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t3\t0.0\t1\t1 2\r\n",
      "24\t18\t0.0\t1\t12 6\r\n",
      "25\t5\t0.0\t1\t1 4\r\n",
      "26\t1629\t0.0\t1\t32 1597\r\n",
      "27\t59109\t0.0\t1\t54934 4175\r\n",
      "28\t202\t0.0\t1\t0 202\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1271.4\t0\t25\r\n",
      "4\t2\t317.9\t0\t2\r\n",
      "5\t3\t79.5\t0\t3\r\n",
      "6\t1\t19.9\t0\t1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2988.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2988.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.55 s (36 us/read; 1.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,261\r\n",
      "Reads with adapters:                     7,006 (5.6%)\r\n",
      "Reads written (passing filters):       125,261 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    37,263,506 bp\r\n",
      "Total written (filtered):     37,170,432 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3883 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "3876 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1957.2\t0\t5\r\n",
      "4\t2\t489.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3862\t1957.2\t0\t3862\r\n",
      "4\t9\t489.3\t0\t9\r\n",
      "5\t5\t122.3\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3123 times.\r\n",
      "2990 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1957.2\t0\t1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2986\t0.0\t1\t0 2986\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t1957.2\t0\t128\r\n",
      "5\t4\t122.3\t0\t4\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2988.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2988.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.93 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,261\r\n",
      "Reads with adapters:                   116,914 (93.3%)\r\n",
      "Reads written (passing filters):       125,261 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    37,292,783 bp\r\n",
      "Total written (filtered):     34,840,997 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 116492 times.\r\n",
      "116465 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t28\t0.0\t0\t28\r\n",
      "20\t159\t0.0\t0\t159\r\n",
      "21\t116271\t0.0\t0\t116271\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1957.2\t0\t12\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t4\t0.0\t0\t4\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 422 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "402 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1957.2\t0\t17\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1957.2\t0\t7\r\n",
      "4\t395\t489.3\t0\t395\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3089.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3089.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.41 s (42 us/read; 1.43 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,370\r\n",
      "Reads with adapters:                    55,936 (68.7%)\r\n",
      "Reads written (passing filters):        81,370 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,238,965 bp\r\n",
      "Total written (filtered):     18,103,841 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 53645 times.\r\n",
      "53552 times, it overlapped the 5' end of a read\r\n",
      "93 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1271.4\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "20\t26\t0.0\t0\t26\r\n",
      "21\t53514\t0.0\t0\t53514\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1271.4\t0\t83\r\n",
      "4\t1\t317.9\t0\t1\r\n",
      "5\t2\t79.5\t0\t2\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "127\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2291 times.\r\n",
      "37 times, it overlapped the 5' end of a read\r\n",
      "2254 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1271.4\t0\t35\r\n",
      "4\t1\t317.9\t0\t1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t1271.4\t0\t180\r\n",
      "4\t2072\t317.9\t0\t2072\r\n",
      "5\t2\t79.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3096.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3096.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.18 s (29 us/read; 2.06 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  75,065\r\n",
      "Reads with adapters:                    63,610 (84.7%)\r\n",
      "Reads written (passing filters):        75,065 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,702,770 bp\r\n",
      "Total written (filtered):     20,393,003 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62101 times.\r\n",
      "62002 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1172.9\t0\t1\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t31\t0.0\t0\t31\r\n",
      "21\t61966\t0.0\t0\t61966\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t1172.9\t0\t86\r\n",
      "4\t4\t293.2\t0\t4\r\n",
      "5\t1\t73.3\t0\t1\r\n",
      "7\t2\t4.6\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1509 times.\r\n",
      "35 times, it overlapped the 5' end of a read\r\n",
      "1474 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1172.9\t0\t34\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t322\t1172.9\t0\t322\r\n",
      "4\t1152\t293.2\t0\t1152\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3096.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3096.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.20 s (29 us/read; 2.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  75,065\r\n",
      "Reads with adapters:                    73,425 (97.8%)\r\n",
      "Reads written (passing filters):        75,065 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,643,442 bp\r\n",
      "Total written (filtered):     19,661,222 bp (90.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1172.9\t0\t8\r\n",
      "4\t4\t293.2\t0\t4\r\n",
      "5\t2\t73.3\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1172.9\t0\t22\r\n",
      "4\t1\t293.2\t0\t1\r\n",
      "5\t1\t73.3\t0\t1\r\n",
      "6\t1\t18.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 73385 times.\r\n",
      "73362 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t5\t0.0\t0\t5\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t2\t0.0\t1\t0 2\r\n",
      "24\t9\t0.0\t1\t4 5\r\n",
      "25\t11\t0.0\t1\t5 6\r\n",
      "26\t1767\t0.0\t1\t68 1699\r\n",
      "27\t71446\t0.0\t1\t67602 3844\r\n",
      "28\t120\t0.0\t1\t0 120\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1172.9\t0\t12\r\n",
      "5\t1\t73.3\t0\t1\r\n",
      "297\t1\t0.0\t1\t1\r\n",
      "300\t9\t0.0\t1\t8 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3208.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3208.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.79 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  37,132\r\n",
      "Reads with adapters:                    30,861 (83.1%)\r\n",
      "Reads written (passing filters):        37,132 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,143,480 bp\r\n",
      "Total written (filtered):      9,540,869 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 28172 times.\r\n",
      "28156 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t16\t0.0\t0\t16\r\n",
      "21\t28131\t0.0\t0\t28131\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t580.2\t0\t14\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2689 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "2670 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t580.2\t0\t18\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t580.2\t0\t29\r\n",
      "4\t2641\t145.0\t0\t2641\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.36 s (27 us/read; 2.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                     5,105 (5.9%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,518,505 bp\r\n",
      "Total written (filtered):     25,444,522 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2561 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2553 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1342.8\t0\t7\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2543\t1342.8\t0\t2543\r\n",
      "4\t9\t335.7\t0\t9\r\n",
      "5\t1\t83.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2544 times.\r\n",
      "2445 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1342.8\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2439\t0.0\t1\t0 2439\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t96\t1342.8\t0\t96\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "5\t2\t83.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3208.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3208.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.82 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  37,132\r\n",
      "Reads with adapters:                    30,794 (82.9%)\r\n",
      "Reads written (passing filters):        37,132 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,143,103 bp\r\n",
      "Total written (filtered):      9,314,348 bp (91.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 47 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t580.2\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t580.2\t0\t39\r\n",
      "4\t1\t145.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 30747 times.\r\n",
      "30735 times, it overlapped the 5' end of a read\r\n",
      "12 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t580.2\t0\t1\r\n",
      "23\t2\t0.0\t1\t1 1\r\n",
      "24\t4\t0.0\t1\t1 3\r\n",
      "25\t5\t0.0\t1\t1 4\r\n",
      "26\t1574\t0.0\t1\t12 1562\r\n",
      "27\t29125\t0.0\t1\t23423 5702\r\n",
      "28\t24\t0.0\t1\t0 24\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t580.2\t0\t9\r\n",
      "4\t1\t145.0\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.92 s (46 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                    81,198 (94.5%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,568,415 bp\r\n",
      "Total written (filtered):     23,862,680 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80995 times.\r\n",
      "80966 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t136\t0.0\t0\t136\r\n",
      "21\t80808\t0.0\t0\t80808\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1342.8\t0\t13\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 203 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "174 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1342.8\t0\t28\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1342.8\t0\t6\r\n",
      "4\t168\t335.7\t0\t168\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3143.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3143.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.84 s (32 us/read; 1.86 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,315\r\n",
      "Reads with adapters:                     5,899 (4.9%)\r\n",
      "Reads written (passing filters):       119,315 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,602,624 bp\r\n",
      "Total written (filtered):     35,519,594 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3049 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "3039 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1864.3\t0\t7\r\n",
      "4\t3\t466.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3014\t1864.3\t0\t3014\r\n",
      "4\t17\t466.1\t0\t17\r\n",
      "5\t8\t116.5\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2850 times.\r\n",
      "2721 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t466.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2718\t0.0\t1\t0 2718\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t1864.3\t0\t120\r\n",
      "4\t1\t466.1\t0\t1\r\n",
      "5\t7\t116.5\t0\t7\r\n",
      "6\t1\t29.1\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3143.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3143.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.95 s (41 us/read; 1.45 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 119,315\r\n",
      "Reads with adapters:                   115,121 (96.5%)\r\n",
      "Reads written (passing filters):       119,315 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,648,503 bp\r\n",
      "Total written (filtered):     33,236,209 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 114749 times.\r\n",
      "114730 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1864.3\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t33\t0.0\t0\t33\r\n",
      "20\t146\t0.0\t0\t146\r\n",
      "21\t114538\t0.0\t0\t114538\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1864.3\t0\t12\r\n",
      "6\t1\t29.1\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 372 times.\r\n",
      "164 times, it overlapped the 5' end of a read\r\n",
      "208 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1864.3\t0\t161\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1864.3\t0\t3\r\n",
      "4\t205\t466.1\t0\t205\r\n",
      "\r\n",
      "\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 179706 (89853 pairs)\n",
      "FastQ single records kept: 4472 (from PE1: 3896, from PE2: 576)\n",
      "FastQ paired records discarded: 804 (402 pairs)\n",
      "FastQ single records discarded: 4472 (from PE1: 576, from PE2: 3896)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 151822 (75911 pairs)\n",
      "FastQ single records kept: 5137 (from PE1: 3780, from PE2: 1357)\n",
      "FastQ paired records discarded: 2106 (1053 pairs)\n",
      "FastQ single records discarded: 5137 (from PE1: 1357, from PE2: 3780)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 232102 (116051 pairs)\n",
      "FastQ single records kept: 3036 (from PE1: 2495, from PE2: 541)\n",
      "FastQ paired records discarded: 456 (228 pairs)\n",
      "FastQ single records discarded: 3036 (from PE1: 541, from PE2: 2495)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 243418 (121709 pairs)\n",
      "FastQ single records kept: 3339 (from PE1: 2898, from PE2: 441)\n",
      "FastQ paired records discarded: 426 (213 pairs)\n",
      "FastQ single records discarded: 3339 (from PE1: 441, from PE2: 2898)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 104572 (52286 pairs)\n",
      "FastQ single records kept: 4426 (from PE1: 3977, from PE2: 449)\n",
      "FastQ paired records discarded: 990 (495 pairs)\n",
      "FastQ single records discarded: 4426 (from PE1: 449, from PE2: 3977)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 145722 (72861 pairs)\n",
      "FastQ single records kept: 1873 (from PE1: 1306, from PE2: 567)\n",
      "FastQ paired records discarded: 662 (331 pairs)\n",
      "FastQ single records discarded: 1873 (from PE1: 567, from PE2: 1306)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 98108 (49054 pairs)\n",
      "FastQ single records kept: 13212 (from PE1: 12355, from PE2: 857)\n",
      "FastQ paired records discarded: 1694 (847 pairs)\n",
      "FastQ single records discarded: 13212 (from PE1: 857, from PE2: 12355)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 64764 (32382 pairs)\n",
      "FastQ single records kept: 2262 (from PE1: 1992, from PE2: 270)\n",
      "FastQ paired records discarded: 534 (267 pairs)\n",
      "FastQ single records discarded: 2262 (from PE1: 270, from PE2: 1992)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 166752 (83376 pairs)\n",
      "FastQ single records kept: 16051 (from PE1: 15068, from PE2: 983)\n",
      "FastQ paired records discarded: 3536 (1768 pairs)\n",
      "FastQ single records discarded: 16051 (from PE1: 983, from PE2: 15068)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 166882 (83441 pairs)\n",
      "FastQ single records kept: 2272 (from PE1: 1696, from PE2: 576)\n",
      "FastQ paired records discarded: 448 (224 pairs)\n",
      "FastQ single records discarded: 2272 (from PE1: 576, from PE2: 1696)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 167258 (83629 pairs)\n",
      "FastQ single records kept: 1432 (from PE1: 1186, from PE2: 246)\n",
      "FastQ paired records discarded: 390 (195 pairs)\n",
      "FastQ single records discarded: 1432 (from PE1: 246, from PE2: 1186)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 68904 (34452 pairs)\n",
      "FastQ single records kept: 2301 (from PE1: 2029, from PE2: 272)\n",
      "FastQ paired records discarded: 758 (379 pairs)\n",
      "FastQ single records discarded: 2301 (from PE1: 272, from PE2: 2029)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 86968 (43484 pairs)\n",
      "FastQ single records kept: 2310 (from PE1: 1814, from PE2: 496)\n",
      "FastQ paired records discarded: 1154 (577 pairs)\n",
      "FastQ single records discarded: 2310 (from PE1: 496, from PE2: 1814)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 138612 (69306 pairs)\n",
      "FastQ single records kept: 3826 (from PE1: 2576, from PE2: 1250)\n",
      "FastQ paired records discarded: 1616 (808 pairs)\n",
      "FastQ single records discarded: 3826 (from PE1: 1250, from PE2: 2576)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 207648 (103824 pairs)\n",
      "FastQ single records kept: 1736 (from PE1: 1323, from PE2: 413)\n",
      "FastQ paired records discarded: 508 (254 pairs)\n",
      "FastQ single records discarded: 1736 (from PE1: 413, from PE2: 1323)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 169064 (84532 pairs)\n",
      "FastQ single records kept: 9279 (from PE1: 6681, from PE2: 2598)\n",
      "FastQ paired records discarded: 3908 (1954 pairs)\n",
      "FastQ single records discarded: 9279 (from PE1: 2598, from PE2: 6681)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 124168 (62084 pairs)\n",
      "FastQ single records kept: 2491 (from PE1: 1891, from PE2: 600)\n",
      "FastQ paired records discarded: 33590 (16795 pairs)\n",
      "FastQ single records discarded: 2491 (from PE1: 600, from PE2: 1891)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 202156 (101078 pairs)\n",
      "FastQ single records kept: 1194 (from PE1: 987, from PE2: 207)\n",
      "FastQ paired records discarded: 212 (106 pairs)\n",
      "FastQ single records discarded: 1194 (from PE1: 207, from PE2: 987)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 261878 (130939 pairs)\n",
      "FastQ single records kept: 26378 (from PE1: 21421, from PE2: 4957)\n",
      "FastQ paired records discarded: 5324 (2662 pairs)\n",
      "FastQ single records discarded: 26378 (from PE1: 4957, from PE2: 21421)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 208018 (104009 pairs)\n",
      "FastQ single records kept: 1333 (from PE1: 1102, from PE2: 231)\n",
      "FastQ paired records discarded: 286 (143 pairs)\n",
      "FastQ single records discarded: 1333 (from PE1: 231, from PE2: 1102)\n",
      "\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2622.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2622.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 76,112 / 89,853 (84.707%)\n",
      "Discarded reads ...................: 0 / 89,853 (0.000%)\n",
      "Not assembled reads ...............: 13,741 / 89,853 (15.293%)\n",
      "Assembled reads file...............: /code/sequences/2622.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2622.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2622.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2622.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2988.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2988.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 68%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 115,145 / 121,709 (94.607%)\n",
      "Discarded reads ...................: 0 / 121,709 (0.000%)\n",
      "Not assembled reads ...............: 6,564 / 121,709 (5.393%)\n",
      "Assembled reads file...............: /code/sequences/2988.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2988.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2988.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2988.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2866.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2866.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 48,972 / 52,286 (93.662%)\n",
      "Discarded reads ...................: 0 / 52,286 (0.000%)\n",
      "Not assembled reads ...............: 3,314 / 52,286 (6.338%)\n",
      "Assembled reads file...............: /code/sequences/2866.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2866.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2866.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2866.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3141.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3141.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 99%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 79,733 / 83,441 (95.556%)\n",
      "Discarded reads ...................: 0 / 83,441 (0.000%)\n",
      "Not assembled reads ...............: 3,708 / 83,441 (4.444%)\n",
      "Assembled reads file...............: /code/sequences/3141.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3141.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3141.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3141.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2753.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2753.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 90%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 90,501 / 104,009 (87.013%)\n",
      "Discarded reads ...................: 0 / 104,009 (0.000%)\n",
      "Not assembled reads ...............: 13,508 / 104,009 (12.987%)\n",
      "Assembled reads file...............: /code/sequences/2753.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2753.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2753.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2753.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2628.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2628.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 32,648 / 49,054 (66.555%)\n",
      "Discarded reads ...................: 0 / 49,054 (0.000%)\n",
      "Not assembled reads ...............: 16,406 / 49,054 (33.445%)\n",
      "Assembled reads file...............: /code/sequences/2628.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2628.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2628.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2628.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2973.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2973.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 91%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 89,085 / 103,824 (85.804%)\n",
      "Discarded reads ...................: 0 / 103,824 (0.000%)\n",
      "Not assembled reads ...............: 14,739 / 103,824 (14.196%)\n",
      "Assembled reads file...............: /code/sequences/2973.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2973.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2973.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2973.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3208.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3208.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,775 / 34,452 (89.327%)\n",
      "Discarded reads ...................: 0 / 34,452 (0.000%)\n",
      "Not assembled reads ...............: 3,677 / 34,452 (10.673%)\n",
      "Assembled reads file...............: /code/sequences/3208.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3208.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3208.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3208.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2966.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2966.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,584 / 75,911 (66.636%)\n",
      "Discarded reads ...................: 0 / 75,911 (0.000%)\n",
      "Not assembled reads ...............: 25,327 / 75,911 (33.364%)\n",
      "Assembled reads file...............: /code/sequences/2966.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2966.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2966.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2966.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2532.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2532.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,766 / 130,939 (27.315%)\n",
      "Discarded reads ...................: 0 / 130,939 (0.000%)\n",
      "Not assembled reads ...............: 95,173 / 130,939 (72.685%)\n",
      "Assembled reads file...............: /code/sequences/2532.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2532.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2532.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2532.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2743.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2743.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 71,309 / 83,629 (85.268%)\n",
      "Discarded reads ...................: 0 / 83,629 (0.000%)\n",
      "Not assembled reads ...............: 12,320 / 83,629 (14.732%)\n",
      "Assembled reads file...............: /code/sequences/2743.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2743.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2743.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2743.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3143.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3143.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 69%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 111,913 / 116,051 (96.434%)\n",
      "Discarded reads ...................: 0 / 116,051 (0.000%)\n",
      "Not assembled reads ...............: 4,138 / 116,051 (3.566%)\n",
      "Assembled reads file...............: /code/sequences/3143.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3143.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3143.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3143.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2385.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2385.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,063 / 43,484 (87.533%)\n",
      "Discarded reads ...................: 0 / 43,484 (0.000%)\n",
      "Not assembled reads ...............: 5,421 / 43,484 (12.467%)\n",
      "Assembled reads file...............: /code/sequences/2385.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2385.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2385.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2385.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3088.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3088.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 51,144 / 69,306 (73.794%)\n",
      "Discarded reads ...................: 0 / 69,306 (0.000%)\n",
      "Not assembled reads ...............: 18,162 / 69,306 (26.206%)\n",
      "Assembled reads file...............: /code/sequences/3088.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3088.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3088.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3088.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3059.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3059.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,432 / 32,382 (93.978%)\n",
      "Discarded reads ...................: 0 / 32,382 (0.000%)\n",
      "Not assembled reads ...............: 1,950 / 32,382 (6.022%)\n",
      "Assembled reads file...............: /code/sequences/3059.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3059.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3059.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3059.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2947.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2947.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 47,969 / 84,532 (56.747%)\n",
      "Discarded reads ...................: 0 / 84,532 (0.000%)\n",
      "Not assembled reads ...............: 36,563 / 84,532 (43.253%)\n",
      "Assembled reads file...............: /code/sequences/2947.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2947.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2947.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2947.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3096.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3096.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,867 / 72,861 (80.794%)\n",
      "Discarded reads ...................: 0 / 72,861 (0.000%)\n",
      "Not assembled reads ...............: 13,994 / 72,861 (19.206%)\n",
      "Assembled reads file...............: /code/sequences/3096.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3096.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3096.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3096.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3089.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3089.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 52,091 / 62,084 (83.904%)\n",
      "Discarded reads ...................: 0 / 62,084 (0.000%)\n",
      "Not assembled reads ...............: 9,993 / 62,084 (16.096%)\n",
      "Assembled reads file...............: /code/sequences/3089.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3089.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3089.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3089.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2478.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2478.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 98%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 75,935 / 83,376 (91.075%)\n",
      "Discarded reads ...................: 0 / 83,376 (0.000%)\n",
      "Not assembled reads ...............: 7,441 / 83,376 (8.925%)\n",
      "Assembled reads file...............: /code/sequences/2478.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2478.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2478.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2478.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2823.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2823.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 91%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 88,209 / 101,078 (87.268%)\n",
      "Discarded reads ...................: 0 / 101,078 (0.000%)\n",
      "Not assembled reads ...............: 12,869 / 101,078 (12.732%)\n",
      "Assembled reads file...............: /code/sequences/2823.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2823.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2823.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2823.unassembled.reverse.fastq\n",
      "\n",
      "\n",
      "FastQ records kept: 23102\n",
      "FastQ records discarded: 12664\n",
      "\n",
      "\n",
      "FastQ records kept: 26114\n",
      "FastQ records discarded: 6534\n",
      "\n",
      "\n",
      "FastQ records kept: 31356\n",
      "FastQ records discarded: 6707\n",
      "\n",
      "\n",
      "FastQ records kept: 56129\n",
      "FastQ records discarded: 15180\n",
      "\n",
      "\n",
      "FastQ records kept: 57267\n",
      "FastQ records discarded: 18845\n",
      "\n",
      "\n",
      "FastQ records kept: 34319\n",
      "FastQ records discarded: 13650\n",
      "\n",
      "\n",
      "FastQ records kept: 43740\n",
      "FastQ records discarded: 5232\n",
      "\n",
      "\n",
      "FastQ records kept: 67567\n",
      "FastQ records discarded: 8368\n",
      "\n",
      "\n",
      "FastQ records kept: 36838\n",
      "FastQ records discarded: 13746\n",
      "\n",
      "\n",
      "FastQ records kept: 69614\n",
      "FastQ records discarded: 18595\n",
      "\n",
      "\n",
      "FastQ records kept: 72314\n",
      "FastQ records discarded: 18187\n",
      "\n",
      "\n",
      "FastQ records kept: 27429\n",
      "FastQ records discarded: 3003\n",
      "\n",
      "\n",
      "FastQ records kept: 36178\n",
      "FastQ records discarded: 14966\n",
      "\n",
      "\n",
      "FastQ records kept: 39959\n",
      "FastQ records discarded: 12132\n",
      "\n",
      "\n",
      "FastQ records kept: 25318\n",
      "FastQ records discarded: 5457\n",
      "\n",
      "\n",
      "FastQ records kept: 41921\n",
      "FastQ records discarded: 16946\n",
      "\n",
      "\n",
      "FastQ records kept: 70597\n",
      "FastQ records discarded: 18488\n",
      "\n",
      "\n",
      "FastQ records kept: 77948\n",
      "FastQ records discarded: 1785\n",
      "\n",
      "\n",
      "FastQ records kept: 113653\n",
      "FastQ records discarded: 1492\n",
      "\n",
      "\n",
      "FastQ records kept: 110133\n",
      "FastQ records discarded: 1780\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Histogram /code/sequences/readyForQiime.allsamples.fasta.histogram.svg finished\n",
      "Histogram /code/sequences/3141.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3059.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2966.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2973.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2385.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2532.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2988.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3143.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3208.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2866.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3096.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2823.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2753.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2628.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2947.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2478.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3088.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2622.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3089.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2743.passedQC.fastq.histogram.svg finished\n",
      "FINISHED Amplicon quality control!\n",
      "CPU times: user 16 s, sys: 3.78 s, total: 19.7 s\n",
      "Wall time: 18min 8s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./ampliconQC.py --data sequences --forward ATGCGTTGGAGAGARCGTTTC --reverse GATCACCTTCTAATTTACCWACAACTG --threads 8 --histograms --qiime"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4. Generate sequence counts file used in final step to produce reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 40 ms, sys: 20 ms, total: 60 ms\n",
      "Wall time: 2.96 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!for file in sequences/*.passedQC.fastq; \\\n",
    "do \\\n",
    "  awk 'NR%4==2{sum+=1}END{print FILENAME,sum}' $file >> sequences/diatomSequenceCounts.txt; \\\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5. Assign similar sequences to OTUs using user-defined similarity threshold\n",
    "#### Default clustering algorithm is UCLUST, O(m\\*n), which is a greedy algorithm and is dependent on ordering of sequences in readyForQiime.allsamples.fasta file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/miniconda/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.\n",
      "  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')\n",
      "CPU times: user 8.83 s, sys: 1.78 s, total: 10.6 s\n",
      "Wall time: 10min 44s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!pick_otus.py -i sequences/readyForQiime.allsamples.fasta -o sequences/picked_otus_97"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  6. Pick a representative set of sequences. For each OTU, one sequence will be used in subsequent analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 360 ms, sys: 100 ms, total: 460 ms\n",
      "Wall time: 27 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!pick_rep_set.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -f sequences/readyForQiime.allsamples.fasta \\\n",
    "  -o sequences/repset.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Query BLAST database with OTU representatives"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 58.5 s, sys: 10.3 s, total: 1min 8s\n",
      "Wall time: 1h 12min 34s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!blastn -db diatoms -query sequences/repset.fasta \\\n",
    "  -out sequences/repset.diatoms.blastn \\\n",
    "  -task blastn -max_target_seqs 1 -num_threads 8 -outfmt 6 -evalue 0.01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir sequences/assigned_taxonomy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Create taxonomy assignments from BLAST outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 0 ns, sys: 30 ms, total: 30 ms\n",
      "Wall time: 1.89 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./create_taxonomy_assignments_from_blast.py --taxonomy diatoms.taxonomy.FINAL2017.txt \\\n",
    "  --percid 95.0 --blast sequences/repset.diatoms.blastn --output sequences/assigned_taxonomy/repset.taxonomy.txt "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 9. Report how often an OTU is found in each sample and add the taxonomic predictions for each OTU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 210 ms, sys: 20 ms, total: 230 ms\n",
      "Wall time: 14.2 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!make_otu_table.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -t sequences/assigned_taxonomy/repset.taxonomy.txt \\\n",
    "  -o sequences/otu_table.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 10. Filter the OTU table based on taxonomic metadata excluding specific taxa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 190 ms, sys: 70 ms, total: 260 ms\n",
      "Wall time: 16.5 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!filter_taxa_from_otu_table.py -i sequences/otu_table.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom \\\n",
    "  -n MARINE,NOT_DIATOM,Yellow_green_Algae,None"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 11. Sort OTU table by sample id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 220 ms, sys: 50 ms, total: 270 ms\n",
      "Wall time: 15.7 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!sort_otu_table.py -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 12. Provide summary information of the representation of taxonomic groups within each sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 300 ms, sys: 50 ms, total: 350 ms\n",
      "Wall time: 21.3 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!summarize_taxa.py -L 1 \\\n",
    "  -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/visualised_taxonomy -a"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 13. Produce Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reports completed\n",
      "CPU times: user 0 ns, sys: 20 ms, total: 20 ms\n",
      "Wall time: 1.41 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./produceDiatomReports.py --folder sequences --lookup lookuptable.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 13. Inspect producted Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>101522615</th>\n",
       "      <th>101608272</th>\n",
       "      <th>101247590</th>\n",
       "      <th>100966229</th>\n",
       "      <th>100775150</th>\n",
       "      <th>101481781</th>\n",
       "      <th>101493216</th>\n",
       "      <th>101611476</th>\n",
       "      <th>101607102</th>\n",
       "      <th>...</th>\n",
       "      <th>101712111</th>\n",
       "      <th>101365185</th>\n",
       "      <th>100985450</th>\n",
       "      <th>101537286</th>\n",
       "      <th>101712096</th>\n",
       "      <th>100757042</th>\n",
       "      <th>101712098</th>\n",
       "      <th>101560493</th>\n",
       "      <th>101280443</th>\n",
       "      <th>101581652</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>0 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [Taxonomy, 101522615, 101608272, 101247590, 100966229, 100775150, 101481781, 101493216, 101611476, 101607102, 101306260, 101712111, 101365185, 100985450, 101537286, 101712096, 100757042, 101712098, 101560493, 101280443, 101581652]\n",
       "Index: []\n",
       "\n",
       "[0 rows x 21 columns]"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('sequences/Abundances.fail.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Taxonomy</th>\n",
       "      <th>100966229</th>\n",
       "      <th>101712111</th>\n",
       "      <th>101280443</th>\n",
       "      <th>101712098</th>\n",
       "      <th>101607102</th>\n",
       "      <th>101560493</th>\n",
       "      <th>101712096</th>\n",
       "      <th>100775150</th>\n",
       "      <th>101581652</th>\n",
       "      <th>...</th>\n",
       "      <th>101306260</th>\n",
       "      <th>101481781</th>\n",
       "      <th>100985450</th>\n",
       "      <th>101522615</th>\n",
       "      <th>101493216</th>\n",
       "      <th>101537286</th>\n",
       "      <th>101247590</th>\n",
       "      <th>101365185</th>\n",
       "      <th>100757042</th>\n",
       "      <th>101611476</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>AC023A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>18.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>12.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>AC143A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>827.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3799.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>5336.0</td>\n",
       "      <td>166.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>...</td>\n",
       "      <td>771.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>606.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>38.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AC166A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ACHD-02</td>\n",
       "      <td>844.0</td>\n",
       "      <td>22.0</td>\n",
       "      <td>1352.0</td>\n",
       "      <td>282.0</td>\n",
       "      <td>1552.0</td>\n",
       "      <td>701.0</td>\n",
       "      <td>52.0</td>\n",
       "      <td>42.0</td>\n",
       "      <td>23.0</td>\n",
       "      <td>...</td>\n",
       "      <td>127.0</td>\n",
       "      <td>335.0</td>\n",
       "      <td>674.0</td>\n",
       "      <td>216.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>124.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>58.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ACHD-09</td>\n",
       "      <td>0.0</td>\n",
       "      <td>40.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>ACHD-11</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>ACHD-12</td>\n",
       "      <td>434.0</td>\n",
       "      <td>82.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>74.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>195.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>76.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>ADLA-01</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ADLA-03</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>ADLA-05</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>AM001A</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>68.0</td>\n",
       "      <td>34.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>183.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>AM012B</td>\n",
       "      <td>269.0</td>\n",
       "      <td>727.0</td>\n",
       "      <td>44.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>105.0</td>\n",
       "      <td>95.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>116.0</td>\n",
       "      <td>...</td>\n",
       "      <td>30.0</td>\n",
       "      <td>948.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>6884.0</td>\n",
       "      <td>1315.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>601.0</td>\n",
       "      <td>872.0</td>\n",
       "      <td>255.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>AM9999</td>\n",
       "      <td>15.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>...</td>\n",
       "      <td>4.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>32.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>110.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>AMPH-09</td>\n",
       "      <td>15.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>31.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>...</td>\n",
       "      <td>14.0</td>\n",
       "      <td>357.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>123.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>63.0</td>\n",
       "      <td>29.0</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>AMPH-10</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>AMPH-15</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>11.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>AP001A</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>37.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>AS001A</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>26.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>AU002A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>AU003D</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>12.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>51.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>AU020A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>76.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>758.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>AU9999</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>BR001A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>BR010A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>88.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>BREB-01</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>370.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>CA035A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>16.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>CC001A</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>CC002A</td>\n",
       "      <td>10.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>CI002A</td>\n",
       "      <td>30.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>CI004A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>7.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>SU004A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>SU005C</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>SU024A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>SU073A</td>\n",
       "      <td>278.0</td>\n",
       "      <td>41.0</td>\n",
       "      <td>2910.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>23.0</td>\n",
       "      <td>552.0</td>\n",
       "      <td>400.0</td>\n",
       "      <td>143.0</td>\n",
       "      <td>132.0</td>\n",
       "      <td>...</td>\n",
       "      <td>73.0</td>\n",
       "      <td>113.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>169.0</td>\n",
       "      <td>449.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>45.0</td>\n",
       "      <td>498.0</td>\n",
       "      <td>15.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>SU074A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>202.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>SU9999</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>24.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>38.0</td>\n",
       "      <td>1954.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>102.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>SURI-01</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>SURI-06</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>SURI-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>TA001A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>186.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>225.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>179.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>53.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>TE9999</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>TF003A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>11.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>TF9999</td>\n",
       "      <td>12.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>119.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>TH031A</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>46.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>23.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>THAL-01</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>96.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>95.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>72.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>THAL-12</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>ULNA-01</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>125.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>...</td>\n",
       "      <td>2.0</td>\n",
       "      <td>28.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>78.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>ULNA-02</td>\n",
       "      <td>193.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>81.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>137.0</td>\n",
       "      <td>11762.0</td>\n",
       "      <td>627.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>168.0</td>\n",
       "      <td>...</td>\n",
       "      <td>3467.0</td>\n",
       "      <td>3061.0</td>\n",
       "      <td>5716.0</td>\n",
       "      <td>107.0</td>\n",
       "      <td>57.0</td>\n",
       "      <td>594.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>334.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1112.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>YH001A</td>\n",
       "      <td>0.0</td>\n",
       "      <td>19.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>40.0</td>\n",
       "      <td>29.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>277.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>ZZZ842</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>17.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>12.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>ZZZ845</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>ZZZ871</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>16.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>ZZZ896</td>\n",
       "      <td>352.0</td>\n",
       "      <td>224.0</td>\n",
       "      <td>137.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>199.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>20.0</td>\n",
       "      <td>20.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>12.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>6.0</td>\n",
       "      <td>12.0</td>\n",
       "      <td>40.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>ZZZ897</td>\n",
       "      <td>5582.0</td>\n",
       "      <td>355.0</td>\n",
       "      <td>1425.0</td>\n",
       "      <td>50.0</td>\n",
       "      <td>25.0</td>\n",
       "      <td>225.0</td>\n",
       "      <td>409.0</td>\n",
       "      <td>682.0</td>\n",
       "      <td>46.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>95.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>29.0</td>\n",
       "      <td>124.0</td>\n",
       "      <td>21.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>52.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>833.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>ZZZ900</td>\n",
       "      <td>9.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>16.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>ZZZ910</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>14.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>104.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>29.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>ZZZ922</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>ZZZ930</td>\n",
       "      <td>67.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>88.0</td>\n",
       "      <td>10.0</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>ZZZ959</td>\n",
       "      <td>2168.0</td>\n",
       "      <td>98.0</td>\n",
       "      <td>13737.0</td>\n",
       "      <td>602.0</td>\n",
       "      <td>6866.0</td>\n",
       "      <td>3724.0</td>\n",
       "      <td>875.0</td>\n",
       "      <td>101.0</td>\n",
       "      <td>209.0</td>\n",
       "      <td>...</td>\n",
       "      <td>8349.0</td>\n",
       "      <td>814.0</td>\n",
       "      <td>13155.0</td>\n",
       "      <td>584.0</td>\n",
       "      <td>328.0</td>\n",
       "      <td>472.0</td>\n",
       "      <td>8.0</td>\n",
       "      <td>310.0</td>\n",
       "      <td>143.0</td>\n",
       "      <td>61.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>ZZZ985</td>\n",
       "      <td>2.0</td>\n",
       "      <td>46.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>227 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Taxonomy  100966229  101712111  101280443  101712098  101607102  \\\n",
       "0     AC023A        0.0        0.0        0.0        0.0        0.0   \n",
       "1     AC143A        0.0      827.0        1.0     3799.0        0.0   \n",
       "2     AC166A        0.0        0.0        1.0        0.0        0.0   \n",
       "3    ACHD-02      844.0       22.0     1352.0      282.0     1552.0   \n",
       "4    ACHD-09        0.0       40.0        0.0        0.0        0.0   \n",
       "5    ACHD-11        1.0        2.0        4.0        0.0        0.0   \n",
       "6    ACHD-12      434.0       82.0        7.0       74.0        0.0   \n",
       "7    ADLA-01        5.0        0.0        1.0        0.0        0.0   \n",
       "8    ADLA-03        2.0        0.0        0.0        0.0        0.0   \n",
       "9    ADLA-05        0.0        0.0        1.0        0.0        0.0   \n",
       "10    AM001A        1.0        0.0        0.0        0.0        0.0   \n",
       "11    AM012B      269.0      727.0       44.0        0.0        9.0   \n",
       "12    AM9999       15.0        2.0        0.0        0.0        0.0   \n",
       "13   AMPH-09       15.0        0.0        3.0        0.0        0.0   \n",
       "14   AMPH-10        0.0        0.0        0.0        0.0        0.0   \n",
       "15   AMPH-15        0.0        0.0        0.0        0.0        0.0   \n",
       "16    AP001A        1.0        0.0        0.0        0.0       37.0   \n",
       "17    AS001A        3.0        0.0       13.0        0.0        0.0   \n",
       "18    AU002A        0.0        0.0        0.0        0.0        0.0   \n",
       "19    AU003D        0.0        0.0        0.0        0.0       12.0   \n",
       "20    AU020A        0.0        0.0        2.0        0.0        0.0   \n",
       "21    AU9999        1.0        0.0        0.0        0.0        0.0   \n",
       "22    BR001A        0.0        0.0        0.0        0.0        0.0   \n",
       "23    BR010A        0.0        0.0        0.0        5.0        0.0   \n",
       "24   BREB-01        3.0        0.0        0.0        0.0        0.0   \n",
       "25    CA035A        0.0        0.0        0.0        2.0        0.0   \n",
       "26    CC001A        1.0        0.0        0.0        0.0        0.0   \n",
       "27    CC002A       10.0        0.0        0.0        0.0        0.0   \n",
       "28    CI002A       30.0        6.0        0.0        0.0        0.0   \n",
       "29    CI004A        0.0        0.0        0.0        0.0        0.0   \n",
       "..       ...        ...        ...        ...        ...        ...   \n",
       "197   SU004A        0.0        0.0        0.0        0.0        0.0   \n",
       "198   SU005C        0.0        0.0        0.0        0.0        0.0   \n",
       "199   SU024A        0.0        0.0        0.0        0.0        0.0   \n",
       "200   SU073A      278.0       41.0     2910.0        0.0       23.0   \n",
       "201   SU074A        0.0        0.0        0.0        0.0        0.0   \n",
       "202   SU9999        0.0        0.0        0.0        0.0        0.0   \n",
       "203  SURI-01        0.0        0.0        0.0       13.0        0.0   \n",
       "204  SURI-06        0.0        0.0        0.0        0.0        0.0   \n",
       "205  SURI-07        0.0        0.0        0.0        0.0        0.0   \n",
       "206   TA001A        0.0       19.0        2.0      186.0        0.0   \n",
       "207   TE9999        0.0        0.0        0.0        4.0        0.0   \n",
       "208   TF003A        0.0        0.0        0.0        0.0        0.0   \n",
       "209   TF9999       12.0        0.0        0.0        0.0        0.0   \n",
       "210   TH031A        4.0        0.0        0.0        0.0       46.0   \n",
       "211  THAL-01        0.0        0.0        0.0        0.0        0.0   \n",
       "212  THAL-12        3.0        0.0        0.0        0.0        0.0   \n",
       "213  ULNA-01        6.0        0.0        6.0        0.0      125.0   \n",
       "214  ULNA-02      193.0        1.0       81.0        0.0      137.0   \n",
       "215   YH001A        0.0       19.0        1.0        0.0        0.0   \n",
       "216   ZZZ842        0.0        0.0        0.0        0.0       21.0   \n",
       "217   ZZZ845        1.0        0.0        1.0        0.0        0.0   \n",
       "218   ZZZ871        1.0        0.0       16.0        0.0        0.0   \n",
       "219   ZZZ896      352.0      224.0      137.0        0.0        0.0   \n",
       "220   ZZZ897     5582.0      355.0     1425.0       50.0       25.0   \n",
       "221   ZZZ900        9.0        2.0        0.0        0.0        1.0   \n",
       "222   ZZZ910        0.0        0.0       14.0        0.0        0.0   \n",
       "223   ZZZ922        0.0        0.0        2.0        0.0        0.0   \n",
       "224   ZZZ930       67.0        0.0        3.0        0.0       88.0   \n",
       "225   ZZZ959     2168.0       98.0    13737.0      602.0     6866.0   \n",
       "226   ZZZ985        2.0       46.0        0.0        0.0        0.0   \n",
       "\n",
       "     101560493  101712096  100775150  101581652    ...      101306260  \\\n",
       "0          0.0        0.0        0.0        1.0    ...            0.0   \n",
       "1          7.0     5336.0      166.0        7.0    ...          771.0   \n",
       "2          2.0        0.0        0.0        0.0    ...            0.0   \n",
       "3        701.0       52.0       42.0       23.0    ...          127.0   \n",
       "4          0.0        0.0        0.0        0.0    ...            0.0   \n",
       "5          1.0        0.0        0.0        0.0    ...            0.0   \n",
       "6        195.0        0.0       76.0        0.0    ...            0.0   \n",
       "7          2.0        5.0        0.0        0.0    ...            0.0   \n",
       "8          2.0        0.0        0.0        2.0    ...            0.0   \n",
       "9          0.0       14.0        0.0        0.0    ...            0.0   \n",
       "10         0.0        2.0        0.0        1.0    ...            1.0   \n",
       "11       105.0       95.0        2.0      116.0    ...           30.0   \n",
       "12         0.0        3.0        0.0        2.0    ...            4.0   \n",
       "13         2.0       31.0        1.0        5.0    ...           14.0   \n",
       "14         1.0        0.0        0.0        0.0    ...            0.0   \n",
       "15         2.0        0.0        0.0        0.0    ...            0.0   \n",
       "16         0.0        0.0        0.0        5.0    ...            0.0   \n",
       "17         4.0       17.0        0.0        0.0    ...            0.0   \n",
       "18         0.0        9.0        0.0        0.0    ...            0.0   \n",
       "19         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "20         6.0       76.0        0.0       15.0    ...            0.0   \n",
       "21         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "22         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "23         0.0       88.0        0.0        0.0    ...            0.0   \n",
       "24         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "25         0.0       16.0        0.0        0.0    ...            0.0   \n",
       "26         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "27         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "28         0.0        0.0        0.0        0.0    ...            0.0   \n",
       "29         0.0        0.0        0.0        7.0    ...            0.0   \n",
       "..         ...        ...        ...        ...    ...            ...   \n",
       "197        0.0        0.0        0.0        3.0    ...            0.0   \n",
       "198        0.0        1.0        0.0        0.0    ...            0.0   \n",
       "199        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "200      552.0      400.0      143.0      132.0    ...           73.0   \n",
       "201        0.0        0.0        0.0        3.0    ...            0.0   \n",
       "202        0.0       24.0        0.0        6.0    ...            0.0   \n",
       "203        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "204        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "205        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "206        3.0      225.0        1.0        3.0    ...            0.0   \n",
       "207        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "208        0.0        0.0        0.0        4.0    ...            0.0   \n",
       "209        0.0       19.0        0.0        0.0    ...            0.0   \n",
       "210        1.0        0.0        0.0        0.0    ...            0.0   \n",
       "211        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "212        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "213        4.0        1.0        0.0       21.0    ...            2.0   \n",
       "214    11762.0      627.0        1.0      168.0    ...         3467.0   \n",
       "215       40.0       29.0        0.0        0.0    ...            0.0   \n",
       "216       17.0        1.0        0.0        0.0    ...            0.0   \n",
       "217        0.0        0.0        0.0        2.0    ...            1.0   \n",
       "218        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "219      199.0       21.0       20.0       20.0    ...            0.0   \n",
       "220      225.0      409.0      682.0       46.0    ...            0.0   \n",
       "221        1.0        5.0        0.0        1.0    ...           16.0   \n",
       "222      104.0        0.0        0.0        0.0    ...            0.0   \n",
       "223        1.0        0.0        0.0        1.0    ...            0.0   \n",
       "224       10.0        4.0        0.0        1.0    ...            3.0   \n",
       "225     3724.0      875.0      101.0      209.0    ...         8349.0   \n",
       "226        0.0        0.0        0.0        0.0    ...            0.0   \n",
       "\n",
       "     101481781  100985450  101522615  101493216  101537286  101247590  \\\n",
       "0          0.0        0.0       18.0        2.0        0.0        0.0   \n",
       "1          1.0        1.0        0.0        0.0      606.0        0.0   \n",
       "2          0.0        0.0        0.0        1.0        0.0        0.0   \n",
       "3        335.0      674.0      216.0       19.0      124.0        4.0   \n",
       "4          0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "5          0.0        4.0        0.0        0.0        0.0        0.0   \n",
       "6          0.0       17.0        1.0        0.0        1.0        0.0   \n",
       "7          0.0        0.0        0.0        0.0        3.0        0.0   \n",
       "8          0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "9          0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "10         0.0        0.0       68.0       34.0        0.0        0.0   \n",
       "11       948.0        5.0     6884.0     1315.0        0.0        3.0   \n",
       "12         7.0        0.0       32.0        5.0        0.0        0.0   \n",
       "13       357.0        0.0      123.0        1.0        0.0        0.0   \n",
       "14         2.0        0.0        1.0        0.0        0.0        0.0   \n",
       "15         0.0        0.0       11.0        0.0        0.0        0.0   \n",
       "16         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "17         0.0        0.0        0.0       26.0        0.0        0.0   \n",
       "18         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "19         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "20         0.0        0.0        0.0      758.0        0.0        0.0   \n",
       "21         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "22         0.0        2.0        0.0        0.0        0.0        0.0   \n",
       "23         0.0        7.0        0.0        0.0        0.0        0.0   \n",
       "24         0.0        0.0        0.0      370.0        0.0        0.0   \n",
       "25         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "26         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "27         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "28         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "29         0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "..         ...        ...        ...        ...        ...        ...   \n",
       "197        0.0        0.0        0.0        1.0        0.0        0.0   \n",
       "198        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "199        0.0        0.0        1.0        0.0        0.0        0.0   \n",
       "200      113.0        1.0        3.0      169.0      449.0        2.0   \n",
       "201        0.0        0.0        0.0      202.0        0.0        0.0   \n",
       "202        0.0        0.0       38.0     1954.0        0.0        0.0   \n",
       "203        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "204        0.0        0.0        1.0        1.0        0.0        0.0   \n",
       "205        0.0        0.0        0.0        1.0        0.0        0.0   \n",
       "206        0.0      179.0        0.0        0.0       53.0        0.0   \n",
       "207        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "208        0.0        0.0        0.0        3.0        0.0        0.0   \n",
       "209        0.0        0.0        9.0        0.0        0.0        0.0   \n",
       "210        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "211       96.0        0.0        0.0        5.0        0.0        0.0   \n",
       "212        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "213       28.0        2.0        0.0        1.0        0.0        0.0   \n",
       "214     3061.0     5716.0      107.0       57.0      594.0        1.0   \n",
       "215        0.0        0.0        0.0        0.0      277.0        0.0   \n",
       "216        0.0       12.0        0.0        0.0        6.0        0.0   \n",
       "217        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "218        0.0        1.0        0.0        0.0        0.0        0.0   \n",
       "219        0.0        1.0       12.0       14.0        0.0        5.0   \n",
       "220       95.0       10.0       29.0      124.0       21.0        1.0   \n",
       "221        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "222        0.0        1.0        0.0        0.0       29.0        0.0   \n",
       "223        0.0        0.0        1.0        1.0        0.0        3.0   \n",
       "224        0.0        0.0        1.0        2.0        1.0        0.0   \n",
       "225      814.0    13155.0      584.0      328.0      472.0        8.0   \n",
       "226        0.0        0.0        0.0        0.0        0.0        0.0   \n",
       "\n",
       "     101365185  100757042  101611476  \n",
       "0          0.0       12.0        0.0  \n",
       "1          3.0        0.0       38.0  \n",
       "2          0.0        0.0        0.0  \n",
       "3         58.0        5.0       16.0  \n",
       "4          0.0        0.0        0.0  \n",
       "5          0.0        0.0        0.0  \n",
       "6          0.0        0.0        0.0  \n",
       "7          0.0        0.0        0.0  \n",
       "8          0.0        0.0        0.0  \n",
       "9          0.0        0.0        0.0  \n",
       "10         0.0      183.0        0.0  \n",
       "11       601.0      872.0      255.0  \n",
       "12         4.0      110.0        0.0  \n",
       "13        63.0       29.0        4.0  \n",
       "14         2.0        2.0        0.0  \n",
       "15         1.0        3.0        0.0  \n",
       "16         0.0        0.0        0.0  \n",
       "17         0.0        0.0        0.0  \n",
       "18         0.0        0.0        0.0  \n",
       "19         0.0       51.0        0.0  \n",
       "20         0.0        7.0        0.0  \n",
       "21         0.0        0.0        0.0  \n",
       "22         0.0        0.0        0.0  \n",
       "23         0.0        0.0        0.0  \n",
       "24         0.0        8.0        0.0  \n",
       "25         0.0        0.0        0.0  \n",
       "26         0.0        7.0        0.0  \n",
       "27         0.0        0.0        0.0  \n",
       "28         0.0        0.0        0.0  \n",
       "29         0.0        0.0        1.0  \n",
       "..         ...        ...        ...  \n",
       "197        0.0        0.0        0.0  \n",
       "198        0.0        0.0        0.0  \n",
       "199        0.0        0.0        0.0  \n",
       "200       45.0      498.0       15.0  \n",
       "201        0.0        0.0        0.0  \n",
       "202        0.0      102.0        0.0  \n",
       "203        0.0        0.0        0.0  \n",
       "204        0.0        0.0        0.0  \n",
       "205        0.0        0.0        0.0  \n",
       "206        0.0        0.0        0.0  \n",
       "207        0.0        5.0        0.0  \n",
       "208        0.0       11.0        0.0  \n",
       "209        1.0      119.0        0.0  \n",
       "210        0.0       23.0        0.0  \n",
       "211       95.0        1.0       72.0  \n",
       "212        0.0        0.0        0.0  \n",
       "213       78.0        0.0        6.0  \n",
       "214      334.0        1.0     1112.0  \n",
       "215        0.0        0.0        6.0  \n",
       "216        0.0        0.0        0.0  \n",
       "217        0.0        0.0        1.0  \n",
       "218        0.0        0.0        0.0  \n",
       "219        6.0       12.0       40.0  \n",
       "220       52.0        0.0      833.0  \n",
       "221        0.0        9.0        0.0  \n",
       "222        1.0        0.0        0.0  \n",
       "223        0.0        0.0        0.0  \n",
       "224        0.0        1.0        8.0  \n",
       "225      310.0      143.0       61.0  \n",
       "226        0.0        0.0        0.0  \n",
       "\n",
       "[227 rows x 21 columns]"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('sequences/Abundances.pass.csv')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
