{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Diatom Analysis Pipeline - OTU\n",
    "\n",
    "#### A notebook  for the diatom analysis pipeline using OTU classification."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1. Create BLAST reference database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 06/25/2019 13:29:11\n",
      "New DB name:   /code/diatoms\n",
      "New DB title:  diatoms.sequences.FINAL2017.fasta\n",
      "Sequence type: Nucleotide\n",
      "Keep Linkouts: T\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2701 sequences in 0.189666 seconds.\n",
      "CPU times: user 20 ms, sys: 0 ns, total: 20 ms\n",
      "Wall time: 1.04 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!makeblastdb -in diatoms.sequences.FINAL2017.fasta -out diatoms -dbtype nucl"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Quality Control\n",
    "* *Cutadapt:* Trim primers from sequences\n",
    "* *SicklePE:* Trim off bad quality 3' bases\n",
    "* *Pear:* Merge R1 and R2 reads\n",
    "* *SickleSE:* Remove post-merging bad quality sequences\n",
    "* *Histogram Generation:* Plots sequence length against number of sequences\n",
    "* *QIIME Prep:* Pepare passed QC files for QIIME processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2381.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2381.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.86 s (24 us/read; 2.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  35,894\r\n",
      "Reads with adapters:                    31,336 (87.3%)\r\n",
      "Reads written (passing filters):        35,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,357,358 bp\r\n",
      "Total written (filtered):      9,511,755 bp (91.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 26 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t560.8\t0\t2\r\n",
      "4\t2\t140.2\t0\t2\r\n",
      "5\t1\t35.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t560.8\t0\t21\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 31310 times.\r\n",
      "31302 times, it overlapped the 5' end of a read\r\n",
      "8 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t5\t0.0\t1\t2 3\r\n",
      "25\t2\t0.0\t1\t1 1\r\n",
      "26\t579\t0.0\t1\t32 547\r\n",
      "27\t30670\t0.0\t1\t25079 5591\r\n",
      "28\t46\t0.0\t1\t0 46\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t560.8\t0\t3\r\n",
      "5\t2\t35.1\t0\t2\r\n",
      "300\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2426.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2426.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.09 s (21 us/read; 2.80 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  51,003\r\n",
      "Reads with adapters:                       991 (1.9%)\r\n",
      "Reads written (passing filters):        51,003 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,754,702 bp\r\n",
      "Total written (filtered):     13,751,697 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 925 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "924 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t796.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t907\t796.9\t0\t907\r\n",
      "4\t5\t199.2\t0\t5\r\n",
      "5\t11\t49.8\t0\t11\r\n",
      "6\t1\t12.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 66 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "65 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t796.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t796.9\t0\t63\r\n",
      "4\t2\t199.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2425.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2425.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.37 s (22 us/read; 2.69 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,428\r\n",
      "Reads with adapters:                     1,504 (2.4%)\r\n",
      "Reads written (passing filters):        61,428 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,772,972 bp\r\n",
      "Total written (filtered):     16,768,366 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1240 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1236 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t959.8\t0\t2\r\n",
      "4\t2\t240.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1213\t959.8\t0\t1213\r\n",
      "4\t10\t240.0\t0\t10\r\n",
      "5\t8\t60.0\t0\t8\r\n",
      "6\t5\t15.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 264 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "264 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t257\t959.8\t0\t257\r\n",
      "4\t1\t240.0\t0\t1\r\n",
      "5\t3\t60.0\t0\t3\r\n",
      "6\t2\t15.0\t0\t2\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2381.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2381.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.82 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  35,894\r\n",
      "Reads with adapters:                    30,837 (85.9%)\r\n",
      "Reads written (passing filters):        35,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,371,907 bp\r\n",
      "Total written (filtered):      9,727,893 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 30620 times.\r\n",
      "30609 times, it overlapped the 5' end of a read\r\n",
      "11 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t13\t0.0\t0\t13\r\n",
      "21\t30586\t0.0\t0\t30586\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t560.8\t0\t8\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 217 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "200 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t560.8\t0\t17\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t560.8\t0\t11\r\n",
      "4\t189\t140.2\t0\t189\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2397.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2397.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.06 s (26 us/read; 2.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,290\r\n",
      "Reads with adapters:                    75,921 (95.8%)\r\n",
      "Reads written (passing filters):        79,290 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,088,090 bp\r\n",
      "Total written (filtered):     20,038,557 bp (90.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 20 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1238.9\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t16\t1238.9\t0\t16\r\n",
      "5\t1\t77.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 75901 times.\r\n",
      "75873 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1238.9\t0\t1\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "23\t9\t0.0\t1\t6 3\r\n",
      "24\t23\t0.0\t1\t14 9\r\n",
      "25\t7\t0.0\t1\t3 4\r\n",
      "26\t3061\t0.0\t1\t52 3009\r\n",
      "27\t72573\t0.0\t1\t66718 5855\r\n",
      "28\t193\t0.0\t1\t0 193\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1238.9\t0\t14\r\n",
      "5\t1\t77.4\t0\t1\r\n",
      "300\t13\t0.0\t1\t7 6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2426.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2426.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.54 s (50 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  51,003\r\n",
      "Reads with adapters:                    44,874 (88.0%)\r\n",
      "Reads written (passing filters):        51,003 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,773,439 bp\r\n",
      "Total written (filtered):     12,882,889 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41863 times.\r\n",
      "41754 times, it overlapped the 5' end of a read\r\n",
      "109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t59\t0.0\t0\t59\r\n",
      "21\t41680\t0.0\t0\t41680\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t796.9\t0\t102\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "57\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3011 times.\r\n",
      "145 times, it overlapped the 5' end of a read\r\n",
      "2866 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t144\t796.9\t0\t144\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t796.9\t0\t28\r\n",
      "4\t2836\t199.2\t0\t2836\r\n",
      "5\t2\t49.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2425.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2425.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.05 s (50 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,428\r\n",
      "Reads with adapters:                    43,134 (70.2%)\r\n",
      "Reads written (passing filters):        61,428 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,835,267 bp\r\n",
      "Total written (filtered):     15,998,696 bp (95.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 39346 times.\r\n",
      "39097 times, it overlapped the 5' end of a read\r\n",
      "249 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t959.8\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t6\t0.0\t0\t6\r\n",
      "20\t50\t0.0\t0\t50\r\n",
      "21\t39035\t0.0\t0\t39035\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t247\t959.8\t0\t247\r\n",
      "4\t1\t240.0\t0\t1\r\n",
      "105\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3788 times.\r\n",
      "272 times, it overlapped the 5' end of a read\r\n",
      "3516 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t271\t959.8\t0\t271\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t125\t959.8\t0\t125\r\n",
      "4\t3390\t240.0\t0\t3390\r\n",
      "5\t1\t60.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2431.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2431.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.10 s (25 us/read; 2.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,138\r\n",
      "Reads with adapters:                     2,310 (2.8%)\r\n",
      "Reads written (passing filters):        83,138 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,706,581 bp\r\n",
      "Total written (filtered):     24,699,624 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2250 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2247 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t324.8\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2235\t1299.0\t0\t2235\r\n",
      "4\t4\t324.8\t0\t4\r\n",
      "5\t5\t81.2\t0\t5\r\n",
      "6\t3\t20.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 60 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "59 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1299.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1299.0\t0\t58\r\n",
      "4\t1\t324.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2397.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2397.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.75 s (47 us/read; 1.27 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,290\r\n",
      "Reads with adapters:                    75,820 (95.6%)\r\n",
      "Reads written (passing filters):        79,290 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,138,723 bp\r\n",
      "Total written (filtered):     20,616,396 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 71679 times.\r\n",
      "71630 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t68\t0.0\t0\t68\r\n",
      "21\t71556\t0.0\t0\t71556\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1238.9\t0\t42\r\n",
      "7\t1\t4.8\t0\t1\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4141 times.\r\n",
      "50 times, it overlapped the 5' end of a read\r\n",
      "4091 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1238.9\t0\t50\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1238.9\t0\t18\r\n",
      "4\t4073\t309.7\t0\t4073\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2455.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2455.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.35 s (22 us/read; 2.73 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,333\r\n",
      "Reads with adapters:                     1,228 (2.0%)\r\n",
      "Reads written (passing filters):        61,333 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,228,772 bp\r\n",
      "Total written (filtered):     17,225,056 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1158 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1156 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t239.6\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1139\t958.3\t0\t1139\r\n",
      "4\t11\t239.6\t0\t11\r\n",
      "5\t5\t59.9\t0\t5\r\n",
      "6\t1\t15.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 70 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t239.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t958.3\t0\t66\r\n",
      "4\t1\t239.6\t0\t1\r\n",
      "5\t2\t59.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2431.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2431.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.86 s (46 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,138\r\n",
      "Reads with adapters:                    65,216 (78.4%)\r\n",
      "Reads written (passing filters):        83,138 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,705,760 bp\r\n",
      "Total written (filtered):     23,349,572 bp (94.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64648 times.\r\n",
      "64378 times, it overlapped the 5' end of a read\r\n",
      "270 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1299.0\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t19\t0.0\t0\t19\r\n",
      "20\t92\t0.0\t0\t92\r\n",
      "21\t64258\t0.0\t0\t64258\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t263\t1299.0\t0\t263\r\n",
      "4\t1\t324.8\t0\t1\r\n",
      "5\t1\t81.2\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 568 times.\r\n",
      "98 times, it overlapped the 5' end of a read\r\n",
      "470 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t97\t1299.0\t0\t97\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t68\t1299.0\t0\t68\r\n",
      "4\t402\t324.8\t0\t402\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2455.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2455.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.11 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,333\r\n",
      "Reads with adapters:                    37,243 (60.7%)\r\n",
      "Reads written (passing filters):        61,333 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,227,519 bp\r\n",
      "Total written (filtered):     16,479,352 bp (95.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35437 times.\r\n",
      "35174 times, it overlapped the 5' end of a read\r\n",
      "263 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t6\t0.0\t0\t6\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t35117\t0.0\t0\t35117\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t255\t958.3\t0\t255\r\n",
      "4\t1\t239.6\t0\t1\r\n",
      "5\t1\t59.9\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1806 times.\r\n",
      "73 times, it overlapped the 5' end of a read\r\n",
      "1733 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t958.3\t0\t73\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t958.3\t0\t128\r\n",
      "4\t1605\t239.6\t0\t1605\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2456.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2456.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.21 s (22 us/read; 2.79 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,725\r\n",
      "Reads with adapters:                     2,286 (2.2%)\r\n",
      "Reads written (passing filters):       102,725 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,204,014 bp\r\n",
      "Total written (filtered):     29,196,828 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2168 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2165 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t3\t401.3\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2141\t1605.1\t0\t2141\r\n",
      "4\t18\t401.3\t0\t18\r\n",
      "5\t6\t100.3\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 118 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "112 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t6\t401.3\t0\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t101\t1605.1\t0\t101\r\n",
      "4\t3\t401.3\t0\t3\r\n",
      "5\t4\t100.3\t0\t4\r\n",
      "6\t3\t25.1\t0\t3\r\n",
      "272\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2459.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2459.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.93 s (22 us/read; 2.75 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,371\r\n",
      "Reads with adapters:                     1,902 (2.2%)\r\n",
      "Reads written (passing filters):        88,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,548,531 bp\r\n",
      "Total written (filtered):     25,542,755 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1729 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1724 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1380.8\t0\t4\r\n",
      "4\t1\t345.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1693\t1380.8\t0\t1693\r\n",
      "4\t15\t345.2\t0\t15\r\n",
      "5\t12\t86.3\t0\t12\r\n",
      "6\t4\t21.6\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 173 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "169 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1380.8\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t1380.8\t0\t160\r\n",
      "4\t5\t345.2\t0\t5\r\n",
      "5\t2\t86.3\t0\t2\r\n",
      "6\t1\t21.6\t0\t1\r\n",
      "9\t1\t0.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2456.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2456.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.15 s (40 us/read; 1.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 102,725\r\n",
      "Reads with adapters:                    89,030 (86.7%)\r\n",
      "Reads written (passing filters):       102,725 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,236,312 bp\r\n",
      "Total written (filtered):     27,409,253 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 86749 times.\r\n",
      "86450 times, it overlapped the 5' end of a read\r\n",
      "299 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1605.1\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t17\t0.0\t0\t17\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t92\t0.0\t0\t92\r\n",
      "21\t86327\t0.0\t0\t86327\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t290\t1605.1\t0\t290\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t2\t0.0\t0\t2\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2281 times.\r\n",
      "82 times, it overlapped the 5' end of a read\r\n",
      "2199 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t82\t1605.1\t0\t82\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t65\t1605.1\t0\t65\r\n",
      "4\t2134\t401.3\t0\t2134\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2475.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2475.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.28 s (22 us/read; 2.74 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,304\r\n",
      "Reads with adapters:                     1,741 (3.0%)\r\n",
      "Reads written (passing filters):        58,304 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,800,385 bp\r\n",
      "Total written (filtered):     15,795,136 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1672 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1667 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t911.0\t0\t4\r\n",
      "4\t1\t227.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1656\t911.0\t0\t1656\r\n",
      "4\t7\t227.8\t0\t7\r\n",
      "5\t4\t56.9\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 69 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t911.0\t0\t2\r\n",
      "4\t5\t227.8\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t911.0\t0\t59\r\n",
      "4\t1\t227.8\t0\t1\r\n",
      "5\t2\t56.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2475.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2475.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.06 s (35 us/read; 1.70 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,304\r\n",
      "Reads with adapters:                    50,754 (87.1%)\r\n",
      "Reads written (passing filters):        58,304 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,790,648 bp\r\n",
      "Total written (filtered):     14,778,152 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 47724 times.\r\n",
      "47637 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t911.0\t0\t3\r\n",
      "4\t1\t227.8\t0\t1\r\n",
      "12\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t80\t0.0\t0\t80\r\n",
      "21\t47531\t0.0\t0\t47531\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t75\t911.0\t0\t75\r\n",
      "4\t6\t227.8\t0\t6\r\n",
      "5\t5\t56.9\t0\t5\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3030 times.\r\n",
      "86 times, it overlapped the 5' end of a read\r\n",
      "2944 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t84\t911.0\t0\t84\r\n",
      "4\t1\t227.8\t0\t1\r\n",
      "5\t1\t56.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t911.0\t0\t26\r\n",
      "4\t2918\t227.8\t0\t2918\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2459.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2459.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.37 s (50 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,371\r\n",
      "Reads with adapters:                    74,509 (84.3%)\r\n",
      "Reads written (passing filters):        88,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,560,490 bp\r\n",
      "Total written (filtered):     24,027,678 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 72767 times.\r\n",
      "72528 times, it overlapped the 5' end of a read\r\n",
      "239 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t109\t0.0\t0\t109\r\n",
      "21\t72385\t0.0\t0\t72385\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t231\t1380.8\t0\t231\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1742 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "1634 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t107\t1380.8\t0\t107\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1380.8\t0\t63\r\n",
      "4\t1571\t345.2\t0\t1571\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2463.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2463.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.63 s (36 us/read; 1.67 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,371\r\n",
      "Reads with adapters:                     4,966 (4.9%)\r\n",
      "Reads written (passing filters):       101,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,819,644 bp\r\n",
      "Total written (filtered):     28,751,087 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2630 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2625 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1583.9\t0\t4\r\n",
      "4\t1\t396.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2616\t1583.9\t0\t2616\r\n",
      "4\t4\t396.0\t0\t4\r\n",
      "5\t5\t99.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2336 times.\r\n",
      "2235 times, it overlapped the 5' end of a read\r\n",
      "101 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t1\t0.0\t1\t0 1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2232\t0.0\t1\t0 2232\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t95\t1583.9\t0\t95\r\n",
      "4\t3\t396.0\t0\t3\r\n",
      "5\t3\t99.0\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2463.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2463.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.91 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,371\r\n",
      "Reads with adapters:                    93,513 (92.2%)\r\n",
      "Reads written (passing filters):       101,371 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,906,719 bp\r\n",
      "Total written (filtered):     26,985,919 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90901 times.\r\n",
      "90872 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1583.9\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t122\t0.0\t0\t122\r\n",
      "21\t90723\t0.0\t0\t90723\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1583.9\t0\t21\r\n",
      "51\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2612 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "2545 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1583.9\t0\t62\r\n",
      "27\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1583.9\t0\t18\r\n",
      "4\t2527\t396.0\t0\t2527\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2540.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2540.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.96 s (22 us/read; 2.67 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,122\r\n",
      "Reads with adapters:                     4,275 (4.9%)\r\n",
      "Reads written (passing filters):        87,122 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,744,340 bp\r\n",
      "Total written (filtered):     22,685,702 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2273 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "2262 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1361.3\t0\t8\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2258\t1361.3\t0\t2258\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "5\t1\t85.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2002 times.\r\n",
      "1911 times, it overlapped the 5' end of a read\r\n",
      "91 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t340.3\t0\t2\r\n",
      "5\t1\t85.1\t0\t1\r\n",
      "27\t1908\t0.0\t1\t0 1908\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t1361.3\t0\t85\r\n",
      "4\t3\t340.3\t0\t3\r\n",
      "5\t3\t85.1\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2481.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2481.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.29 s (32 us/read; 1.86 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,281\r\n",
      "Reads with adapters:                     9,863 (7.4%)\r\n",
      "Reads written (passing filters):       133,281 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,476,064 bp\r\n",
      "Total written (filtered):     39,368,847 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 6483 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "6479 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2082.5\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6466\t2082.5\t0\t6466\r\n",
      "4\t6\t520.6\t0\t6\r\n",
      "5\t7\t130.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3380 times.\r\n",
      "3237 times, it overlapped the 5' end of a read\r\n",
      "143 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2082.5\t0\t1\r\n",
      "4\t2\t520.6\t0\t2\r\n",
      "5\t1\t130.2\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t3230\t0.0\t1\t0 3230\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t135\t2082.5\t0\t135\r\n",
      "4\t1\t520.6\t0\t1\r\n",
      "5\t7\t130.2\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2497.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2497.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.62 s (29 us/read; 2.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,306\r\n",
      "Reads with adapters:                     5,447 (4.3%)\r\n",
      "Reads written (passing filters):       126,306 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,315,581 bp\r\n",
      "Total written (filtered):     31,236,586 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2667 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2666 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1973.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2643\t1973.5\t0\t2643\r\n",
      "4\t13\t493.4\t0\t13\r\n",
      "5\t10\t123.3\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2780 times.\r\n",
      "2611 times, it overlapped the 5' end of a read\r\n",
      "169 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1973.5\t0\t1\r\n",
      "4\t1\t493.4\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2606\t0.0\t1\t0 2606\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t164\t1973.5\t0\t164\r\n",
      "4\t3\t493.4\t0\t3\r\n",
      "5\t2\t123.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2549.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2549.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.94 s (21 us/read; 2.85 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,410\r\n",
      "Reads with adapters:                     4,026 (4.4%)\r\n",
      "Reads written (passing filters):        92,410 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,775,552 bp\r\n",
      "Total written (filtered):     21,721,001 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2106 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "2091 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1443.9\t0\t13\r\n",
      "4\t2\t361.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2080\t1443.9\t0\t2080\r\n",
      "4\t9\t361.0\t0\t9\r\n",
      "5\t2\t90.2\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1920 times.\r\n",
      "1768 times, it overlapped the 5' end of a read\r\n",
      "152 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t361.0\t0\t1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1765\t0.0\t1\t0 1765\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t1443.9\t0\t146\r\n",
      "4\t2\t361.0\t0\t2\r\n",
      "5\t3\t90.2\t0\t3\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2540.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2540.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.83 s (44 us/read; 1.37 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  87,122\r\n",
      "Reads with adapters:                    76,164 (87.4%)\r\n",
      "Reads written (passing filters):        87,122 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,863,123 bp\r\n",
      "Total written (filtered):     21,405,450 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67768 times.\r\n",
      "67732 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t14\t0.0\t0\t14\r\n",
      "20\t87\t0.0\t0\t87\r\n",
      "21\t67624\t0.0\t0\t67624\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1361.3\t0\t29\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8396 times.\r\n",
      "63 times, it overlapped the 5' end of a read\r\n",
      "8333 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1361.3\t0\t63\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1361.3\t0\t29\r\n",
      "4\t8304\t340.3\t0\t8304\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2481.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2481.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.86 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 133,281\r\n",
      "Reads with adapters:                   128,217 (96.2%)\r\n",
      "Reads written (passing filters):       133,281 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,534,014 bp\r\n",
      "Total written (filtered):     36,846,124 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 127623 times.\r\n",
      "127590 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t18\t0.0\t0\t18\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t164\t0.0\t0\t164\r\n",
      "21\t127385\t0.0\t0\t127385\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t2082.5\t0\t12\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t20\t0.0\t0\t20\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 594 times.\r\n",
      "37 times, it overlapped the 5' end of a read\r\n",
      "557 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t2082.5\t0\t33\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t2082.5\t0\t4\r\n",
      "4\t553\t520.6\t0\t553\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2549.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2549.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.62 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,410\r\n",
      "Reads with adapters:                    81,427 (88.1%)\r\n",
      "Reads written (passing filters):        92,410 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,956,022 bp\r\n",
      "Total written (filtered):     20,512,902 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 65716 times.\r\n",
      "65654 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "6\t1\t22.6\t0\t1\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t91\t0.0\t0\t91\r\n",
      "21\t65539\t0.0\t0\t65539\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1443.9\t0\t54\r\n",
      "4\t1\t361.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 15711 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "15668 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1443.9\t0\t42\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1443.9\t0\t35\r\n",
      "4\t15632\t361.0\t0\t15632\r\n",
      "5\t1\t90.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2559.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2559.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.71 s (21 us/read; 2.91 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,970\r\n",
      "Reads with adapters:                     1,699 (2.0%)\r\n",
      "Reads written (passing filters):        82,970 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,462,530 bp\r\n",
      "Total written (filtered):     20,457,374 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1619 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1617 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1296.4\t0\t1\r\n",
      "4\t1\t324.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1594\t1296.4\t0\t1594\r\n",
      "4\t5\t324.1\t0\t5\r\n",
      "5\t15\t81.0\t0\t15\r\n",
      "6\t3\t20.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 80 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "79 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1296.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1296.4\t0\t74\r\n",
      "4\t1\t324.1\t0\t1\r\n",
      "5\t2\t81.0\t0\t2\r\n",
      "6\t1\t20.3\t0\t1\r\n",
      "9\t1\t0.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2497.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2497.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.45 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,306\r\n",
      "Reads with adapters:                   110,870 (87.8%)\r\n",
      "Reads written (passing filters):       126,306 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,472,642 bp\r\n",
      "Total written (filtered):     29,425,581 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 94184 times.\r\n",
      "94115 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t13\t0.0\t0\t13\r\n",
      "20\t152\t0.0\t0\t152\r\n",
      "21\t93944\t0.0\t0\t93944\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1973.5\t0\t52\r\n",
      "4\t2\t493.4\t0\t2\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "55\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 16686 times.\r\n",
      "53 times, it overlapped the 5' end of a read\r\n",
      "16633 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1973.5\t0\t52\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1973.5\t0\t45\r\n",
      "4\t16587\t493.4\t0\t16587\r\n",
      "6\t1\t30.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2559.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2559.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.06 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,970\r\n",
      "Reads with adapters:                    55,297 (66.6%)\r\n",
      "Reads written (passing filters):        82,970 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,537,855 bp\r\n",
      "Total written (filtered):     19,501,656 bp (95.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 48299 times.\r\n",
      "47962 times, it overlapped the 5' end of a read\r\n",
      "337 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1296.4\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t72\t0.0\t0\t72\r\n",
      "21\t47874\t0.0\t0\t47874\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t332\t1296.4\t0\t332\r\n",
      "5\t3\t81.0\t0\t3\r\n",
      "57\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6998 times.\r\n",
      "99 times, it overlapped the 5' end of a read\r\n",
      "6899 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t98\t1296.4\t0\t98\r\n",
      "4\t1\t324.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t142\t1296.4\t0\t142\r\n",
      "4\t6756\t324.1\t0\t6756\r\n",
      "5\t1\t81.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2561.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2561.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.29 s (35 us/read; 1.71 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,674\r\n",
      "Reads with adapters:                     1,119 (1.2%)\r\n",
      "Reads written (passing filters):        93,674 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,653,303 bp\r\n",
      "Total written (filtered):     15,649,837 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 937 times.\r\n",
      "12 times, it overlapped the 5' end of a read\r\n",
      "925 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1463.7\t0\t9\r\n",
      "4\t2\t365.9\t0\t2\r\n",
      "5\t1\t91.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t902\t1463.7\t0\t902\r\n",
      "4\t6\t365.9\t0\t6\r\n",
      "5\t13\t91.5\t0\t13\r\n",
      "6\t4\t22.9\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 182 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "179 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1463.7\t0\t2\r\n",
      "4\t1\t365.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t164\t1463.7\t0\t164\r\n",
      "4\t7\t365.9\t0\t7\r\n",
      "5\t6\t91.5\t0\t6\r\n",
      "6\t1\t22.9\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2561.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2561.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.58 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,674\r\n",
      "Reads with adapters:                    69,151 (73.8%)\r\n",
      "Reads written (passing filters):        93,674 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,108,943 bp\r\n",
      "Total written (filtered):     15,100,101 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 43380 times.\r\n",
      "43063 times, it overlapped the 5' end of a read\r\n",
      "317 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t86\t0.0\t0\t86\r\n",
      "21\t42957\t0.0\t0\t42957\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t312\t1463.7\t0\t312\r\n",
      "4\t1\t365.9\t0\t1\r\n",
      "48\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "247\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25771 times.\r\n",
      "78 times, it overlapped the 5' end of a read\r\n",
      "25693 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t68\t1463.7\t0\t68\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t6\t0.0\t1\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t187\t1463.7\t0\t187\r\n",
      "4\t25504\t365.9\t0\t25504\r\n",
      "5\t2\t91.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2592.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2592.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.70 s (33 us/read; 1.82 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,982\r\n",
      "Reads with adapters:                     4,898 (4.4%)\r\n",
      "Reads written (passing filters):       111,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,542,600 bp\r\n",
      "Total written (filtered):     33,467,397 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2250 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "2246 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1749.7\t0\t3\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2237\t1749.7\t0\t2237\r\n",
      "4\t4\t437.4\t0\t4\r\n",
      "5\t5\t109.4\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2648 times.\r\n",
      "2521 times, it overlapped the 5' end of a read\r\n",
      "127 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "27\t2520\t0.0\t1\t0 2520\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t119\t1749.7\t0\t119\r\n",
      "4\t3\t437.4\t0\t3\r\n",
      "5\t5\t109.4\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2599.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2599.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.98 s (22 us/read; 2.78 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,120\r\n",
      "Reads with adapters:                     1,659 (1.8%)\r\n",
      "Reads written (passing filters):        92,120 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,147,115 bp\r\n",
      "Total written (filtered):     25,142,065 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1466 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1458 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1439.4\t0\t6\r\n",
      "4\t2\t359.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1423\t1439.4\t0\t1423\r\n",
      "4\t18\t359.8\t0\t18\r\n",
      "5\t17\t90.0\t0\t17\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 193 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "193 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t183\t1439.4\t0\t183\r\n",
      "4\t2\t359.8\t0\t2\r\n",
      "5\t7\t90.0\t0\t7\r\n",
      "6\t1\t22.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2579.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2579.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.56 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                   106,203 (87.0%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,563,591 bp\r\n",
      "Total written (filtered):     34,335,356 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 105945 times.\r\n",
      "105912 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t13\t0.0\t0\t13\r\n",
      "19\t23\t0.0\t0\t23\r\n",
      "20\t133\t0.0\t0\t133\r\n",
      "21\t105740\t0.0\t0\t105740\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1907.5\t0\t22\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 258 times.\r\n",
      "118 times, it overlapped the 5' end of a read\r\n",
      "140 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t116\t1907.5\t0\t116\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1907.5\t0\t9\r\n",
      "4\t131\t476.9\t0\t131\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2579.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2579.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.57 s (46 us/read; 1.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 122,078\r\n",
      "Reads with adapters:                     6,509 (5.3%)\r\n",
      "Reads written (passing filters):       122,078 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    36,520,467 bp\r\n",
      "Total written (filtered):     36,433,341 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3573 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "3572 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1907.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3564\t1907.5\t0\t3564\r\n",
      "4\t4\t476.9\t0\t4\r\n",
      "5\t4\t119.2\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2936 times.\r\n",
      "2823 times, it overlapped the 5' end of a read\r\n",
      "113 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1907.5\t0\t4\r\n",
      "4\t2\t476.9\t0\t2\r\n",
      "5\t1\t119.2\t0\t1\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2812\t0.0\t1\t0 2812\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1907.5\t0\t108\r\n",
      "4\t3\t476.9\t0\t3\r\n",
      "5\t2\t119.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2592.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2592.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.22 s (56 us/read; 1.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 111,982\r\n",
      "Reads with adapters:                   108,500 (96.9%)\r\n",
      "Reads written (passing filters):       111,982 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,576,097 bp\r\n",
      "Total written (filtered):     31,298,691 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 108325 times.\r\n",
      "108309 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t131\t0.0\t0\t131\r\n",
      "21\t108152\t0.0\t0\t108152\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1749.7\t0\t7\r\n",
      "4\t1\t437.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 175 times.\r\n",
      "54 times, it overlapped the 5' end of a read\r\n",
      "121 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1749.7\t0\t52\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1749.7\t0\t2\r\n",
      "4\t119\t437.4\t0\t119\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2599.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2599.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.01 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,120\r\n",
      "Reads with adapters:                    78,213 (84.9%)\r\n",
      "Reads written (passing filters):        92,120 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,167,371 bp\r\n",
      "Total written (filtered):     23,628,224 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 72316 times.\r\n",
      "72055 times, it overlapped the 5' end of a read\r\n",
      "261 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1439.4\t0\t1\r\n",
      "17\t4\t0.0\t0\t4\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t21\t0.0\t0\t21\r\n",
      "20\t126\t0.0\t0\t126\r\n",
      "21\t71897\t0.0\t0\t71897\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t251\t1439.4\t0\t251\r\n",
      "5\t1\t90.0\t0\t1\r\n",
      "6\t1\t22.5\t0\t1\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5897 times.\r\n",
      "224 times, it overlapped the 5' end of a read\r\n",
      "5673 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t223\t1439.4\t0\t223\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t90\t1439.4\t0\t90\r\n",
      "4\t5583\t359.8\t0\t5583\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2593.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2593.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.43 s (43 us/read; 1.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,836\r\n",
      "Reads with adapters:                     4,771 (3.8%)\r\n",
      "Reads written (passing filters):       126,836 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,303,051 bp\r\n",
      "Total written (filtered):     29,235,837 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2400 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "2394 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1981.8\t0\t5\r\n",
      "4\t1\t495.5\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2377\t1981.8\t0\t2377\r\n",
      "4\t13\t495.5\t0\t13\r\n",
      "5\t4\t123.9\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2371 times.\r\n",
      "2203 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2198\t0.0\t1\t0 2198\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1981.8\t0\t161\r\n",
      "4\t3\t495.5\t0\t3\r\n",
      "5\t3\t123.9\t0\t3\r\n",
      "6\t1\t31.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2593.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2593.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.82 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,836\r\n",
      "Reads with adapters:                   120,525 (95.0%)\r\n",
      "Reads written (passing filters):       126,836 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,607,504 bp\r\n",
      "Total written (filtered):     27,503,256 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 95477 times.\r\n",
      "95401 times, it overlapped the 5' end of a read\r\n",
      "76 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1981.8\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t154\t0.0\t0\t154\r\n",
      "21\t95225\t0.0\t0\t95225\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t1981.8\t0\t70\r\n",
      "4\t1\t495.5\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25048 times.\r\n",
      "77 times, it overlapped the 5' end of a read\r\n",
      "24971 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t76\t1981.8\t0\t76\r\n",
      "26\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t1981.8\t0\t64\r\n",
      "4\t24907\t495.5\t0\t24907\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2606.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2606.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.96 s (23 us/read; 2.62 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,445\r\n",
      "Reads with adapters:                     1,981 (2.3%)\r\n",
      "Reads written (passing filters):        85,445 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,452,711 bp\r\n",
      "Total written (filtered):     24,446,189 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1892 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "1863 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1335.1\t0\t27\r\n",
      "4\t2\t333.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1845\t1335.1\t0\t1845\r\n",
      "4\t7\t333.8\t0\t7\r\n",
      "5\t10\t83.4\t0\t10\r\n",
      "6\t1\t20.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 89 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "89 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t77\t1335.1\t0\t77\r\n",
      "4\t2\t333.8\t0\t2\r\n",
      "5\t3\t83.4\t0\t3\r\n",
      "6\t5\t20.9\t0\t5\r\n",
      "256\t1\t0.0\t1\t1\r\n",
      "274\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2618.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2618.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.03 s (24 us/read; 2.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,439\r\n",
      "Reads with adapters:                    78,721 (93.2%)\r\n",
      "Reads written (passing filters):        84,439 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,488,541 bp\r\n",
      "Total written (filtered):     19,368,479 bp (90.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 24 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1319.4\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1319.4\t0\t14\r\n",
      "4\t3\t329.8\t0\t3\r\n",
      "5\t2\t82.5\t0\t2\r\n",
      "6\t2\t20.6\t0\t2\r\n",
      "8\t1\t1.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 78697 times.\r\n",
      "78648 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "7\t1\t5.2\t0\t1\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "22\t4\t0.0\t1\t4\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t12\t0.0\t1\t5 7\r\n",
      "25\t16\t0.0\t1\t7 9\r\n",
      "26\t6053\t0.0\t1\t66 5987\r\n",
      "27\t71933\t0.0\t1\t63581 8352\r\n",
      "28\t627\t0.0\t1\t0 627\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1319.4\t0\t36\r\n",
      "4\t2\t329.8\t0\t2\r\n",
      "5\t3\t82.5\t0\t3\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "52\t1\t0.0\t1\t1\r\n",
      "297\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2620.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2620.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.74 s (25 us/read; 2.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,894\r\n",
      "Reads with adapters:                    64,733 (92.6%)\r\n",
      "Reads written (passing filters):        69,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,549,221 bp\r\n",
      "Total written (filtered):     15,805,843 bp (90.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 15 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t273.0\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1092.1\t0\t10\r\n",
      "4\t2\t273.0\t0\t2\r\n",
      "5\t1\t68.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 64718 times.\r\n",
      "64689 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1092.1\t0\t1\r\n",
      "4\t1\t273.0\t0\t1\r\n",
      "23\t1\t0.0\t1\t1\r\n",
      "24\t6\t0.0\t1\t1 5\r\n",
      "25\t6\t0.0\t1\t2 4\r\n",
      "26\t5466\t0.0\t1\t50 5416\r\n",
      "27\t58923\t0.0\t1\t51843 7080\r\n",
      "28\t285\t0.0\t1\t0 285\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1092.1\t0\t19\r\n",
      "5\t1\t68.3\t0\t1\r\n",
      "6\t1\t17.1\t0\t1\r\n",
      "51\t2\t0.0\t1\t2\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2606.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2606.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.61 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,445\r\n",
      "Reads with adapters:                    73,617 (86.2%)\r\n",
      "Reads written (passing filters):        85,445 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,435,502 bp\r\n",
      "Total written (filtered):     22,929,580 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 71401 times.\r\n",
      "71187 times, it overlapped the 5' end of a read\r\n",
      "214 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1335.1\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t127\t0.0\t0\t127\r\n",
      "21\t71038\t0.0\t0\t71038\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t207\t1335.1\t0\t207\r\n",
      "4\t1\t333.8\t0\t1\r\n",
      "264\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2216 times.\r\n",
      "108 times, it overlapped the 5' end of a read\r\n",
      "2108 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t104\t1335.1\t0\t104\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1335.1\t0\t47\r\n",
      "4\t2060\t333.8\t0\t2060\r\n",
      "5\t1\t83.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2631.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2631.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.41 s (24 us/read; 2.49 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,764\r\n",
      "Reads with adapters:                    56,201 (95.6%)\r\n",
      "Reads written (passing filters):        58,764 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,063,183 bp\r\n",
      "Total written (filtered):     14,549,922 bp (90.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 25 times.\r\n",
      "12 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t918.2\t0\t10\r\n",
      "4\t1\t229.5\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t918.2\t0\t12\r\n",
      "4\t1\t229.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 56176 times.\r\n",
      "56158 times, it overlapped the 5' end of a read\r\n",
      "18 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t918.2\t0\t2\r\n",
      "23\t10\t0.0\t1\t3 7\r\n",
      "24\t11\t0.0\t1\t3 8\r\n",
      "25\t12\t0.0\t1\t4 8\r\n",
      "26\t3371\t0.0\t1\t36 3335\r\n",
      "27\t52688\t0.0\t1\t49658 3030\r\n",
      "28\t64\t0.0\t1\t0 64\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t918.2\t0\t15\r\n",
      "4\t1\t229.5\t0\t1\r\n",
      "5\t1\t57.4\t0\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2631.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2631.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.41 s (41 us/read; 1.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  58,764\r\n",
      "Reads with adapters:                    45,140 (76.8%)\r\n",
      "Reads written (passing filters):        58,764 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,139,385 bp\r\n",
      "Total written (filtered):     15,254,880 bp (94.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41511 times.\r\n",
      "41382 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "20\t23\t0.0\t0\t23\r\n",
      "21\t41351\t0.0\t0\t41351\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t121\t918.2\t0\t121\r\n",
      "7\t6\t3.6\t0\t6\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3629 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "3603 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t918.2\t0\t24\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t918.2\t0\t12\r\n",
      "4\t3591\t229.5\t0\t3591\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2618.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2618.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.64 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  84,439\r\n",
      "Reads with adapters:                    79,510 (94.2%)\r\n",
      "Reads written (passing filters):        84,439 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,537,654 bp\r\n",
      "Total written (filtered):     20,004,264 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 71533 times.\r\n",
      "71470 times, it overlapped the 5' end of a read\r\n",
      "63 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "20\t46\t0.0\t0\t46\r\n",
      "21\t71414\t0.0\t0\t71414\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1319.4\t0\t59\r\n",
      "7\t1\t5.2\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "217\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7977 times.\r\n",
      "50 times, it overlapped the 5' end of a read\r\n",
      "7927 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t49\t1319.4\t0\t49\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1319.4\t0\t35\r\n",
      "4\t7891\t329.8\t0\t7891\r\n",
      "5\t1\t82.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2620.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2620.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.98 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,894\r\n",
      "Reads with adapters:                    63,636 (91.0%)\r\n",
      "Reads written (passing filters):        69,894 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,670,079 bp\r\n",
      "Total written (filtered):     16,468,193 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55740 times.\r\n",
      "55665 times, it overlapped the 5' end of a read\r\n",
      "75 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t55626\t0.0\t0\t55626\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t69\t1092.1\t0\t69\r\n",
      "7\t2\t4.3\t0\t2\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7896 times.\r\n",
      "41 times, it overlapped the 5' end of a read\r\n",
      "7855 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1092.1\t0\t40\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1092.1\t0\t20\r\n",
      "4\t7834\t273.0\t0\t7834\r\n",
      "5\t1\t68.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2634.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2634.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.39 s (39 us/read; 1.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,279\r\n",
      "Reads with adapters:                    57,800 (94.3%)\r\n",
      "Reads written (passing filters):        61,279 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,344,093 bp\r\n",
      "Total written (filtered):     12,321,671 bp (92.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 46525 times.\r\n",
      "46481 times, it overlapped the 5' end of a read\r\n",
      "44 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "20\t31\t0.0\t0\t31\r\n",
      "21\t46448\t0.0\t0\t46448\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t957.5\t0\t36\r\n",
      "4\t2\t239.4\t0\t2\r\n",
      "5\t1\t59.8\t0\t1\r\n",
      "7\t2\t3.7\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11275 times.\r\n",
      "32 times, it overlapped the 5' end of a read\r\n",
      "11243 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t957.5\t0\t30\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t957.5\t0\t29\r\n",
      "4\t11211\t239.4\t0\t11211\r\n",
      "6\t2\t15.0\t0\t2\r\n",
      "237\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2632.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2632.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.24 s (46 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,425\r\n",
      "Reads with adapters:                    80,429 (88.0%)\r\n",
      "Reads written (passing filters):        91,425 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,904,435 bp\r\n",
      "Total written (filtered):     11,752,178 bp (84.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1428.5\t0\t10\r\n",
      "4\t3\t357.1\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1428.5\t0\t27\r\n",
      "4\t3\t357.1\t0\t3\r\n",
      "5\t1\t89.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 80384 times.\r\n",
      "80329 times, it overlapped the 5' end of a read\r\n",
      "55 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t357.1\t0\t1\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "23\t4\t0.0\t1\t2 2\r\n",
      "24\t16\t0.0\t1\t8 8\r\n",
      "25\t31\t0.0\t1\t16 15\r\n",
      "26\t19128\t0.0\t1\t47 19081\r\n",
      "27\t60493\t0.0\t1\t57354 3139\r\n",
      "28\t654\t0.0\t1\t0 654\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1428.5\t0\t41\r\n",
      "4\t1\t357.1\t0\t1\r\n",
      "5\t3\t89.3\t0\t3\r\n",
      "6\t1\t22.3\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "52\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t0 1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2634.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2634.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.93 s (48 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  61,279\r\n",
      "Reads with adapters:                    56,039 (91.4%)\r\n",
      "Reads written (passing filters):        61,279 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    13,049,284 bp\r\n",
      "Total written (filtered):     11,544,056 bp (88.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 24 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t957.5\t0\t2\r\n",
      "4\t2\t239.4\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t957.5\t0\t18\r\n",
      "4\t1\t239.4\t0\t1\r\n",
      "6\t1\t15.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 56015 times.\r\n",
      "55988 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t6\t0.0\t1\t2 4\r\n",
      "24\t4\t0.0\t1\t2 2\r\n",
      "25\t6\t0.0\t1\t3 3\r\n",
      "26\t7806\t0.0\t1\t43 7763\r\n",
      "27\t47647\t0.0\t1\t43788 3859\r\n",
      "28\t519\t0.0\t1\t0 519\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t957.5\t0\t20\r\n",
      "4\t1\t239.4\t0\t1\r\n",
      "5\t1\t59.8\t0\t1\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "60\t1\t0.0\t1\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "300\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2632.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2632.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.60 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  91,425\r\n",
      "Reads with adapters:                    85,794 (93.8%)\r\n",
      "Reads written (passing filters):        91,425 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,738,308 bp\r\n",
      "Total written (filtered):     13,407,752 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58138 times.\r\n",
      "58050 times, it overlapped the 5' end of a read\r\n",
      "88 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "20\t33\t0.0\t0\t33\r\n",
      "21\t58012\t0.0\t0\t58012\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t82\t1428.5\t0\t82\r\n",
      "5\t1\t89.3\t0\t1\r\n",
      "7\t1\t5.6\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "74\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 27656 times.\r\n",
      "32 times, it overlapped the 5' end of a read\r\n",
      "27624 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1428.5\t0\t30\r\n",
      "4\t1\t357.1\t0\t1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1428.5\t0\t59\r\n",
      "4\t27562\t357.1\t0\t27562\r\n",
      "5\t3\t89.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2642.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2642.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.86 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,870\r\n",
      "Reads with adapters:                    75,552 (91.2%)\r\n",
      "Reads written (passing filters):        82,870 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,886,501 bp\r\n",
      "Total written (filtered):     16,854,494 bp (89.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 24 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t323.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1294.8\t0\t20\r\n",
      "4\t2\t323.7\t0\t2\r\n",
      "5\t1\t80.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 75528 times.\r\n",
      "75476 times, it overlapped the 5' end of a read\r\n",
      "52 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1294.8\t0\t4\r\n",
      "4\t1\t323.7\t0\t1\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "23\t5\t0.0\t1\t4 1\r\n",
      "24\t11\t0.0\t1\t7 4\r\n",
      "25\t18\t0.0\t1\t8 10\r\n",
      "26\t10608\t0.0\t1\t57 10551\r\n",
      "27\t64301\t0.0\t1\t60542 3759\r\n",
      "28\t527\t0.0\t1\t0 527\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1294.8\t0\t27\r\n",
      "4\t2\t323.7\t0\t2\r\n",
      "5\t1\t80.9\t0\t1\r\n",
      "6\t1\t20.2\t0\t1\r\n",
      "41\t3\t0.0\t1\t0 3\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "48\t2\t0.0\t1\t1 1\r\n",
      "51\t1\t0.0\t1\t0 1\r\n",
      "58\t1\t0.0\t1\t1\r\n",
      "300\t13\t0.0\t1\t9 4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2642.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2642.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.20 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,870\r\n",
      "Reads with adapters:                    78,795 (95.1%)\r\n",
      "Reads written (passing filters):        82,870 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,129,490 bp\r\n",
      "Total written (filtered):     17,725,029 bp (92.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64114 times.\r\n",
      "64065 times, it overlapped the 5' end of a read\r\n",
      "49 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "20\t37\t0.0\t0\t37\r\n",
      "21\t64016\t0.0\t0\t64016\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t1294.8\t0\t46\r\n",
      "7\t2\t5.1\t0\t2\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 14681 times.\r\n",
      "44 times, it overlapped the 5' end of a read\r\n",
      "14637 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1294.8\t0\t41\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1294.8\t0\t35\r\n",
      "4\t14600\t323.7\t0\t14600\r\n",
      "5\t2\t80.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2649.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2649.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.96 s (24 us/read; 2.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,579\r\n",
      "Reads with adapters:                   122,604 (97.6%)\r\n",
      "Reads written (passing filters):       125,579 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,538,969 bp\r\n",
      "Total written (filtered):     32,230,076 bp (90.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 27 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t490.5\t0\t1\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1962.2\t0\t18\r\n",
      "4\t1\t490.5\t0\t1\r\n",
      "5\t3\t122.6\t0\t3\r\n",
      "7\t1\t7.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 122577 times.\r\n",
      "122538 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1962.2\t0\t1\r\n",
      "4\t3\t490.5\t0\t3\r\n",
      "7\t1\t7.7\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t2\t0.0\t1\t0 2\r\n",
      "23\t3\t0.0\t1\t0 3\r\n",
      "24\t24\t0.0\t1\t13 11\r\n",
      "25\t16\t0.0\t1\t3 13\r\n",
      "26\t4231\t0.0\t1\t99 4132\r\n",
      "27\t118153\t0.0\t1\t113105 5048\r\n",
      "28\t100\t0.0\t1\t0 100\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1962.2\t0\t20\r\n",
      "4\t2\t490.5\t0\t2\r\n",
      "5\t1\t122.6\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "300\t15\t0.0\t1\t9 6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2673.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2673.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.94 s (25 us/read; 2.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,314\r\n",
      "Reads with adapters:                    73,106 (94.6%)\r\n",
      "Reads written (passing filters):        77,314 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,629,173 bp\r\n",
      "Total written (filtered):     17,662,178 bp (90.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 27 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1208.0\t0\t3\r\n",
      "4\t1\t302.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1208.0\t0\t20\r\n",
      "4\t2\t302.0\t0\t2\r\n",
      "5\t1\t75.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 73079 times.\r\n",
      "73057 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1208.0\t0\t1\r\n",
      "4\t1\t302.0\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t6\t0.0\t1\t5 1\r\n",
      "23\t7\t0.0\t1\t0 7\r\n",
      "24\t13\t0.0\t1\t3 10\r\n",
      "25\t20\t0.0\t1\t9 11\r\n",
      "26\t7277\t0.0\t1\t66 7211\r\n",
      "27\t65580\t0.0\t1\t60562 5018\r\n",
      "28\t148\t0.0\t1\t0 148\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1208.0\t0\t13\r\n",
      "5\t1\t75.5\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "61\t2\t0.0\t1\t1 1\r\n",
      "300\t5\t0.0\t1\t2 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2675.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2675.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.24 s (21 us/read; 2.87 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,177\r\n",
      "Reads with adapters:                     1,892 (1.8%)\r\n",
      "Reads written (passing filters):       107,177 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,004,122 bp\r\n",
      "Total written (filtered):     24,998,383 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1753 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1748 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1674.6\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1723\t1674.6\t0\t1723\r\n",
      "4\t5\t418.7\t0\t5\r\n",
      "5\t19\t104.7\t0\t19\r\n",
      "6\t1\t26.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 139 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "139 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t130\t1674.6\t0\t130\r\n",
      "4\t3\t418.7\t0\t3\r\n",
      "5\t4\t104.7\t0\t4\r\n",
      "6\t2\t26.2\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2649.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2649.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.13 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 125,579\r\n",
      "Reads with adapters:                   120,702 (96.1%)\r\n",
      "Reads written (passing filters):       125,579 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    35,709,189 bp\r\n",
      "Total written (filtered):     33,227,564 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 117397 times.\r\n",
      "117340 times, it overlapped the 5' end of a read\r\n",
      "57 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "20\t56\t0.0\t0\t56\r\n",
      "21\t117271\t0.0\t0\t117271\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1962.2\t0\t41\r\n",
      "7\t1\t7.7\t0\t1\r\n",
      "55\t1\t0.0\t0\t1\r\n",
      "300\t14\t0.0\t0\t14\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3305 times.\r\n",
      "74 times, it overlapped the 5' end of a read\r\n",
      "3231 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t71\t1962.2\t0\t71\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1962.2\t0\t22\r\n",
      "4\t3208\t490.5\t0\t3208\r\n",
      "5\t1\t122.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2673.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2673.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.99 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,314\r\n",
      "Reads with adapters:                    73,366 (94.9%)\r\n",
      "Reads written (passing filters):        77,314 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,797,150 bp\r\n",
      "Total written (filtered):     18,409,272 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64394 times.\r\n",
      "64350 times, it overlapped the 5' end of a read\r\n",
      "44 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t30\t0.0\t0\t30\r\n",
      "21\t64310\t0.0\t0\t64310\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1208.0\t0\t40\r\n",
      "4\t1\t302.0\t0\t1\r\n",
      "7\t1\t4.7\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8972 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "8934 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1208.0\t0\t36\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1208.0\t0\t40\r\n",
      "4\t8894\t302.0\t0\t8894\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2658.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2658.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.38 s (49 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,871\r\n",
      "Reads with adapters:                    93,183 (85.6%)\r\n",
      "Reads written (passing filters):       108,871 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,286,755 bp\r\n",
      "Total written (filtered):     18,907,559 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 59273 times.\r\n",
      "59087 times, it overlapped the 5' end of a read\r\n",
      "186 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t59044\t0.0\t0\t59044\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t172\t1701.1\t0\t172\r\n",
      "4\t1\t425.3\t0\t1\r\n",
      "5\t1\t106.3\t0\t1\r\n",
      "7\t2\t6.6\t0\t2\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "48\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "74\t1\t0.0\t0\t1\r\n",
      "296\t2\t0.0\t0\t2\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 33910 times.\r\n",
      "52 times, it overlapped the 5' end of a read\r\n",
      "33858 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1701.1\t0\t45\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t102\t1701.1\t0\t102\r\n",
      "4\t33752\t425.3\t0\t33752\r\n",
      "5\t3\t106.3\t0\t3\r\n",
      "247\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2676.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2676.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.63 s (22 us/read; 2.76 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  74,909\r\n",
      "Reads with adapters:                     1,526 (2.0%)\r\n",
      "Reads written (passing filters):        74,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,853,350 bp\r\n",
      "Total written (filtered):     19,848,746 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1432 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "1427 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1170.5\t0\t2\r\n",
      "4\t3\t292.6\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1413\t1170.5\t0\t1413\r\n",
      "4\t8\t292.6\t0\t8\r\n",
      "5\t6\t73.2\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 94 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "92 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1170.5\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t90\t1170.5\t0\t90\r\n",
      "4\t1\t292.6\t0\t1\r\n",
      "5\t1\t73.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2658.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2658.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.08 s (47 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 108,871\r\n",
      "Reads with adapters:                    91,806 (84.3%)\r\n",
      "Reads written (passing filters):       108,871 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,834,620 bp\r\n",
      "Total written (filtered):     17,378,343 bp (87.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 46 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1701.1\t0\t1\r\n",
      "4\t2\t425.3\t0\t2\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1701.1\t0\t34\r\n",
      "4\t1\t425.3\t0\t1\r\n",
      "5\t5\t106.3\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 91760 times.\r\n",
      "91687 times, it overlapped the 5' end of a read\r\n",
      "73 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1701.1\t0\t2\r\n",
      "4\t5\t425.3\t0\t5\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "23\t3\t0.0\t1\t2 1\r\n",
      "24\t18\t0.0\t1\t13 5\r\n",
      "25\t35\t0.0\t1\t6 29\r\n",
      "26\t23373\t0.0\t1\t52 23321\r\n",
      "27\t67577\t0.0\t1\t63570 4007\r\n",
      "28\t671\t0.0\t1\t0 671\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t1701.1\t0\t46\r\n",
      "4\t2\t425.3\t0\t2\r\n",
      "5\t6\t106.3\t0\t6\r\n",
      "40\t1\t0.0\t1\t0 1\r\n",
      "41\t8\t0.0\t1\t0 8\r\n",
      "298\t1\t0.0\t1\t0 1\r\n",
      "300\t9\t0.0\t1\t9\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2676.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2676.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.07 s (54 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  74,909\r\n",
      "Reads with adapters:                    64,001 (85.4%)\r\n",
      "Reads written (passing filters):        74,909 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,837,539 bp\r\n",
      "Total written (filtered):     18,598,630 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58014 times.\r\n",
      "57802 times, it overlapped the 5' end of a read\r\n",
      "212 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t100\t0.0\t0\t100\r\n",
      "21\t57682\t0.0\t0\t57682\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t209\t1170.5\t0\t209\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5987 times.\r\n",
      "206 times, it overlapped the 5' end of a read\r\n",
      "5781 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t205\t1170.5\t0\t205\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t78\t1170.5\t0\t78\r\n",
      "4\t5703\t292.6\t0\t5703\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2675.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2675.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.80 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 107,177\r\n",
      "Reads with adapters:                    94,890 (88.5%)\r\n",
      "Reads written (passing filters):       107,177 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,104,075 bp\r\n",
      "Total written (filtered):     23,321,594 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 82650 times.\r\n",
      "82456 times, it overlapped the 5' end of a read\r\n",
      "194 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t16\t0.0\t0\t16\r\n",
      "20\t123\t0.0\t0\t123\r\n",
      "21\t82311\t0.0\t0\t82311\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t185\t1674.6\t0\t185\r\n",
      "4\t2\t418.7\t0\t2\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12240 times.\r\n",
      "98 times, it overlapped the 5' end of a read\r\n",
      "12142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t1674.6\t0\t93\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1674.6\t0\t62\r\n",
      "4\t12077\t418.7\t0\t12077\r\n",
      "5\t3\t104.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2684.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2684.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.74 s (39 us/read; 1.53 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  70,071\r\n",
      "Reads with adapters:                     1,219 (1.7%)\r\n",
      "Reads written (passing filters):        70,071 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,835,223 bp\r\n",
      "Total written (filtered):     14,831,465 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1028 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "1025 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1094.9\t0\t1\r\n",
      "4\t2\t273.7\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t996\t1094.9\t0\t996\r\n",
      "4\t13\t273.7\t0\t13\r\n",
      "5\t16\t68.4\t0\t16\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 191 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "191 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t182\t1094.9\t0\t182\r\n",
      "4\t5\t273.7\t0\t5\r\n",
      "5\t3\t68.4\t0\t3\r\n",
      "46\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2684.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2684.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.65 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  70,071\r\n",
      "Reads with adapters:                    60,462 (86.3%)\r\n",
      "Reads written (passing filters):        70,071 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,030,411 bp\r\n",
      "Total written (filtered):     13,969,854 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 48315 times.\r\n",
      "48141 times, it overlapped the 5' end of a read\r\n",
      "174 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t84\t0.0\t0\t84\r\n",
      "21\t48040\t0.0\t0\t48040\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t169\t1094.9\t0\t169\r\n",
      "6\t1\t17.1\t0\t1\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 12147 times.\r\n",
      "82 times, it overlapped the 5' end of a read\r\n",
      "12065 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t1094.9\t0\t79\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t78\t1094.9\t0\t78\r\n",
      "4\t11986\t273.7\t0\t11986\r\n",
      "5\t1\t68.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2706.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2706.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.94 s (20 us/read; 2.94 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,061\r\n",
      "Reads with adapters:                     1,806 (1.9%)\r\n",
      "Reads written (passing filters):        95,061 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,045,455 bp\r\n",
      "Total written (filtered):     22,039,866 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1673 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "1671 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1485.3\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1649\t1485.3\t0\t1649\r\n",
      "4\t10\t371.3\t0\t10\r\n",
      "5\t10\t92.8\t0\t10\r\n",
      "6\t2\t23.2\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 133 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "129 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1485.3\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t118\t1485.3\t0\t118\r\n",
      "4\t3\t371.3\t0\t3\r\n",
      "5\t5\t92.8\t0\t5\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "49\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2715.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2715.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 0.82 s (24 us/read; 2.48 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  34,079\r\n",
      "Reads with adapters:                       598 (1.8%)\r\n",
      "Reads written (passing filters):        34,079 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,013,286 bp\r\n",
      "Total written (filtered):      9,011,469 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 550 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "544 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t532.5\t0\t4\r\n",
      "4\t2\t133.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t531\t532.5\t0\t531\r\n",
      "4\t9\t133.1\t0\t9\r\n",
      "5\t4\t33.3\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 48 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t532.5\t0\t46\r\n",
      "5\t2\t33.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2712.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2712.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.58 s (22 us/read; 2.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,247\r\n",
      "Reads with adapters:                     1,473 (2.0%)\r\n",
      "Reads written (passing filters):        73,247 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,233,639 bp\r\n",
      "Total written (filtered):     20,229,119 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1303 times.\r\n",
      "16 times, it overlapped the 5' end of a read\r\n",
      "1287 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1144.5\t0\t15\r\n",
      "4\t1\t286.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1262\t1144.5\t0\t1262\r\n",
      "4\t10\t286.1\t0\t10\r\n",
      "5\t10\t71.5\t0\t10\r\n",
      "6\t5\t17.9\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 170 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "170 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t147\t1144.5\t0\t147\r\n",
      "4\t3\t286.1\t0\t3\r\n",
      "5\t8\t71.5\t0\t8\r\n",
      "6\t12\t17.9\t0\t12\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2715.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2715.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.70 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  34,079\r\n",
      "Reads with adapters:                    27,922 (81.9%)\r\n",
      "Reads written (passing filters):        34,079 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,027,823 bp\r\n",
      "Total written (filtered):      8,483,180 bp (94.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 25579 times.\r\n",
      "25444 times, it overlapped the 5' end of a read\r\n",
      "135 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t39\t0.0\t0\t39\r\n",
      "21\t25398\t0.0\t0\t25398\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t132\t532.5\t0\t132\r\n",
      "5\t1\t33.3\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2343 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "2305 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t532.5\t0\t35\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t532.5\t0\t30\r\n",
      "4\t2275\t133.1\t0\t2275\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2706.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2706.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.59 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  95,061\r\n",
      "Reads with adapters:                    65,855 (69.3%)\r\n",
      "Reads written (passing filters):        95,061 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,029,041 bp\r\n",
      "Total written (filtered):     20,875,385 bp (94.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 52690 times.\r\n",
      "52291 times, it overlapped the 5' end of a read\r\n",
      "399 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1485.3\t0\t1\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t77\t0.0\t0\t77\r\n",
      "21\t52201\t0.0\t0\t52201\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t387\t1485.3\t0\t387\r\n",
      "4\t1\t371.3\t0\t1\r\n",
      "5\t2\t92.8\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "292\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 13165 times.\r\n",
      "252 times, it overlapped the 5' end of a read\r\n",
      "12913 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t251\t1485.3\t0\t251\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t1485.3\t0\t146\r\n",
      "4\t12763\t371.3\t0\t12763\r\n",
      "5\t4\t92.8\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2712.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2712.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.00 s (55 us/read; 1.10 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,247\r\n",
      "Reads with adapters:                    51,975 (71.0%)\r\n",
      "Reads written (passing filters):        73,247 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,254,322 bp\r\n",
      "Total written (filtered):     19,206,346 bp (94.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 49634 times.\r\n",
      "49303 times, it overlapped the 5' end of a read\r\n",
      "331 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "11\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t63\t0.0\t0\t63\r\n",
      "21\t49224\t0.0\t0\t49224\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t322\t1144.5\t0\t322\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2341 times.\r\n",
      "260 times, it overlapped the 5' end of a read\r\n",
      "2081 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t259\t1144.5\t0\t259\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t74\t1144.5\t0\t74\r\n",
      "4\t2005\t286.1\t0\t2005\r\n",
      "5\t1\t71.5\t0\t1\r\n",
      "247\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2717.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2717.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.91 s (22 us/read; 2.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,237\r\n",
      "Reads with adapters:                     1,609 (1.8%)\r\n",
      "Reads written (passing filters):        88,237 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,809,717 bp\r\n",
      "Total written (filtered):     22,804,798 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1411 times.\r\n",
      "16 times, it overlapped the 5' end of a read\r\n",
      "1395 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1378.7\t0\t13\r\n",
      "4\t3\t344.7\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1366\t1378.7\t0\t1366\r\n",
      "4\t11\t344.7\t0\t11\r\n",
      "5\t13\t86.2\t0\t13\r\n",
      "6\t4\t21.5\t0\t4\r\n",
      "8\t1\t1.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 198 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "198 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t182\t1378.7\t0\t182\r\n",
      "4\t5\t344.7\t0\t5\r\n",
      "5\t6\t86.2\t0\t6\r\n",
      "6\t4\t21.5\t0\t4\r\n",
      "9\t1\t0.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2700.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2700.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.64 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 172,077\r\n",
      "Reads with adapters:                     2,570 (1.5%)\r\n",
      "Reads written (passing filters):       172,077 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,663,430 bp\r\n",
      "Total written (filtered):     28,655,067 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2311 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "2292 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t2688.7\t0\t15\r\n",
      "4\t4\t672.2\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2258\t2688.7\t0\t2258\r\n",
      "4\t14\t672.2\t0\t14\r\n",
      "5\t18\t168.0\t0\t18\r\n",
      "6\t1\t42.0\t0\t1\r\n",
      "7\t1\t10.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 259 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "255 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t2688.7\t0\t3\r\n",
      "4\t1\t672.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t227\t2688.7\t0\t227\r\n",
      "4\t17\t672.2\t0\t17\r\n",
      "5\t6\t168.0\t0\t6\r\n",
      "6\t1\t42.0\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "46\t1\t0.0\t1\t0 1\r\n",
      "242\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2718.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2718.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.17 s (23 us/read; 2.57 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,067\r\n",
      "Reads with adapters:                     1,528 (1.6%)\r\n",
      "Reads written (passing filters):        93,067 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,803,772 bp\r\n",
      "Total written (filtered):     22,797,979 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1325 times.\r\n",
      "7 times, it overlapped the 5' end of a read\r\n",
      "1318 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1454.2\t0\t7\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1269\t1454.2\t0\t1269\r\n",
      "4\t24\t363.5\t0\t24\r\n",
      "5\t23\t90.9\t0\t23\r\n",
      "6\t2\t22.7\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 203 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "202 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1454.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t190\t1454.2\t0\t190\r\n",
      "4\t3\t363.5\t0\t3\r\n",
      "5\t2\t90.9\t0\t2\r\n",
      "46\t1\t0.0\t1\t0 1\r\n",
      "47\t2\t0.0\t1\t1 1\r\n",
      "241\t1\t0.0\t1\t1\r\n",
      "247\t2\t0.0\t1\t2\r\n",
      "272\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2717.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2717.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.56 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,237\r\n",
      "Reads with adapters:                    65,883 (74.7%)\r\n",
      "Reads written (passing filters):        88,237 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,864,260 bp\r\n",
      "Total written (filtered):     21,605,188 bp (94.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58835 times.\r\n",
      "58529 times, it overlapped the 5' end of a read\r\n",
      "306 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t10\t0.0\t0\t10\r\n",
      "20\t105\t0.0\t0\t105\r\n",
      "21\t58407\t0.0\t0\t58407\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t301\t1378.7\t0\t301\r\n",
      "4\t2\t344.7\t0\t2\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7048 times.\r\n",
      "101 times, it overlapped the 5' end of a read\r\n",
      "6947 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t98\t1378.7\t0\t98\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "28\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1378.7\t0\t108\r\n",
      "4\t6838\t344.7\t0\t6838\r\n",
      "246\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2700.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2700.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.45 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 172,077\r\n",
      "Reads with adapters:                   154,289 (89.7%)\r\n",
      "Reads written (passing filters):       172,077 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,067,518 bp\r\n",
      "Total written (filtered):     26,573,942 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 110536 times.\r\n",
      "110269 times, it overlapped the 5' end of a read\r\n",
      "267 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2688.7\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t11\t0.0\t0\t11\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t172\t0.0\t0\t172\r\n",
      "21\t110068\t0.0\t0\t110068\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t248\t2688.7\t0\t248\r\n",
      "4\t3\t672.2\t0\t3\r\n",
      "47\t4\t0.0\t0\t4\r\n",
      "49\t1\t0.0\t0\t1\r\n",
      "51\t1\t0.0\t0\t1\r\n",
      "52\t2\t0.0\t0\t2\r\n",
      "71\t2\t0.0\t0\t2\r\n",
      "78\t1\t0.0\t0\t1\r\n",
      "250\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 43753 times.\r\n",
      "149 times, it overlapped the 5' end of a read\r\n",
      "43604 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t130\t2688.7\t0\t130\r\n",
      "26\t4\t0.0\t1\t0 4\r\n",
      "27\t15\t0.0\t1\t15\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t190\t2688.7\t0\t190\r\n",
      "4\t43410\t672.2\t0\t43410\r\n",
      "5\t3\t168.0\t0\t3\r\n",
      "246\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2719.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2719.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.29 s (23 us/read; 2.59 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,053\r\n",
      "Reads with adapters:                     1,895 (1.9%)\r\n",
      "Reads written (passing filters):        99,053 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,000,676 bp\r\n",
      "Total written (filtered):     24,994,931 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1611 times.\r\n",
      "12 times, it overlapped the 5' end of a read\r\n",
      "1599 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1547.7\t0\t10\r\n",
      "4\t2\t386.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1575\t1547.7\t0\t1575\r\n",
      "4\t8\t386.9\t0\t8\r\n",
      "5\t12\t96.7\t0\t12\r\n",
      "6\t4\t24.2\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 284 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "284 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t276\t1547.7\t0\t276\r\n",
      "4\t3\t386.9\t0\t3\r\n",
      "5\t4\t96.7\t0\t4\r\n",
      "6\t1\t24.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2718.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2718.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.52 s (49 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,067\r\n",
      "Reads with adapters:                    72,139 (77.5%)\r\n",
      "Reads written (passing filters):        93,067 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,844,755 bp\r\n",
      "Total written (filtered):     21,491,777 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 62886 times.\r\n",
      "62556 times, it overlapped the 5' end of a read\r\n",
      "330 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t363.5\t0\t1\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t104\t0.0\t0\t104\r\n",
      "21\t62436\t0.0\t0\t62436\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t318\t1454.2\t0\t318\r\n",
      "4\t1\t363.5\t0\t1\r\n",
      "5\t3\t90.9\t0\t3\r\n",
      "47\t3\t0.0\t0\t3\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9253 times.\r\n",
      "103 times, it overlapped the 5' end of a read\r\n",
      "9150 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t99\t1454.2\t0\t99\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "27\t3\t0.0\t1\t2 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t154\t1454.2\t0\t154\r\n",
      "4\t8995\t363.5\t0\t8995\r\n",
      "5\t1\t90.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2719.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2719.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.26 s (43 us/read; 1.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  99,053\r\n",
      "Reads with adapters:                    69,827 (70.5%)\r\n",
      "Reads written (passing filters):        99,053 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,090,905 bp\r\n",
      "Total written (filtered):     23,777,313 bp (94.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61209 times.\r\n",
      "60810 times, it overlapped the 5' end of a read\r\n",
      "399 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1547.7\t0\t1\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t90\t0.0\t0\t90\r\n",
      "21\t60704\t0.0\t0\t60704\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t391\t1547.7\t0\t391\r\n",
      "5\t2\t96.7\t0\t2\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8618 times.\r\n",
      "134 times, it overlapped the 5' end of a read\r\n",
      "8484 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t129\t1547.7\t0\t129\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t1547.7\t0\t211\r\n",
      "4\t8273\t386.9\t0\t8273\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2741.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2741.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.66 s (28 us/read; 2.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  96,606\r\n",
      "Reads with adapters:                    92,567 (95.8%)\r\n",
      "Reads written (passing filters):        96,606 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,401,251 bp\r\n",
      "Total written (filtered):     24,904,704 bp (90.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1509.5\t0\t3\r\n",
      "4\t3\t377.4\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1509.5\t0\t34\r\n",
      "8\t1\t1.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 92526 times.\r\n",
      "92494 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1509.5\t0\t2\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "23\t5\t0.0\t1\t5\r\n",
      "24\t12\t0.0\t1\t10 2\r\n",
      "25\t6\t0.0\t1\t2 4\r\n",
      "26\t2905\t0.0\t1\t57 2848\r\n",
      "27\t89332\t0.0\t1\t82063 7269\r\n",
      "28\t229\t0.0\t1\t0 229\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1509.5\t0\t19\r\n",
      "4\t1\t377.4\t0\t1\r\n",
      "5\t6\t94.3\t0\t6\r\n",
      "297\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t4 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2741.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2741.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.29 s (34 us/read; 1.76 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  96,606\r\n",
      "Reads with adapters:                    88,614 (91.7%)\r\n",
      "Reads written (passing filters):        96,606 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,488,776 bp\r\n",
      "Total written (filtered):     25,680,151 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 85603 times.\r\n",
      "85524 times, it overlapped the 5' end of a read\r\n",
      "79 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t85482\t0.0\t0\t85482\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t72\t1509.5\t0\t72\r\n",
      "4\t1\t377.4\t0\t1\r\n",
      "5\t2\t94.3\t0\t2\r\n",
      "7\t2\t5.9\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3011 times.\r\n",
      "56 times, it overlapped the 5' end of a read\r\n",
      "2955 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t55\t1509.5\t0\t55\r\n",
      "9\t1\t0.4\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t160\t1509.5\t0\t160\r\n",
      "4\t2793\t377.4\t0\t2793\r\n",
      "5\t2\t94.3\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2755.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2755.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.66 s (32 us/read; 1.88 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,228\r\n",
      "Reads with adapters:                    54,843 (65.9%)\r\n",
      "Reads written (passing filters):        83,228 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,767,030 bp\r\n",
      "Total written (filtered):     22,660,077 bp (95.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 52322 times.\r\n",
      "52161 times, it overlapped the 5' end of a read\r\n",
      "161 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t37\t0.0\t0\t37\r\n",
      "21\t52118\t0.0\t0\t52118\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t151\t1300.4\t0\t151\r\n",
      "5\t1\t81.3\t0\t1\r\n",
      "7\t4\t5.1\t0\t4\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2521 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "2474 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1300.4\t0\t43\r\n",
      "27\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t493\t1300.4\t0\t493\r\n",
      "4\t1981\t325.1\t0\t1981\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2745.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2745.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.55 s (26 us/read; 2.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,821\r\n",
      "Reads with adapters:                    95,941 (98.1%)\r\n",
      "Reads written (passing filters):        97,821 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,077,805 bp\r\n",
      "Total written (filtered):     26,487,290 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 39 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1528.5\t0\t5\r\n",
      "4\t3\t382.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1528.5\t0\t29\r\n",
      "4\t2\t382.1\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 95902 times.\r\n",
      "95875 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1528.5\t0\t2\r\n",
      "4\t3\t382.1\t0\t3\r\n",
      "5\t1\t95.5\t0\t1\r\n",
      "23\t6\t0.0\t1\t4 2\r\n",
      "24\t6\t0.0\t1\t5 1\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1154\t0.0\t1\t56 1098\r\n",
      "27\t94664\t0.0\t1\t89424 5240\r\n",
      "28\t38\t0.0\t1\t0 38\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1528.5\t0\t9\r\n",
      "5\t8\t95.5\t0\t8\r\n",
      "300\t10\t0.0\t1\t6 4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2745.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2745.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.33 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,821\r\n",
      "Reads with adapters:                    88,020 (90.0%)\r\n",
      "Reads written (passing filters):        97,821 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,108,499 bp\r\n",
      "Total written (filtered):     27,268,454 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 87542 times.\r\n",
      "87494 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t55\t0.0\t0\t55\r\n",
      "21\t87434\t0.0\t0\t87434\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1528.5\t0\t44\r\n",
      "5\t1\t95.5\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 478 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "452 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1528.5\t0\t26\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t181\t1528.5\t0\t181\r\n",
      "4\t271\t382.1\t0\t271\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2755.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2755.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.50 s (30 us/read; 2.00 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,228\r\n",
      "Reads with adapters:                    81,831 (98.3%)\r\n",
      "Reads written (passing filters):        83,228 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,697,199 bp\r\n",
      "Total written (filtered):     21,490,224 bp (90.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 27 times.\r\n",
      "10 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1300.4\t0\t5\r\n",
      "4\t1\t325.1\t0\t1\r\n",
      "5\t2\t81.3\t0\t2\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1300.4\t0\t14\r\n",
      "5\t3\t81.3\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 81804 times.\r\n",
      "81790 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t325.1\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t1\t0.0\t1\t1\r\n",
      "23\t8\t0.0\t1\t5 3\r\n",
      "24\t1\t0.0\t1\t1\r\n",
      "25\t8\t0.0\t1\t6 2\r\n",
      "26\t2659\t0.0\t1\t57 2602\r\n",
      "27\t79083\t0.0\t1\t75863 3220\r\n",
      "28\t28\t0.0\t1\t0 28\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1300.4\t0\t8\r\n",
      "4\t1\t325.1\t0\t1\r\n",
      "5\t1\t81.3\t0\t1\r\n",
      "300\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2766.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2766.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.85 s (27 us/read; 2.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,170\r\n",
      "Reads with adapters:                    64,533 (93.3%)\r\n",
      "Reads written (passing filters):        69,170 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,670,224 bp\r\n",
      "Total written (filtered):     19,319,044 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64265 times.\r\n",
      "64219 times, it overlapped the 5' end of a read\r\n",
      "46 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "20\t24\t0.0\t0\t24\r\n",
      "21\t64192\t0.0\t0\t64192\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1080.8\t0\t38\r\n",
      "4\t1\t270.2\t0\t1\r\n",
      "7\t2\t4.2\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 268 times.\r\n",
      "46 times, it overlapped the 5' end of a read\r\n",
      "222 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t44\t1080.8\t0\t44\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t88\t1080.8\t0\t88\r\n",
      "4\t134\t270.2\t0\t134\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2758.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2758.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.26 s (26 us/read; 2.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,347\r\n",
      "Reads with adapters:                    86,450 (97.9%)\r\n",
      "Reads written (passing filters):        88,347 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,517,583 bp\r\n",
      "Total written (filtered):     24,183,728 bp (91.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "42 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1380.4\t0\t15\r\n",
      "4\t4\t345.1\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1380.4\t0\t39\r\n",
      "4\t2\t345.1\t0\t2\r\n",
      "5\t1\t86.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 86389 times.\r\n",
      "86360 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t1\t0.0\t1\t1\r\n",
      "23\t4\t0.0\t1\t3 1\r\n",
      "24\t10\t0.0\t1\t4 6\r\n",
      "25\t10\t0.0\t1\t3 7\r\n",
      "26\t758\t0.0\t1\t56 702\r\n",
      "27\t85559\t0.0\t1\t82039 3520\r\n",
      "28\t16\t0.0\t1\t0 16\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1380.4\t0\t15\r\n",
      "5\t5\t86.3\t0\t5\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2766.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2766.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.10 s (30 us/read; 1.97 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,170\r\n",
      "Reads with adapters:                    67,912 (98.2%)\r\n",
      "Reads written (passing filters):        69,170 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,664,156 bp\r\n",
      "Total written (filtered):     18,828,041 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 22 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1080.8\t0\t9\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1080.8\t0\t12\r\n",
      "5\t1\t67.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 67890 times.\r\n",
      "67860 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1080.8\t0\t1\r\n",
      "4\t1\t270.2\t0\t1\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t2\t0.0\t1\t0 2\r\n",
      "23\t4\t0.0\t1\t1 3\r\n",
      "24\t4\t0.0\t1\t2 2\r\n",
      "25\t12\t0.0\t1\t4 8\r\n",
      "26\t640\t0.0\t1\t37 603\r\n",
      "27\t67134\t0.0\t1\t63356 3778\r\n",
      "28\t57\t0.0\t1\t0 57\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1080.8\t0\t12\r\n",
      "5\t3\t67.5\t0\t3\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t14\t0.0\t1\t9 5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2758.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2758.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.20 s (47 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  88,347\r\n",
      "Reads with adapters:                    82,732 (93.6%)\r\n",
      "Reads written (passing filters):        88,347 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,526,830 bp\r\n",
      "Total written (filtered):     24,792,897 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 82513 times.\r\n",
      "82473 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "20\t40\t0.0\t0\t40\r\n",
      "21\t82428\t0.0\t0\t82428\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1380.4\t0\t31\r\n",
      "4\t1\t345.1\t0\t1\r\n",
      "5\t1\t86.3\t0\t1\r\n",
      "7\t3\t5.4\t0\t3\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 219 times.\r\n",
      "51 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t51\t1380.4\t0\t51\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t1380.4\t0\t93\r\n",
      "4\t75\t345.1\t0\t75\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2773.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2773.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.16 s (28 us/read; 2.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,832\r\n",
      "Reads with adapters:                    75,408 (96.9%)\r\n",
      "Reads written (passing filters):        77,832 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,599,691 bp\r\n",
      "Total written (filtered):     20,564,164 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 49 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t304.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1216.1\t0\t45\r\n",
      "4\t1\t304.0\t0\t1\r\n",
      "5\t1\t76.0\t0\t1\r\n",
      "6\t1\t19.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 75359 times.\r\n",
      "75339 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1216.1\t0\t1\r\n",
      "4\t1\t304.0\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "24\t8\t0.0\t1\t3 5\r\n",
      "25\t10\t0.0\t1\t6 4\r\n",
      "26\t1042\t0.0\t1\t47 995\r\n",
      "27\t74050\t0.0\t1\t66623 7427\r\n",
      "28\t225\t0.0\t1\t0 225\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1216.1\t0\t10\r\n",
      "4\t2\t304.0\t0\t2\r\n",
      "5\t1\t76.0\t0\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2773.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2773.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.01 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  77,832\r\n",
      "Reads with adapters:                    58,419 (75.1%)\r\n",
      "Reads written (passing filters):        77,832 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,637,621 bp\r\n",
      "Total written (filtered):     21,434,912 bp (94.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 57073 times.\r\n",
      "56950 times, it overlapped the 5' end of a read\r\n",
      "123 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t45\t0.0\t0\t45\r\n",
      "21\t56901\t0.0\t0\t56901\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t100\t1216.1\t0\t100\r\n",
      "4\t13\t304.0\t0\t13\r\n",
      "7\t4\t4.8\t0\t4\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1346 times.\r\n",
      "30 times, it overlapped the 5' end of a read\r\n",
      "1316 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t1216.1\t0\t30\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t463\t1216.1\t0\t463\r\n",
      "4\t852\t304.0\t0\t852\r\n",
      "5\t1\t76.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2788.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2788.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.23 s (27 us/read; 2.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,084\r\n",
      "Reads with adapters:                    78,452 (95.6%)\r\n",
      "Reads written (passing filters):        82,084 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,714,876 bp\r\n",
      "Total written (filtered):     20,599,708 bp (90.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 38 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1282.6\t0\t1\r\n",
      "4\t2\t320.6\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1282.6\t0\t31\r\n",
      "4\t1\t320.6\t0\t1\r\n",
      "5\t2\t80.2\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 78414 times.\r\n",
      "78391 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1282.6\t0\t12\r\n",
      "23\t6\t0.0\t1\t3 3\r\n",
      "24\t9\t0.0\t1\t5 4\r\n",
      "25\t12\t0.0\t1\t5 7\r\n",
      "26\t3485\t0.0\t1\t35 3450\r\n",
      "27\t74740\t0.0\t1\t70996 3744\r\n",
      "28\t127\t0.0\t1\t0 127\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1282.6\t0\t11\r\n",
      "4\t1\t320.6\t0\t1\r\n",
      "5\t3\t80.2\t0\t3\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "300\t7\t0.0\t1\t5 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2772.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2772.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.96 s (47 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,375\r\n",
      "Reads with adapters:                    76,932 (92.3%)\r\n",
      "Reads written (passing filters):        83,375 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,026,002 bp\r\n",
      "Total written (filtered):     21,951,218 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "55 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1302.7\t0\t2\r\n",
      "4\t1\t325.7\t0\t1\r\n",
      "7\t1\t5.1\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1302.7\t0\t52\r\n",
      "5\t3\t81.4\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 76871 times.\r\n",
      "76810 times, it overlapped the 5' end of a read\r\n",
      "61 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1302.7\t0\t2\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t5\t0.0\t1\t1 4\r\n",
      "23\t5\t0.0\t1\t3 2\r\n",
      "24\t6\t0.0\t1\t4 2\r\n",
      "25\t10\t0.0\t1\t7 3\r\n",
      "26\t1175\t0.0\t1\t54 1121\r\n",
      "27\t75581\t0.0\t1\t62917 12664\r\n",
      "28\t24\t0.0\t1\t0 24\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1302.7\t0\t47\r\n",
      "4\t1\t325.7\t0\t1\r\n",
      "5\t7\t81.4\t0\t7\r\n",
      "300\t6\t0.0\t1\t2 4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2772.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2772.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.31 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,375\r\n",
      "Reads with adapters:                    69,522 (83.4%)\r\n",
      "Reads written (passing filters):        83,375 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    24,060,819 bp\r\n",
      "Total written (filtered):     22,632,871 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 67625 times.\r\n",
      "67556 times, it overlapped the 5' end of a read\r\n",
      "69 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t67519\t0.0\t0\t67519\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1302.7\t0\t59\r\n",
      "4\t1\t325.7\t0\t1\r\n",
      "7\t3\t5.1\t0\t3\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1897 times.\r\n",
      "36 times, it overlapped the 5' end of a read\r\n",
      "1861 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1302.7\t0\t34\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t287\t1302.7\t0\t287\r\n",
      "4\t1573\t325.7\t0\t1573\r\n",
      "5\t1\t81.4\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2788.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2788.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.47 s (42 us/read; 1.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  82,084\r\n",
      "Reads with adapters:                    47,681 (58.1%)\r\n",
      "Reads written (passing filters):        82,084 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,797,352 bp\r\n",
      "Total written (filtered):     21,903,756 bp (96.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41630 times.\r\n",
      "41409 times, it overlapped the 5' end of a read\r\n",
      "221 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t21\t0.0\t0\t21\r\n",
      "21\t41387\t0.0\t0\t41387\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t1282.6\t0\t211\r\n",
      "4\t2\t320.6\t0\t2\r\n",
      "5\t2\t80.2\t0\t2\r\n",
      "7\t6\t5.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6051 times.\r\n",
      "41 times, it overlapped the 5' end of a read\r\n",
      "6010 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1282.6\t0\t39\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t873\t1282.6\t0\t873\r\n",
      "4\t5137\t320.6\t0\t5137\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2794.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2794.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.20 s (24 us/read; 2.47 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,862\r\n",
      "Reads with adapters:                    88,915 (97.9%)\r\n",
      "Reads written (passing filters):        90,862 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,244,841 bp\r\n",
      "Total written (filtered):     24,842,981 bp (91.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 15 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "11 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t354.9\t0\t1\r\n",
      "5\t1\t88.7\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1419.7\t0\t10\r\n",
      "5\t1\t88.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 88900 times.\r\n",
      "88876 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1419.7\t0\t4\r\n",
      "22\t8\t0.0\t1\t6 2\r\n",
      "23\t5\t0.0\t1\t1 4\r\n",
      "24\t9\t0.0\t1\t8 1\r\n",
      "25\t8\t0.0\t1\t4 4\r\n",
      "26\t757\t0.0\t1\t37 720\r\n",
      "27\t88057\t0.0\t1\t83790 4267\r\n",
      "28\t28\t0.0\t1\t0 28\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1419.7\t0\t10\r\n",
      "5\t4\t88.7\t0\t4\r\n",
      "300\t10\t0.0\t1\t9 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2794.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2794.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.25 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,862\r\n",
      "Reads with adapters:                    74,309 (81.8%)\r\n",
      "Reads written (passing filters):        90,862 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,249,783 bp\r\n",
      "Total written (filtered):     25,697,865 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 73841 times.\r\n",
      "73759 times, it overlapped the 5' end of a read\r\n",
      "82 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "20\t44\t0.0\t0\t44\r\n",
      "21\t73707\t0.0\t0\t73707\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t1419.7\t0\t73\r\n",
      "5\t1\t88.7\t0\t1\r\n",
      "6\t1\t22.2\t0\t1\r\n",
      "7\t3\t5.5\t0\t3\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 468 times.\r\n",
      "48 times, it overlapped the 5' end of a read\r\n",
      "420 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t46\t1419.7\t0\t46\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t276\t1419.7\t0\t276\r\n",
      "4\t144\t354.9\t0\t144\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2797.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2797.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.58 s (24 us/read; 2.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,874\r\n",
      "Reads with adapters:                    64,883 (98.5%)\r\n",
      "Reads written (passing filters):        65,874 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,706,049 bp\r\n",
      "Total written (filtered):     17,954,294 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 11 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "10 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t257.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1029.3\t0\t9\r\n",
      "4\t1\t257.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 64872 times.\r\n",
      "64859 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t3\t0.0\t1\t2 1\r\n",
      "24\t11\t0.0\t1\t5 6\r\n",
      "25\t8\t0.0\t1\t5 3\r\n",
      "26\t655\t0.0\t1\t39 616\r\n",
      "27\t64168\t0.0\t1\t61682 2486\r\n",
      "28\t14\t0.0\t1\t0 14\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1029.3\t0\t8\r\n",
      "5\t1\t64.3\t0\t1\r\n",
      "300\t4\t0.0\t1\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2866.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2866.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.26 s (22 us/read; 2.72 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,207\r\n",
      "Reads with adapters:                     1,619 (2.8%)\r\n",
      "Reads written (passing filters):        57,207 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,200,674 bp\r\n",
      "Total written (filtered):     16,195,758 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1528 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "1520 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t893.9\t0\t7\r\n",
      "5\t1\t55.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1490\t893.9\t0\t1490\r\n",
      "4\t8\t223.5\t0\t8\r\n",
      "5\t20\t55.9\t0\t20\r\n",
      "6\t2\t14.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 91 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "90 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t893.9\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t88\t893.9\t0\t88\r\n",
      "4\t1\t223.5\t0\t1\r\n",
      "5\t1\t55.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2866.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2866.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.78 s (31 us/read; 1.93 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,207\r\n",
      "Reads with adapters:                    47,247 (82.6%)\r\n",
      "Reads written (passing filters):        57,207 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,234,156 bp\r\n",
      "Total written (filtered):     15,279,531 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45137 times.\r\n",
      "44950 times, it overlapped the 5' end of a read\r\n",
      "187 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t81\t0.0\t0\t81\r\n",
      "21\t44857\t0.0\t0\t44857\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t178\t893.9\t0\t178\r\n",
      "4\t2\t223.5\t0\t2\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2110 times.\r\n",
      "259 times, it overlapped the 5' end of a read\r\n",
      "1851 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t259\t893.9\t0\t259\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t893.9\t0\t35\r\n",
      "4\t1816\t223.5\t0\t1816\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2797.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2797.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.45 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,874\r\n",
      "Reads with adapters:                    57,320 (87.0%)\r\n",
      "Reads written (passing filters):        65,874 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,717,675 bp\r\n",
      "Total written (filtered):     18,519,744 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 56954 times.\r\n",
      "56892 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t25\t0.0\t0\t25\r\n",
      "21\t56863\t0.0\t0\t56863\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1029.3\t0\t54\r\n",
      "5\t1\t64.3\t0\t1\r\n",
      "7\t1\t4.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 366 times.\r\n",
      "28 times, it overlapped the 5' end of a read\r\n",
      "338 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1029.3\t0\t28\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t174\t1029.3\t0\t174\r\n",
      "4\t164\t257.3\t0\t164\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2801.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2801.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.56 s (35 us/read; 1.72 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,531\r\n",
      "Reads with adapters:                    71,759 (97.6%)\r\n",
      "Reads written (passing filters):        73,531 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,461,669 bp\r\n",
      "Total written (filtered):     19,524,130 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 19 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "13 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1148.9\t0\t3\r\n",
      "4\t1\t287.2\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1148.9\t0\t11\r\n",
      "4\t1\t287.2\t0\t1\r\n",
      "5\t1\t71.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 71740 times.\r\n",
      "71721 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "24\t6\t0.0\t1\t3 3\r\n",
      "25\t15\t0.0\t1\t5 10\r\n",
      "26\t1708\t0.0\t1\t48 1660\r\n",
      "27\t69965\t0.0\t1\t67127 2838\r\n",
      "28\t23\t0.0\t1\t0 23\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1148.9\t0\t8\r\n",
      "4\t1\t287.2\t0\t1\r\n",
      "5\t1\t71.8\t0\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2801.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2801.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.81 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,531\r\n",
      "Reads with adapters:                    60,782 (82.7%)\r\n",
      "Reads written (passing filters):        73,531 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,496,423 bp\r\n",
      "Total written (filtered):     20,251,125 bp (94.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58964 times.\r\n",
      "58901 times, it overlapped the 5' end of a read\r\n",
      "63 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "20\t37\t0.0\t0\t37\r\n",
      "21\t58861\t0.0\t0\t58861\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1148.9\t0\t59\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1818 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "1796 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1148.9\t0\t20\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t252\t1148.9\t0\t252\r\n",
      "4\t1544\t287.2\t0\t1544\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2874.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2874.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.79 s (39 us/read; 1.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,478\r\n",
      "Reads with adapters:                     4,867 (4.9%)\r\n",
      "Reads written (passing filters):        98,478 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,392,836 bp\r\n",
      "Total written (filtered):     28,322,974 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2435 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2432 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1538.7\t0\t2\r\n",
      "4\t1\t384.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2417\t1538.7\t0\t2417\r\n",
      "4\t7\t384.7\t0\t7\r\n",
      "5\t8\t96.2\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2432 times.\r\n",
      "2302 times, it overlapped the 5' end of a read\r\n",
      "130 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t384.7\t0\t1\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2298\t0.0\t1\t0 2298\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t121\t1538.7\t0\t121\r\n",
      "4\t3\t384.7\t0\t3\r\n",
      "5\t6\t96.2\t0\t6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2872.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2872.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.58 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,951\r\n",
      "Reads with adapters:                     5,295 (5.1%)\r\n",
      "Reads written (passing filters):       103,951 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,941,785 bp\r\n",
      "Total written (filtered):     29,862,325 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2522 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "2521 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2500\t1624.2\t0\t2500\r\n",
      "4\t8\t406.1\t0\t8\r\n",
      "5\t12\t101.5\t0\t12\r\n",
      "6\t1\t25.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2773 times.\r\n",
      "2649 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1624.2\t0\t1\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2645\t0.0\t1\t0 2645\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1624.2\t0\t114\r\n",
      "4\t8\t406.1\t0\t8\r\n",
      "5\t2\t101.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2872.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2872.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.71 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 103,951\r\n",
      "Reads with adapters:                    76,158 (73.3%)\r\n",
      "Reads written (passing filters):       103,951 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,029,127 bp\r\n",
      "Total written (filtered):     28,466,353 bp (94.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 73826 times.\r\n",
      "73756 times, it overlapped the 5' end of a read\r\n",
      "70 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1624.2\t0\t2\r\n",
      "18\t12\t0.0\t0\t12\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t96\t0.0\t0\t96\r\n",
      "21\t73631\t0.0\t0\t73631\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1624.2\t0\t52\r\n",
      "4\t1\t406.1\t0\t1\r\n",
      "7\t1\t6.3\t0\t1\r\n",
      "266\t2\t0.0\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2332 times.\r\n",
      "129 times, it overlapped the 5' end of a read\r\n",
      "2203 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t128\t1624.2\t0\t128\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1624.2\t0\t21\r\n",
      "4\t2181\t406.1\t0\t2181\r\n",
      "5\t1\t101.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2874.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2874.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.21 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,478\r\n",
      "Reads with adapters:                    60,292 (61.2%)\r\n",
      "Reads written (passing filters):        98,478 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,472,098 bp\r\n",
      "Total written (filtered):     27,236,766 bp (95.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58481 times.\r\n",
      "58425 times, it overlapped the 5' end of a read\r\n",
      "56 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1538.7\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "19\t9\t0.0\t0\t9\r\n",
      "20\t70\t0.0\t0\t70\r\n",
      "21\t58335\t0.0\t0\t58335\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1538.7\t0\t48\r\n",
      "5\t1\t96.2\t0\t1\r\n",
      "7\t3\t6.0\t0\t3\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1811 times.\r\n",
      "67 times, it overlapped the 5' end of a read\r\n",
      "1744 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t66\t1538.7\t0\t66\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1538.7\t0\t26\r\n",
      "4\t1718\t384.7\t0\t1718\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2881.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2881.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.33 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,535\r\n",
      "Reads with adapters:                     4,960 (4.9%)\r\n",
      "Reads written (passing filters):       101,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,416,410 bp\r\n",
      "Total written (filtered):     30,342,646 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2372 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "2369 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1586.5\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2357\t1586.5\t0\t2357\r\n",
      "4\t5\t396.6\t0\t5\r\n",
      "5\t7\t99.2\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2588 times.\r\n",
      "2464 times, it overlapped the 5' end of a read\r\n",
      "124 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1586.5\t0\t11\r\n",
      "4\t1\t396.6\t0\t1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t2451\t0.0\t1\t0 2451\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t114\t1586.5\t0\t114\r\n",
      "4\t3\t396.6\t0\t3\r\n",
      "5\t7\t99.2\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2876.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2876.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.14 s (36 us/read; 1.66 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,277\r\n",
      "Reads with adapters:                     7,299 (5.1%)\r\n",
      "Reads written (passing filters):       142,277 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,654,832 bp\r\n",
      "Total written (filtered):     41,553,599 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3818 times.\r\n",
      "11 times, it overlapped the 5' end of a read\r\n",
      "3807 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t2223.1\t0\t9\r\n",
      "4\t2\t555.8\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3789\t2223.1\t0\t3789\r\n",
      "4\t11\t555.8\t0\t11\r\n",
      "5\t7\t138.9\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3481 times.\r\n",
      "3314 times, it overlapped the 5' end of a read\r\n",
      "167 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2223.1\t0\t8\r\n",
      "4\t2\t555.8\t0\t2\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t3302\t0.0\t1\t0 3302\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t157\t2223.1\t0\t157\r\n",
      "4\t7\t555.8\t0\t7\r\n",
      "5\t3\t138.9\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2881.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2881.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.93 s (58 us/read; 1.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 101,535\r\n",
      "Reads with adapters:                    97,596 (96.1%)\r\n",
      "Reads written (passing filters):       101,535 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    30,446,922 bp\r\n",
      "Total written (filtered):     28,397,709 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97421 times.\r\n",
      "97403 times, it overlapped the 5' end of a read\r\n",
      "18 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t396.6\t0\t2\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t27\t0.0\t0\t27\r\n",
      "20\t141\t0.0\t0\t141\r\n",
      "21\t97222\t0.0\t0\t97222\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1586.5\t0\t7\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 175 times.\r\n",
      "47 times, it overlapped the 5' end of a read\r\n",
      "128 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1586.5\t0\t45\r\n",
      "27\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1586.5\t0\t5\r\n",
      "4\t123\t396.6\t0\t123\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2889.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2889.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.35 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  62,042\r\n",
      "Reads with adapters:                    58,152 (93.7%)\r\n",
      "Reads written (passing filters):        62,042 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,582,165 bp\r\n",
      "Total written (filtered):     13,492,716 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 50418 times.\r\n",
      "50370 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t33\t0.0\t0\t33\r\n",
      "21\t50328\t0.0\t0\t50328\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t969.4\t0\t45\r\n",
      "4\t1\t242.4\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 7734 times.\r\n",
      "42 times, it overlapped the 5' end of a read\r\n",
      "7692 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t969.4\t0\t37\r\n",
      "27\t5\t0.0\t1\t4 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t969.4\t0\t11\r\n",
      "4\t7681\t242.4\t0\t7681\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2889.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2889.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.41 s (55 us/read; 1.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  62,042\r\n",
      "Reads with adapters:                    56,930 (91.8%)\r\n",
      "Reads written (passing filters):        62,042 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,457,334 bp\r\n",
      "Total written (filtered):     12,925,942 bp (89.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t969.4\t0\t4\r\n",
      "4\t1\t242.4\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t969.4\t0\t29\r\n",
      "4\t1\t242.4\t0\t1\r\n",
      "5\t2\t60.6\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 56889 times.\r\n",
      "56848 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t969.4\t0\t6\r\n",
      "21\t1\t0.0\t0\t0 1\r\n",
      "22\t3\t0.0\t1\t2 1\r\n",
      "23\t2\t0.0\t1\t2\r\n",
      "24\t6\t0.0\t1\t2 4\r\n",
      "25\t16\t0.0\t1\t8 8\r\n",
      "26\t5469\t0.0\t1\t52 5417\r\n",
      "27\t51148\t0.0\t1\t45440 5708\r\n",
      "28\t197\t0.0\t1\t0 197\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t969.4\t0\t28\r\n",
      "5\t4\t60.6\t0\t4\r\n",
      "6\t1\t15.1\t0\t1\r\n",
      "52\t2\t0.0\t1\t0 2\r\n",
      "76\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2876.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2876.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.20 s (58 us/read; 1.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 142,277\r\n",
      "Reads with adapters:                   133,188 (93.6%)\r\n",
      "Reads written (passing filters):       142,277 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    41,728,624 bp\r\n",
      "Total written (filtered):     38,952,488 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 131774 times.\r\n",
      "131740 times, it overlapped the 5' end of a read\r\n",
      "34 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2223.1\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t16\t0.0\t0\t16\r\n",
      "19\t39\t0.0\t0\t39\r\n",
      "20\t139\t0.0\t0\t139\r\n",
      "21\t131544\t0.0\t0\t131544\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t2223.1\t0\t19\r\n",
      "4\t1\t555.8\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t12\t0.0\t0\t12\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1414 times.\r\n",
      "96 times, it overlapped the 5' end of a read\r\n",
      "1318 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t93\t2223.1\t0\t93\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2223.1\t0\t8\r\n",
      "4\t1310\t555.8\t0\t1310\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2901.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2901.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.73 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,453\r\n",
      "Reads with adapters:                    78,258 (93.8%)\r\n",
      "Reads written (passing filters):        83,453 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,950,704 bp\r\n",
      "Total written (filtered):     17,840,903 bp (89.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 23 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1304.0\t0\t1\r\n",
      "4\t2\t326.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1304.0\t0\t17\r\n",
      "4\t3\t326.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 78235 times.\r\n",
      "78192 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t326.0\t0\t2\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t4\t0.0\t1\t3 1\r\n",
      "23\t3\t0.0\t1\t1 2\r\n",
      "24\t13\t0.0\t1\t8 5\r\n",
      "25\t21\t0.0\t1\t7 14\r\n",
      "26\t6398\t0.0\t1\t52 6346\r\n",
      "27\t71315\t0.0\t1\t66118 5197\r\n",
      "28\t434\t0.0\t1\t0 434\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1304.0\t0\t23\r\n",
      "4\t1\t326.0\t0\t1\r\n",
      "5\t1\t81.5\t0\t1\r\n",
      "8\t1\t1.3\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "54\t1\t0.0\t1\t1\r\n",
      "300\t15\t0.0\t1\t10 5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2898.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2898.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.74 s (59 us/read; 1.02 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,518\r\n",
      "Reads with adapters:                    76,153 (94.6%)\r\n",
      "Reads written (passing filters):        80,518 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,319,499 bp\r\n",
      "Total written (filtered):     19,823,854 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 70056 times.\r\n",
      "70004 times, it overlapped the 5' end of a read\r\n",
      "52 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t48\t0.0\t0\t48\r\n",
      "21\t69941\t0.0\t0\t69941\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t48\t1258.1\t0\t48\r\n",
      "282\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6097 times.\r\n",
      "40 times, it overlapped the 5' end of a read\r\n",
      "6057 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1258.1\t0\t40\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1258.1\t0\t22\r\n",
      "4\t6034\t314.5\t0\t6034\r\n",
      "5\t1\t78.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2898.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2898.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.27 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  80,518\r\n",
      "Reads with adapters:                    76,035 (94.4%)\r\n",
      "Reads written (passing filters):        80,518 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,219,301 bp\r\n",
      "Total written (filtered):     19,169,021 bp (90.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 22 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1258.1\t0\t19\r\n",
      "4\t1\t314.5\t0\t1\r\n",
      "5\t2\t78.6\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 76013 times.\r\n",
      "75975 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "23\t3\t0.0\t1\t1 2\r\n",
      "24\t9\t0.0\t1\t7 2\r\n",
      "25\t12\t0.0\t1\t5 7\r\n",
      "26\t4576\t0.0\t1\t59 4517\r\n",
      "27\t71161\t0.0\t1\t66845 4316\r\n",
      "28\t212\t0.0\t1\t0 212\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1258.1\t0\t21\r\n",
      "4\t1\t314.5\t0\t1\r\n",
      "5\t3\t78.6\t0\t3\r\n",
      "48\t1\t0.0\t1\t1\r\n",
      "96\t2\t0.0\t1\t2\r\n",
      "300\t10\t0.0\t1\t7 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2901.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2901.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.45 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  83,453\r\n",
      "Reads with adapters:                    78,087 (93.6%)\r\n",
      "Reads written (passing filters):        83,453 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,252,604 bp\r\n",
      "Total written (filtered):     18,753,848 bp (92.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 69817 times.\r\n",
      "69755 times, it overlapped the 5' end of a read\r\n",
      "62 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t40\t0.0\t0\t40\r\n",
      "21\t69708\t0.0\t0\t69708\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t1304.0\t0\t57\r\n",
      "7\t1\t5.1\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "54\t1\t0.0\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8270 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "8227 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t40\t1304.0\t0\t40\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1304.0\t0\t24\r\n",
      "4\t8203\t326.0\t0\t8203\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2910.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2910.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.54 s (37 us/read; 1.63 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,717\r\n",
      "Reads with adapters:                    38,325 (91.9%)\r\n",
      "Reads written (passing filters):        41,717 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,706,060 bp\r\n",
      "Total written (filtered):      8,676,908 bp (89.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 21 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "19 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t2\t163.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t651.8\t0\t18\r\n",
      "7\t1\t2.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 38304 times.\r\n",
      "38279 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t2\t0.0\t1\t1 1\r\n",
      "24\t7\t0.0\t1\t5 2\r\n",
      "25\t7\t0.0\t1\t0 7\r\n",
      "26\t4647\t0.0\t1\t26 4621\r\n",
      "27\t33546\t0.0\t1\t29515 4031\r\n",
      "28\t70\t0.0\t1\t0 70\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t651.8\t0\t19\r\n",
      "5\t4\t40.7\t0\t4\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "52\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2903.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2903.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.75 s (40 us/read; 1.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,999\r\n",
      "Reads with adapters:                    78,904 (83.9%)\r\n",
      "Reads written (passing filters):        93,999 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,179,213 bp\r\n",
      "Total written (filtered):     24,595,007 bp (93.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 74620 times.\r\n",
      "74492 times, it overlapped the 5' end of a read\r\n",
      "128 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "20\t46\t0.0\t0\t46\r\n",
      "21\t74441\t0.0\t0\t74441\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t113\t1468.7\t0\t113\r\n",
      "4\t2\t367.2\t0\t2\r\n",
      "7\t3\t5.7\t0\t3\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4284 times.\r\n",
      "55 times, it overlapped the 5' end of a read\r\n",
      "4229 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t54\t1468.7\t0\t54\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1468.7\t0\t29\r\n",
      "4\t4200\t367.2\t0\t4200\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2910.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2910.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.53 s (61 us/read; 0.99 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,717\r\n",
      "Reads with adapters:                    39,611 (95.0%)\r\n",
      "Reads written (passing filters):        41,717 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,855,046 bp\r\n",
      "Total written (filtered):      9,114,439 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 34244 times.\r\n",
      "34216 times, it overlapped the 5' end of a read\r\n",
      "28 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "20\t29\t0.0\t0\t29\r\n",
      "21\t34181\t0.0\t0\t34181\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t651.8\t0\t25\r\n",
      "6\t1\t10.2\t0\t1\r\n",
      "300\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 5367 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "5350 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t651.8\t0\t17\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t651.8\t0\t14\r\n",
      "4\t5336\t163.0\t0\t5336\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2903.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2903.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.57 s (38 us/read; 1.58 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,999\r\n",
      "Reads with adapters:                    87,045 (92.6%)\r\n",
      "Reads written (passing filters):        93,999 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,150,544 bp\r\n",
      "Total written (filtered):     23,804,463 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "44 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t367.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1468.7\t0\t36\r\n",
      "5\t7\t91.8\t0\t7\r\n",
      "6\t1\t22.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 87000 times.\r\n",
      "86929 times, it overlapped the 5' end of a read\r\n",
      "71 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1468.7\t0\t3\r\n",
      "20\t1\t0.0\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t7\t0.0\t1\t7\r\n",
      "23\t7\t0.0\t1\t6 1\r\n",
      "24\t15\t0.0\t1\t10 5\r\n",
      "25\t11\t0.0\t1\t5 6\r\n",
      "26\t3954\t0.0\t1\t63 3891\r\n",
      "27\t82800\t0.0\t1\t70639 12161\r\n",
      "28\t129\t0.0\t1\t0 129\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t1468.7\t0\t58\r\n",
      "4\t1\t367.2\t0\t1\r\n",
      "5\t3\t91.8\t0\t3\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "300\t8\t0.0\t1\t5 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2909.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2909.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.60 s (58 us/read; 1.04 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,440\r\n",
      "Reads with adapters:                    65,971 (67.7%)\r\n",
      "Reads written (passing filters):        97,440 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,163,120 bp\r\n",
      "Total written (filtered):     20,082,553 bp (94.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 48350 times.\r\n",
      "48011 times, it overlapped the 5' end of a read\r\n",
      "339 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "20\t35\t0.0\t0\t35\r\n",
      "21\t47973\t0.0\t0\t47973\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t328\t1522.5\t0\t328\r\n",
      "4\t3\t380.6\t0\t3\r\n",
      "7\t5\t5.9\t0\t5\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17621 times.\r\n",
      "46 times, it overlapped the 5' end of a read\r\n",
      "17575 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1522.5\t0\t43\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t62\t1522.5\t0\t62\r\n",
      "4\t17512\t380.6\t0\t17512\r\n",
      "5\t1\t95.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2909.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2909.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.21 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  97,440\r\n",
      "Reads with adapters:                    88,393 (90.7%)\r\n",
      "Reads written (passing filters):        97,440 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,727,413 bp\r\n",
      "Total written (filtered):     18,352,295 bp (88.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 32 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "24 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1522.5\t0\t3\r\n",
      "4\t2\t380.6\t0\t2\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1522.5\t0\t22\r\n",
      "4\t1\t380.6\t0\t1\r\n",
      "6\t1\t23.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 88361 times.\r\n",
      "88325 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t1\t95.2\t0\t1\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t4\t0.0\t1\t3 1\r\n",
      "23\t9\t0.0\t1\t0 9\r\n",
      "24\t15\t0.0\t1\t11 4\r\n",
      "25\t17\t0.0\t1\t5 12\r\n",
      "26\t12488\t0.0\t1\t55 12433\r\n",
      "27\t75729\t0.0\t1\t69673 6056\r\n",
      "28\t59\t0.0\t1\t0 59\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1522.5\t0\t22\r\n",
      "5\t4\t95.2\t0\t4\r\n",
      "6\t1\t23.8\t0\t1\r\n",
      "299\t1\t0.0\t1\t0 1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2933.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2933.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.69 s (39 us/read; 1.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,794\r\n",
      "Reads with adapters:                    62,305 (89.3%)\r\n",
      "Reads written (passing filters):        69,794 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,807,847 bp\r\n",
      "Total written (filtered):     14,131,802 bp (89.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 29 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1090.5\t0\t2\r\n",
      "4\t1\t272.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1090.5\t0\t21\r\n",
      "4\t1\t272.6\t0\t1\r\n",
      "5\t3\t68.2\t0\t3\r\n",
      "6\t1\t17.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 62276 times.\r\n",
      "62250 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t6\t0.0\t1\t5 1\r\n",
      "25\t6\t0.0\t1\t4 2\r\n",
      "26\t7028\t0.0\t1\t26 7002\r\n",
      "27\t55099\t0.0\t1\t51797 3302\r\n",
      "28\t108\t0.0\t1\t0 108\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t15\t1090.5\t0\t15\r\n",
      "4\t2\t272.6\t0\t2\r\n",
      "5\t2\t68.2\t0\t2\r\n",
      "300\t7\t0.0\t1\t5 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2926.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2926.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.37 s (34 us/read; 1.75 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,593\r\n",
      "Reads with adapters:                     4,350 (4.4%)\r\n",
      "Reads written (passing filters):        98,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,206,424 bp\r\n",
      "Total written (filtered):     28,147,941 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2279 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "2274 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1540.5\t0\t3\r\n",
      "4\t2\t385.1\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2259\t1540.5\t0\t2259\r\n",
      "4\t8\t385.1\t0\t8\r\n",
      "5\t7\t96.3\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2071 times.\r\n",
      "1903 times, it overlapped the 5' end of a read\r\n",
      "168 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t11\t1540.5\t0\t11\r\n",
      "24\t2\t0.0\t1\t0 2\r\n",
      "25\t1\t0.0\t1\t0 1\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1888\t0.0\t1\t0 1888\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t161\t1540.5\t0\t161\r\n",
      "4\t4\t385.1\t0\t4\r\n",
      "5\t3\t96.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2913.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2913.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.01 s (49 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,637\r\n",
      "Reads with adapters:                   115,062 (94.6%)\r\n",
      "Reads written (passing filters):       121,637 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,663,170 bp\r\n",
      "Total written (filtered):     30,365,297 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 108141 times.\r\n",
      "108075 times, it overlapped the 5' end of a read\r\n",
      "66 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t15\t0.0\t0\t15\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t55\t0.0\t0\t55\r\n",
      "21\t108002\t0.0\t0\t108002\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t64\t1900.6\t0\t64\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6921 times.\r\n",
      "69 times, it overlapped the 5' end of a read\r\n",
      "6852 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t68\t1900.6\t0\t68\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1900.6\t0\t21\r\n",
      "4\t6829\t475.1\t0\t6829\r\n",
      "5\t2\t118.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2933.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2933.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.67 s (53 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  69,794\r\n",
      "Reads with adapters:                    66,220 (94.9%)\r\n",
      "Reads written (passing filters):        69,794 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    16,006,884 bp\r\n",
      "Total written (filtered):     14,806,631 bp (92.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 55037 times.\r\n",
      "54997 times, it overlapped the 5' end of a read\r\n",
      "40 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "20\t26\t0.0\t0\t26\r\n",
      "21\t54970\t0.0\t0\t54970\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t1090.5\t0\t39\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11183 times.\r\n",
      "26 times, it overlapped the 5' end of a read\r\n",
      "11157 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1090.5\t0\t25\r\n",
      "5\t1\t68.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1090.5\t0\t23\r\n",
      "4\t11133\t272.6\t0\t11133\r\n",
      "248\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2913.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2913.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.43 s (53 us/read; 1.13 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 121,637\r\n",
      "Reads with adapters:                   115,179 (94.7%)\r\n",
      "Reads written (passing filters):       121,637 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    32,444,872 bp\r\n",
      "Total written (filtered):     29,339,233 bp (90.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 53 times.\r\n",
      "3 times, it overlapped the 5' end of a read\r\n",
      "50 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1900.6\t0\t1\r\n",
      "4\t1\t475.1\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t1900.6\t0\t47\r\n",
      "4\t1\t475.1\t0\t1\r\n",
      "6\t2\t29.7\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 115126 times.\r\n",
      "115047 times, it overlapped the 5' end of a read\r\n",
      "79 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1900.6\t0\t1\r\n",
      "20\t2\t0.0\t0\t2\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t4\t0.0\t1\t2 2\r\n",
      "23\t5\t0.0\t1\t4 1\r\n",
      "24\t17\t0.0\t1\t8 9\r\n",
      "25\t28\t0.0\t1\t5 23\r\n",
      "26\t6036\t0.0\t1\t76 5960\r\n",
      "27\t108479\t0.0\t1\t76036 32443\r\n",
      "28\t472\t0.0\t1\t0 472\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t56\t1900.6\t0\t56\r\n",
      "5\t2\t118.8\t0\t2\r\n",
      "6\t1\t29.7\t0\t1\r\n",
      "47\t3\t0.0\t1\t3\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "73\t1\t0.0\t1\t1\r\n",
      "297\t1\t0.0\t1\t0 1\r\n",
      "298\t2\t0.0\t1\t0 2\r\n",
      "300\t12\t0.0\t1\t9 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2926.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2926.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.66 s (57 us/read; 1.05 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,593\r\n",
      "Reads with adapters:                    81,672 (82.8%)\r\n",
      "Reads written (passing filters):        98,593 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    28,299,238 bp\r\n",
      "Total written (filtered):     26,625,771 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 79171 times.\r\n",
      "79135 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "19\t22\t0.0\t0\t22\r\n",
      "20\t126\t0.0\t0\t126\r\n",
      "21\t78979\t0.0\t0\t78979\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1540.5\t0\t29\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2501 times.\r\n",
      "116 times, it overlapped the 5' end of a read\r\n",
      "2385 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t115\t1540.5\t0\t115\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1540.5\t0\t18\r\n",
      "4\t2367\t385.1\t0\t2367\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2939.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2939.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.18 s (30 us/read; 2.03 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,673\r\n",
      "Reads with adapters:                    62,638 (85.0%)\r\n",
      "Reads written (passing filters):        73,673 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,630,912 bp\r\n",
      "Total written (filtered):     13,948,534 bp (89.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 21 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1151.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1151.1\t0\t18\r\n",
      "4\t1\t287.8\t0\t1\r\n",
      "5\t1\t71.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 62617 times.\r\n",
      "62579 times, it overlapped the 5' end of a read\r\n",
      "38 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "5\t2\t71.9\t0\t2\r\n",
      "23\t3\t0.0\t1\t2 1\r\n",
      "24\t8\t0.0\t1\t3 5\r\n",
      "25\t18\t0.0\t1\t6 12\r\n",
      "26\t10394\t0.0\t1\t40 10354\r\n",
      "27\t51835\t0.0\t1\t48391 3444\r\n",
      "28\t319\t0.0\t1\t0 319\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t26\t1151.1\t0\t26\r\n",
      "4\t1\t287.8\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "300\t9\t0.0\t1\t3 6\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2939.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2939.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.94 s (53 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  73,673\r\n",
      "Reads with adapters:                    63,615 (86.3%)\r\n",
      "Reads written (passing filters):        73,673 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    15,874,100 bp\r\n",
      "Total written (filtered):     14,837,421 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 46113 times.\r\n",
      "46006 times, it overlapped the 5' end of a read\r\n",
      "107 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "20\t29\t0.0\t0\t29\r\n",
      "21\t45971\t0.0\t0\t45971\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t104\t1151.1\t0\t104\r\n",
      "6\t1\t18.0\t0\t1\r\n",
      "7\t1\t4.5\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 17502 times.\r\n",
      "34 times, it overlapped the 5' end of a read\r\n",
      "17468 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t31\t1151.1\t0\t31\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t63\t1151.1\t0\t63\r\n",
      "4\t17404\t287.8\t0\t17404\r\n",
      "5\t1\t71.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2951.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2951.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.14 s (33 us/read; 1.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,017\r\n",
      "Reads with adapters:                   115,827 (91.9%)\r\n",
      "Reads written (passing filters):       126,017 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,965,593 bp\r\n",
      "Total written (filtered):     17,056,760 bp (89.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 85207 times.\r\n",
      "85026 times, it overlapped the 5' end of a read\r\n",
      "181 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1969.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "20\t52\t0.0\t0\t52\r\n",
      "21\t84963\t0.0\t0\t84963\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t168\t1969.0\t0\t168\r\n",
      "5\t2\t123.1\t0\t2\r\n",
      "7\t5\t7.7\t0\t5\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "50\t1\t0.0\t0\t1\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "131\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 30620 times.\r\n",
      "32 times, it overlapped the 5' end of a read\r\n",
      "30588 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1969.0\t0\t32\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t250\t1969.0\t0\t250\r\n",
      "4\t30333\t492.3\t0\t30333\r\n",
      "5\t5\t123.1\t0\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2938.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2938.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.39 s (54 us/read; 1.12 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 118,858\r\n",
      "Reads with adapters:                   112,310 (94.5%)\r\n",
      "Reads written (passing filters):       118,858 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,818,264 bp\r\n",
      "Total written (filtered):     27,708,398 bp (92.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 97551 times.\r\n",
      "97451 times, it overlapped the 5' end of a read\r\n",
      "100 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "20\t41\t0.0\t0\t41\r\n",
      "21\t97401\t0.0\t0\t97401\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t87\t1857.2\t0\t87\r\n",
      "4\t2\t464.3\t0\t2\r\n",
      "7\t1\t7.3\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 14759 times.\r\n",
      "53 times, it overlapped the 5' end of a read\r\n",
      "14706 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t52\t1857.2\t0\t52\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1857.2\t0\t50\r\n",
      "4\t14635\t464.3\t0\t14635\r\n",
      "5\t1\t116.1\t0\t1\r\n",
      "10\t1\t0.1\t0\t1\r\n",
      "59\t17\t0.0\t1\t15 2\r\n",
      "276\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2938.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2938.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.93 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 118,858\r\n",
      "Reads with adapters:                    99,851 (84.0%)\r\n",
      "Reads written (passing filters):       118,858 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    29,563,542 bp\r\n",
      "Total written (filtered):     26,887,195 bp (90.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 285 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "279 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1857.2\t0\t4\r\n",
      "4\t1\t464.3\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t261\t1857.2\t0\t261\r\n",
      "4\t3\t464.3\t0\t3\r\n",
      "5\t13\t116.1\t0\t13\r\n",
      "6\t2\t29.0\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 99566 times.\r\n",
      "99320 times, it overlapped the 5' end of a read\r\n",
      "246 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1857.2\t0\t1\r\n",
      "4\t2\t464.3\t0\t2\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t4\t0.0\t1\t1 3\r\n",
      "23\t5\t0.0\t1\t1 4\r\n",
      "24\t5\t0.0\t1\t2 3\r\n",
      "25\t20\t0.0\t1\t2 18\r\n",
      "26\t11241\t0.0\t1\t72 11169\r\n",
      "27\t87815\t0.0\t1\t77089 10726\r\n",
      "28\t225\t0.0\t1\t0 225\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t218\t1857.2\t0\t218\r\n",
      "4\t2\t464.3\t0\t2\r\n",
      "5\t10\t116.1\t0\t10\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "297\t1\t0.0\t1\t0 1\r\n",
      "300\t13\t0.0\t1\t8 5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2956.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2956.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.08 s (45 us/read; 1.34 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,910\r\n",
      "Reads with adapters:                    65,392 (71.9%)\r\n",
      "Reads written (passing filters):        90,910 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,859,947 bp\r\n",
      "Total written (filtered):     19,117,598 bp (91.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 638 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "632 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1420.5\t0\t1\r\n",
      "4\t3\t355.1\t0\t3\r\n",
      "7\t1\t5.5\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t606\t1420.5\t0\t606\r\n",
      "4\t4\t355.1\t0\t4\r\n",
      "5\t14\t88.8\t0\t14\r\n",
      "6\t6\t22.2\t0\t6\r\n",
      "7\t1\t5.5\t0\t1\r\n",
      "8\t1\t1.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 64754 times.\r\n",
      "64541 times, it overlapped the 5' end of a read\r\n",
      "213 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "7\t1\t5.5\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t7\t0.0\t1\t2 5\r\n",
      "25\t13\t0.0\t1\t4 9\r\n",
      "26\t5127\t0.0\t1\t39 5088\r\n",
      "27\t58876\t0.0\t1\t44816 14060\r\n",
      "28\t514\t0.0\t1\t0 514\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t140\t1420.5\t0\t140\r\n",
      "4\t7\t355.1\t0\t7\r\n",
      "5\t55\t88.8\t0\t55\r\n",
      "6\t2\t22.2\t0\t2\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "59\t2\t0.0\t1\t0 2\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t4\t0.0\t1\t1 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2964.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2964.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.01 s (22 us/read; 2.70 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,392\r\n",
      "Reads with adapters:                    84,001 (92.9%)\r\n",
      "Reads written (passing filters):        90,392 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,528,637 bp\r\n",
      "Total written (filtered):     18,264,245 bp (89.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1412.4\t0\t1\r\n",
      "4\t3\t353.1\t0\t3\r\n",
      "5\t1\t88.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t19\t1412.4\t0\t19\r\n",
      "4\t4\t353.1\t0\t4\r\n",
      "5\t5\t88.3\t0\t5\r\n",
      "6\t2\t22.1\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83966 times.\r\n",
      "83934 times, it overlapped the 5' end of a read\r\n",
      "32 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t4\t353.1\t0\t4\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "23\t7\t0.0\t1\t2 5\r\n",
      "24\t10\t0.0\t1\t4 6\r\n",
      "25\t13\t0.0\t1\t4 9\r\n",
      "26\t5135\t0.0\t1\t61 5074\r\n",
      "27\t78009\t0.0\t1\t73478 4531\r\n",
      "28\t753\t0.0\t1\t0 753\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1412.4\t0\t17\r\n",
      "4\t3\t353.1\t0\t3\r\n",
      "5\t1\t88.3\t0\t1\r\n",
      "47\t2\t0.0\t1\t2\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t6\t0.0\t1\t3 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2956.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2956.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.07 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,910\r\n",
      "Reads with adapters:                    86,039 (94.6%)\r\n",
      "Reads written (passing filters):        90,910 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,125,127 bp\r\n",
      "Total written (filtered):     19,464,154 bp (92.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 77411 times.\r\n",
      "77359 times, it overlapped the 5' end of a read\r\n",
      "52 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "20\t59\t0.0\t0\t59\r\n",
      "21\t77292\t0.0\t0\t77292\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1420.5\t0\t38\r\n",
      "4\t3\t355.1\t0\t3\r\n",
      "5\t1\t88.8\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "59\t2\t0.0\t0\t2\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "248\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8628 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "8590 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t37\t1420.5\t0\t37\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1420.5\t0\t83\r\n",
      "4\t8506\t355.1\t0\t8506\r\n",
      "5\t1\t88.8\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2951.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2951.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.29 s (50 us/read; 1.20 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 126,017\r\n",
      "Reads with adapters:                   108,685 (86.2%)\r\n",
      "Reads written (passing filters):       126,017 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,618,543 bp\r\n",
      "Total written (filtered):     14,699,893 bp (83.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 52 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "48 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1969.0\t0\t1\r\n",
      "4\t3\t492.3\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t1969.0\t0\t41\r\n",
      "4\t3\t492.3\t0\t3\r\n",
      "5\t3\t123.1\t0\t3\r\n",
      "6\t1\t30.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 108633 times.\r\n",
      "108578 times, it overlapped the 5' end of a read\r\n",
      "55 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1969.0\t0\t2\r\n",
      "4\t1\t492.3\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "23\t6\t0.0\t1\t0 6\r\n",
      "24\t17\t0.0\t1\t9 8\r\n",
      "25\t19\t0.0\t1\t4 15\r\n",
      "26\t17596\t0.0\t1\t52 17544\r\n",
      "27\t88053\t0.0\t1\t80896 7157\r\n",
      "28\t2883\t0.0\t1\t0 2883\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t33\t1969.0\t0\t33\r\n",
      "4\t2\t492.3\t0\t2\r\n",
      "5\t4\t123.1\t0\t4\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "46\t1\t0.0\t1\t0 1\r\n",
      "47\t3\t0.0\t1\t3\r\n",
      "55\t1\t0.0\t1\t1\r\n",
      "56\t1\t0.0\t1\t1\r\n",
      "60\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "69\t1\t0.0\t1\t1\r\n",
      "87\t1\t0.0\t1\t1\r\n",
      "93\t1\t0.0\t1\t0 1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2964.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2964.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.70 s (52 us/read; 1.16 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  90,392\r\n",
      "Reads with adapters:                    68,738 (76.0%)\r\n",
      "Reads written (passing filters):        90,392 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,828,625 bp\r\n",
      "Total written (filtered):     19,547,442 bp (93.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 59357 times.\r\n",
      "59194 times, it overlapped the 5' end of a read\r\n",
      "163 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "20\t31\t0.0\t0\t31\r\n",
      "21\t59154\t0.0\t0\t59154\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t146\t1412.4\t0\t146\r\n",
      "4\t5\t353.1\t0\t5\r\n",
      "5\t4\t88.3\t0\t4\r\n",
      "6\t4\t22.1\t0\t4\r\n",
      "7\t2\t5.5\t0\t2\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9381 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "9352 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1412.4\t0\t28\r\n",
      "4\t1\t353.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t450\t1412.4\t0\t450\r\n",
      "4\t8897\t353.1\t0\t8897\r\n",
      "5\t4\t88.3\t0\t4\r\n",
      "9\t1\t0.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2965.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2965.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.98 s (25 us/read; 2.39 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,080\r\n",
      "Reads with adapters:                    77,360 (97.8%)\r\n",
      "Reads written (passing filters):        79,080 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,156,128 bp\r\n",
      "Total written (filtered):     21,067,278 bp (91.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 20 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t17\t1235.6\t0\t17\r\n",
      "4\t1\t308.9\t0\t1\r\n",
      "5\t1\t77.2\t0\t1\r\n",
      "6\t1\t19.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 77340 times.\r\n",
      "77315 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1235.6\t0\t1\r\n",
      "4\t3\t308.9\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "23\t2\t0.0\t1\t1 1\r\n",
      "24\t7\t0.0\t1\t4 3\r\n",
      "25\t17\t0.0\t1\t10 7\r\n",
      "26\t955\t0.0\t1\t47 908\r\n",
      "27\t76224\t0.0\t1\t72119 4105\r\n",
      "28\t105\t0.0\t1\t0 105\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t12\t1235.6\t0\t12\r\n",
      "4\t1\t308.9\t0\t1\r\n",
      "5\t2\t77.2\t0\t2\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "49\t1\t0.0\t1\t0 1\r\n",
      "51\t1\t0.0\t1\t1\r\n",
      "297\t2\t0.0\t1\t2\r\n",
      "300\t5\t0.0\t1\t5\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2965.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2965.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.68 s (34 us/read; 1.77 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  79,080\r\n",
      "Reads with adapters:                    46,586 (58.9%)\r\n",
      "Reads written (passing filters):        79,080 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,181,023 bp\r\n",
      "Total written (filtered):     22,226,969 bp (95.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 45300 times.\r\n",
      "45136 times, it overlapped the 5' end of a read\r\n",
      "164 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "20\t30\t0.0\t0\t30\r\n",
      "21\t45101\t0.0\t0\t45101\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t150\t1235.6\t0\t150\r\n",
      "4\t1\t308.9\t0\t1\r\n",
      "5\t2\t77.2\t0\t2\r\n",
      "7\t7\t4.8\t0\t7\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1286 times.\r\n",
      "32 times, it overlapped the 5' end of a read\r\n",
      "1254 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t1235.6\t0\t32\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t580\t1235.6\t0\t580\r\n",
      "4\t674\t308.9\t0\t674\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2983.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2983.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.90 s (36 us/read; 1.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,834\r\n",
      "Reads with adapters:                     5,945 (5.6%)\r\n",
      "Reads written (passing filters):       106,834 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,695,751 bp\r\n",
      "Total written (filtered):     31,614,149 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3143 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "3138 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t1669.3\t0\t5\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3126\t1669.3\t0\t3126\r\n",
      "4\t7\t417.3\t0\t7\r\n",
      "5\t4\t104.3\t0\t4\r\n",
      "6\t1\t26.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2802 times.\r\n",
      "2660 times, it overlapped the 5' end of a read\r\n",
      "142 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1669.3\t0\t2\r\n",
      "4\t2\t417.3\t0\t2\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t2653\t0.0\t1\t0 2653\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t139\t1669.3\t0\t139\r\n",
      "5\t3\t104.3\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2983.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2983.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.09 s (48 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 106,834\r\n",
      "Reads with adapters:                   103,420 (96.8%)\r\n",
      "Reads written (passing filters):       106,834 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    31,740,487 bp\r\n",
      "Total written (filtered):     29,577,874 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 102753 times.\r\n",
      "102738 times, it overlapped the 5' end of a read\r\n",
      "15 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t9\t0.0\t0\t9\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t128\t0.0\t0\t128\r\n",
      "21\t102585\t0.0\t0\t102585\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1669.3\t0\t6\r\n",
      "300\t9\t0.0\t0\t9\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 667 times.\r\n",
      "125 times, it overlapped the 5' end of a read\r\n",
      "542 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t123\t1669.3\t0\t123\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1669.3\t0\t2\r\n",
      "4\t540\t417.3\t0\t540\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2974.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2974.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.73 s (36 us/read; 1.65 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 157,459\r\n",
      "Reads with adapters:                   146,650 (93.1%)\r\n",
      "Reads written (passing filters):       157,459 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,449,303 bp\r\n",
      "Total written (filtered):     35,495,938 bp (90.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 64 times.\r\n",
      "19 times, it overlapped the 5' end of a read\r\n",
      "45 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t2460.3\t0\t8\r\n",
      "4\t10\t615.1\t0\t10\r\n",
      "5\t1\t153.8\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t2460.3\t0\t38\r\n",
      "4\t4\t615.1\t0\t4\r\n",
      "5\t2\t153.8\t0\t2\r\n",
      "6\t1\t38.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 146586 times.\r\n",
      "146520 times, it overlapped the 5' end of a read\r\n",
      "66 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2460.3\t0\t1\r\n",
      "4\t1\t615.1\t0\t1\r\n",
      "19\t2\t0.0\t0\t2\r\n",
      "20\t2\t0.0\t0\t2\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t2\t0.0\t1\t1 1\r\n",
      "23\t6\t0.0\t1\t4 2\r\n",
      "24\t17\t0.0\t1\t9 8\r\n",
      "25\t28\t0.0\t1\t9 19\r\n",
      "26\t7781\t0.0\t1\t103 7678\r\n",
      "27\t137662\t0.0\t1\t126937 10725\r\n",
      "28\t1015\t0.0\t1\t0 1015\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t2460.3\t0\t35\r\n",
      "4\t3\t615.1\t0\t3\r\n",
      "5\t11\t153.8\t0\t11\r\n",
      "47\t1\t0.0\t1\t0 1\r\n",
      "56\t1\t0.0\t1\t0 1\r\n",
      "59\t1\t0.0\t1\t1\r\n",
      "90\t2\t0.0\t1\t2\r\n",
      "298\t1\t0.0\t1\t0 1\r\n",
      "300\t11\t0.0\t1\t10 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2974.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2974.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 7.40 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 157,459\r\n",
      "Reads with adapters:                   127,355 (80.9%)\r\n",
      "Reads written (passing filters):       157,459 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    39,542,761 bp\r\n",
      "Total written (filtered):     37,070,739 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 115515 times.\r\n",
      "115276 times, it overlapped the 5' end of a read\r\n",
      "239 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t70\t0.0\t0\t70\r\n",
      "21\t115188\t0.0\t0\t115188\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t2460.3\t0\t211\r\n",
      "4\t1\t615.1\t0\t1\r\n",
      "5\t4\t153.8\t0\t4\r\n",
      "6\t1\t38.4\t0\t1\r\n",
      "7\t8\t9.6\t0\t8\r\n",
      "8\t1\t2.4\t0\t1\r\n",
      "299\t2\t0.0\t0\t2\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11840 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "11802 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t2460.3\t0\t38\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t592\t2460.3\t0\t592\r\n",
      "4\t11208\t615.1\t0\t11208\r\n",
      "5\t2\t153.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2987.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2987.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.78 s (32 us/read; 1.86 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 148,349\r\n",
      "Reads with adapters:                     7,070 (4.8%)\r\n",
      "Reads written (passing filters):       148,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,467,797 bp\r\n",
      "Total written (filtered):     43,367,567 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 3590 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "3588 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2318.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3570\t2318.0\t0\t3570\r\n",
      "4\t10\t579.5\t0\t10\r\n",
      "5\t8\t144.9\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3480 times.\r\n",
      "3291 times, it overlapped the 5' end of a read\r\n",
      "189 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t3288\t0.0\t1\t0 3288\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t181\t2318.0\t0\t181\r\n",
      "4\t1\t579.5\t0\t1\r\n",
      "5\t7\t144.9\t0\t7\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2991.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2991.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.74 s (44 us/read; 1.38 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 131,756\r\n",
      "Reads with adapters:                   116,394 (88.3%)\r\n",
      "Reads written (passing filters):       131,756 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,457,186 bp\r\n",
      "Total written (filtered):     31,184,505 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 106459 times.\r\n",
      "106261 times, it overlapped the 5' end of a read\r\n",
      "198 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "8\t1\t2.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "20\t62\t0.0\t0\t62\r\n",
      "21\t106188\t0.0\t0\t106188\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t192\t2058.7\t0\t192\r\n",
      "4\t1\t514.7\t0\t1\r\n",
      "6\t1\t32.2\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9935 times.\r\n",
      "75 times, it overlapped the 5' end of a read\r\n",
      "9860 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t69\t2058.7\t0\t69\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t4\t0.0\t1\t3 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t39\t2058.7\t0\t39\r\n",
      "4\t9817\t514.7\t0\t9817\r\n",
      "5\t4\t128.7\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2991.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2991.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 6.37 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 131,756\r\n",
      "Reads with adapters:                   115,684 (87.8%)\r\n",
      "Reads written (passing filters):       131,756 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    33,139,552 bp\r\n",
      "Total written (filtered):     30,033,370 bp (90.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 239 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "231 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2058.7\t0\t6\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t2058.7\t0\t211\r\n",
      "4\t4\t514.7\t0\t4\r\n",
      "5\t11\t128.7\t0\t11\r\n",
      "6\t5\t32.2\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 115445 times.\r\n",
      "115238 times, it overlapped the 5' end of a read\r\n",
      "207 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t514.7\t0\t1\r\n",
      "5\t1\t128.7\t0\t1\r\n",
      "20\t2\t0.0\t0\t2\r\n",
      "21\t4\t0.0\t0\t3 1\r\n",
      "22\t5\t0.0\t1\t4 1\r\n",
      "23\t8\t0.0\t1\t2 6\r\n",
      "24\t5\t0.0\t1\t2 3\r\n",
      "25\t18\t0.0\t1\t7 11\r\n",
      "26\t8341\t0.0\t1\t66 8275\r\n",
      "27\t106523\t0.0\t1\t91706 14817\r\n",
      "28\t330\t0.0\t1\t0 330\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t185\t2058.7\t0\t185\r\n",
      "4\t6\t514.7\t0\t6\r\n",
      "5\t4\t128.7\t0\t4\r\n",
      "6\t1\t32.2\t0\t1\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "51\t2\t0.0\t1\t0 2\r\n",
      "57\t1\t0.0\t1\t1\r\n",
      "61\t2\t0.0\t1\t0 2\r\n",
      "298\t2\t0.0\t1\t2\r\n",
      "300\t2\t0.0\t1\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3012.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3012.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.52 s (21 us/read; 2.83 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,717\r\n",
      "Reads with adapters:                     1,665 (2.3%)\r\n",
      "Reads written (passing filters):        71,717 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,291,334 bp\r\n",
      "Total written (filtered):     20,286,304 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1580 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "1576 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1120.6\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1557\t1120.6\t0\t1557\r\n",
      "4\t7\t280.1\t0\t7\r\n",
      "5\t11\t70.0\t0\t11\r\n",
      "6\t1\t17.5\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 85 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "84 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t280.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1120.6\t0\t83\r\n",
      "5\t1\t70.0\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3016.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3016.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.03 s (52 us/read; 1.14 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  38,652\r\n",
      "Reads with adapters:                    35,917 (92.9%)\r\n",
      "Reads written (passing filters):        38,652 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,303,669 bp\r\n",
      "Total written (filtered):      9,561,686 bp (92.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35155 times.\r\n",
      "35147 times, it overlapped the 5' end of a read\r\n",
      "8 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t11\t0.0\t0\t11\r\n",
      "20\t11\t0.0\t0\t11\r\n",
      "21\t35122\t0.0\t0\t35122\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t603.9\t0\t4\r\n",
      "4\t1\t151.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 762 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "742 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t603.9\t0\t20\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t603.9\t0\t7\r\n",
      "4\t734\t151.0\t0\t734\r\n",
      "5\t1\t37.7\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3016.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3016.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.50 s (39 us/read; 1.55 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  38,652\r\n",
      "Reads with adapters:                    32,757 (84.7%)\r\n",
      "Reads written (passing filters):        38,652 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,288,127 bp\r\n",
      "Total written (filtered):      9,404,870 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 60 times.\r\n",
      "13 times, it overlapped the 5' end of a read\r\n",
      "47 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t603.9\t0\t8\r\n",
      "4\t4\t151.0\t0\t4\r\n",
      "5\t1\t37.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t47\t603.9\t0\t47\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 32697 times.\r\n",
      "32687 times, it overlapped the 5' end of a read\r\n",
      "10 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t603.9\t0\t1\r\n",
      "22\t1\t0.0\t1\t1\r\n",
      "23\t4\t0.0\t1\t1 3\r\n",
      "24\t9\t0.0\t1\t5 4\r\n",
      "25\t5\t0.0\t1\t3 2\r\n",
      "26\t662\t0.0\t1\t29 633\r\n",
      "27\t31960\t0.0\t1\t25881 6079\r\n",
      "28\t45\t0.0\t1\t0 45\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5\t603.9\t0\t5\r\n",
      "6\t1\t9.4\t0\t1\r\n",
      "300\t4\t0.0\t1\t2 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2992.R2.fastq.gz.trimmed.fastq.gz /code/sequences/2992.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.41 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,754\r\n",
      "Reads with adapters:                     5,117 (5.5%)\r\n",
      "Reads written (passing filters):        93,754 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,638,480 bp\r\n",
      "Total written (filtered):     27,570,347 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2830 times.\r\n",
      "2 times, it overlapped the 5' end of a read\r\n",
      "2828 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1464.9\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2814\t1464.9\t0\t2814\r\n",
      "4\t10\t366.2\t0\t10\r\n",
      "5\t4\t91.6\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2287 times.\r\n",
      "2200 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1464.9\t0\t1\r\n",
      "4\t1\t366.2\t0\t1\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2196\t0.0\t1\t0 2196\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t1464.9\t0\t80\r\n",
      "4\t2\t366.2\t0\t2\r\n",
      "5\t4\t91.6\t0\t4\r\n",
      "6\t1\t22.9\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3012.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3012.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.67 s (51 us/read; 1.17 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  71,717\r\n",
      "Reads with adapters:                    63,634 (88.7%)\r\n",
      "Reads written (passing filters):        71,717 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    20,323,726 bp\r\n",
      "Total written (filtered):     19,028,433 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61273 times.\r\n",
      "61066 times, it overlapped the 5' end of a read\r\n",
      "207 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t12\t0.0\t0\t12\r\n",
      "20\t79\t0.0\t0\t79\r\n",
      "21\t60971\t0.0\t0\t60971\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t195\t1120.6\t0\t195\r\n",
      "7\t1\t4.4\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t10\t0.0\t0\t10\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2361 times.\r\n",
      "298 times, it overlapped the 5' end of a read\r\n",
      "2063 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t297\t1120.6\t0\t297\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t1120.6\t0\t42\r\n",
      "4\t2018\t280.1\t0\t2018\r\n",
      "5\t3\t70.0\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2987.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2987.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.34 s (56 us/read; 1.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 148,349\r\n",
      "Reads with adapters:                   143,102 (96.5%)\r\n",
      "Reads written (passing filters):       148,349 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    43,529,358 bp\r\n",
      "Total written (filtered):     40,544,190 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 141567 times.\r\n",
      "141537 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t24\t0.0\t0\t24\r\n",
      "20\t202\t0.0\t0\t202\r\n",
      "21\t141301\t0.0\t0\t141301\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t2318.0\t0\t6\r\n",
      "4\t1\t579.5\t0\t1\r\n",
      "300\t23\t0.0\t0\t23\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1535 times.\r\n",
      "78 times, it overlapped the 5' end of a read\r\n",
      "1457 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t2318.0\t0\t70\r\n",
      "27\t8\t0.0\t1\t8\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t2318.0\t0\t1\r\n",
      "4\t1456\t579.5\t0\t1456\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/2992.R1.fastq.gz.trimmed.fastq.gz /code/sequences/2992.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.05 s (54 us/read; 1.11 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,754\r\n",
      "Reads with adapters:                    90,737 (96.8%)\r\n",
      "Reads written (passing filters):        93,754 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    27,685,581 bp\r\n",
      "Total written (filtered):     25,787,085 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 90218 times.\r\n",
      "90202 times, it overlapped the 5' end of a read\r\n",
      "16 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t2\t0.0\t0\t2\r\n",
      "18\t14\t0.0\t0\t14\r\n",
      "19\t18\t0.0\t0\t18\r\n",
      "20\t100\t0.0\t0\t100\r\n",
      "21\t90068\t0.0\t0\t90068\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t1464.9\t0\t8\r\n",
      "300\t8\t0.0\t0\t8\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 519 times.\r\n",
      "128 times, it overlapped the 5' end of a read\r\n",
      "391 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t125\t1464.9\t0\t125\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t1464.9\t0\t4\r\n",
      "4\t387\t366.2\t0\t387\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3028.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3028.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.53 s (21 us/read; 2.86 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,971\r\n",
      "Reads with adapters:                     1,451 (2.0%)\r\n",
      "Reads written (passing filters):        72,971 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,185,383 bp\r\n",
      "Total written (filtered):     19,180,974 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1331 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "1316 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1140.2\t0\t14\r\n",
      "4\t1\t285.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1299\t1140.2\t0\t1299\r\n",
      "4\t5\t285.0\t0\t5\r\n",
      "5\t10\t71.3\t0\t10\r\n",
      "6\t2\t17.8\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 120 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "119 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1140.2\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t107\t1140.2\t0\t107\r\n",
      "4\t2\t285.0\t0\t2\r\n",
      "5\t8\t71.3\t0\t8\r\n",
      "6\t2\t17.8\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3031.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3031.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.83 s (24 us/read; 2.51 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  76,656\r\n",
      "Reads with adapters:                    73,865 (96.4%)\r\n",
      "Reads written (passing filters):        76,656 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,886,072 bp\r\n",
      "Total written (filtered):     19,893,883 bp (90.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1197.8\t0\t7\r\n",
      "4\t1\t299.4\t0\t1\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t23\t1197.8\t0\t23\r\n",
      "4\t1\t299.4\t0\t1\r\n",
      "5\t1\t74.9\t0\t1\r\n",
      "6\t1\t18.7\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 73830 times.\r\n",
      "73795 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1197.8\t0\t2\r\n",
      "4\t1\t299.4\t0\t1\r\n",
      "20\t3\t0.0\t0\t3\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "23\t2\t0.0\t1\t0 2\r\n",
      "24\t6\t0.0\t1\t1 5\r\n",
      "25\t13\t0.0\t1\t4 9\r\n",
      "26\t2272\t0.0\t1\t44 2228\r\n",
      "27\t71361\t0.0\t1\t61721 9640\r\n",
      "28\t133\t0.0\t1\t0 133\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1197.8\t0\t28\r\n",
      "5\t1\t74.9\t0\t1\r\n",
      "300\t6\t0.0\t1\t5 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3028.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3028.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.38 s (46 us/read; 1.29 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,971\r\n",
      "Reads with adapters:                    54,732 (75.0%)\r\n",
      "Reads written (passing filters):        72,971 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,164,753 bp\r\n",
      "Total written (filtered):     18,071,555 bp (94.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 51587 times.\r\n",
      "51329 times, it overlapped the 5' end of a read\r\n",
      "258 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "15\t1\t0.0\t0\t1\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t8\t0.0\t0\t8\r\n",
      "20\t88\t0.0\t0\t88\r\n",
      "21\t51228\t0.0\t0\t51228\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t248\t1140.2\t0\t248\r\n",
      "4\t1\t285.0\t0\t1\r\n",
      "5\t2\t71.3\t0\t2\r\n",
      "300\t7\t0.0\t0\t7\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3145 times.\r\n",
      "77 times, it overlapped the 5' end of a read\r\n",
      "3068 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t72\t1140.2\t0\t72\r\n",
      "27\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t70\t1140.2\t0\t70\r\n",
      "4\t2998\t285.0\t0\t2998\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3045.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3045.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.24 s (24 us/read; 2.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,896\r\n",
      "Reads with adapters:                    92,159 (97.1%)\r\n",
      "Reads written (passing filters):        94,896 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,251,196 bp\r\n",
      "Total written (filtered):     23,765,104 bp (90.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 28 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "23 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1482.8\t0\t1\r\n",
      "4\t3\t370.7\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1482.8\t0\t20\r\n",
      "4\t3\t370.7\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 92131 times.\r\n",
      "92100 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "21\t4\t0.0\t0\t4\r\n",
      "22\t4\t0.0\t1\t3 1\r\n",
      "23\t8\t0.0\t1\t5 3\r\n",
      "24\t6\t0.0\t1\t6\r\n",
      "25\t14\t0.0\t1\t6 8\r\n",
      "26\t3484\t0.0\t1\t76 3408\r\n",
      "27\t88451\t0.0\t1\t83525 4926\r\n",
      "28\t128\t0.0\t1\t0 128\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1482.8\t0\t18\r\n",
      "4\t3\t370.7\t0\t3\r\n",
      "5\t1\t92.7\t0\t1\r\n",
      "300\t9\t0.0\t1\t8 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3031.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3031.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.64 s (48 us/read; 1.26 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  76,656\r\n",
      "Reads with adapters:                    72,503 (94.6%)\r\n",
      "Reads written (passing filters):        76,656 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    21,931,563 bp\r\n",
      "Total written (filtered):     20,447,156 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 70219 times.\r\n",
      "70178 times, it overlapped the 5' end of a read\r\n",
      "41 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "20\t42\t0.0\t0\t42\r\n",
      "21\t70130\t0.0\t0\t70130\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1197.8\t0\t36\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2284 times.\r\n",
      "55 times, it overlapped the 5' end of a read\r\n",
      "2229 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t53\t1197.8\t0\t53\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1197.8\t0\t6\r\n",
      "4\t2223\t299.4\t0\t2223\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3064.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3064.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.46 s (25 us/read; 2.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,365\r\n",
      "Reads with adapters:                     1,119 (2.0%)\r\n",
      "Reads written (passing filters):        57,365 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,559,728 bp\r\n",
      "Total written (filtered):     14,556,331 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1028 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "1022 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t896.3\t0\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1001\t896.3\t0\t1001\r\n",
      "4\t10\t224.1\t0\t10\r\n",
      "5\t11\t56.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 91 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "90 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t896.3\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t86\t896.3\t0\t86\r\n",
      "5\t4\t56.0\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3067.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3067.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.63 s (39 us/read; 1.53 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,636\r\n",
      "Reads with adapters:                    36,495 (87.7%)\r\n",
      "Reads written (passing filters):        41,636 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,758,705 bp\r\n",
      "Total written (filtered):     11,014,258 bp (93.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35223 times.\r\n",
      "35139 times, it overlapped the 5' end of a read\r\n",
      "84 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t5\t0.0\t0\t5\r\n",
      "20\t39\t0.0\t0\t39\r\n",
      "21\t35091\t0.0\t0\t35091\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t650.6\t0\t79\r\n",
      "140\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1272 times.\r\n",
      "43 times, it overlapped the 5' end of a read\r\n",
      "1229 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t650.6\t0\t43\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t32\t650.6\t0\t32\r\n",
      "4\t1197\t162.6\t0\t1197\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3064.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3064.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.98 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  57,365\r\n",
      "Reads with adapters:                    50,282 (87.7%)\r\n",
      "Reads written (passing filters):        57,365 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,561,271 bp\r\n",
      "Total written (filtered):     13,613,193 bp (93.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 44073 times.\r\n",
      "43940 times, it overlapped the 5' end of a read\r\n",
      "133 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t3\t0.0\t0\t3\r\n",
      "19\t6\t0.0\t0\t6\r\n",
      "20\t63\t0.0\t0\t63\r\n",
      "21\t43867\t0.0\t0\t43867\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t131\t896.3\t0\t131\r\n",
      "5\t1\t56.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6209 times.\r\n",
      "65 times, it overlapped the 5' end of a read\r\n",
      "6144 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t65\t896.3\t0\t65\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t42\t896.3\t0\t42\r\n",
      "4\t6102\t224.1\t0\t6102\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3067.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3067.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.03 s (25 us/read; 2.42 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,636\r\n",
      "Reads with adapters:                     1,193 (2.9%)\r\n",
      "Reads written (passing filters):        41,636 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,760,425 bp\r\n",
      "Total written (filtered):     11,756,791 bp (100.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 1110 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "1109 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t650.6\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1080\t650.6\t0\t1080\r\n",
      "4\t9\t162.6\t0\t9\r\n",
      "5\t20\t40.7\t0\t20\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 83 times.\r\n",
      "0 times, it overlapped the 5' end of a read\r\n",
      "83 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t80\t650.6\t0\t80\r\n",
      "5\t3\t40.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3045.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3045.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.38 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  94,896\r\n",
      "Reads with adapters:                    89,932 (94.8%)\r\n",
      "Reads written (passing filters):        94,896 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    26,377,591 bp\r\n",
      "Total written (filtered):     24,546,990 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 86474 times.\r\n",
      "86423 times, it overlapped the 5' end of a read\r\n",
      "51 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t8\t0.0\t0\t8\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t48\t0.0\t0\t48\r\n",
      "21\t86366\t0.0\t0\t86366\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1482.8\t0\t45\r\n",
      "300\t6\t0.0\t0\t6\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3458 times.\r\n",
      "63 times, it overlapped the 5' end of a read\r\n",
      "3395 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t1482.8\t0\t59\r\n",
      "26\t2\t0.0\t1\t0 2\r\n",
      "27\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1482.8\t0\t9\r\n",
      "4\t3384\t370.7\t0\t3384\r\n",
      "5\t1\t92.7\t0\t1\r\n",
      "6\t1\t23.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3047.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3047.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.99 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,223\r\n",
      "Reads with adapters:                    92,652 (94.3%)\r\n",
      "Reads written (passing filters):        98,223 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,639,833 bp\r\n",
      "Total written (filtered):     16,132,212 bp (91.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 66862 times.\r\n",
      "66764 times, it overlapped the 5' end of a read\r\n",
      "98 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1534.7\t0\t1\r\n",
      "15\t2\t0.0\t0\t2\r\n",
      "16\t2\t0.0\t0\t2\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "20\t45\t0.0\t0\t45\r\n",
      "21\t66707\t0.0\t0\t66707\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t79\t1534.7\t0\t79\r\n",
      "4\t1\t383.7\t0\t1\r\n",
      "7\t2\t6.0\t0\t2\r\n",
      "53\t2\t0.0\t0\t2\r\n",
      "56\t1\t0.0\t0\t1\r\n",
      "61\t5\t0.0\t0\t5\r\n",
      "67\t1\t0.0\t0\t1\r\n",
      "93\t1\t0.0\t0\t1\r\n",
      "250\t1\t0.0\t0\t1\r\n",
      "300\t5\t0.0\t0\t5\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 25790 times.\r\n",
      "30 times, it overlapped the 5' end of a read\r\n",
      "25760 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1534.7\t0\t29\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t106\t1534.7\t0\t106\r\n",
      "4\t25650\t383.7\t0\t25650\r\n",
      "5\t4\t95.9\t0\t4\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3047.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3047.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.70 s (48 us/read; 1.25 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  98,223\r\n",
      "Reads with adapters:                    88,045 (89.6%)\r\n",
      "Reads written (passing filters):        98,223 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    17,026,902 bp\r\n",
      "Total written (filtered):     14,666,126 bp (86.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 34 times.\r\n",
      "4 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1534.7\t0\t2\r\n",
      "4\t2\t383.7\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t1534.7\t0\t22\r\n",
      "4\t2\t383.7\t0\t2\r\n",
      "5\t5\t95.9\t0\t5\r\n",
      "6\t1\t24.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 88011 times.\r\n",
      "87958 times, it overlapped the 5' end of a read\r\n",
      "53 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t3\t0.0\t0\t3\r\n",
      "22\t1\t0.0\t1\t1\r\n",
      "23\t6\t0.0\t1\t4 2\r\n",
      "24\t8\t0.0\t1\t2 6\r\n",
      "25\t18\t0.0\t1\t9 9\r\n",
      "26\t17702\t0.0\t1\t60 17642\r\n",
      "27\t69762\t0.0\t1\t65596 4166\r\n",
      "28\t458\t0.0\t1\t0 458\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1534.7\t0\t29\r\n",
      "4\t1\t383.7\t0\t1\r\n",
      "5\t2\t95.9\t0\t2\r\n",
      "41\t3\t0.0\t1\t0 3\r\n",
      "42\t1\t0.0\t1\t1\r\n",
      "47\t2\t0.0\t1\t1 1\r\n",
      "51\t3\t0.0\t1\t3\r\n",
      "56\t1\t0.0\t1\t1\r\n",
      "59\t1\t0.0\t1\t1\r\n",
      "61\t2\t0.0\t1\t2\r\n",
      "297\t2\t0.0\t1\t1 1\r\n",
      "298\t1\t0.0\t1\t1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3089.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3089.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.84 s (23 us/read; 2.66 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,370\r\n",
      "Reads with adapters:                    61,082 (75.1%)\r\n",
      "Reads written (passing filters):        81,370 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,115,972 bp\r\n",
      "Total written (filtered):     17,468,560 bp (91.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 72 times.\r\n",
      "17 times, it overlapped the 5' end of a read\r\n",
      "55 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t1271.4\t0\t10\r\n",
      "4\t6\t317.9\t0\t6\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t45\t1271.4\t0\t45\r\n",
      "4\t4\t317.9\t0\t4\r\n",
      "5\t5\t79.5\t0\t5\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 61010 times.\r\n",
      "60971 times, it overlapped the 5' end of a read\r\n",
      "39 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t1271.4\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t2\t0.0\t1\t2\r\n",
      "23\t3\t0.0\t1\t1 2\r\n",
      "24\t18\t0.0\t1\t12 6\r\n",
      "25\t5\t0.0\t1\t1 4\r\n",
      "26\t1629\t0.0\t1\t32 1597\r\n",
      "27\t59109\t0.0\t1\t54934 4175\r\n",
      "28\t202\t0.0\t1\t0 202\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1271.4\t0\t25\r\n",
      "4\t2\t317.9\t0\t2\r\n",
      "5\t3\t79.5\t0\t3\r\n",
      "6\t1\t19.9\t0\t1\r\n",
      "300\t8\t0.0\t1\t6 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3091.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3091.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.90 s (29 us/read; 2.08 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,793\r\n",
      "Reads with adapters:                    61,422 (93.4%)\r\n",
      "Reads written (passing filters):        65,793 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,718,197 bp\r\n",
      "Total written (filtered):     17,056,456 bp (91.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 35 times.\r\n",
      "14 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1028.0\t0\t13\r\n",
      "4\t1\t257.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t20\t1028.0\t0\t20\r\n",
      "4\t1\t257.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 61387 times.\r\n",
      "61331 times, it overlapped the 5' end of a read\r\n",
      "56 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1028.0\t0\t1\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t9\t0.0\t1\t4 5\r\n",
      "25\t8\t0.0\t1\t1 7\r\n",
      "26\t1126\t0.0\t1\t32 1094\r\n",
      "27\t60018\t0.0\t1\t53780 6238\r\n",
      "28\t164\t0.0\t1\t0 164\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1028.0\t0\t25\r\n",
      "5\t7\t64.3\t0\t7\r\n",
      "7\t1\t4.0\t0\t1\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "297\t1\t0.0\t1\t0 1\r\n",
      "298\t2\t0.0\t1\t2\r\n",
      "299\t2\t0.0\t1\t2\r\n",
      "300\t17\t0.0\t1\t14 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3098.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3098.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.39 s (26 us/read; 2.31 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  53,734\r\n",
      "Reads with adapters:                    49,297 (91.7%)\r\n",
      "Reads written (passing filters):        53,734 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,350,147 bp\r\n",
      "Total written (filtered):     13,356,707 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 46915 times.\r\n",
      "46854 times, it overlapped the 5' end of a read\r\n",
      "61 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t839.6\t0\t2\r\n",
      "5\t1\t52.5\t0\t1\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "20\t30\t0.0\t0\t30\r\n",
      "21\t46820\t0.0\t0\t46820\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t59\t839.6\t0\t59\r\n",
      "4\t1\t209.9\t0\t1\r\n",
      "52\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2382 times.\r\n",
      "50 times, it overlapped the 5' end of a read\r\n",
      "2332 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t839.6\t0\t50\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t120\t839.6\t0\t120\r\n",
      "4\t2211\t209.9\t0\t2211\r\n",
      "5\t1\t52.5\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3098.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3098.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.50 s (28 us/read; 2.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  53,734\r\n",
      "Reads with adapters:                    51,349 (95.6%)\r\n",
      "Reads written (passing filters):        53,734 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    14,204,546 bp\r\n",
      "Total written (filtered):     12,819,983 bp (90.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 37 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "15 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t839.6\t0\t10\r\n",
      "4\t8\t209.9\t0\t8\r\n",
      "5\t4\t52.5\t0\t4\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t839.6\t0\t14\r\n",
      "4\t1\t209.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 51312 times.\r\n",
      "51298 times, it overlapped the 5' end of a read\r\n",
      "14 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t839.6\t0\t3\r\n",
      "4\t1\t209.9\t0\t1\r\n",
      "22\t3\t0.0\t1\t1 2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t5\t0.0\t1\t4 1\r\n",
      "25\t11\t0.0\t1\t6 5\r\n",
      "26\t1594\t0.0\t1\t41 1553\r\n",
      "27\t49479\t0.0\t1\t46230 3249\r\n",
      "28\t201\t0.0\t1\t0 201\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t10\t839.6\t0\t10\r\n",
      "4\t1\t209.9\t0\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t2\t0.0\t1\t1 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3086.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3086.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.31 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,044\r\n",
      "Reads with adapters:                    41,488 (48.8%)\r\n",
      "Reads written (passing filters):        85,044 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,532,892 bp\r\n",
      "Total written (filtered):     21,746,500 bp (96.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 36689 times.\r\n",
      "36475 times, it overlapped the 5' end of a read\r\n",
      "214 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "20\t21\t0.0\t0\t21\r\n",
      "21\t36450\t0.0\t0\t36450\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t188\t1328.8\t0\t188\r\n",
      "4\t9\t332.2\t0\t9\r\n",
      "5\t5\t83.1\t0\t5\r\n",
      "6\t1\t20.8\t0\t1\r\n",
      "7\t6\t5.2\t0\t6\r\n",
      "59\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4799 times.\r\n",
      "24 times, it overlapped the 5' end of a read\r\n",
      "4775 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1328.8\t0\t24\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t656\t1328.8\t0\t656\r\n",
      "4\t4117\t332.2\t0\t4117\r\n",
      "5\t2\t83.1\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3091.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3091.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.00 s (46 us/read; 1.32 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  65,793\r\n",
      "Reads with adapters:                    62,482 (95.0%)\r\n",
      "Reads written (passing filters):        65,793 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    18,765,855 bp\r\n",
      "Total written (filtered):     17,479,115 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 61012 times.\r\n",
      "60981 times, it overlapped the 5' end of a read\r\n",
      "31 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t4\t0.0\t0\t4\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t31\t0.0\t0\t31\r\n",
      "21\t60945\t0.0\t0\t60945\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1028.0\t0\t29\r\n",
      "5\t1\t64.3\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 1470 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "1432 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t38\t1028.0\t0\t38\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1028.0\t0\t50\r\n",
      "4\t1382\t257.0\t0\t1382\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3086.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3086.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.16 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,044\r\n",
      "Reads with adapters:                    81,564 (95.9%)\r\n",
      "Reads written (passing filters):        85,044 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    22,414,892 bp\r\n",
      "Total written (filtered):     20,218,023 bp (90.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 52 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t1328.8\t0\t18\r\n",
      "4\t3\t332.2\t0\t3\r\n",
      "5\t1\t83.1\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t27\t1328.8\t0\t27\r\n",
      "4\t3\t332.2\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 81512 times.\r\n",
      "81477 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t1\t332.2\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t5\t0.0\t1\t3 2\r\n",
      "23\t3\t0.0\t1\t2 1\r\n",
      "24\t7\t0.0\t1\t4 3\r\n",
      "25\t17\t0.0\t1\t8 9\r\n",
      "26\t5131\t0.0\t1\t45 5086\r\n",
      "27\t76037\t0.0\t1\t63320 12717\r\n",
      "28\t274\t0.0\t1\t0 274\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t1328.8\t0\t21\r\n",
      "4\t4\t332.2\t0\t4\r\n",
      "5\t2\t83.1\t0\t2\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "61\t1\t0.0\t1\t1\r\n",
      "120\t1\t0.0\t1\t0 1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t4\t0.0\t1\t1 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3089.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3089.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.13 s (51 us/read; 1.18 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  81,370\r\n",
      "Reads with adapters:                    55,936 (68.7%)\r\n",
      "Reads written (passing filters):        81,370 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,238,965 bp\r\n",
      "Total written (filtered):     18,103,841 bp (94.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 53645 times.\r\n",
      "53552 times, it overlapped the 5' end of a read\r\n",
      "93 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1271.4\t0\t1\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "18\t10\t0.0\t0\t10\r\n",
      "20\t26\t0.0\t0\t26\r\n",
      "21\t53514\t0.0\t0\t53514\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t83\t1271.4\t0\t83\r\n",
      "4\t1\t317.9\t0\t1\r\n",
      "5\t2\t79.5\t0\t2\r\n",
      "7\t1\t5.0\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "127\t1\t0.0\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2291 times.\r\n",
      "37 times, it overlapped the 5' end of a read\r\n",
      "2254 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t1271.4\t0\t35\r\n",
      "4\t1\t317.9\t0\t1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t180\t1271.4\t0\t180\r\n",
      "4\t2072\t317.9\t0\t2072\r\n",
      "5\t2\t79.5\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3100.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3100.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.69 s (23 us/read; 2.58 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,882\r\n",
      "Reads with adapters:                    56,592 (77.6%)\r\n",
      "Reads written (passing filters):        72,882 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,325,032 bp\r\n",
      "Total written (filtered):     17,810,677 bp (92.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 464 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "458 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t1138.8\t0\t3\r\n",
      "4\t2\t284.7\t0\t2\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t443\t1138.8\t0\t443\r\n",
      "4\t1\t284.7\t0\t1\r\n",
      "5\t7\t71.2\t0\t7\r\n",
      "6\t5\t17.8\t0\t5\r\n",
      "7\t1\t4.4\t0\t1\r\n",
      "8\t1\t1.1\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 56128 times.\r\n",
      "55980 times, it overlapped the 5' end of a read\r\n",
      "148 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "24\t6\t0.0\t1\t4 2\r\n",
      "25\t10\t0.0\t1\t3 7\r\n",
      "26\t1893\t0.0\t1\t32 1861\r\n",
      "27\t53869\t0.0\t1\t41434 12435\r\n",
      "28\t202\t0.0\t1\t0 202\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t108\t1138.8\t0\t108\r\n",
      "4\t3\t284.7\t0\t3\r\n",
      "5\t28\t71.2\t0\t28\r\n",
      "297\t1\t0.0\t1\t1\r\n",
      "300\t8\t0.0\t1\t7 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3101.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3101.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.22 s (24 us/read; 2.50 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,556\r\n",
      "Reads with adapters:                    86,647 (93.6%)\r\n",
      "Reads written (passing filters):        92,556 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,831,809 bp\r\n",
      "Total written (filtered):     21,494,094 bp (90.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 42 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "33 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1446.2\t0\t6\r\n",
      "4\t1\t361.5\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t1446.2\t0\t29\r\n",
      "4\t2\t361.5\t0\t2\r\n",
      "5\t2\t90.4\t0\t2\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 86605 times.\r\n",
      "86559 times, it overlapped the 5' end of a read\r\n",
      "46 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1446.2\t0\t1\r\n",
      "20\t3\t0.0\t0\t3\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "23\t8\t0.0\t1\t6 2\r\n",
      "24\t13\t0.0\t1\t6 7\r\n",
      "25\t11\t0.0\t1\t6 5\r\n",
      "26\t3455\t0.0\t1\t58 3397\r\n",
      "27\t82627\t0.0\t1\t76442 6185\r\n",
      "28\t440\t0.0\t1\t0 440\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t1446.2\t0\t24\r\n",
      "4\t2\t361.5\t0\t2\r\n",
      "5\t5\t90.4\t0\t5\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "47\t1\t0.0\t1\t1\r\n",
      "49\t1\t0.0\t1\t1\r\n",
      "71\t1\t0.0\t1\t1\r\n",
      "299\t1\t0.0\t1\t0 1\r\n",
      "300\t10\t0.0\t1\t8 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3111.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3111.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.56 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  52,511\r\n",
      "Reads with adapters:                    47,024 (89.6%)\r\n",
      "Reads written (passing filters):        52,511 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,024,170 bp\r\n",
      "Total written (filtered):      8,401,362 bp (93.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 25552 times.\r\n",
      "25523 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t820.5\t0\t2\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t14\t0.0\t0\t14\r\n",
      "21\t25503\t0.0\t0\t25503\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t820.5\t0\t22\r\n",
      "4\t2\t205.1\t0\t2\r\n",
      "5\t1\t51.3\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 21472 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "21452 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t820.5\t0\t18\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t58\t820.5\t0\t58\r\n",
      "4\t21393\t205.1\t0\t21393\r\n",
      "5\t1\t51.3\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3111.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3111.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.31 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  52,511\r\n",
      "Reads with adapters:                    36,224 (69.0%)\r\n",
      "Reads written (passing filters):        52,511 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     8,837,750 bp\r\n",
      "Total written (filtered):      7,870,333 bp (89.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 34 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t820.5\t0\t2\r\n",
      "4\t3\t205.1\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t820.5\t0\t28\r\n",
      "5\t1\t51.3\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 36190 times.\r\n",
      "36160 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t820.5\t0\t1\r\n",
      "22\t3\t0.0\t1\t2 1\r\n",
      "23\t2\t0.0\t1\t1 1\r\n",
      "24\t5\t0.0\t1\t1 4\r\n",
      "25\t9\t0.0\t1\t0 9\r\n",
      "26\t10441\t0.0\t1\t17 10424\r\n",
      "27\t25547\t0.0\t1\t20440 5107\r\n",
      "28\t152\t0.0\t1\t0 152\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t820.5\t0\t22\r\n",
      "4\t1\t205.1\t0\t1\r\n",
      "7\t1\t3.2\t0\t1\r\n",
      "41\t2\t0.0\t1\t0 2\r\n",
      "300\t4\t0.0\t1\t2 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3100.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3100.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.55 s (49 us/read; 1.23 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  72,882\r\n",
      "Reads with adapters:                    62,018 (85.1%)\r\n",
      "Reads written (passing filters):        72,882 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    19,497,165 bp\r\n",
      "Total written (filtered):     18,258,645 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58295 times.\r\n",
      "58216 times, it overlapped the 5' end of a read\r\n",
      "79 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "14\t1\t0.0\t0\t1\r\n",
      "16\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "20\t37\t0.0\t0\t37\r\n",
      "21\t58171\t0.0\t0\t58171\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t73\t1138.8\t0\t73\r\n",
      "5\t1\t71.2\t0\t1\r\n",
      "7\t1\t4.4\t0\t1\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3723 times.\r\n",
      "45 times, it overlapped the 5' end of a read\r\n",
      "3678 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t43\t1138.8\t0\t43\r\n",
      "5\t1\t71.2\t0\t1\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t250\t1138.8\t0\t250\r\n",
      "4\t3427\t284.7\t0\t3427\r\n",
      "5\t1\t71.2\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3103.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3103.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.29 s (24 us/read; 2.46 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,949\r\n",
      "Reads with adapters:                    90,552 (96.4%)\r\n",
      "Reads written (passing filters):        93,949 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,705,419 bp\r\n",
      "Total written (filtered):     23,261,636 bp (90.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 40 times.\r\n",
      "15 times, it overlapped the 5' end of a read\r\n",
      "25 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t1468.0\t0\t9\r\n",
      "4\t5\t367.0\t0\t5\r\n",
      "5\t1\t91.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t1468.0\t0\t25\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 90512 times.\r\n",
      "90486 times, it overlapped the 5' end of a read\r\n",
      "26 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "22\t3\t0.0\t1\t1 2\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t13\t0.0\t1\t8 5\r\n",
      "25\t8\t0.0\t1\t6 2\r\n",
      "26\t2380\t0.0\t1\t92 2288\r\n",
      "27\t87602\t0.0\t1\t82811 4791\r\n",
      "28\t478\t0.0\t1\t0 478\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t1468.0\t0\t14\r\n",
      "4\t1\t367.0\t0\t1\r\n",
      "5\t1\t91.7\t0\t1\r\n",
      "6\t1\t22.9\t0\t1\r\n",
      "60\t1\t0.0\t1\t1\r\n",
      "299\t1\t0.0\t1\t1\r\n",
      "300\t7\t0.0\t1\t6 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3125.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3125.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.33 s (50 us/read; 1.21 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  46,811\r\n",
      "Reads with adapters:                    25,766 (55.0%)\r\n",
      "Reads written (passing filters):        46,811 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,105,123 bp\r\n",
      "Total written (filtered):     10,715,161 bp (96.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 16896 times.\r\n",
      "16869 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t731.4\t0\t1\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t10\t0.0\t0\t10\r\n",
      "21\t16854\t0.0\t0\t16854\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t731.4\t0\t25\r\n",
      "5\t1\t45.7\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 8870 times.\r\n",
      "35 times, it overlapped the 5' end of a read\r\n",
      "8835 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t731.4\t0\t35\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t78\t731.4\t0\t78\r\n",
      "4\t8754\t182.9\t0\t8754\r\n",
      "5\t3\t45.7\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3125.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3125.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.04 s (22 us/read; 2.69 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  46,811\r\n",
      "Reads with adapters:                    36,255 (77.4%)\r\n",
      "Reads written (passing filters):        46,811 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,045,222 bp\r\n",
      "Total written (filtered):     10,070,314 bp (91.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 28 times.\r\n",
      "6 times, it overlapped the 5' end of a read\r\n",
      "22 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t731.4\t0\t3\r\n",
      "4\t3\t182.9\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t731.4\t0\t22\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 36227 times.\r\n",
      "36210 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t731.4\t0\t3\r\n",
      "9\t1\t0.2\t0\t1\r\n",
      "21\t2\t0.0\t0\t2\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "24\t2\t0.0\t1\t2\r\n",
      "25\t8\t0.0\t1\t0 8\r\n",
      "26\t4619\t0.0\t1\t20 4599\r\n",
      "27\t31548\t0.0\t1\t25599 5949\r\n",
      "28\t26\t0.0\t1\t0 26\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t731.4\t0\t9\r\n",
      "5\t1\t45.7\t0\t1\r\n",
      "41\t1\t0.0\t1\t0 1\r\n",
      "300\t6\t0.0\t1\t3 3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3101.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3101.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.55 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  92,556\r\n",
      "Reads with adapters:                    84,311 (91.1%)\r\n",
      "Reads written (passing filters):        92,556 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    23,925,374 bp\r\n",
      "Total written (filtered):     22,257,785 bp (93.0%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 78240 times.\r\n",
      "78175 times, it overlapped the 5' end of a read\r\n",
      "65 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "16\t3\t0.0\t0\t3\r\n",
      "17\t1\t0.0\t0\t1\r\n",
      "18\t6\t0.0\t0\t6\r\n",
      "20\t47\t0.0\t0\t47\r\n",
      "21\t78118\t0.0\t0\t78118\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t50\t1446.2\t0\t50\r\n",
      "5\t4\t90.4\t0\t4\r\n",
      "7\t3\t5.6\t0\t3\r\n",
      "47\t2\t0.0\t0\t2\r\n",
      "61\t1\t0.0\t0\t1\r\n",
      "299\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6071 times.\r\n",
      "38 times, it overlapped the 5' end of a read\r\n",
      "6033 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t36\t1446.2\t0\t36\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t143\t1446.2\t0\t143\r\n",
      "4\t5887\t361.5\t0\t5887\r\n",
      "5\t3\t90.4\t0\t3\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3103.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3103.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.90 s (52 us/read; 1.15 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  93,949\r\n",
      "Reads with adapters:                    83,479 (88.9%)\r\n",
      "Reads written (passing filters):        93,949 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,777,277 bp\r\n",
      "Total written (filtered):     24,080,565 bp (93.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80182 times.\r\n",
      "80118 times, it overlapped the 5' end of a read\r\n",
      "64 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t5\t0.0\t0\t5\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t43\t0.0\t0\t43\r\n",
      "21\t80069\t0.0\t0\t80069\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t57\t1468.0\t0\t57\r\n",
      "5\t1\t91.7\t0\t1\r\n",
      "7\t1\t5.7\t0\t1\r\n",
      "47\t1\t0.0\t0\t1\r\n",
      "300\t4\t0.0\t0\t4\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 3297 times.\r\n",
      "34 times, it overlapped the 5' end of a read\r\n",
      "3263 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t1468.0\t0\t34\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t290\t1468.0\t0\t290\r\n",
      "4\t2973\t367.0\t0\t2973\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 3.08 s (36 us/read; 1.67 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                    81,198 (94.5%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,568,415 bp\r\n",
      "Total written (filtered):     23,862,680 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 80995 times.\r\n",
      "80966 times, it overlapped the 5' end of a read\r\n",
      "29 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t7\t0.0\t0\t7\r\n",
      "19\t15\t0.0\t0\t15\r\n",
      "20\t136\t0.0\t0\t136\r\n",
      "21\t80808\t0.0\t0\t80808\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t1342.8\t0\t13\r\n",
      "296\t1\t0.0\t0\t1\r\n",
      "300\t15\t0.0\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 203 times.\r\n",
      "29 times, it overlapped the 5' end of a read\r\n",
      "174 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t28\t1342.8\t0\t28\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t1342.8\t0\t6\r\n",
      "4\t168\t335.7\t0\t168\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3154.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3154.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.71 s (39 us/read; 1.56 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  44,263\r\n",
      "Reads with adapters:                    29,243 (66.1%)\r\n",
      "Reads written (passing filters):        44,263 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,470,566 bp\r\n",
      "Total written (filtered):     11,892,037 bp (95.4%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 27126 times.\r\n",
      "27105 times, it overlapped the 5' end of a read\r\n",
      "21 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t15\t0.0\t0\t15\r\n",
      "21\t27082\t0.0\t0\t27082\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t691.6\t0\t18\r\n",
      "300\t3\t0.0\t0\t3\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2117 times.\r\n",
      "25 times, it overlapped the 5' end of a read\r\n",
      "2092 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t691.6\t0\t25\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t691.6\t0\t41\r\n",
      "4\t2051\t172.9\t0\t2051\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3154.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3154.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.27 s (29 us/read; 2.09 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  44,263\r\n",
      "Reads with adapters:                    37,401 (84.5%)\r\n",
      "Reads written (passing filters):        44,263 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    12,468,916 bp\r\n",
      "Total written (filtered):     11,460,697 bp (91.9%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 58 times.\r\n",
      "22 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t14\t691.6\t0\t14\r\n",
      "4\t6\t172.9\t0\t6\r\n",
      "5\t2\t43.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t691.6\t0\t35\r\n",
      "5\t1\t43.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 37343 times.\r\n",
      "37335 times, it overlapped the 5' end of a read\r\n",
      "8 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t1\t0.0\t1\t0 1\r\n",
      "24\t4\t0.0\t1\t3 1\r\n",
      "25\t6\t0.0\t1\t2 4\r\n",
      "26\t1524\t0.0\t1\t20 1504\r\n",
      "27\t35769\t0.0\t1\t28623 7146\r\n",
      "28\t31\t0.0\t1\t0 31\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t3\t691.6\t0\t3\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3141.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3141.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 4.17 s (48 us/read; 1.24 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  85,937\r\n",
      "Reads with adapters:                     5,105 (5.9%)\r\n",
      "Reads written (passing filters):        85,937 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    25,518,505 bp\r\n",
      "Total written (filtered):     25,444,522 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 2561 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "2553 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t7\t1342.8\t0\t7\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2543\t1342.8\t0\t2543\r\n",
      "4\t9\t335.7\t0\t9\r\n",
      "5\t1\t83.9\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2544 times.\r\n",
      "2445 times, it overlapped the 5' end of a read\r\n",
      "99 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t1342.8\t0\t1\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t2439\t0.0\t1\t0 2439\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t96\t1342.8\t0\t96\r\n",
      "4\t1\t335.7\t0\t1\r\n",
      "5\t2\t83.9\t0\t2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3216.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3216.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.04 s (25 us/read; 2.41 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,574\r\n",
      "Reads with adapters:                    32,923 (79.2%)\r\n",
      "Reads written (passing filters):        41,574 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,218,771 bp\r\n",
      "Total written (filtered):      9,333,441 bp (91.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 41 times.\r\n",
      "5 times, it overlapped the 5' end of a read\r\n",
      "36 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t649.6\t0\t4\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t35\t649.6\t0\t35\r\n",
      "5\t1\t40.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 32882 times.\r\n",
      "32865 times, it overlapped the 5' end of a read\r\n",
      "17 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "23\t4\t0.0\t1\t0 4\r\n",
      "24\t2\t0.0\t1\t2\r\n",
      "25\t5\t0.0\t1\t1 4\r\n",
      "26\t3479\t0.0\t1\t19 3460\r\n",
      "27\t29271\t0.0\t1\t23316 5955\r\n",
      "28\t104\t0.0\t1\t0 104\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t13\t649.6\t0\t13\r\n",
      "300\t4\t0.0\t1\t3 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3127.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3127.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 5.25 s (29 us/read; 2.07 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 181,429\r\n",
      "Reads with adapters:                     9,115 (5.0%)\r\n",
      "Reads written (passing filters):       181,429 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,605,019 bp\r\n",
      "Total written (filtered):     52,468,543 bp (99.7%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 4320 times.\r\n",
      "24 times, it overlapped the 5' end of a read\r\n",
      "4296 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t2834.8\t0\t21\r\n",
      "4\t3\t708.7\t0\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4251\t2834.8\t0\t4251\r\n",
      "4\t30\t708.7\t0\t30\r\n",
      "5\t15\t177.2\t0\t15\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4795 times.\r\n",
      "4533 times, it overlapped the 5' end of a read\r\n",
      "262 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t2834.8\t0\t2\r\n",
      "25\t2\t0.0\t1\t0 2\r\n",
      "26\t5\t0.0\t1\t0 5\r\n",
      "27\t4524\t0.0\t1\t0 4524\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t241\t2834.8\t0\t241\r\n",
      "4\t4\t708.7\t0\t4\r\n",
      "5\t16\t177.2\t0\t16\r\n",
      "300\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3216.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3216.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.09 s (50 us/read; 1.19 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  41,574\r\n",
      "Reads with adapters:                    19,675 (47.3%)\r\n",
      "Reads written (passing filters):        41,574 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    10,215,160 bp\r\n",
      "Total written (filtered):      9,920,626 bp (97.1%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 12708 times.\r\n",
      "12681 times, it overlapped the 5' end of a read\r\n",
      "27 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t1\t0.0\t0\t1\r\n",
      "19\t1\t0.0\t0\t1\r\n",
      "20\t4\t0.0\t0\t4\r\n",
      "21\t12675\t0.0\t0\t12675\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t649.6\t0\t22\r\n",
      "4\t3\t162.4\t0\t3\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "300\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 6967 times.\r\n",
      "20 times, it overlapped the 5' end of a read\r\n",
      "6947 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t649.6\t0\t18\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t85\t649.6\t0\t85\r\n",
      "4\t6861\t162.4\t0\t6861\r\n",
      "5\t1\t40.6\t0\t1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3221.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3221.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 2.09 s (46 us/read; 1.30 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  45,376\r\n",
      "Reads with adapters:                    24,079 (53.1%)\r\n",
      "Reads written (passing filters):        45,376 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,842,003 bp\r\n",
      "Total written (filtered):      9,496,733 bp (96.5%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 14671 times.\r\n",
      "14636 times, it overlapped the 5' end of a read\r\n",
      "35 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t1\t709.0\t0\t1\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "19\t3\t0.0\t0\t3\r\n",
      "20\t6\t0.0\t0\t6\r\n",
      "21\t14624\t0.0\t0\t14624\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t30\t709.0\t0\t30\r\n",
      "4\t2\t177.2\t0\t2\r\n",
      "5\t2\t44.3\t0\t2\r\n",
      "7\t1\t2.8\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 9408 times.\r\n",
      "23 times, it overlapped the 5' end of a read\r\n",
      "9385 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t21\t709.0\t0\t21\r\n",
      "26\t1\t0.0\t1\t0 1\r\n",
      "27\t1\t0.0\t1\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t65\t709.0\t0\t65\r\n",
      "4\t9318\t177.2\t0\t9318\r\n",
      "5\t1\t44.3\t0\t1\r\n",
      "247\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3221.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3221.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.77 s (39 us/read; 1.54 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  45,376\r\n",
      "Reads with adapters:                    30,086 (66.3%)\r\n",
      "Reads written (passing filters):        45,376 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:     9,837,328 bp\r\n",
      "Total written (filtered):      9,029,983 bp (91.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 50 times.\r\n",
      "8 times, it overlapped the 5' end of a read\r\n",
      "42 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t6\t709.0\t0\t6\r\n",
      "4\t2\t177.2\t0\t2\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t709.0\t0\t41\r\n",
      "4\t1\t177.2\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 30036 times.\r\n",
      "30006 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t2\t709.0\t0\t2\r\n",
      "22\t2\t0.0\t1\t1 1\r\n",
      "23\t1\t0.0\t1\t1\r\n",
      "24\t4\t0.0\t1\t1 3\r\n",
      "25\t7\t0.0\t1\t3 4\r\n",
      "26\t4712\t0.0\t1\t26 4686\r\n",
      "27\t25034\t0.0\t1\t18918 6116\r\n",
      "28\t244\t0.0\t1\t0 244\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t18\t709.0\t0\t18\r\n",
      "4\t4\t177.2\t0\t4\r\n",
      "5\t2\t44.3\t0\t2\r\n",
      "8\t1\t0.7\t0\t1\r\n",
      "299\t2\t0.0\t1\t2\r\n",
      "300\t3\t0.0\t1\t1 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3228.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3228.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.06 s (25 us/read; 2.40 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  42,394\r\n",
      "Reads with adapters:                    34,799 (82.1%)\r\n",
      "Reads written (passing filters):        42,394 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,222,236 bp\r\n",
      "Total written (filtered):     10,284,488 bp (91.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 31 times.\r\n",
      "1 times, it overlapped the 5' end of a read\r\n",
      "30 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "21\t1\t0.0\t0\t1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t29\t662.4\t0\t29\r\n",
      "5\t1\t41.4\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 34768 times.\r\n",
      "34748 times, it overlapped the 5' end of a read\r\n",
      "20 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "22\t1\t0.0\t1\t0 1\r\n",
      "24\t3\t0.0\t1\t1 2\r\n",
      "25\t6\t0.0\t1\t2 4\r\n",
      "26\t2408\t0.0\t1\t17 2391\r\n",
      "27\t32309\t0.0\t1\t26143 6166\r\n",
      "28\t21\t0.0\t1\t0 21\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t8\t662.4\t0\t8\r\n",
      "4\t2\t165.6\t0\t2\r\n",
      "5\t3\t41.4\t0\t3\r\n",
      "6\t1\t10.4\t0\t1\r\n",
      "299\t1\t0.0\t1\t0 1\r\n",
      "300\t5\t0.0\t1\t3 2\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3228.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3228.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 1.98 s (47 us/read; 1.28 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                  42,394\r\n",
      "Reads with adapters:                    38,199 (90.1%)\r\n",
      "Reads written (passing filters):        42,394 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    11,257,922 bp\r\n",
      "Total written (filtered):     10,532,061 bp (93.6%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 33721 times.\r\n",
      "33715 times, it overlapped the 5' end of a read\r\n",
      "6 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "18\t2\t0.0\t0\t2\r\n",
      "19\t7\t0.0\t0\t7\r\n",
      "20\t26\t0.0\t0\t26\r\n",
      "21\t33680\t0.0\t0\t33680\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t4\t662.4\t0\t4\r\n",
      "4\t1\t165.6\t0\t1\r\n",
      "7\t1\t2.6\t0\t1\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4478 times.\r\n",
      "42 times, it overlapped the 5' end of a read\r\n",
      "4436 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t41\t662.4\t0\t41\r\n",
      "27\t1\t0.0\t1\t0 1\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t25\t662.4\t0\t25\r\n",
      "4\t4411\t165.6\t0\t4411\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3127.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3127.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.95 s (49 us/read; 1.22 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 181,429\r\n",
      "Reads with adapters:                   172,799 (95.2%)\r\n",
      "Reads written (passing filters):       181,429 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    52,795,430 bp\r\n",
      "Total written (filtered):     49,200,148 bp (93.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 170612 times.\r\n",
      "170569 times, it overlapped the 5' end of a read\r\n",
      "43 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "13\t1\t0.0\t0\t1\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t29\t0.0\t0\t29\r\n",
      "19\t34\t0.0\t0\t34\r\n",
      "20\t230\t0.0\t0\t230\r\n",
      "21\t170272\t0.0\t0\t170272\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t24\t2834.8\t0\t24\r\n",
      "7\t2\t11.1\t0\t2\r\n",
      "300\t17\t0.0\t0\t17\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 2187 times.\r\n",
      "246 times, it overlapped the 5' end of a read\r\n",
      "1941 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t243\t2834.8\t0\t243\r\n",
      "27\t3\t0.0\t1\t3\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t22\t2834.8\t0\t22\r\n",
      "4\t1919\t708.7\t0\t1919\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3146.R2.fastq.gz.trimmed.fastq.gz /code/sequences/3146.R2.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 8.46 s (36 us/read; 1.68 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 237,589\r\n",
      "Reads with adapters:                    10,075 (4.2%)\r\n",
      "Reads written (passing filters):       237,589 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    64,592,138 bp\r\n",
      "Total written (filtered):     64,454,396 bp (99.8%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 5274 times.\r\n",
      "9 times, it overlapped the 5' end of a read\r\n",
      "5265 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t9\t3712.3\t0\t9\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t5215\t3712.3\t0\t5215\r\n",
      "4\t26\t928.1\t0\t26\r\n",
      "5\t24\t232.0\t0\t24\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 4801 times.\r\n",
      "4479 times, it overlapped the 5' end of a read\r\n",
      "322 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "4\t4\t928.1\t0\t4\r\n",
      "26\t3\t0.0\t1\t0 3\r\n",
      "27\t4472\t0.0\t1\t0 4472\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t297\t3712.3\t0\t297\r\n",
      "4\t8\t928.1\t0\t8\r\n",
      "5\t17\t232.0\t0\t17\r\n",
      "\r\n",
      "This is cutadapt 1.9.1 with Python 2.7.16\r\n",
      "Command line parameters: -e 0.047619047619 -b ATGCGTTGGAGAGARCGTTTC -b GATCACCTTCTAATTTACCWACAACTG -o /code/sequences/3146.R1.fastq.gz.trimmed.fastq.gz /code/sequences/3146.R1.fastq.gz\r\n",
      "Trimming 2 adapters with at most 4.8% errors in single-end mode ...\r\n",
      "Finished in 10.47 s (44 us/read; 1.36 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:                 237,589\r\n",
      "Reads with adapters:                   217,638 (91.6%)\r\n",
      "Reads written (passing filters):       237,589 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed:    64,887,697 bp\r\n",
      "Total written (filtered):     60,516,161 bp (93.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: ATGCGTTGGAGAGARCGTTTC; Type: variable 5'/3'; Length: 21; Trimmed: 205813 times.\r\n",
      "205726 times, it overlapped the 5' end of a read\r\n",
      "87 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-21 bp: 0\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "17\t3\t0.0\t0\t3\r\n",
      "18\t28\t0.0\t0\t28\r\n",
      "19\t38\t0.0\t0\t38\r\n",
      "20\t280\t0.0\t0\t280\r\n",
      "21\t205377\t0.0\t0\t205377\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t61\t3712.3\t0\t61\r\n",
      "4\t9\t928.1\t0\t9\r\n",
      "5\t1\t232.0\t0\t1\r\n",
      "53\t1\t0.0\t0\t1\r\n",
      "74\t1\t0.0\t0\t1\r\n",
      "299\t3\t0.0\t0\t3\r\n",
      "300\t11\t0.0\t0\t11\r\n",
      "\r\n",
      "=== Adapter 2 ===\r\n",
      "\r\n",
      "Sequence: GATCACCTTCTAATTTACCWACAACTG; Type: variable 5'/3'; Length: 27; Trimmed: 11825 times.\r\n",
      "217 times, it overlapped the 5' end of a read\r\n",
      "11608 times, it overlapped the 3' end or was within the read\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-20 bp: 0; 21-27 bp: 1\r\n",
      "\r\n",
      "Overview of removed sequences (5')\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t211\t3712.3\t0\t211\r\n",
      "27\t6\t0.0\t1\t6\r\n",
      "\r\n",
      "\r\n",
      "Overview of removed sequences (3' or within)\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "3\t34\t3712.3\t0\t34\r\n",
      "4\t11573\t928.1\t0\t11573\r\n",
      "5\t1\t232.0\t0\t1\r\n",
      "\r\n",
      "\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 155116 (77558 pairs)\n",
      "FastQ single records kept: 1328 (from PE1: 1060, from PE2: 268)\n",
      "FastQ paired records discarded: 388 (194 pairs)\n",
      "FastQ single records discarded: 1328 (from PE1: 268, from PE2: 1060)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 165710 (82855 pairs)\n",
      "FastQ single records kept: 6187 (from PE1: 4317, from PE2: 1870)\n",
      "FastQ paired records discarded: 2700 (1350 pairs)\n",
      "FastQ single records discarded: 6187 (from PE1: 1870, from PE2: 4317)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 161992 (80996 pairs)\n",
      "FastQ single records kept: 25332 (from PE1: 23557, from PE2: 1775)\n",
      "FastQ paired records discarded: 5086 (2543 pairs)\n",
      "FastQ single records discarded: 25332 (from PE1: 1775, from PE2: 23557)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 147908 (73954 pairs)\n",
      "FastQ single records kept: 12765 (from PE1: 11930, from PE2: 835)\n",
      "FastQ paired records discarded: 3036 (1518 pairs)\n",
      "FastQ single records discarded: 12765 (from PE1: 835, from PE2: 11930)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 58558 (29279 pairs)\n",
      "FastQ single records kept: 4299 (from PE1: 3998, from PE2: 301)\n",
      "FastQ paired records discarded: 1002 (501 pairs)\n",
      "FastQ single records discarded: 4299 (from PE1: 301, from PE2: 3998)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148742 (74371 pairs)\n",
      "FastQ single records kept: 16643 (from PE1: 15426, from PE2: 1217)\n",
      "FastQ paired records discarded: 4106 (2053 pairs)\n",
      "FastQ single records discarded: 16643 (from PE1: 1217, from PE2: 15426)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 162760 (81380 pairs)\n",
      "FastQ single records kept: 15766 (from PE1: 14529, from PE2: 1237)\n",
      "FastQ paired records discarded: 3814 (1907 pairs)\n",
      "FastQ single records discarded: 15766 (from PE1: 1237, from PE2: 14529)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 179010 (89505 pairs)\n",
      "FastQ single records kept: 4617 (from PE1: 3578, from PE2: 1039)\n",
      "FastQ paired records discarded: 1548 (774 pairs)\n",
      "FastQ single records discarded: 4617 (from PE1: 1039, from PE2: 3578)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148622 (74311 pairs)\n",
      "FastQ single records kept: 21643 (from PE1: 19388, from PE2: 2255)\n",
      "FastQ paired records discarded: 4538 (2269 pairs)\n",
      "FastQ single records discarded: 21643 (from PE1: 2255, from PE2: 19388)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 349520 (174760 pairs)\n",
      "FastQ single records kept: 6093 (from PE1: 4614, from PE2: 1479)\n",
      "FastQ paired records discarded: 1152 (576 pairs)\n",
      "FastQ single records discarded: 6093 (from PE1: 1479, from PE2: 4614)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 79784 (39892 pairs)\n",
      "FastQ single records kept: 5981 (from PE1: 5570, from PE2: 411)\n",
      "FastQ paired records discarded: 1876 (938 pairs)\n",
      "FastQ single records discarded: 5981 (from PE1: 411, from PE2: 5570)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 101180 (50590 pairs)\n",
      "FastQ single records kept: 6924 (from PE1: 6400, from PE2: 524)\n",
      "FastQ paired records discarded: 1580 (790 pairs)\n",
      "FastQ single records discarded: 6924 (from PE1: 524, from PE2: 6400)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 71506 (35753 pairs)\n",
      "FastQ single records kept: 5332 (from PE1: 4641, from PE2: 691)\n",
      "FastQ paired records discarded: 1264 (632 pairs)\n",
      "FastQ single records discarded: 5332 (from PE1: 691, from PE2: 4641)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 224660 (112330 pairs)\n",
      "FastQ single records kept: 8155 (from PE1: 6818, from PE2: 1337)\n",
      "FastQ paired records discarded: 2304 (1152 pairs)\n",
      "FastQ single records discarded: 8155 (from PE1: 1337, from PE2: 6818)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 193950 (96975 pairs)\n",
      "FastQ single records kept: 4011 (from PE1: 3164, from PE2: 847)\n",
      "FastQ paired records discarded: 770 (385 pairs)\n",
      "FastQ single records discarded: 4011 (from PE1: 847, from PE2: 3164)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 144388 (72194 pairs)\n",
      "FastQ single records kept: 20507 (from PE1: 19378, from PE2: 1129)\n",
      "FastQ paired records discarded: 4720 (2360 pairs)\n",
      "FastQ single records discarded: 20507 (from PE1: 1129, from PE2: 19378)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 192080 (96040 pairs)\n",
      "FastQ single records kept: 67315 (from PE1: 63209, from PE2: 4106)\n",
      "FastQ paired records discarded: 17444 (8722 pairs)\n",
      "FastQ single records discarded: 67315 (from PE1: 4106, from PE2: 63209)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 130774 (65387 pairs)\n",
      "FastQ single records kept: 6988 (from PE1: 6369, from PE2: 619)\n",
      "FastQ paired records discarded: 1744 (872 pairs)\n",
      "FastQ single records discarded: 6988 (from PE1: 619, from PE2: 6369)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 153096 (76548 pairs)\n",
      "FastQ single records kept: 7021 (from PE1: 6091, from PE2: 930)\n",
      "FastQ paired records discarded: 1740 (870 pairs)\n",
      "FastQ single records discarded: 7021 (from PE1: 930, from PE2: 6091)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 133878 (66939 pairs)\n",
      "FastQ single records kept: 14178 (from PE1: 13072, from PE2: 1106)\n",
      "FastQ paired records discarded: 3706 (1853 pairs)\n",
      "FastQ single records discarded: 14178 (from PE1: 1106, from PE2: 13072)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 181838 (90919 pairs)\n",
      "FastQ single records kept: 2635 (from PE1: 2202, from PE2: 433)\n",
      "FastQ paired records discarded: 400 (200 pairs)\n",
      "FastQ single records discarded: 2635 (from PE1: 433, from PE2: 2202)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 145984 (72992 pairs)\n",
      "FastQ single records kept: 3227 (from PE1: 2763, from PE2: 464)\n",
      "FastQ paired records discarded: 874 (437 pairs)\n",
      "FastQ single records discarded: 3227 (from PE1: 464, from PE2: 2763)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 236500 (118250 pairs)\n",
      "FastQ single records kept: 7395 (from PE1: 6073, from PE2: 1322)\n",
      "FastQ paired records discarded: 1322 (661 pairs)\n",
      "FastQ single records discarded: 7395 (from PE1: 1322, from PE2: 6073)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 150510 (75255 pairs)\n",
      "FastQ single records kept: 3618 (from PE1: 3107, from PE2: 511)\n",
      "FastQ paired records discarded: 834 (417 pairs)\n",
      "FastQ single records discarded: 3618 (from PE1: 511, from PE2: 3107)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 238708 (119354 pairs)\n",
      "FastQ single records kept: 10852 (from PE1: 9316, from PE2: 1536)\n",
      "FastQ paired records discarded: 3100 (1550 pairs)\n",
      "FastQ single records discarded: 10852 (from PE1: 1536, from PE2: 9316)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 129980 (64990 pairs)\n",
      "FastQ single records kept: 794 (from PE1: 631, from PE2: 163)\n",
      "FastQ paired records discarded: 180 (90 pairs)\n",
      "FastQ single records discarded: 794 (from PE1: 163, from PE2: 631)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 179310 (89655 pairs)\n",
      "FastQ single records kept: 1118 (from PE1: 901, from PE2: 217)\n",
      "FastQ paired records discarded: 178 (89 pairs)\n",
      "FastQ single records discarded: 1118 (from PE1: 217, from PE2: 901)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 177410 (88705 pairs)\n",
      "FastQ single records kept: 4731 (from PE1: 3969, from PE2: 762)\n",
      "FastQ paired records discarded: 1126 (563 pairs)\n",
      "FastQ single records discarded: 4731 (from PE1: 762, from PE2: 3969)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148048 (74024 pairs)\n",
      "FastQ single records kept: 8222 (from PE1: 6860, from PE2: 1362)\n",
      "FastQ paired records discarded: 2414 (1207 pairs)\n",
      "FastQ single records discarded: 8222 (from PE1: 1362, from PE2: 6860)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 200190 (100095 pairs)\n",
      "FastQ single records kept: 3546 (from PE1: 2782, from PE2: 764)\n",
      "FastQ paired records discarded: 620 (310 pairs)\n",
      "FastQ single records discarded: 3546 (from PE1: 764, from PE2: 2782)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 161872 (80936 pairs)\n",
      "FastQ single records kept: 2137 (from PE1: 1663, from PE2: 474)\n",
      "FastQ paired records discarded: 604 (302 pairs)\n",
      "FastQ single records discarded: 2137 (from PE1: 474, from PE2: 1663)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 152250 (76125 pairs)\n",
      "FastQ single records kept: 1416 (from PE1: 946, from PE2: 470)\n",
      "FastQ paired records discarded: 582 (291 pairs)\n",
      "FastQ single records discarded: 1416 (from PE1: 470, from PE2: 946)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 166882 (83441 pairs)\n",
      "FastQ single records kept: 2272 (from PE1: 1696, from PE2: 576)\n",
      "FastQ paired records discarded: 448 (224 pairs)\n",
      "FastQ single records discarded: 2272 (from PE1: 576, from PE2: 1696)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 449386 (224693 pairs)\n",
      "FastQ single records kept: 11685 (from PE1: 9260, from PE2: 2425)\n",
      "FastQ paired records discarded: 2422 (1211 pairs)\n",
      "FastQ single records discarded: 11685 (from PE1: 2425, from PE2: 9260)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 163426 (81713 pairs)\n",
      "FastQ single records kept: 4939 (from PE1: 3992, from PE2: 947)\n",
      "FastQ paired records discarded: 940 (470 pairs)\n",
      "FastQ single records discarded: 4939 (from PE1: 947, from PE2: 3992)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 121636 (60818 pairs)\n",
      "FastQ single records kept: 6401 (from PE1: 5844, from PE2: 557)\n",
      "FastQ paired records discarded: 11504 (5752 pairs)\n",
      "FastQ single records discarded: 6401 (from PE1: 557, from PE2: 5844)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 207568 (103784 pairs)\n",
      "FastQ single records kept: 2841 (from PE1: 2423, from PE2: 418)\n",
      "FastQ paired records discarded: 418 (209 pairs)\n",
      "FastQ single records discarded: 2841 (from PE1: 418, from PE2: 2423)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 170484 (85242 pairs)\n",
      "FastQ single records kept: 6519 (from PE1: 5418, from PE2: 1101)\n",
      "FastQ paired records discarded: 1298 (649 pairs)\n",
      "FastQ single records discarded: 6519 (from PE1: 1101, from PE2: 5418)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 288006 (144003 pairs)\n",
      "FastQ single records kept: 4040 (from PE1: 3440, from PE2: 600)\n",
      "FastQ paired records discarded: 612 (306 pairs)\n",
      "FastQ single records discarded: 4040 (from PE1: 600, from PE2: 3440)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 155556 (77778 pairs)\n",
      "FastQ single records kept: 4035 (from PE1: 3583, from PE2: 452)\n",
      "FastQ paired records discarded: 542 (271 pairs)\n",
      "FastQ single records discarded: 4035 (from PE1: 452, from PE2: 3583)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 68868 (34434 pairs)\n",
      "FastQ single records kept: 1214 (from PE1: 963, from PE2: 251)\n",
      "FastQ paired records discarded: 492 (246 pairs)\n",
      "FastQ single records discarded: 1214 (from PE1: 251, from PE2: 963)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 157154 (78577 pairs)\n",
      "FastQ single records kept: 6219 (from PE1: 5753, from PE2: 466)\n",
      "FastQ paired records discarded: 1298 (649 pairs)\n",
      "FastQ single records discarded: 6219 (from PE1: 466, from PE2: 5753)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 209256 (104628 pairs)\n",
      "FastQ single records kept: 13120 (from PE1: 11906, from PE2: 1214)\n",
      "FastQ paired records discarded: 2220 (1110 pairs)\n",
      "FastQ single records discarded: 13120 (from PE1: 1214, from PE2: 11906)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 123182 (61591 pairs)\n",
      "FastQ single records kept: 11335 (from PE1: 10361, from PE2: 974)\n",
      "FastQ paired records discarded: 1494 (747 pairs)\n",
      "FastQ single records discarded: 11335 (from PE1: 974, from PE2: 10361)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 136296 (68148 pairs)\n",
      "FastQ single records kept: 912 (from PE1: 674, from PE2: 238)\n",
      "FastQ paired records discarded: 220 (110 pairs)\n",
      "FastQ single records discarded: 912 (from PE1: 238, from PE2: 674)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 121272 (60636 pairs)\n",
      "FastQ single records kept: 8363 (from PE1: 7429, from PE2: 934)\n",
      "FastQ paired records discarded: 1590 (795 pairs)\n",
      "FastQ single records discarded: 8363 (from PE1: 934, from PE2: 7429)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 83072 (41536 pairs)\n",
      "FastQ single records kept: 2372 (from PE1: 2158, from PE2: 214)\n",
      "FastQ paired records discarded: 710 (355 pairs)\n",
      "FastQ single records discarded: 2372 (from PE1: 214, from PE2: 2158)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 237372 (118686 pairs)\n",
      "FastQ single records kept: 3246 (from PE1: 2938, from PE2: 308)\n",
      "FastQ paired records discarded: 292 (146 pairs)\n",
      "FastQ single records discarded: 3246 (from PE1: 308, from PE2: 2938)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 130814 (65407 pairs)\n",
      "FastQ single records kept: 5603 (from PE1: 4957, from PE2: 646)\n",
      "FastQ paired records discarded: 1414 (707 pairs)\n",
      "FastQ single records discarded: 5603 (from PE1: 646, from PE2: 4957)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 71492 (35746 pairs)\n",
      "FastQ single records kept: 2194 (from PE1: 1688, from PE2: 506)\n",
      "FastQ paired records discarded: 1424 (712 pairs)\n",
      "FastQ single records discarded: 2194 (from PE1: 506, from PE2: 1688)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 136264 (68132 pairs)\n",
      "FastQ single records kept: 8371 (from PE1: 7557, from PE2: 814)\n",
      "FastQ paired records discarded: 1622 (811 pairs)\n",
      "FastQ single records discarded: 8371 (from PE1: 814, from PE2: 7557)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 127322 (63661 pairs)\n",
      "FastQ single records kept: 1831 (from PE1: 1355, from PE2: 476)\n",
      "FastQ paired records discarded: 602 (301 pairs)\n",
      "FastQ single records discarded: 1831 (from PE1: 476, from PE2: 1355)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 164346 (82173 pairs)\n",
      "FastQ single records kept: 22258 (from PE1: 20703, from PE2: 1555)\n",
      "FastQ paired records discarded: 5492 (2746 pairs)\n",
      "FastQ single records discarded: 22258 (from PE1: 1555, from PE2: 20703)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 127390 (63695 pairs)\n",
      "FastQ single records kept: 10245 (from PE1: 9674, from PE2: 571)\n",
      "FastQ paired records discarded: 1938 (969 pairs)\n",
      "FastQ single records discarded: 10245 (from PE1: 571, from PE2: 9674)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 101874 (50937 pairs)\n",
      "FastQ single records kept: 2407 (from PE1: 1496, from PE2: 911)\n",
      "FastQ paired records discarded: 780 (390 pairs)\n",
      "FastQ single records discarded: 2407 (from PE1: 911, from PE2: 1496)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 165196 (82598 pairs)\n",
      "FastQ single records kept: 5278 (from PE1: 4843, from PE2: 435)\n",
      "FastQ paired records discarded: 990 (495 pairs)\n",
      "FastQ single records discarded: 5278 (from PE1: 435, from PE2: 4843)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 218462 (109231 pairs)\n",
      "FastQ single records kept: 2627 (from PE1: 2328, from PE2: 299)\n",
      "FastQ paired records discarded: 248 (124 pairs)\n",
      "FastQ single records discarded: 2627 (from PE1: 299, from PE2: 2328)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 233770 (116885 pairs)\n",
      "FastQ single records kept: 9166 (from PE1: 7866, from PE2: 1300)\n",
      "FastQ paired records discarded: 1570 (785 pairs)\n",
      "FastQ single records discarded: 9166 (from PE1: 1300, from PE2: 7866)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 111218 (55609 pairs)\n",
      "FastQ single records kept: 5189 (from PE1: 4828, from PE2: 361)\n",
      "FastQ paired records discarded: 1070 (535 pairs)\n",
      "FastQ single records discarded: 5189 (from PE1: 361, from PE2: 4828)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 187474 (93737 pairs)\n",
      "FastQ single records kept: 8034 (from PE1: 7257, from PE2: 777)\n",
      "FastQ paired records discarded: 1908 (954 pairs)\n",
      "FastQ single records discarded: 8034 (from PE1: 777, from PE2: 7257)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 162958 (81479 pairs)\n",
      "FastQ single records kept: 9605 (from PE1: 8917, from PE2: 688)\n",
      "FastQ paired records discarded: 2072 (1036 pairs)\n",
      "FastQ single records discarded: 9605 (from PE1: 688, from PE2: 8917)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 189316 (94658 pairs)\n",
      "FastQ single records kept: 3626 (from PE1: 2842, from PE2: 784)\n",
      "FastQ paired records discarded: 618 (309 pairs)\n",
      "FastQ single records discarded: 3626 (from PE1: 784, from PE2: 2842)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 142328 (71164 pairs)\n",
      "FastQ single records kept: 2197 (from PE1: 1920, from PE2: 277)\n",
      "FastQ paired records discarded: 340 (170 pairs)\n",
      "FastQ single records discarded: 2197 (from PE1: 277, from PE2: 1920)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 73126 (36563 pairs)\n",
      "FastQ single records kept: 4348 (from PE1: 4014, from PE2: 334)\n",
      "FastQ paired records discarded: 1326 (663 pairs)\n",
      "FastQ single records discarded: 4348 (from PE1: 334, from PE2: 4014)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 198282 (99141 pairs)\n",
      "FastQ single records kept: 2267 (from PE1: 2041, from PE2: 226)\n",
      "FastQ paired records discarded: 254 (127 pairs)\n",
      "FastQ single records discarded: 2267 (from PE1: 226, from PE2: 2041)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 106368 (53184 pairs)\n",
      "FastQ single records kept: 35859 (from PE1: 33747, from PE2: 2112)\n",
      "FastQ paired records discarded: 9262 (4631 pairs)\n",
      "FastQ single records discarded: 35859 (from PE1: 2112, from PE2: 33747)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 108486 (54243 pairs)\n",
      "FastQ single records kept: 6906 (from PE1: 5855, from PE2: 1051)\n",
      "FastQ paired records discarded: 1786 (893 pairs)\n",
      "FastQ single records discarded: 6906 (from PE1: 1051, from PE2: 5855)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 124168 (62084 pairs)\n",
      "FastQ single records kept: 2491 (from PE1: 1891, from PE2: 600)\n",
      "FastQ paired records discarded: 33590 (16795 pairs)\n",
      "FastQ single records discarded: 2491 (from PE1: 600, from PE2: 1891)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 96304 (48152 pairs)\n",
      "FastQ single records kept: 19309 (from PE1: 17978, from PE2: 1331)\n",
      "FastQ paired records discarded: 5220 (2610 pairs)\n",
      "FastQ single records discarded: 19309 (from PE1: 1331, from PE2: 17978)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 161226 (80613 pairs)\n",
      "FastQ single records kept: 3776 (from PE1: 2886, from PE2: 890)\n",
      "FastQ paired records discarded: 1310 (655 pairs)\n",
      "FastQ single records discarded: 3776 (from PE1: 890, from PE2: 2886)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 161038 (80519 pairs)\n",
      "FastQ single records kept: 15364 (from PE1: 13733, from PE2: 1631)\n",
      "FastQ paired records discarded: 3114 (1557 pairs)\n",
      "FastQ single records discarded: 15364 (from PE1: 1631, from PE2: 13733)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "FastQ paired records kept: 126686 (63343 pairs)\n",
      "FastQ single records kept: 6047 (from PE1: 5414, from PE2: 633)\n",
      "FastQ paired records discarded: 1008 (504 pairs)\n",
      "FastQ single records discarded: 6047 (from PE1: 633, from PE2: 5414)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 107816 (53908 pairs)\n",
      "FastQ single records kept: 6688 (from PE1: 5966, from PE2: 722)\n",
      "FastQ paired records discarded: 1664 (832 pairs)\n",
      "FastQ single records discarded: 6688 (from PE1: 722, from PE2: 5966)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 89172 (44586 pairs)\n",
      "FastQ single records kept: 5814 (from PE1: 5388, from PE2: 426)\n",
      "FastQ paired records discarded: 1206 (603 pairs)\n",
      "FastQ single records discarded: 5814 (from PE1: 426, from PE2: 5388)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 165206 (82603 pairs)\n",
      "FastQ single records kept: 6927 (from PE1: 5244, from PE2: 1683)\n",
      "FastQ paired records discarded: 2760 (1380 pairs)\n",
      "FastQ single records discarded: 6927 (from PE1: 1683, from PE2: 5244)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 201898 (100949 pairs)\n",
      "FastQ single records kept: 21677 (from PE1: 15513, from PE2: 6164)\n",
      "FastQ paired records discarded: 6782 (3391 pairs)\n",
      "FastQ single records discarded: 21677 (from PE1: 6164, from PE2: 15513)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 148140 (74070 pairs)\n",
      "FastQ single records kept: 5703 (from PE1: 4956, from PE2: 747)\n",
      "FastQ paired records discarded: 1490 (745 pairs)\n",
      "FastQ single records discarded: 5703 (from PE1: 747, from PE2: 4956)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 259174 (129587 pairs)\n",
      "FastQ single records kept: 3455 (from PE1: 2879, from PE2: 576)\n",
      "FastQ paired records discarded: 478 (239 pairs)\n",
      "FastQ single records discarded: 3455 (from PE1: 576, from PE2: 2879)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 159822 (79911 pairs)\n",
      "FastQ single records kept: 3007 (from PE1: 2659, from PE2: 348)\n",
      "FastQ paired records discarded: 620 (310 pairs)\n",
      "FastQ single records discarded: 3007 (from PE1: 348, from PE2: 2659)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 174234 (87117 pairs)\n",
      "FastQ single records kept: 1104 (from PE1: 893, from PE2: 211)\n",
      "FastQ paired records discarded: 252 (126 pairs)\n",
      "FastQ single records discarded: 1104 (from PE1: 211, from PE2: 893)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 104572 (52286 pairs)\n",
      "FastQ single records kept: 4426 (from PE1: 3977, from PE2: 449)\n",
      "FastQ paired records discarded: 990 (495 pairs)\n",
      "FastQ single records discarded: 4426 (from PE1: 449, from PE2: 3977)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 102084 (51042 pairs)\n",
      "FastQ single records kept: 9534 (from PE1: 8401, from PE2: 1133)\n",
      "FastQ paired records discarded: 1406 (703 pairs)\n",
      "FastQ single records discarded: 9534 (from PE1: 1133, from PE2: 8401)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 158632 (79316 pairs)\n",
      "FastQ single records kept: 3549 (from PE1: 3312, from PE2: 237)\n",
      "FastQ paired records discarded: 546 (273 pairs)\n",
      "FastQ single records discarded: 3549 (from PE1: 237, from PE2: 3312)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 109418 (54709 pairs)\n",
      "FastQ single records kept: 3666 (from PE1: 3215, from PE2: 451)\n",
      "FastQ paired records discarded: 778 (389 pairs)\n",
      "FastQ single records discarded: 3666 (from PE1: 451, from PE2: 3215)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 132554 (66277 pairs)\n",
      "FastQ single records kept: 22827 (from PE1: 20279, from PE2: 2548)\n",
      "FastQ paired records discarded: 4642 (2321 pairs)\n",
      "FastQ single records discarded: 22827 (from PE1: 2548, from PE2: 20279)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 75234 (37617 pairs)\n",
      "FastQ single records kept: 3585 (from PE1: 3316, from PE2: 269)\n",
      "FastQ paired records discarded: 868 (434 pairs)\n",
      "FastQ single records discarded: 3585 (from PE1: 269, from PE2: 3316)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 94256 (47128 pairs)\n",
      "FastQ single records kept: 9309 (from PE1: 8786, from PE2: 523)\n",
      "FastQ paired records discarded: 1856 (928 pairs)\n",
      "FastQ single records discarded: 9309 (from PE1: 523, from PE2: 8786)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 137928 (68964 pairs)\n",
      "FastQ single records kept: 3308 (from PE1: 2287, from PE2: 1021)\n",
      "FastQ paired records discarded: 1220 (610 pairs)\n",
      "FastQ single records discarded: 3308 (from PE1: 1021, from PE2: 2287)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 173932 (86966 pairs)\n",
      "FastQ single records kept: 4732 (from PE1: 3609, from PE2: 1123)\n",
      "FastQ paired records discarded: 1716 (858 pairs)\n",
      "FastQ single records discarded: 4732 (from PE1: 1123, from PE2: 3609)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 180272 (90136 pairs)\n",
      "FastQ single records kept: 3210 (from PE1: 2353, from PE2: 857)\n",
      "FastQ paired records discarded: 1206 (603 pairs)\n",
      "FastQ single records discarded: 3210 (from PE1: 857, from PE2: 2353)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 185088 (92544 pairs)\n",
      "FastQ single records kept: 3581 (from PE1: 2867, from PE2: 714)\n",
      "FastQ paired records discarded: 962 (481 pairs)\n",
      "FastQ single records discarded: 3581 (from PE1: 714, from PE2: 2867)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 192496 (96248 pairs)\n",
      "FastQ single records kept: 1400 (from PE1: 1110, from PE2: 290)\n",
      "FastQ paired records discarded: 346 (173 pairs)\n",
      "FastQ single records discarded: 1400 (from PE1: 290, from PE2: 1110)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 237746 (118873 pairs)\n",
      "FastQ single records kept: 5746 (from PE1: 4679, from PE2: 1067)\n",
      "FastQ paired records discarded: 1920 (960 pairs)\n",
      "FastQ single records discarded: 5746 (from PE1: 1067, from PE2: 4679)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 274794 (137397 pairs)\n",
      "FastQ single records kept: 4565 (from PE1: 3795, from PE2: 770)\n",
      "FastQ paired records discarded: 630 (315 pairs)\n",
      "FastQ single records discarded: 4565 (from PE1: 770, from PE2: 3795)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 189204 (94602 pairs)\n",
      "FastQ single records kept: 3630 (from PE1: 3072, from PE2: 558)\n",
      "FastQ paired records discarded: 492 (246 pairs)\n",
      "FastQ single records discarded: 3630 (from PE1: 558, from PE2: 3072)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 139568 (69784 pairs)\n",
      "FastQ single records kept: 12048 (from PE1: 10955, from PE2: 1093)\n",
      "FastQ paired records discarded: 2076 (1038 pairs)\n",
      "FastQ single records discarded: 12048 (from PE1: 1093, from PE2: 10955)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 294396 (147198 pairs)\n",
      "FastQ single records kept: 8453 (from PE1: 5973, from PE2: 2480)\n",
      "FastQ paired records discarded: 3616 (1808 pairs)\n",
      "FastQ single records discarded: 8453 (from PE1: 2480, from PE2: 5973)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 76982 (38491 pairs)\n",
      "FastQ single records kept: 3397 (from PE1: 3136, from PE2: 261)\n",
      "FastQ paired records discarded: 1012 (506 pairs)\n",
      "FastQ single records discarded: 3397 (from PE1: 261, from PE2: 3136)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 71588 (35794 pairs)\n",
      "FastQ single records kept: 5403 (from PE1: 4945, from PE2: 458)\n",
      "FastQ paired records discarded: 8358 (4179 pairs)\n",
      "FastQ single records discarded: 5403 (from PE1: 458, from PE2: 4945)\n",
      "\n",
      "\n",
      "\n",
      "FastQ paired records kept: 74596 (37298 pairs)\n",
      "FastQ single records kept: 13061 (from PE1: 12182, from PE2: 879)\n",
      "FastQ paired records discarded: 4304 (2152 pairs)\n",
      "FastQ single records discarded: 13061 (from PE1: 879, from PE2: 12182)\n",
      "\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2712.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2712.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61,325 / 65,387 (93.788%)\n",
      "Discarded reads ...................: 0 / 65,387 (0.000%)\n",
      "Not assembled reads ...............: 4,062 / 65,387 (6.212%)\n",
      "Assembled reads file...............: /code/sequences/2712.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2712.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2712.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2712.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2964.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2964.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 53,542 / 82,855 (64.621%)\n",
      "Discarded reads ...................: 0 / 82,855 (0.000%)\n",
      "Not assembled reads ...............: 29,313 / 82,855 (35.379%)\n",
      "Assembled reads file...............: /code/sequences/2964.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2964.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2964.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2964.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2658.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2658.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 48,316 / 80,996 (59.652%)\n",
      "Discarded reads ...................: 0 / 80,996 (0.000%)\n",
      "Not assembled reads ...............: 32,680 / 80,996 (40.348%)\n",
      "Assembled reads file...............: /code/sequences/2658.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2658.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2658.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2658.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2717.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2717.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 67,672 / 73,954 (91.506%)\n",
      "Discarded reads ...................: 0 / 73,954 (0.000%)\n",
      "Not assembled reads ...............: 6,282 / 73,954 (8.494%)\n",
      "Assembled reads file...............: /code/sequences/2717.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2717.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2717.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2717.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2715.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2715.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,067 / 29,279 (92.445%)\n",
      "Discarded reads ...................: 0 / 29,279 (0.000%)\n",
      "Not assembled reads ...............: 2,212 / 29,279 (7.555%)\n",
      "Assembled reads file...............: /code/sequences/2715.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2715.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2715.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2715.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2718.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2718.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 66,666 / 74,371 (89.640%)\n",
      "Discarded reads ...................: 0 / 74,371 (0.000%)\n",
      "Not assembled reads ...............: 7,705 / 74,371 (10.360%)\n",
      "Assembled reads file...............: /code/sequences/2718.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2718.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2718.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2718.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2719.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2719.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 72,981 / 81,380 (89.679%)\n",
      "Discarded reads ...................: 0 / 81,380 (0.000%)\n",
      "Not assembled reads ...............: 8,399 / 81,380 (10.321%)\n",
      "Assembled reads file...............: /code/sequences/2719.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2719.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2719.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2719.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3045.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3045.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 74,717 / 89,505 (83.478%)\n",
      "Discarded reads ...................: 0 / 89,505 (0.000%)\n",
      "Not assembled reads ...............: 14,788 / 89,505 (16.522%)\n",
      "Assembled reads file...............: /code/sequences/3045.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3045.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3045.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3045.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3047.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3047.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,825 / 74,311 (52.247%)\n",
      "Discarded reads ...................: 0 / 74,311 (0.000%)\n",
      "Not assembled reads ...............: 35,486 / 74,311 (47.753%)\n",
      "Assembled reads file...............: /code/sequences/3047.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3047.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3047.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3047.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3127.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3127.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 48%\n",
      "Assemblying reads: 95%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 163,247 / 174,760 (93.412%)\n",
      "Discarded reads ...................: 0 / 174,760 (0.000%)\n",
      "Not assembled reads ...............: 11,513 / 174,760 (6.588%)\n",
      "Assembled reads file...............: /code/sequences/3127.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3127.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3127.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3127.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2481.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2481.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 63%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 124,015 / 129,587 (95.700%)\n",
      "Discarded reads ...................: 0 / 129,587 (0.000%)\n",
      "Not assembled reads ...............: 5,572 / 129,587 (4.300%)\n",
      "Assembled reads file...............: /code/sequences/2481.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2481.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2481.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2481.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2910.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2910.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,792 / 35,753 (72.139%)\n",
      "Discarded reads ...................: 0 / 35,753 (0.000%)\n",
      "Not assembled reads ...............: 9,961 / 35,753 (27.861%)\n",
      "Assembled reads file...............: /code/sequences/2910.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2910.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2910.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2910.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2913.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2913.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 90%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 90,219 / 112,330 (80.316%)\n",
      "Discarded reads ...................: 0 / 112,330 (0.000%)\n",
      "Not assembled reads ...............: 22,111 / 112,330 (19.684%)\n",
      "Assembled reads file...............: /code/sequences/2913.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2913.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2913.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2913.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2463.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2463.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 86%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 89,273 / 96,975 (92.058%)\n",
      "Discarded reads ...................: 0 / 96,975 (0.000%)\n",
      "Not assembled reads ...............: 7,702 / 96,975 (7.942%)\n",
      "Assembled reads file...............: /code/sequences/2463.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2463.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2463.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2463.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2706.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2706.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 63,054 / 72,194 (87.340%)\n",
      "Discarded reads ...................: 0 / 72,194 (0.000%)\n",
      "Not assembled reads ...............: 9,140 / 72,194 (12.660%)\n",
      "Assembled reads file...............: /code/sequences/2706.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2706.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2706.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2706.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2700.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2700.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 71,973 / 96,040 (74.941%)\n",
      "Discarded reads ...................: 0 / 96,040 (0.000%)\n",
      "Not assembled reads ...............: 24,067 / 96,040 (25.059%)\n",
      "Assembled reads file...............: /code/sequences/2700.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2700.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2700.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2700.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2599.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2599.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 99%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 75,898 / 81,479 (93.150%)\n",
      "Discarded reads ...................: 0 / 81,479 (0.000%)\n",
      "Not assembled reads ...............: 5,581 / 81,479 (6.850%)\n",
      "Assembled reads file...............: /code/sequences/2599.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2599.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2599.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2599.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2926.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2926.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 89%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 87,593 / 94,658 (92.536%)\n",
      "Discarded reads ...................: 0 / 94,658 (0.000%)\n",
      "Not assembled reads ...............: 7,065 / 94,658 (7.464%)\n",
      "Assembled reads file...............: /code/sequences/2926.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2926.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2926.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2926.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2559.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2559.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 59,771 / 66,939 (89.292%)\n",
      "Discarded reads ...................: 0 / 66,939 (0.000%)\n",
      "Not assembled reads ...............: 7,168 / 66,939 (10.708%)\n",
      "Assembled reads file...............: /code/sequences/2559.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2559.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2559.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2559.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2992.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2992.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 90%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 86,549 / 90,919 (95.194%)\n",
      "Discarded reads ...................: 0 / 90,919 (0.000%)\n",
      "Not assembled reads ...............: 4,370 / 90,919 (4.806%)\n",
      "Assembled reads file...............: /code/sequences/2992.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2992.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2992.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2992.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3031.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3031.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 62,773 / 72,992 (86.000%)\n",
      "Discarded reads ...................: 0 / 72,992 (0.000%)\n",
      "Not assembled reads ...............: 10,219 / 72,992 (14.000%)\n",
      "Assembled reads file...............: /code/sequences/3031.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3031.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3031.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3031.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2497.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2497.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 80%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 95,489 / 118,250 (80.752%)\n",
      "Discarded reads ...................: 0 / 118,250 (0.000%)\n",
      "Not assembled reads ...............: 22,761 / 118,250 (19.248%)\n",
      "Assembled reads file...............: /code/sequences/2497.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2497.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2497.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2497.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2397.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2397.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 64,386 / 75,255 (85.557%)\n",
      "Discarded reads ...................: 0 / 75,255 (0.000%)\n",
      "Not assembled reads ...............: 10,869 / 75,255 (14.443%)\n",
      "Assembled reads file...............: /code/sequences/2397.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2397.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2397.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2397.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2991.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2991.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 89%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 90,902 / 119,354 (76.162%)\n",
      "Discarded reads ...................: 0 / 119,354 (0.000%)\n",
      "Not assembled reads ...............: 28,452 / 119,354 (23.838%)\n",
      "Assembled reads file...............: /code/sequences/2991.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2991.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2991.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2991.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2797.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2797.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 56,724 / 64,990 (87.281%)\n",
      "Discarded reads ...................: 0 / 64,990 (0.000%)\n",
      "Not assembled reads ...............: 8,266 / 64,990 (12.719%)\n",
      "Assembled reads file...............: /code/sequences/2797.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2797.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2797.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2797.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2794.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2794.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 98%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 76,227 / 89,655 (85.023%)\n",
      "Discarded reads ...................: 0 / 89,655 (0.000%)\n",
      "Not assembled reads ...............: 13,428 / 89,655 (14.977%)\n",
      "Assembled reads file...............: /code/sequences/2794.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2794.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2794.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2794.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2903.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2903.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 72,275 / 88,705 (81.478%)\n",
      "Discarded reads ...................: 0 / 88,705 (0.000%)\n",
      "Not assembled reads ...............: 16,430 / 88,705 (18.522%)\n",
      "Assembled reads file...............: /code/sequences/2903.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2903.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2903.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2903.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2475.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2475.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 45,138 / 50,590 (89.223%)\n",
      "Discarded reads ...................: 0 / 50,590 (0.000%)\n",
      "Not assembled reads ...............: 5,452 / 50,590 (10.777%)\n",
      "Assembled reads file...............: /code/sequences/2475.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2475.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2475.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2475.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2649.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2649.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 84%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 101,430 / 118,873 (85.326%)\n",
      "Discarded reads ...................: 0 / 118,873 (0.000%)\n",
      "Not assembled reads ...............: 17,443 / 118,873 (14.674%)\n",
      "Assembled reads file...............: /code/sequences/2649.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2649.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2649.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2649.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2772.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2772.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 64,837 / 80,936 (80.109%)\n",
      "Discarded reads ...................: 0 / 80,936 (0.000%)\n",
      "Not assembled reads ...............: 16,099 / 80,936 (19.891%)\n",
      "Assembled reads file...............: /code/sequences/2772.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2772.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2772.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2772.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2773.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2773.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 60,771 / 76,125 (79.831%)\n",
      "Discarded reads ...................: 0 / 76,125 (0.000%)\n",
      "Not assembled reads ...............: 15,354 / 76,125 (20.169%)\n",
      "Assembled reads file...............: /code/sequences/2773.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2773.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2773.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2773.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3141.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3141.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 99%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 79,733 / 83,441 (95.556%)\n",
      "Discarded reads ...................: 0 / 83,441 (0.000%)\n",
      "Not assembled reads ...............: 3,708 / 83,441 (4.444%)\n",
      "Assembled reads file...............: /code/sequences/3141.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3141.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3141.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3141.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3146.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3146.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 39%\n",
      "Assemblying reads: 77%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 200,116 / 224,693 (89.062%)\n",
      "Discarded reads ...................: 0 / 224,693 (0.000%)\n",
      "Not assembled reads ...............: 24,577 / 224,693 (10.938%)\n",
      "Assembled reads file...............: /code/sequences/3146.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3146.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3146.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3146.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2540.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2540.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 68,752 / 81,713 (84.138%)\n",
      "Discarded reads ...................: 0 / 81,713 (0.000%)\n",
      "Not assembled reads ...............: 12,961 / 81,713 (15.862%)\n",
      "Assembled reads file...............: /code/sequences/2540.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2540.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2540.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2540.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3028.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3028.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 57,472 / 60,818 (94.498%)\n",
      "Discarded reads ...................: 0 / 60,818 (0.000%)\n",
      "Not assembled reads ...............: 3,346 / 60,818 (5.502%)\n",
      "Assembled reads file...............: /code/sequences/3028.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3028.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3028.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3028.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2983.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2983.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 80%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 99,211 / 103,784 (95.594%)\n",
      "Discarded reads ...................: 0 / 103,784 (0.000%)\n",
      "Not assembled reads ...............: 4,573 / 103,784 (4.406%)\n",
      "Assembled reads file...............: /code/sequences/2983.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2983.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2983.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2983.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2549.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2549.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 64,865 / 85,242 (76.095%)\n",
      "Discarded reads ...................: 0 / 85,242 (0.000%)\n",
      "Not assembled reads ...............: 20,377 / 85,242 (23.905%)\n",
      "Assembled reads file...............: /code/sequences/2549.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2549.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2549.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2549.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2987.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2987.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 57%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 137,033 / 144,003 (95.160%)\n",
      "Discarded reads ...................: 0 / 144,003 (0.000%)\n",
      "Not assembled reads ...............: 6,970 / 144,003 (4.840%)\n",
      "Assembled reads file...............: /code/sequences/2987.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2987.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2987.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2987.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2788.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2788.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 62,041 / 77,778 (79.767%)\n",
      "Discarded reads ...................: 0 / 77,778 (0.000%)\n",
      "Not assembled reads ...............: 15,737 / 77,778 (20.233%)\n",
      "Assembled reads file...............: /code/sequences/2788.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2788.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2788.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2788.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2381.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2381.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,739 / 34,434 (92.173%)\n",
      "Discarded reads ...................: 0 / 34,434 (0.000%)\n",
      "Not assembled reads ...............: 2,695 / 34,434 (7.827%)\n",
      "Assembled reads file...............: /code/sequences/2381.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2381.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2381.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2381.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2606.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2606.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 74,353 / 78,577 (94.624%)\n",
      "Discarded reads ...................: 0 / 78,577 (0.000%)\n",
      "Not assembled reads ...............: 4,224 / 78,577 (5.376%)\n",
      "Assembled reads file...............: /code/sequences/2606.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2606.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2606.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2606.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2938.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2938.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 95%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 80,821 / 104,628 (77.246%)\n",
      "Discarded reads ...................: 0 / 104,628 (0.000%)\n",
      "Not assembled reads ...............: 23,807 / 104,628 (22.754%)\n",
      "Assembled reads file...............: /code/sequences/2938.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2938.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2938.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2938.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2939.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2939.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 41,196 / 61,591 (66.886%)\n",
      "Discarded reads ...................: 0 / 61,591 (0.000%)\n",
      "Not assembled reads ...............: 20,395 / 61,591 (33.114%)\n",
      "Assembled reads file...............: /code/sequences/2939.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2939.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2939.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2939.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2933.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2933.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 42,875 / 60,636 (70.709%)\n",
      "Discarded reads ...................: 0 / 60,636 (0.000%)\n",
      "Not assembled reads ...............: 17,761 / 60,636 (29.291%)\n",
      "Assembled reads file...............: /code/sequences/2933.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2933.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2933.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2933.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2766.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2766.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,198 / 68,148 (85.399%)\n",
      "Discarded reads ...................: 0 / 68,148 (0.000%)\n",
      "Not assembled reads ...............: 9,950 / 68,148 (14.601%)\n",
      "Assembled reads file...............: /code/sequences/2766.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2766.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2766.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2766.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3154.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3154.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,600 / 41,536 (90.524%)\n",
      "Discarded reads ...................: 0 / 41,536 (0.000%)\n",
      "Not assembled reads ...............: 3,936 / 41,536 (9.476%)\n",
      "Assembled reads file...............: /code/sequences/3154.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3154.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3154.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3154.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2579.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2579.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 68%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 114,334 / 118,686 (96.333%)\n",
      "Discarded reads ...................: 0 / 118,686 (0.000%)\n",
      "Not assembled reads ...............: 4,352 / 118,686 (3.667%)\n",
      "Assembled reads file...............: /code/sequences/2579.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2579.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2579.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2579.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3012.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3012.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61,705 / 65,407 (94.340%)\n",
      "Discarded reads ...................: 0 / 65,407 (0.000%)\n",
      "Not assembled reads ...............: 3,702 / 65,407 (5.660%)\n",
      "Assembled reads file...............: /code/sequences/3012.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3012.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3012.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3012.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3016.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3016.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,601 / 35,746 (85.607%)\n",
      "Discarded reads ...................: 0 / 35,746 (0.000%)\n",
      "Not assembled reads ...............: 5,145 / 35,746 (14.393%)\n",
      "Assembled reads file...............: /code/sequences/3016.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3016.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3016.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3016.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2673.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2673.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 54,276 / 68,132 (79.663%)\n",
      "Discarded reads ...................: 0 / 68,132 (0.000%)\n",
      "Not assembled reads ...............: 13,856 / 68,132 (20.337%)\n",
      "Assembled reads file...............: /code/sequences/2673.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2673.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2673.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2673.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3091.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3091.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 52,235 / 63,661 (82.052%)\n",
      "Discarded reads ...................: 0 / 63,661 (0.000%)\n",
      "Not assembled reads ...............: 11,426 / 63,661 (17.948%)\n",
      "Assembled reads file...............: /code/sequences/3091.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3091.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3091.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3091.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2675.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2675.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 71,669 / 82,173 (87.217%)\n",
      "Discarded reads ...................: 0 / 82,173 (0.000%)\n",
      "Not assembled reads ...............: 10,504 / 82,173 (12.783%)\n",
      "Assembled reads file...............: /code/sequences/2675.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2675.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2675.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2675.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2676.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2676.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,554 / 63,695 (91.929%)\n",
      "Discarded reads ...................: 0 / 63,695 (0.000%)\n",
      "Not assembled reads ...............: 5,141 / 63,695 (8.071%)\n",
      "Assembled reads file...............: /code/sequences/2676.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2676.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2676.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2676.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3098.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3098.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 36,991 / 50,937 (72.621%)\n",
      "Discarded reads ...................: 0 / 50,937 (0.000%)\n",
      "Not assembled reads ...............: 13,946 / 50,937 (27.379%)\n",
      "Assembled reads file...............: /code/sequences/3098.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3098.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3098.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3098.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2459.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2459.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 97%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 78,881 / 82,598 (95.500%)\n",
      "Discarded reads ...................: 0 / 82,598 (0.000%)\n",
      "Not assembled reads ...............: 3,717 / 82,598 (4.500%)\n",
      "Assembled reads file...............: /code/sequences/2459.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2459.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2459.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2459.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2592.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2592.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 74%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 105,895 / 109,231 (96.946%)\n",
      "Discarded reads ...................: 0 / 109,231 (0.000%)\n",
      "Not assembled reads ...............: 3,336 / 109,231 (3.054%)\n",
      "Assembled reads file...............: /code/sequences/2592.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2592.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2592.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2592.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2593.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2593.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 85%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 86,311 / 116,885 (73.843%)\n",
      "Discarded reads ...................: 0 / 116,885 (0.000%)\n",
      "Not assembled reads ...............: 30,574 / 116,885 (26.157%)\n",
      "Assembled reads file...............: /code/sequences/2593.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2593.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2593.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2593.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2455.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2455.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 52,103 / 55,609 (93.695%)\n",
      "Discarded reads ...................: 0 / 55,609 (0.000%)\n",
      "Not assembled reads ...............: 3,506 / 55,609 (6.305%)\n",
      "Assembled reads file...............: /code/sequences/2455.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2455.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2455.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2455.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2456.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2456.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 87%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 88,739 / 93,737 (94.668%)\n",
      "Discarded reads ...................: 0 / 93,737 (0.000%)\n",
      "Not assembled reads ...............: 4,998 / 93,737 (5.332%)\n",
      "Assembled reads file...............: /code/sequences/2456.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2456.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2456.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2456.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2901.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2901.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 53,605 / 74,024 (72.416%)\n",
      "Discarded reads ...................: 0 / 74,024 (0.000%)\n",
      "Not assembled reads ...............: 20,419 / 74,024 (27.584%)\n",
      "Assembled reads file...............: /code/sequences/2901.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2901.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2901.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2901.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2618.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2618.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,917 / 76,548 (76.967%)\n",
      "Discarded reads ...................: 0 / 76,548 (0.000%)\n",
      "Not assembled reads ...............: 17,631 / 76,548 (23.033%)\n",
      "Assembled reads file...............: /code/sequences/2618.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2618.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2618.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2618.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2801.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2801.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61,238 / 71,164 (86.052%)\n",
      "Discarded reads ...................: 0 / 71,164 (0.000%)\n",
      "Not assembled reads ...............: 9,926 / 71,164 (13.948%)\n",
      "Assembled reads file...............: /code/sequences/2801.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2801.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2801.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2801.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3216.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3216.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 29,608 / 36,563 (80.978%)\n",
      "Discarded reads ...................: 0 / 36,563 (0.000%)\n",
      "Not assembled reads ...............: 6,955 / 36,563 (19.022%)\n",
      "Assembled reads file...............: /code/sequences/3216.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3216.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3216.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3216.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2881.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2881.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 82%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 95,979 / 99,141 (96.811%)\n",
      "Discarded reads ...................: 0 / 99,141 (0.000%)\n",
      "Not assembled reads ...............: 3,162 / 99,141 (3.189%)\n",
      "Assembled reads file...............: /code/sequences/2881.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2881.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2881.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2881.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2561.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2561.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 39,014 / 53,184 (73.357%)\n",
      "Discarded reads ...................: 0 / 53,184 (0.000%)\n",
      "Not assembled reads ...............: 14,170 / 53,184 (26.643%)\n",
      "Assembled reads file...............: /code/sequences/2561.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2561.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2561.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2561.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3125.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3125.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,244 / 39,892 (78.321%)\n",
      "Discarded reads ...................: 0 / 39,892 (0.000%)\n",
      "Not assembled reads ...............: 8,648 / 39,892 (21.679%)\n",
      "Assembled reads file...............: /code/sequences/3125.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3125.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3125.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3125.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2889.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2889.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 39,408 / 54,243 (72.651%)\n",
      "Discarded reads ...................: 0 / 54,243 (0.000%)\n",
      "Not assembled reads ...............: 14,835 / 54,243 (27.349%)\n",
      "Assembled reads file...............: /code/sequences/2889.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2889.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2889.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2889.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3089.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3089.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 52,091 / 62,084 (83.904%)\n",
      "Discarded reads ...................: 0 / 62,084 (0.000%)\n",
      "Not assembled reads ...............: 9,993 / 62,084 (16.096%)\n",
      "Assembled reads file...............: /code/sequences/3089.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3089.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3089.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3089.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2684.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2684.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 40,517 / 48,152 (84.144%)\n",
      "Discarded reads ...................: 0 / 48,152 (0.000%)\n",
      "Not assembled reads ...............: 7,635 / 48,152 (15.856%)\n",
      "Assembled reads file...............: /code/sequences/2684.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2684.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2684.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2684.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3086.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3086.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61,811 / 80,613 (76.676%)\n",
      "Discarded reads ...................: 0 / 80,613 (0.000%)\n",
      "Not assembled reads ...............: 18,802 / 80,613 (23.324%)\n",
      "Assembled reads file...............: /code/sequences/3086.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3086.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3086.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3086.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2909.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2909.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 53,422 / 80,519 (66.347%)\n",
      "Discarded reads ...................: 0 / 80,519 (0.000%)\n",
      "Not assembled reads ...............: 27,097 / 80,519 (33.653%)\n",
      "Assembled reads file...............: /code/sequences/2909.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2909.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2909.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2909.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2620.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2620.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 48,887 / 63,343 (77.178%)\n",
      "Discarded reads ...................: 0 / 63,343 (0.000%)\n",
      "Not assembled reads ...............: 14,456 / 63,343 (22.822%)\n",
      "Assembled reads file...............: /code/sequences/2620.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2620.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2620.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2620.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2425.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2425.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 49,458 / 53,908 (91.745%)\n",
      "Discarded reads ...................: 0 / 53,908 (0.000%)\n",
      "Not assembled reads ...............: 4,450 / 53,908 (8.255%)\n",
      "Assembled reads file...............: /code/sequences/2425.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2425.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2425.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2425.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2426.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2426.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 41,307 / 44,586 (92.646%)\n",
      "Discarded reads ...................: 0 / 44,586 (0.000%)\n",
      "Not assembled reads ...............: 3,279 / 44,586 (7.354%)\n",
      "Assembled reads file...............: /code/sequences/2426.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2426.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2426.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2426.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2956.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2956.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 52,520 / 82,603 (63.581%)\n",
      "Discarded reads ...................: 0 / 82,603 (0.000%)\n",
      "Not assembled reads ...............: 30,083 / 82,603 (36.419%)\n",
      "Assembled reads file...............: /code/sequences/2956.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2956.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2956.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2956.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2951.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2951.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,171 / 100,949 (37.812%)\n",
      "Discarded reads ...................: 0 / 100,949 (0.000%)\n",
      "Not assembled reads ...............: 62,778 / 100,949 (62.188%)\n",
      "Assembled reads file...............: /code/sequences/2951.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2951.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2951.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2951.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2898.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2898.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 59,177 / 74,070 (79.893%)\n",
      "Discarded reads ...................: 0 / 74,070 (0.000%)\n",
      "Not assembled reads ...............: 14,893 / 74,070 (20.107%)\n",
      "Assembled reads file...............: /code/sequences/2898.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2898.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2898.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2898.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2755.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2755.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 66,400 / 79,911 (83.092%)\n",
      "Discarded reads ...................: 0 / 79,911 (0.000%)\n",
      "Not assembled reads ...............: 13,511 / 79,911 (16.908%)\n",
      "Assembled reads file...............: /code/sequences/2755.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2755.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2755.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2755.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2758.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2758.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 99%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 76,778 / 87,117 (88.132%)\n",
      "Discarded reads ...................: 0 / 87,117 (0.000%)\n",
      "Not assembled reads ...............: 10,339 / 87,117 (11.868%)\n",
      "Assembled reads file...............: /code/sequences/2758.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2758.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2758.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2758.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2866.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2866.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 48,972 / 52,286 (93.662%)\n",
      "Discarded reads ...................: 0 / 52,286 (0.000%)\n",
      "Not assembled reads ...............: 3,314 / 52,286 (6.338%)\n",
      "Assembled reads file...............: /code/sequences/2866.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2866.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2866.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2866.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2634.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2634.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,064 / 51,042 (66.737%)\n",
      "Discarded reads ...................: 0 / 51,042 (0.000%)\n",
      "Not assembled reads ...............: 16,978 / 51,042 (33.263%)\n",
      "Assembled reads file...............: /code/sequences/2634.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2634.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2634.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2634.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2431.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2431.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 98%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 76,114 / 79,316 (95.963%)\n",
      "Discarded reads ...................: 0 / 79,316 (0.000%)\n",
      "Not assembled reads ...............: 3,202 / 79,316 (4.037%)\n",
      "Assembled reads file...............: /code/sequences/2431.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2431.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2431.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2431.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2631.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2631.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 45,764 / 54,709 (83.650%)\n",
      "Discarded reads ...................: 0 / 54,709 (0.000%)\n",
      "Not assembled reads ...............: 8,945 / 54,709 (16.350%)\n",
      "Assembled reads file...............: /code/sequences/2631.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2631.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2631.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2631.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2632.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2632.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,297 / 66,277 (45.713%)\n",
      "Discarded reads ...................: 0 / 66,277 (0.000%)\n",
      "Not assembled reads ...............: 35,980 / 66,277 (54.287%)\n",
      "Assembled reads file...............: /code/sequences/2632.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2632.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2632.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2632.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3067.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3067.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,116 / 37,617 (93.351%)\n",
      "Discarded reads ...................: 0 / 37,617 (0.000%)\n",
      "Not assembled reads ...............: 2,501 / 37,617 (6.649%)\n",
      "Assembled reads file...............: /code/sequences/3067.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3067.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3067.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3067.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3064.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3064.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 42,835 / 47,128 (90.891%)\n",
      "Discarded reads ...................: 0 / 47,128 (0.000%)\n",
      "Not assembled reads ...............: 4,293 / 47,128 (9.109%)\n",
      "Assembled reads file...............: /code/sequences/3064.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3064.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3064.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3064.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3100.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3100.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 51,395 / 68,964 (74.524%)\n",
      "Discarded reads ...................: 0 / 68,964 (0.000%)\n",
      "Not assembled reads ...............: 17,569 / 68,964 (25.476%)\n",
      "Assembled reads file...............: /code/sequences/3100.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3100.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3100.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3100.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3101.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3101.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 65,368 / 86,966 (75.165%)\n",
      "Discarded reads ...................: 0 / 86,966 (0.000%)\n",
      "Not assembled reads ...............: 21,598 / 86,966 (24.835%)\n",
      "Assembled reads file...............: /code/sequences/3101.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3101.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3101.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3101.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3103.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3103.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 72,403 / 90,136 (80.326%)\n",
      "Discarded reads ...................: 0 / 90,136 (0.000%)\n",
      "Not assembled reads ...............: 17,733 / 90,136 (19.674%)\n",
      "Assembled reads file...............: /code/sequences/3103.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3103.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3103.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3103.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2741.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2741.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 99%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 74,901 / 92,544 (80.936%)\n",
      "Discarded reads ...................: 0 / 92,544 (0.000%)\n",
      "Not assembled reads ...............: 17,643 / 92,544 (19.064%)\n",
      "Assembled reads file...............: /code/sequences/2741.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2741.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2741.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2741.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2745.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2745.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 95%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 83,603 / 96,248 (86.862%)\n",
      "Discarded reads ...................: 0 / 96,248 (0.000%)\n",
      "Not assembled reads ...............: 12,645 / 96,248 (13.138%)\n",
      "Assembled reads file...............: /code/sequences/2745.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2745.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2745.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2745.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2965.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2965.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 65,905 / 77,558 (84.975%)\n",
      "Discarded reads ...................: 0 / 77,558 (0.000%)\n",
      "Not assembled reads ...............: 11,653 / 77,558 (15.025%)\n",
      "Assembled reads file...............: /code/sequences/2965.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2965.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2965.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2965.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2872.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2872.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 83%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 91,888 / 100,095 (91.801%)\n",
      "Discarded reads ...................: 0 / 100,095 (0.000%)\n",
      "Not assembled reads ...............: 8,207 / 100,095 (8.199%)\n",
      "Assembled reads file...............: /code/sequences/2872.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2872.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2872.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2872.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2876.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2876.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 60%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 129,214 / 137,397 (94.044%)\n",
      "Discarded reads ...................: 0 / 137,397 (0.000%)\n",
      "Not assembled reads ...............: 8,183 / 137,397 (5.956%)\n",
      "Assembled reads file...............: /code/sequences/2876.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2876.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2876.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2876.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2874.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2874.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 88%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 88,156 / 94,602 (93.186%)\n",
      "Discarded reads ...................: 0 / 94,602 (0.000%)\n",
      "Not assembled reads ...............: 6,446 / 94,602 (6.814%)\n",
      "Assembled reads file...............: /code/sequences/2874.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2874.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2874.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2874.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2642.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2642.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,429 / 69,784 (72.264%)\n",
      "Discarded reads ...................: 0 / 69,784 (0.000%)\n",
      "Not assembled reads ...............: 19,355 / 69,784 (27.736%)\n",
      "Assembled reads file...............: /code/sequences/2642.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2642.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2642.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2642.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/2974.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/2974.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 77%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 108,115 / 147,198 (73.449%)\n",
      "Discarded reads ...................: 0 / 147,198 (0.000%)\n",
      "Not assembled reads ...............: 39,083 / 147,198 (26.551%)\n",
      "Assembled reads file...............: /code/sequences/2974.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/2974.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/2974.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/2974.unassembled.reverse.fastq\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3228.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3228.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 33,565 / 38,491 (87.202%)\n",
      "Discarded reads ...................: 0 / 38,491 (0.000%)\n",
      "Not assembled reads ...............: 4,926 / 38,491 (12.798%)\n",
      "Assembled reads file...............: /code/sequences/3228.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3228.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3228.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3228.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3221.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3221.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,927 / 35,794 (78.021%)\n",
      "Discarded reads ...................: 0 / 35,794 (0.000%)\n",
      "Not assembled reads ...............: 7,867 / 35,794 (21.979%)\n",
      "Assembled reads file...............: /code/sequences/3221.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3221.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3221.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3221.unassembled.reverse.fastq\n",
      "\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.11 [Nov 5, 2017]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: /code/sequences/3111.sickle.trimmed.R1.fastq.gz\n",
      "Reverse reads file.................: /code/sequences/3111.sickle.trimmed.R2.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: NO\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 999999\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 8\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Assemblying reads: 0%\n",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,842 / 37,298 (58.561%)\n",
      "Discarded reads ...................: 0 / 37,298 (0.000%)\n",
      "Not assembled reads ...............: 15,456 / 37,298 (41.439%)\n",
      "Assembled reads file...............: /code/sequences/3111.assembled.fastq\n",
      "Discarded reads file...............: /code/sequences/3111.discarded.fastq\n",
      "Unassembled forward reads file.....: /code/sequences/3111.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: /code/sequences/3111.unassembled.reverse.fastq\n",
      "\n",
      "\n",
      "FastQ records kept: 26228\n",
      "FastQ records discarded: 5511\n",
      "\n",
      "\n",
      "FastQ records kept: 36643\n",
      "FastQ records discarded: 4664\n",
      "\n",
      "\n",
      "FastQ records kept: 47165\n",
      "FastQ records discarded: 4938\n",
      "\n",
      "\n",
      "FastQ records kept: 44363\n",
      "FastQ records discarded: 5095\n",
      "\n",
      "\n",
      "FastQ records kept: 48868\n",
      "FastQ records discarded: 15518\n",
      "\n",
      "\n",
      "FastQ records kept: 69178\n",
      "FastQ records discarded: 6936\n",
      "\n",
      "\n",
      "FastQ records kept: 79871\n",
      "FastQ records discarded: 8868\n",
      "\n",
      "\n",
      "FastQ records kept: 71080\n",
      "FastQ records discarded: 7801\n",
      "\n",
      "\n",
      "FastQ records kept: 40559\n",
      "FastQ records discarded: 4579\n",
      "\n",
      "\n",
      "FastQ records kept: 87320\n",
      "FastQ records discarded: 1953\n",
      "\n",
      "\n",
      "FastQ records kept: 66895\n",
      "FastQ records discarded: 1857\n",
      "\n",
      "\n",
      "FastQ records kept: 31682\n",
      "FastQ records discarded: 7332\n",
      "\n",
      "\n",
      "FastQ records kept: 61366\n",
      "FastQ records discarded: 3499\n",
      "\n",
      "\n",
      "FastQ records kept: 91463\n",
      "FastQ records discarded: 4026\n",
      "\n",
      "\n",
      "FastQ records kept: 52184\n",
      "FastQ records discarded: 7587\n",
      "\n",
      "\n",
      "FastQ records kept: 121974\n",
      "FastQ records discarded: 2041\n",
      "\n",
      "\n",
      "FastQ records kept: 36658\n",
      "FastQ records discarded: 12229\n",
      "\n",
      "\n",
      "FastQ records kept: 43346\n",
      "FastQ records discarded: 15571\n",
      "\n",
      "\n",
      "FastQ records kept: 83869\n",
      "FastQ records discarded: 2442\n",
      "\n",
      "\n",
      "FastQ records kept: 112804\n",
      "FastQ records discarded: 1530\n",
      "\n",
      "\n",
      "FastQ records kept: 68268\n",
      "FastQ records discarded: 7630\n",
      "\n",
      "\n",
      "FastQ records kept: 66786\n",
      "FastQ records discarded: 7567\n",
      "\n",
      "\n",
      "FastQ records kept: 22702\n",
      "FastQ records discarded: 7595\n",
      "\n",
      "\n",
      "FastQ records kept: 35667\n",
      "FastQ records discarded: 10097\n",
      "\n",
      "\n",
      "FastQ records kept: 25716\n",
      "FastQ records discarded: 8348\n",
      "\n",
      "\n",
      "FastQ records kept: 104842\n",
      "FastQ records discarded: 1053\n",
      "\n",
      "\n",
      "FastQ records kept: 39641\n",
      "FastQ records discarded: 10788\n",
      "\n",
      "\n",
      "FastQ records kept: 35609\n",
      "FastQ records discarded: 12707\n",
      "\n",
      "\n",
      "FastQ records kept: 40742\n",
      "FastQ records discarded: 13534\n",
      "\n",
      "\n",
      "FastQ records kept: 35048\n",
      "FastQ records discarded: 5469\n",
      "\n",
      "\n",
      "FastQ records kept: 52650\n",
      "FastQ records discarded: 5904\n",
      "\n",
      "\n",
      "FastQ records kept: 62896\n",
      "FastQ records discarded: 8773\n",
      "\n",
      "\n",
      "FastQ records kept: 24112\n",
      "FastQ records discarded: 2955\n",
      "\n",
      "\n",
      "FastQ records kept: 78913\n",
      "FastQ records discarded: 22517\n",
      "\n",
      "\n",
      "FastQ records kept: 59242\n",
      "FastQ records discarded: 12731\n",
      "\n",
      "\n",
      "FastQ records kept: 55614\n",
      "FastQ records discarded: 5711\n",
      "\n",
      "\n",
      "FastQ records kept: 53561\n",
      "FastQ records discarded: 9493\n",
      "\n",
      "\n",
      "FastQ records kept: 57397\n",
      "FastQ records discarded: 9269\n",
      "\n",
      "\n",
      "FastQ records kept: 60573\n",
      "FastQ records discarded: 7099\n",
      "\n",
      "\n",
      "FastQ records kept: 58488\n",
      "FastQ records discarded: 16413\n",
      "\n",
      "\n",
      "FastQ records kept: 62555\n",
      "FastQ records discarded: 10426\n",
      "\n",
      "\n",
      "FastQ records kept: 45136\n",
      "FastQ records discarded: 13062\n",
      "\n",
      "\n",
      "FastQ records kept: 61538\n",
      "FastQ records discarded: 15240\n",
      "\n",
      "\n",
      "FastQ records kept: 52284\n",
      "FastQ records discarded: 14116\n",
      "\n",
      "\n",
      "FastQ records kept: 66289\n",
      "FastQ records discarded: 17314\n",
      "\n",
      "\n",
      "FastQ records kept: 50233\n",
      "FastQ records discarded: 14604\n",
      "\n",
      "\n",
      "FastQ records kept: 46387\n",
      "FastQ records discarded: 14384\n",
      "\n",
      "\n",
      "FastQ records kept: 48286\n",
      "FastQ records discarded: 13755\n",
      "\n",
      "\n",
      "FastQ records kept: 43691\n",
      "FastQ records discarded: 5281\n",
      "\n",
      "\n",
      "FastQ records kept: 44967\n",
      "FastQ records discarded: 11757\n",
      "\n",
      "\n",
      "FastQ records kept: 48401\n",
      "FastQ records discarded: 12837\n",
      "\n",
      "\n",
      "FastQ records kept: 61079\n",
      "FastQ records discarded: 15148\n",
      "\n",
      "\n",
      "FastQ records kept: 29435\n",
      "FastQ records discarded: 9973\n",
      "\n",
      "\n",
      "FastQ records kept: 45706\n",
      "FastQ records discarded: 13471\n",
      "\n",
      "\n",
      "FastQ records kept: 41430\n",
      "FastQ records discarded: 12175\n",
      "\n",
      "\n",
      "FastQ records kept: 88901\n",
      "FastQ records discarded: 2987\n",
      "\n",
      "\n",
      "FastQ records kept: 19656\n",
      "FastQ records discarded: 6136\n",
      "\n",
      "\n",
      "FastQ records kept: 85857\n",
      "FastQ records discarded: 2299\n",
      "\n",
      "\n",
      "FastQ records kept: 55417\n",
      "FastQ records discarded: 16858\n",
      "\n",
      "\n",
      "FastQ records kept: 41356\n",
      "FastQ records discarded: 12066\n",
      "\n",
      "\n",
      "FastQ records kept: 32865\n",
      "FastQ records discarded: 10010\n",
      "\n",
      "\n",
      "FastQ records kept: 19355\n",
      "FastQ records discarded: 18816\n",
      "\n",
      "\n",
      "FastQ records kept: 94983\n",
      "FastQ records discarded: 996\n",
      "\n",
      "\n",
      "FastQ records kept: 29305\n",
      "FastQ records discarded: 11891\n",
      "\n",
      "\n",
      "FastQ records kept: 127156\n",
      "FastQ records discarded: 2058\n",
      "\n",
      "\n",
      "FastQ records kept: 69680\n",
      "FastQ records discarded: 20539\n",
      "\n",
      "\n",
      "FastQ records kept: 63046\n",
      "FastQ records discarded: 17775\n",
      "\n",
      "\n",
      "FastQ records kept: 37091\n",
      "FastQ records discarded: 15429\n",
      "\n",
      "\n",
      "FastQ records kept: 85589\n",
      "FastQ records discarded: 2004\n",
      "\n",
      "\n",
      "FastQ records kept: 34460\n",
      "FastQ records discarded: 19082\n",
      "\n",
      "\n",
      "FastQ records kept: 51077\n",
      "FastQ records discarded: 14828\n",
      "\n",
      "\n",
      "FastQ records kept: 24954\n",
      "FastQ records discarded: 5647\n",
      "\n",
      "\n",
      "FastQ records kept: 72132\n",
      "FastQ records discarded: 35983\n",
      "\n",
      "\n",
      "FastQ records kept: 55330\n",
      "FastQ records discarded: 6375\n",
      "\n",
      "\n",
      "FastQ records kept: 71193\n",
      "FastQ records discarded: 19709\n",
      "\n",
      "\n",
      "FastQ records kept: 49071\n",
      "FastQ records discarded: 13702\n",
      "\n",
      "\n",
      "FastQ records kept: 30022\n",
      "FastQ records discarded: 8803\n",
      "\n",
      "\n",
      "FastQ records kept: 51427\n",
      "FastQ records discarded: 6045\n",
      "\n",
      "\n",
      "FastQ records kept: 85392\n",
      "FastQ records discarded: 1157\n",
      "\n",
      "\n",
      "FastQ records kept: 97715\n",
      "FastQ records discarded: 1496\n",
      "\n",
      "\n",
      "FastQ records kept: 31701\n",
      "FastQ records discarded: 3415\n",
      "\n",
      "\n",
      "FastQ records kept: 58048\n",
      "FastQ records discarded: 16669\n",
      "\n",
      "\n",
      "FastQ records kept: 38307\n",
      "FastQ records discarded: 4528\n",
      "\n",
      "\n",
      "FastQ records kept: 39907\n",
      "FastQ records discarded: 12184\n",
      "\n",
      "\n",
      "FastQ records kept: 23173\n",
      "FastQ records discarded: 13818\n",
      "\n",
      "\n",
      "FastQ records kept: 45028\n",
      "FastQ records discarded: 16783\n",
      "\n",
      "\n",
      "FastQ records kept: 17094\n",
      "FastQ records discarded: 4748\n",
      "\n",
      "\n",
      "FastQ records kept: 134791\n",
      "FastQ records discarded: 2242\n",
      "\n",
      "\n",
      "FastQ records kept: 39507\n",
      "FastQ records discarded: 12728\n",
      "\n",
      "\n",
      "FastQ records kept: 25433\n",
      "FastQ records discarded: 5811\n",
      "\n",
      "\n",
      "FastQ records kept: 37856\n",
      "FastQ records discarded: 13539\n",
      "\n",
      "\n",
      "FastQ records kept: 20436\n",
      "FastQ records discarded: 7491\n",
      "\n",
      "\n",
      "FastQ records kept: 23910\n",
      "FastQ records discarded: 5698\n",
      "\n",
      "\n",
      "FastQ records kept: 48570\n",
      "FastQ records discarded: 16798\n",
      "\n",
      "\n",
      "FastQ records kept: 31104\n",
      "FastQ records discarded: 6496\n",
      "\n",
      "\n",
      "FastQ records kept: 55342\n",
      "FastQ records discarded: 17061\n",
      "\n",
      "\n",
      "FastQ records kept: 27011\n",
      "FastQ records discarded: 6554\n",
      "\n",
      "\n",
      "FastQ records kept: 77851\n",
      "FastQ records discarded: 1882\n",
      "\n",
      "\n",
      "FastQ records kept: 160069\n",
      "FastQ records discarded: 3178\n",
      "\n",
      "\n",
      "FastQ records kept: 190395\n",
      "FastQ records discarded: 9721\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Histogram /code/sequences/readyForQiime.allsamples.fasta.histogram.svg finished\n",
      "Histogram /code/sequences/2618.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2381.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2706.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2788.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3125.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2903.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3154.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3141.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2881.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2426.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2684.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2876.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2549.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2673.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3146.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2889.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3045.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2974.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2715.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2700.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2910.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2983.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2456.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2540.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3028.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2459.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2712.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2675.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2801.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3216.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2991.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2794.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2898.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2964.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2872.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2606.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2397.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2718.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2773.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2766.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2717.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2425.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2455.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2431.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2913.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3047.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2772.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2874.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2719.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2797.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2909.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2987.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2992.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2676.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2901.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2866.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3127.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2559.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2965.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3031.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3221.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2741.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2642.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2592.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2951.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3091.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2658.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3101.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2634.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2956.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2926.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2933.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3067.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2497.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3016.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3098.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2620.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3100.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2475.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2593.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3228.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2755.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2632.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2579.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2599.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2649.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2745.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2758.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2939.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2561.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2938.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3086.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2463.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3103.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2481.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3064.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3111.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3089.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/2631.passedQC.fastq.histogram.svg finished\n",
      "Histogram /code/sequences/3012.passedQC.fastq.histogram.svg finished\n",
      "FINISHED Amplicon quality control!\n",
      "CPU times: user 1min 57s, sys: 19.3 s, total: 2min 16s\n",
      "Wall time: 1h 40min 54s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!python ./ampliconQC.py --data sequences --forward ATGCGTTGGAGAGARCGTTTC --reverse GATCACCTTCTAATTTACCWACAACTG --threads 8 --histograms --qiime\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. Generate sequence counts file used in final step to produce reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 800 ms, sys: 150 ms, total: 950 ms\n",
      "Wall time: 48.6 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!for file in sequences/*.passedQC.fastq; \\\n",
    "do \\\n",
    "  awk 'NR%4==2{sum+=1}END{print FILENAME,sum}' $file >> sequences/diatomSequenceCounts.txt; \\\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4. Assign similar sequences to OTUs using user-defined similarity threshold\n",
    "#### Default clustering algorithm is UCLUST, O(m\\*n), which is a greedy algorithm and is dependent on ordering of sequences in readyForQiime.allsamples.fasta file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/miniconda/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.\r\n",
      "  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')\r\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "!pick_otus.py -i sequences/readyForQiime.allsamples.fasta -o sequences/picked_otus_97"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  5. Pick a representative set of sequences. For each OTU, one sequence will be used in subsequent analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!pick_rep_set.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -f sequences/readyForQiime.allsamples.fasta \\\n",
    "  -o sequences/repset.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Query BLAST database with OTU representatives"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!blastn -db diatoms -query sequences/repset.fasta \\\n",
    "  -out sequences/repset.diatoms.blastn \\\n",
    "  -task blastn -max_target_seqs 1 -num_threads 8 -outfmt 6 -evalue 0.01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir sequences/assigned_taxonomy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Create taxonomy assignments from BLAST outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!python ./create_taxonomy_assignments_from_blast.py --taxonomy diatoms.taxonomy.FINAL2017.txt \\\n",
    "  --percid 95.0 --blast sequences/repset.diatoms.blastn --output sequences/assigned_taxonomy/repset.taxonomy.txt "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Report how often an OTU is found in each sample and add the taxonomic predictions for each OTU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!make_otu_table.py -i sequences/picked_otus_97/readyForQiime.allsamples_otus.txt \\\n",
    "  -t sequences/assigned_taxonomy/repset.taxonomy.txt \\\n",
    "  -o sequences/otu_table.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 9. Filter the OTU table based on taxonomic metadata excluding specific taxa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!filter_taxa_from_otu_table.py -i sequences/otu_table.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom \\\n",
    "  -n MARINE,NOT_DIATOM,Yellow_green_Algae,None"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 10. Sort OTU table by sample id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!sort_otu_table.py -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/otu_table.diatomsonly.biom"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 11. Provide summary information of the representation of taxonomic groups within each sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!summarize_taxa.py -L 1 \\\n",
    "  -i sequences/otu_table.diatomsonly.biom \\\n",
    "  -o sequences/visualised_taxonomy -a"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 12. Produce Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "!python ./produceDiatomReports.py --folder sequences --lookup lookuptable.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 13. Inspect producted Diatom reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.read_csv('sequences/Abundances.fail.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.read_csv('sequences/Abundances.pass.csv')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
